---
output:
  pdf_document: default
  html_document: default
header-includes:
- \usepackage{booktabs}
- \usepackage{longtable}
- \usepackage{array}
- \usepackage{multirow}
- \usepackage{wrapfig}
- \usepackage{float}
- \usepackage{colortbl}
- \usepackage{pdflscape}
- \usepackage{tabu}
- \usepackage{threeparttable}
- \usepackage{threeparttablex}
- \usepackage[normalem]{ulem}
- \usepackage{makecell}
- \usepackage{xcolor}
- \usepackage{lscape}
---

`r if(knitr:::is_latex_output()) '\\startappendices'`

`r if(!knitr:::is_latex_output()) '# (APPENDIX) Appendix {-}'` 

<!-- If you feel it necessary to include an appendix, it goes here. The first appendix should include the commands above. -->

# Supplementary material

## Chapter 2 {#ch2supp}

### Molecular methods for analysis of ticks

#### DNA extraction

Once ticks were identified they were surface-sterilised by washing in 10 % hypochlorite solution, rinsed in 70 % ethanol and DNA-free PBS, and then air-dried. Genomic DNA was extracted using the DNeasy Blood and Tissue kit (QIAGEN, Germany) for adults or the QIAamp DNA Mini Kit (QIAGEN, Germany) for nymphs and larvae. 
Due to the large number of larvae collected from some animals and the expected low DNA yeild, in comes cases up to 5 larvae were pooled based on host, species and engorgement status. 
Ticks were placed in a 2 mL safe lock Eppendorf tube with a 5mm steel bead, frozen in liquid nitrogen for 1 min and homogenised by shaking at 40 Hz in a Tissue Lyser LT (QIAGEN, Germany). Final elution of DNA was adjusted according to tick size, between 30-150 $\mu L$ of AE buffer was added to the silcon membrane. 
Extraction controls (EXBs) consisting of 200 $\mu L$ sterile DNA free PBS, were included randomly in each extraction batch.

Prior to DNA extraction ticks were washed in 10% bleach solution, followed by 80% ethanol and rinsed in sterile-DNA free PBS. For museum specimens ticks were extracted in a separate dedicated laboratory, away from modern DNA. Ticks were soaked in sterile 1X PBS solution for 24-48 hours to remove ethanol from specimens. For museum and reference samples non-destructive sampling was performed, where ticks were punctured using a sterile pin and soaked in proteinase K and lysis buffer overnight (16-18 hours) at 56$^\circ C$, 700 rpm. 
DNA was then extracted using QIAamp DNA mini kit (Qiagen, Germany) following manusctures recommendations, except for final elution which was reduced to 30 $\mu L$ of buffer AE and incubated for 30 mins, before final centrifugation. 
In the case of modern specimens adults ticks were bisected and half was placed in proteinase K and lysis buffer overnight (16-18 hours) at 56$^\circ C$, 700 rpm, the other half was retained as a voucher and placed back into 90% ethanol. 
The supernatant was then used for subsequent extraction using DNeasy Blood and Tissue kit (Qiagen, Germany) following manufactures recommendations. Strict laboratory hygiene was maintained throughout, equipment and surafces were cleaned with DNAaway in between each sample.

#### Targeted assays and Sanger sequencing

A multilocus sequence typing (MSLT) approach was used to obtain sequence information from 3 mitochondrial genetic markers. These were; cytochrome c oxidase subunit I (COX1), 12S rDNA, 16S rDNA (primers outlined in table \@ref(tab:T2primers). 
All reactions were carried out in 25 $\mu L$ volumes, each containing; 1X buffer (KAPA Biosystems, South Africa), 2.5 mM $MgCl_{2}$, 0.4$\mu M$ of primer, 200 mM of each dNTP, 1.0 U of Taq (KAPA Biosystems, South Africa), and between 1$\mu L$ (modern specimens) 5 $\mu L$ (museum specimens) of gDNA was added. MLST assays on museum DNA were performed in a separate dedicated facility away from modern DNA. 

The COX1 locus was amplified using a hemi-nested assay with primers HCO2064/HCO1215 in the primary assay and HCO2064/HCO1240 from Song et al. [-@songPhylogeneticPhylogeographicRelationships2011]. 
Despite repeated attempts for many specimens of *Amblyomma* amplification using a single round PCR was not successful. Previous research has demonstrated the primer pair HCO2064/HCO1240 is successful at amplifying COX1 gene from tick specimens of the genera *Ixodes*, *Haemaphysalis* and *Rhipicephalus* [@evansAutomaticBarcodeGap2019]. 
However this assay did not achieve reliable repeatable results for *Am. triguttatum*, therefore a hemi-nested PCR assay was used. 
Reactions were carried out in 25$\mu L$ reactions, each containing; 1X buffer (KAPA Biosystems, South Africa), 2.5 mM MgCl~2~, 0.4$\mu M$ of each primer (HCO2064/HCO1215 and HCO2064/HCO1240), 0.25 mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1$\mu L$ of either tick DNA or primary product. Thermal cycling conditions for both assays were as followed: 95$^\circ C$ for 2 mins, followed by 35 (modern specimens) or 50 (museum specimens) cycles of 95$^\circ C$ for 30 secs, 48$^\circ C$ for 30 secs, 72$^\circ C$ for 50 secs, and a final extension of 72$^\circ C$ for 2 mins.

A 450 bp product of the 16S rDNA gene was amplified [@lvDevelopmentDNABarcoding2014]. Reactions were carried out in 25$\mu L$ reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5mM MgCl~2~, 0.4$\mu M$ of each primer (16S-F/16S-R1), 0.25 mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1$\mu L$ of tick gDNA. A touchdown approach was used for the 16S assays were as followed; 95$^\circ C$ for 5 mins, followed by 5 cycles of 95$^\circ C$ for 30 secs, 49$^\circ C$ for 30 secs, 68$^\circ C$ for 30 secs; 5 cycles of 95$^\circ C$ for 30 secs, 47$^\circ C$ for 30 secs, 68$^\circ C$ for 30 secs; 5 cycles of 95$^\circ C$ for 30 secs, 45$^\circ C$ for 30 secs, 68$^\circ C$ for 30 secs; 25 cycles of 95$^\circ C$ for 30 secs, 43$^\circ C$ for 30 secs, 68$^\circ C$ for 30 secs; and a final extension of 68$^\circ C$ for 5 mins.

A 370bp product of the 12S rDNA gene was amplified. Reactions were carried out in 25$\mu L$ reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5mM MgCl~2~, 0.4$\mu M$ of each primer (T1B/T2A), 0.25mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1$\mu L$ of tick gDNA.Thermal cycling conditions were as followed; 95$^\circ C$ for 5 mins, followed by 5 cycles of 95$^\circ C$ for 15 secs, 51$^\circ C$ for 30 secs, 68$^\circ C$ for 30 secs; 25 cycles of 95$^\circ C$ for 30 secs, 53$^\circ C$ for 30 secs, 72$^\circ C$ for 1 min; and a final extension of 72$^\circ C$ for 5 mins.

Amplified products were visualised on a 1% agarose gel stained with SYBR Safe (Invitrogen, USA) and bands of the correct size were excised using a sterile scalpel blade and purified using previously described methods [@yangSpecificQuantitativeDetection2013]. Sanger sequencing was carried out in both directions and sequencing was performed at the Australian Genome Research Facility (Perth, Western Australia) on an Applied Biosystems 3730xl DNA Analyzer using BigDye(TM) Terminator v3.1 Cycle Sequencing Kit.

```{r include=FALSE}
library(readxl)
library(tidyverse)
ch2_primers <- read_excel("tables/Ch2-primers.xlsx")
ch2_primers %>% 
  mutate_all(~ replace_na(.x, ""))
idx <- c(1:4,7:12,21:23)
ch2_primers <- ch2_primers[idx,1:3]
```

(ref:T2primercap) Primers used to amplify mitochondrial genes of ticks (Ixodida). The following loci were targeted; cytochrome c oxidase subunit 1 (COX1), 12S rDNA, 16S rDNA. References **1.** Song et al. [-@songPhylogeneticPhylogeographicRelationships2011], **2.** Beati et al. [-@beatiAnalysisSystematicRelationships2001], **3.** Lv et al. [-@lvDevelopmentDNABarcoding2014].

```{r T2primers, echo=FALSE}
library(kableExtra)
opts <- options(knitr.kable.NA = "")
knitr::kable(ch2_primers, booktabs = TRUE, linesep = "", caption = "(ref:T2primercap)", caption.short = "Primers used to amplify barcoding genes for Australian tick") %>%
  kable_styling(font_size = 8.5) %>%
  row_spec(c(1, 5, 8, 11), bold = T)
```

#### Library prepation and amplicon next-generation sequencing

Based on results from the MLST approached used above a a high-throughput metabarcoding approach was developed to identify tick species.
In addition the short fragment size (~370 bp) and ability to deliminate between species made the 12rDNA loci most suitable for transferring onto the Illumina MiSeq.
The pan-Ixodida primers T1B and T2A [@beatiAnalysisSystematicRelationships2001] were ordered with Illumina MiSeq adapter overhangs.
Amplicon PCRs were carried out in 25 $\mu L$ reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5 mM MgCl~2~, 0.4 $\mu M$ of each primer, 0.25 mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1 $\mu L$ of tick gDNA.Thermal cycling conditions were as followed; 95$^\circ C$ for 5 mins, followed by 5 cycles of 95$^\circ C$ for 15 secs, 51$^\circ C$ for 30 secs, 68$^\circ C$ for 30 secs; 25 cycles of 95$^\circ C$ for 30 secs, 53$^\circ C$ for 30 secs, 72$^\circ C$ for 1 min; and a final extension of 72$^\circ C$ for 5 mins.

Amplicon PCR products were then indexed using the Nextera XT DNA library preparation kit in 25 $\mu L$ volumes following manufacturer's recommendations. 
All PCRs included no template controls (NTC) during each reaction set up and PCRs were performed under strict laboratory conditions. Amplicons were then dual-indexed using the Nextera XT index kit. 
Reactions were performed in 25 $\mu L$ volumes using following manufacturers recommendations. 
Libraries were purified with Axygen PCR clean up beads, pooled in equimolar amounts and quantified using Qubit High Sensitivity dsDNA assay kit (Thermo Fisher Scientific, Waltham, MA, USA) and amplicons were pooled in equimolar amounts. 
Libraries were shipped to Australian Genomic Research Facility (Melbourne, Australia) for final QC and sequenced on the Illumina MiSeq using v2 chemistry (2 x 250 paired end).


#### Bioinformatics

**Multilocus sequence typing data produced by Sanger Sequencing**

Sequences were imported into Geneious 10.2.6 (https://www.geneious.com) for quality inspection. 
Forward and reverse sequenced were aligned using the default parameters in Geneious and primers were trimmed.
Reference sequences were retrieved from GenBank [@bensonGenBank2017] and aligned with sequences obtained in the present study using MUSCLE [@edgarMUSCLEMultipleSequence2004]. 

**High-throughout sequence data produced by Illumina MiSeq**

Illumina MiSeq fastq data was analysed using bioinformatic pipeline with the program USEARCH v11 [@edgarSearchClusteringOrders2010]. 
Paired end reads were merged and sequences matching forward and reverse primers were retrieved (max number of mismatches = 2). 
Sequences were then quality filtered and singletons were removed. 
The unoise3 [@edgarUNOISE2ImprovedErrorcorrection2016] algorithm was used to perform denoising (error-correction) and generate zero-radius taxonomic units (zOTUs) equivalent to amplicon sequence variants (ASVs). 
Resulting zOTUs were subject to BLAST analysis (BLASTN 2.11.0+ [@zhangGreedyAlgorithmAligning2000; @morgulisDatabaseIndexingProduction2008]) against NCBI nucleotide collection (nt) database. 
Taxonomic lineage for each top hit (based on e-value score) was retrieved from NCBI taxonomy database using TaxonKit [@weissHostReproductiveCycle2021]. 
Final taxonomy assignment was then refined to lowest common ancestor based on BLAST results (percentage identity, e-value and query coverage) and sequence alignment method. 

Sequences generated were aligned with reference sequences were retrieved from GenBank [@bensonGenBank2017] using MAFFT [@edgarMUSCLEMultipleSequence2004]. 
Phylogenies were inferred using the maximum likelihood (ML) method. 
The optimal evolutionary model was selected using ModelFinder [@kalyaanamoorthyModelFinderFastModel2017] based on bayesian information criterion. 
Phylogenetic analysis was performed in IQ-TREE v1.6.11 [@nguyenIQTREEFastEffective2015] and bootstrap support was calculated using ultrafast (UFBoot2) method with 10,000 replicates [@hoangUFBoot2ImprovingUltrafast2018]. 
Genetic sequence similarity was calculated using the Kimura Two-Parameter method [@tamuraEstimationNumberNucleotide1993].

\newpage

\begin{landscape}
```{r FA21, out.width='95%', out.align = 'left',fig.scap = "Records of Australian ticks from Atlas of Living Australia", fig.cap = "Occurrence map of ticks (Acari: Ixodida) records sourced from the Atlas of Living Australia (www.ala.org.au), data has not been curated.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-2.1.png")
```

\end{landscape}

\newpage
```{r, include=FALSE}
library(readxl)
library(tidyverse)

App_Ch2_MLST <- read_excel("tables/App-Ch2-tickMLST.xlsx", 
    col_types = c("text", "text", "text", 
        "text", "text", "text", "text", "text", 
        "skip"))
```

```{r T2MLSTseqs, echo=FALSE, message=FALSE, warning=FALSE}
library(kableExtra)
opts <- options(knitr.kable.NA = "")
knitr::kable(App_Ch2_MLST, booktabs = TRUE, longtable = T, linesep = "", caption = "Genetic sequences generated from Australian ticks using a multi-locus sequence typing approach. Nucleotide data has been deposited in GenBank with accession numbers of specimens included in the table.", 
  caption.short = "Genetic sequences generated from Australian ticks") %>%
  kable_styling(font_size = 7.5) %>%
  kable_styling(latex_options = c("hold_position"))%>% 
  column_spec(c(2), italic = T) %>%
  kable_styling(latex_options = c("repeat_header")) %>%
  landscape()
```

\newpage
```{r, include=FALSE}
library(readxl)
library(tidyverse)

App_Ch2_genbank <- read_excel("tables/App-Ch2-tick12SgeneticDistances.xlsx", 
    sheet = "table")
```

```{r T2genbank, echo=FALSE, message=FALSE, warning=FALSE}
library(kableExtra)
opts <- options(knitr.kable.NA = "")
knitr::kable(App_Ch2_genbank, booktabs = TRUE, longtable = T, linesep = "", caption = "GenBank accession numbers and species identification of sequences used to produce phylogenetic tree (Figure 2.5)", 
  caption.short = "GenBank accession numbers and species identification of sequences used to produce phylogenetic tree") %>%
  kable_styling(font_size = 7.5) %>%
  kable_styling(latex_options = c("hold_position"))%>% 
  column_spec(c(2,4), italic = T) %>%
  column_spec (2,border_right = T) %>%
  kable_styling(latex_options = c("repeat_header"))
```

\newpage

## Chapter 3 {#ch3supp}

```{r include=FALSE}
library(readr)
library(tidyverse)
App_Ch3_seqs <- read_csv("tables/Ch3-appendix-sequences.csv")
```

```{r TA31, echo=FALSE}
library(kableExtra)
opts <- options(knitr.kable.NA = "")
knitr::kable(App_Ch3_seqs, booktabs = TRUE, linesep = "", caption = "Summary of sequences obtained from bacterial next-generation sequencing", 
  caption.short = "Summary of sequences obtained from bacterial amplicon sequencing.") %>%
  kable_styling(font_size = 8.5) %>%
  kable_styling(latex_options = c("hold_position"))
```

\newpage

```{r FA31, out.width='95%', out.align = 'left',fig.scap = "Summary of sequences obtained from bacterial 16S rRNA metabarcoding", fig.cap = "Summary of sequences obtained form bacterial 16S rRNA next-generation sequencing (Illumina MiSeq). (A) Library size (i.e. number of sequences) obtained from samples. (B) Prevalence of contaminant ASVs as identified by `decontam` analysis in controls and samples. (C) Distribution of the number of reads in sample categories blood, tick and tissue.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.1.png")
```

\newpage

```{r FA32, out.width='85%', out.align = 'left',fig.scap = "Rarefaction curves of bacterial 16S rRNA sequencing from wildlife samples", fig.cap = "Rarefaction curves (number of ASVs) of wildlife blood, tick and tissue samples from bacterial 16S rRNA profiling (step size = 100). Rarefaction curves were used to determine how adequate sequencing depth was in detecting the complete theoretical suite of bacterial organisms present; of note, rarefaction plots excluded OTUs considered environmental contaminants (described methods).", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.2.png")
```

\newpage

```{r FA33, out.width='95%', out.align = 'left',fig.scap = "Alpha diversity of microbiome community for tick samples", fig.cap = "Boxplot of Alpha-diversity indices for tick samples. Diversity indexes (A) Observed number of ASVs, (B) Chao1 index, (C) Shannon index and (D) inverse Simpson index. Boxplots and violin plots represent the distribution of diversity among tick species. Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test. Pools of different tick species excluded from analysis.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.3.png")
```

\newpage

```{r FA34, out.width='95%', out.align = 'left',fig.scap = "Phylum level distribution plots", fig.cap = "Phylum level distribution plots of bacterial composition in wildlife samples.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.4.png")
```

\newpage

```{r FA35, out.width='95%', out.align = 'left',fig.scap = "Phylum level taxa abundance", fig.cap = "Abundance (no. of sequences) and prevalence in samples of bacterial phyla identified from wildlife samples (A) overall, (B) blood, (C) tick and (D) tissue.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.5.png")
```

\newpage

```{r FA36, out.width='80%', out.align = 'left',fig.scap = "Bacterial family taxa identified in wildlife blood samples", fig.cap = "Bacterial family taxa (top 6 most abundant) identified in wildlife blood samples.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.6.png")
```

\newpage

```{r FA37, out.width='80%', out.align = 'left',fig.scap = "Bacterial family taxa identified in wildlife tick samples", fig.cap = "Bacterial family taxa (top 6 most abundant) identified in wildlife tick samples.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.7.png")
```

\newpage

```{r FA38, out.width='80%', out.align = 'left',fig.scap = "Bacterial family taxa identified in wildlife tissue samples", fig.cap = "Bacterial family taxa (top 6 most abundant) identified in wildlife tissue samples.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.8.png")
```

\newpage

```{r FA39, out.width='95%', out.align = 'left',fig.scap = "Hierarchical cluster analysis of bacterial communities - all samples", fig.cap = "Hierarchical cluster analysis of bacterial communities from wildlife samples. Data points coloured by sample type; blood, tick and tissue. Cluster analysis was performed using euclidean distance measure (average), data was transformed using Hellinger transformation.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.9.png")
```


\newpage

```{r FA310, out.width='95%', out.align = 'left',fig.scap = "Hierarchical cluster analysis of bacterial communities - by sample type", fig.cap = "Hierarchical cluster analysis of bacterial communities from wildlife samples, separated by sample types. (A) Blood, (B) Tick, (C) Tissue. Data points coloured by host species (and tick species (B)). Cluster analysis was performed using euclidean distance measure (average), data was transformed using Hellinger transformation.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-3.10.png")
```

\clearpage

### Statistical analysis{#ch3stats}

Beta diversity statistical analysis of bacterial microbiome composition using analysis of variance (ANOVA) and permutational multivariate analysis of variance (PERMANOVA) methods.
Statistical analysis calculated using the vegan R package. 
Data cleaning and transformation: removed samples with < 1000 sequences, transformed data using Hellinger transformation method. Distance calculated using Bray-Curtis distance measure.  

#### Sample Type{#ch3stats1}

Statistical analysis comparing the bacterial microbiome of blood, tick and tissue samples collected form wildlife.


*A. Analysis of Variance (ANOVA)* 

```
Analysis of Variance Table

Response: Distances
           Df Sum Sq  Mean Sq F value    Pr(>F)    
Groups      2 0.4466 0.223289  36.209 1.364e-15 ***
Residuals 614 3.7863 0.006167                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

*B. Permutational multivariate analysis of variance (PERMANOVA)*

```
Permutation test for homogeneity of multivariate dispersions
Permutation: free
Number of permutations: 999

Response: Distances
           Df Sum Sq  Mean Sq      F N.Perm Pr(>F)    
Groups      2 0.4466 0.223289 36.209    999  0.001 ***
Residuals 614 3.7863 0.006167                         
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Pairwise comparisons:
(Observed p-value below diagonal, permuted p-value above diagonal)
            Blood       Tick Tissue
Blood             4.1600e-01  0.001
Tick   4.0839e-01             0.001
Tissue 8.8347e-17 3.5464e-15       
```

\newpage

#### Tick species{#ch3stats2}

Statistical analysis comparing bacterial microbiome of tick collected from wildlife (all tick species). 

*A. Analysis of Variance (ANOVA)* 

```
Analysis of Variance Table

Response: Distances
           Df Sum Sq Mean Sq F value    Pr(>F)    
Groups      6 2.5929 0.43215   14.01 9.594e-14 ***
Residuals 251 7.7423 0.03085         
```

*B. Permutational multivariate analysis of variance (PERMANOVA)*

```

Permutation test for homogeneity of multivariate dispersions
Permutation: free
Number of permutations: 999

Response: Distances
           Df Sum Sq Mean Sq     F N.Perm Pr(>F)    
Groups      6 2.5929 0.43215 14.01    999  0.001 ***
Residuals 251 7.7423 0.03085                        
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Pairwise comparisons:
(Observed p-value below diagonal, permuted p-value above diagonal)
                 Amtri Ixant      Ixaus      Ixhol Ixhol;Ixtri      Ixtas Ixtri
Amtri                        9.4400e-01 3.6500e-01  2.5700e-01 7.8500e-01 0.001
Ixant                                                                          
Ixaus       9.3797e-01                  5.9300e-01  3.8000e-02 7.3300e-01 0.001
Ixhol       3.4424e-01       5.7323e-01             5.3700e-01 1.0000e-03 0.001
Ixhol;Ixtri 2.4249e-01       4.5601e-02 5.3705e-01             3.8000e-02 0.001
Ixtas       7.8073e-01       7.4761e-01 3.5985e-05  3.8405e-02            0.001
Ixtri       1.0858e-06       4.6854e-09 2.0461e-06  1.2151e-05 5.1524e-15   
```

\newpage

Statistical analysis comparing bacterial microbiome of cohabiting tick species collected from wildlife in Sydney Northern Beaches area, New South Wales (*Ix. holocyclus*, *Ix. tasmani*, and *Ix. trichosuri*).

*A. Analysis of Variance (ANOVA)* 

```
Analysis of Variance Table

Response: Distances
           Df Sum Sq Mean Sq F value    Pr(>F)    
Groups      3 2.3202 0.77341  25.465 2.177e-14 ***
Residuals 247 7.5019 0.03037                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

*B. Permutational multivariate analysis of variance (PERMANOVA)*

```
Permutation test for homogeneity of multivariate dispersions
Permutation: free
Number of permutations: 999

Response: Distances
           Df Sum Sq Mean Sq      F N.Perm Pr(>F)    
Groups      3 2.3202 0.77341 25.465    999  0.001 ***
Residuals 247 7.5019 0.03037                         
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Pairwise comparisons:
(Observed p-value below diagonal, permuted p-value above diagonal)
                 Ixhol Ixhol;Ixtri      Ixtas Ixtri
Ixhol                   5.3000e-01 1.0000e-03 0.001
Ixhol;Ixtri 5.3687e-01             3.9000e-02 0.001
Ixtas       3.5862e-05  3.8383e-02            0.001
Ixtri       2.0485e-06  1.2237e-05 5.1829e-15     
```

\clearpage

### Supplementary Electronic Tables {#ch3-esupp}

The following electronic supplementary files are available for download on [FigShare](https://figshare.com/s/0f5495957bce881caef8), repository doi: 10.6084/m9.figshare.14363627.

#### Supplementary table E1.1

**Supplementary table E1.1:** Detail for taxa of interest with sequence information and sample metadata, including results from NCBI BLAST analysis.
[Taxa of interest sample data and BLAST results](https://ndownloader.figshare.com/files/27452714)

#### Supplementary table E1.2

**Supplementary table E1.2:** Genetic similarity of Anaplasmataceae sequences based on a 1,244 bp alignment of the 16S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data. 
[Anaplasmataceae genetic diversity table](https://ndownloader.figshare.com/files/27452720)

#### Supplementary table E1.3

**Supplementary table E1.3:** Genetic similarity of *Borrelia* sequences based on a 431 bp alignment of the 16S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data. 
[*Borrelia* genetic diversity table](https://ndownloader.figshare.com/files/27452726)

#### Supplementary table E1.4

**Supplementary table E1.4:**  Genetic similarity of *Bartonella* sequences based on a 522 bp alignment of the 16S rRNA-ITS locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data. 
[*Bartonella* genetic diversity table](https://ndownloader.figshare.com/files/27452723)

---

\clearpage

## Chapter 4{#ch4-supp}

Supplementary material for wildlife haemoprotozoa chapter.

```{r FA41, out.width='95%', fig.pos = 'h', out.align = 'left',fig.scap = "Sequence summary of trypanosome metabarcoding", fig.cap = "Summary of sequences obtained from trypanosome 18S rRNA metabatcoding (Illumina MiSeq). (A) Library size (i.e. number of sequences) obtained from samples. (B) Distribution of the number of reads in sample categories blood, tick and tissue.", echo=FALSE}
knitr::include_graphics("figures/ms-figs-appendix/FigA-4.1.png")
```

\clearpage

### Supplementary Electronic Tables {#ch4-esupp}

The following electronic supplementary files are available for download on [FigShare](https://figshare.com/s/9eb3dba4096470af9904), repository doi: 10.6084/m9.figshare.14370965.

#### Supplementary table E2.1

**Supplementary table E2.1:** *Trypanosoma* amplicon sequence data from blood, tick and tissue samples. Data showing zOTUs belonging to the Trypanosomatidae family with results from NCBI BLAST analysis. [Trypanosome zOTU taxonomy and sample data](https://figshare.com/ndownloader/files/30578460)

#### Supplementary table E2.2

**Supplementary table E2.2:** Genetic similarity of piroplasm sequences based on an 852 bp alignment of the 18S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data. 
[Piroplasm genetic diversity table](https://figshare.com/ndownloader/files/30578457)

#### Supplementary table E2.3

**Supplementary table E2.3:** Genetic similarity of *Hepatozoon* sequences based on an 819 bp alignment of the 18S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data. 
[*Hepatozoon* genetic diversity table](https://figshare.com/ndownloader/files/30578463)

---

\clearpage

## Chapter 6{#ch6-supp}

Supplementary material for Tasmanian devil haemoprotozoa chapter.

```{r include=FALSE}
library(readxl)
library(tidyverse)
App_Ch6_seqs <- read_excel("tables/App-Ch6-seqs.xlsx")
```


```{r TA61, echo=FALSE, message=FALSE, warning=FALSE}
library(kableExtra)
opts <- options(knitr.kable.NA = "")
knitr::kable(App_Ch6_seqs, longtable = T, booktabs = TRUE, linesep = "", caption = "\\textit{Trypanosoma} sequences used in phylogenetic analysis in the present study. Sequences produced in the present study indicated in bold. denotes short 18S rRNA sequences only, included for comparison within the \\textit{Trypanosoma cyclops} clade", 
  caption.short = "Sequences used for phylogenetic analysis of \\textit{Trypanosoma} in Tasmanian devils") %>%
  footnote(general = "Country abbreviations: AU - Australia, GB - Brazil, MZ - Mozambique, UK - United Kingdom, MY - Malaysia, SL - Sri Lanka, PNG - Papua New Guinea, DE - Germany, JP- Japan, PL- Poland, BR - Brazil, CA- Canada, PT - Portugal.",
           threeparttable = T) %>%
  kable_styling(full_width = T) %>%
  kable_styling(font_size = 8.0) %>%
  column_spec(1, width = "4em") %>%
  column_spec(2, width = "5em") %>%
  column_spec(3, width = "4em") %>%
  column_spec(4:6, width = "4em") %>%
  column_spec(c(1, 3), italic = T) %>%
  kable_styling(latex_options = c("repeat_header"))  %>% 
  landscape()
```

```{r include=FALSE}
library(readxl)
library(tidyverse)
App_Ch6_pairwise <- read_excel("tables/App-Ch6-pairwise.xlsx")
```

```{r TA62, echo=FALSE, message=FALSE, warning=FALSE}
library(kableExtra)
opts <- options(knitr.kable.NA = "")
knitr::kable(App_Ch6_pairwise, longtable = T, booktabs = TRUE, linesep = "", caption = "Pairwise genetic similarity matrix of the \\textit{Trypanosoma cyclops} clade at the 18S rRNA locus. Analysis conducted over at 559 bp alignment of the V7-8 hypervariable using the Kimura Two-Parameter (K2P) method.", 
  caption.short = "Pairwise genetic similarity matrix of the \\textit{Trypanosoma cyclops} clade") %>%
  kable_styling(font_size = 8.5) %>%
  column_spec(1, italic = T) %>% 
  landscape()
```