<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>2 An update on records of Australian ticks | Ecology of Ticks and Microbes in Australia Wildlife</title>
<meta name="author" content="Siobhon Lee Egan">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="2 An update on records of Australian ticks | Ecology of Ticks and Microbes in Australia Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="2 An update on records of Australian ticks | Ecology of Ticks and Microbes in Australia Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal BMC Research Notes. The following manuscript will be submitted: Egan, S., Evans, M., Fontaine,...">
<meta property="og:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal BMC Research Notes. The following manuscript will be submitted: Egan, S., Evans, M., Fontaine,...">
<meta name="twitter:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal BMC Research Notes. The following manuscript will be submitted: Egan, S., Evans, M., Fontaine,...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="active" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/ulyngs/oxforddown">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="austicks" class="section level1" number="2">
<h1>
<span class="header-section-number">2</span> An update on records of Australian ticks<a class="anchor" aria-label="anchor" href="#austicks"><i class="fas fa-link"></i></a>
</h1>
<embed src="front-and-back-matter/preface/tickmaps.pdf" width="95%" type="application/pdf"></embed><embed src="front-and-back-matter/preface/austicks.pdf" width="95%" type="application/pdf"></embed><div style="page-break-after: always;"></div>
<div id="preface-1" class="section level2 unnumbered">
<h2>Preface<a class="anchor" aria-label="anchor" href="#preface-1"><i class="fas fa-link"></i></a>
</h2>
<p><strong>Attribution Statement</strong></p>
<p>The following chapter has been drafted in accordance with the journal BMC Research Notes.</p>
<p>The following manuscript will be submitted: <strong>Egan, S.</strong>, Evans, M., Fontaine, J., Ryan, U., Irwin, P., and Oskam C. (<em>To be submitted</em>). Australian ticks - Distribution, Hosts and Genetic identification of Ixodida.</p>
<!-- Research note for BMC max 2000 words
Please ensure you adhere to the word limits for research notes:

Introduction, main text and limitations together: 2000 words
List of abbreviation, declarations, references, figures, figure headings, figure legends, tables, table headings and table legends do not count towards the above stated word limits.

https://bmcresnotes.biomedcentral.com/submission-guidelines/preparing-your-manuscript/research-note


Research note is the main article type of BMC Research Notes and is suitable for

Extensions or updates to previously published research
The reporting of additional controls 
Short descriptions of research projects that did not provide publishable results but represent valuable information regarding protocol and data collection
Additions to established (software) tools and experimental or computational methods, e.g. new functionality of user interface, improvements in performance or the release on a new platform
Replication studies
Null results
Data management plans
Authors must clearly acknowledge any work upon which they are building, both published and unpublished. 



--->
<!--
Alternatives 
https://www.springer.com/journal/11686/submission-guidelines#Instructions%20for%20Authors_Types%20of%20Papers
-->
<!--
The following manuscript will be submitted: **Egan, S.**, Evans, M., Ryan, U., Irwin, P., and Oskam C. (In Prep). Distribution and  Australian ticks
-->
<p>The following authors contributed to this manuscript as outlined below<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Contribution indicates the total involvement the author has had in this project. Placing an ‘X’ in the remaining boxes indicates what aspect(s) of the project each author engaged in.&lt;/p&gt;"><sup>3</sup></a>.</p>
<div class="inline-table"><table class="table table" style="font-size: 7px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Authorship order
</th>
<th style="text-align:right;">
Contribution (%)
</th>
<th style="text-align:left;">
Concept Development
</th>
<th style="text-align:left;">
Data Collection
</th>
<th style="text-align:left;">
Data Analyses
</th>
<th style="text-align:left;">
Drafting of manuscript
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Siobhon L. Egan
</td>
<td style="text-align:right;">
80.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
</tr>
<tr>
<td style="text-align:left;">
Megan Evans
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Joseph Fontaine
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Una M. Ryan
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter J. Irwin
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Charlotte L. Oskam
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
<p>By signing this document, the Candidate and Principal Supervisor acknowledge that the information provided is accurate and has been agreed to by all other authors.</p>
<div class="line-block">__________________       __________________<br>
  Candidate                                 Principal Supervisor</div>
<div style="page-break-after: always;"></div>
<p><strong>Chapter linking statement:</strong></p>
<p>This chapter is written as a “data resource” to synthesize information about Australian ticks.
Curation of records has been performed using data collected from a diverse range of sources.
Updated occurrence maps for three highly prevalent species; <em>Amblyomma triguttatum</em>, <em>Ixodes holocyclus</em> and <em>Ixodes tasmani</em> are presented along with an overall map of all 74 tick species present in Australia. This chapter also presents new genetic information for Australian ticks.
Molecular “barcodes” are presented for target mitochondrial loci, providing valuable genetic references for species identification. In addition, a <span class="correction">high-throughput sequencing</span> assay is also described using pan-Ixodida primers.
This provides a high-throughput method to identify ticks by sequencing a ~370 bp product of the 12S rDNA gene on the Illumina MiSeq.
This chapter contains information gathered from a range of resources.
It is presented in the context of a data centric chapter, and will be submitted to journal BMC Research Notes in the format for a ‘Research Note.’</p>
<p>Discussion of the data presented here is provided however to avoid repetition from the previous chapter (literature review) and in accordance with journal requirements (2000 word limit for main text) the overarching aim here, is to provide a central forum for the synthesis of newly generated and curated data.</p>
<p><strong>Acknowledgment statement:</strong>
Thank you to Dr. Bruce Halliday (Australian National Insect Collection, CSIRO) for access to records and loan of specimens, Dr. Mark Harvey (Western Australia Museum) and Dr. Owen Seeman (Queensland Museum) for access to records and Prof. Ian Beveridge (Melbourne University) for providing advice on tick records and collections.
We acknowledge the use of the Atlas of Living Australia, (<a href="https://www.ala.org.au/" class="uri">https://www.ala.org.au/</a>).
We acknowledge that use of records presented in this manuscript has been made possible by the work of previous researchers, we thank them for data collection and curation that has allowed us to conduct this work.</p>
<p><strong>Funding statement:</strong>
This study was part-funded by the Australian Research Council (LP160100200), Bayer HealthCare (Germany) and Bayer Australia.
S.L.E. was supported by an Australian Government Research Training Program (RTP) Scholarship.
This project was also part supported by The Holsworth Wildlife Research Endowment &amp; The Ecological Society of Australia (awarded to S.L.E).</p>
<p><strong>Data availability:</strong></p>
<p>The datasets generated during the current study have been made available in the public repositories and the code used in analysis is available on GitHub.
Illumina MiSeq data generated from the metabarcoding of ticks targeting the 12S rRNA gene has been deposited in the European nucleotide archive under the project accession number PRJEB46056 (ERP130244), which includes the following sample accession numbers: ERS6635126–ERS6635348 (BioSample # SAMEA8952359–SAMEA8952582).
Nucleotide data for a subset of zOTUs generated are available for the molecular identification of ticks and has been uploaded to GenBank under accession numbers MW665133–MW665150.
Nucleotide data for multilocus sequence typing (MLST) barcode sequence data produced by Sanger sequencing has been deposited in GenBank under the following accession numbers: OM791407–OM791437 (<em>COI</em>), OM756760–OM756765 (<em>18S rRNA</em>), OM830716–OM830764 (<em>12S rDNA</em>), OM830384–OM830429 (<em>16S rRNA</em>).
Code used for analysis and supporting data files used for bioinformatics and statistical analysis are available on GitHub repository <a href="https://github.com/siobhon-egan/wildlife-ticks">github.com/siobhon-egan/wildlife-ticks</a> and the <a href="https://siobhonlegan.com/wildlife-ticks/">project website</a>.</p>
<p><strong>Ethics:</strong>
This study was conducted under the compliance of the Australian Code for the Responsibility Conduct of Research (2007) and Australian Code for the Care and Use of Animals for Scientific Purposes, 2013.
No specific animal research was conducted for the purposes of this work. Ticks from wildlife were collected as part of established research projects.</p>
<p><strong>Keywords:</strong> Ixodida, <em>Amblyomma triguttatum</em>, <em>Ixodes holocyclus</em>, <em>Ixodes tasmani</em>, molecular barcoding</p>
<!---
Three to ten keywords representing the main content of the article.

-->
<div style="page-break-after: always;"></div>
</div>
<div id="abstract-1" class="section level2" number="2.1">
<h2>
<span class="header-section-number">2.1</span> Abstract<a class="anchor" aria-label="anchor" href="#abstract-1"><i class="fas fa-link"></i></a>
</h2>
<!-- 
Abstract: 200 words
The abstract should not exceed 200 words. Please minimize the use of abbreviations and do not cite references in the abstract. The abstract must include the following separate sections:

Objective: The purpose and objective of the research presented.
Results: A brief summary of the main findings.
If the data presented is a single observation or the side product of another research project then authors should state this in the abstract under objective.
-->
<p>To date 74 tick species have been recorded in Australia which include 60 Ixodidae (hard ticks) and 14 Argasidae (soft ticks).
Much of the fundamental knowledge of Australian ticks such as species records and occurrence maps are distributed throughout the scientific literature.
This scattered nature can it make it difficult for non-specialists to access basic information, thus, in the present study a synthesis of Australian tick records is provided.
A review of tick records and the incidences of humans as hosts, revealed a total of 28 species biting humans.
Updated occurrence maps revealed that three widespread tick species <em>Amblyomma triguttatum</em>, <em>Ixodes holocyclus</em> and <em>Ixodes tasmani</em> also represent the most common species identified biting humans.
The present study also provides a review of records including curation of those provided in public databases; a <a href="https://siobhonlegan.com/wildlife-ticks/">companion website</a> also provides interactive maps of these records for readers to investigate.
In addition, new genetic data has been generated for mitochondrial loci from Australian ticks.
This information has been deposited in a public database (GenBank) and will assist in future efforts that seek to use molecular tools to confirm species identification.
The analysis of mitochondrial loci identified the 12S rDNA gene (~370 bp product) was particularly useful at delimitating species.
As a result, a <span class="correction">high-throughput sequencing</span> assay was developed for high-throughput species identification.
The curation of records and genetic data generated here provide new insights in Australian ticks.
The findings reported here will be useful to shed new light for future studies on the ecology and systematics of ticks in Australia and be important in assessing public health significance of ticks as vectors for disease.</p>
</div>
<div id="introduction" class="section level2" number="2.2">
<h2>
<span class="header-section-number">2.2</span> Introduction<a class="anchor" aria-label="anchor" href="#introduction"><i class="fas fa-link"></i></a>
</h2>
<!--
The introduction should be brief and provide the motivation/objective for the work presented in the manuscript, e.g.

Where does the data come from?
Why was the data obtained?
If the data presented is a single observation or the side product of another research project then authors should state this in the introduction. This will not negatively impact editorial assessment as BMC Research Notes aims to make single observations available to the scientific community.

We are not looking for a detailed and lengthy introduction to the topic and authors should instead cite relevant review articles. Authors should not provide a general review of the related literature but instead cite relevant work if the manuscript extends previously published or unpublished research.

For data management plans, the introduction should briefly summarize the research project for which the data management plan was written.
-->
<p>Ticks <span class="correction">(Acari: Ixodida)</span> parasitise a range of vertebrate hosts and serve as vectors for numerous pathogens that affect both humans and animals.
Occurrence records of tick species are a valuable tool that is of interest to medical and veterinary industries.</p>
<p>Tick species distribution records provide fundamental information that is required to assess the public health risk in relationship to diseases that ticks can cause.
Curation of these records is a pain-staking process and requires extensive searching not only in published journal articles but also in the ‘grey’ literature and unpublished data sets such as those contained within university theses, government reports, etc <span class="citation">(<a href="references.html#ref-estrada-penaSpeciesOccurrenceTicks2019" role="doc-biblioref">Estrada-Pena et al. 2019</a>)</span>.
Specimen collections housed by museums, universities, and research groups also provide a wealth of information that may not be easily accessible.
Online databases, particularly those where members of the public can submit records, can also provide useful information to build occurrence data <span class="citation">(<a href="references.html#ref-belbinAtlasLivingAustralia2021" role="doc-biblioref">Belbin et al. 2021</a>)</span>.
However, careful consideration of the content of such records may be required where non-experts are providing identifications.</p>
<p>The aim of this study was to provide an update on records of Australian ticks.
The present study provides: (i) a list of tick species in Australia and identification of records for human biting ticks; (ii) updated distribution maps for three widespread tick species and an occurrence map with records of all 74 Australian tick species; (iii) a review of host records for the most widespread tick species in Australia, <em>Amblyomma triguttatum</em>; (iv) new genetic data generated from ticks, useful as reference information for identifications; and finally (v) description and validation of a high-throughput assay for molecular identification of species</p>
<!--

This chapter is designed as a "data resource" for Australian ticks.
Curation of records has been performed using data collected from a diverse range of sources. 
Updated occurrence maps for three highly prevalent species *Amblyomma triguttatum*, *Ixodes holocyclus* and *Ixodes tasmani* are presented along with an overall map of all 74 tick species present in Australia.
This chapter also presents new genetic information for Australian ticks.
Molecular "barcodes" are presented from target mitrochondrial loci which will assist not only in species confirmation but also future phylogenetic and genomic studies of ticks.
A next-generation sequencing sequencing assay is also described using pan-Ixodida primers designed for high-throughput identification of tick specimens.
This assay utilises the Illumina platform and targets a ~370 bp product of the 12S rDNA locus. 
It is hoped that curated records and new data generated in this chapter, including the protocol for high-throughput species identification, will be useful to provide much needed fundamental information on Australian ticks.

While this chapter does contain a wealth of information it is presented in the context of this thesis as data-focused chapter. 
Reflection and discussion of the data presented here is provided, however to avoid repetition from previous chapter (literature review) the over arching aim here is to provide a central forum for the synthesis of newly genera



While the natural history of widespread ticks in the northern hemisphere, such as *Ixodes scapularis* and *Ixodes ricinus*, is well understand, in comparison much less is known about tick species in Australia.

Ticks are classified along side mites Ixodida order comprises three extant families; Ixodidae (hard ticks), Argasidae (soft ticks) and a single species belonging to the Nuttalliellidae family. 


An additional fourth family, Deinocrotonidae, is an extinct taxa that was described after discovery of the newly named species *Deinocroton draculi* in 99 million-year-old Cretaceous amber [@penalverTicksParasitisedFeathered2017].
Although these broad groupings of ticks have been stable for many years now, the phylogeny and systematics of many tick species remains either unclear or lack thorough support. 

All hard and soft ticks have relatively complex life cycles which include four life stages.
As obligate blood feeding arthropods a blood meal is needed for development of immature stages (larvae and nymphs) in order to proceed to the next life stage.
Adult females require a blood meal for development off eggs, and can expand up to 100 times their weight during engorgement [@andersonNaturalHistoryTicks2002].

While the biology and natural history of ticks is understand in a broad context, much of this is based on data obtained from just a handful of species.
Common human biting ticks, *Ixodes ricinus* and *Ixodes scapularis*, are responsible for the majority of tick-borne infections in Europe and North America respectively [@grayWhatWeStill2021;@eisenBlackleggedTickIxodes2018]



In many instances, pathogens acquired by larval feedings are passed to the subsequent life stages, a phenomenon known as trans-stadial transmission. For example, the etiologic agents for Lyme disease, human babesiosis, and Rocky Mountain spotted fever are carried from one feeding stage to another. The duration and phenology of the different motile life stages vary with geographic location, relationship to hosts, and the environmental condi- tions, including the number of hours of daylight to which ticks are exposed.


Molecular tools have helped shed light on the evolutionary history of ticks and as such many changes to tick taxonomy have been made over the years. 
The current working hypothesis of tick phylogeny highlights the uniqueness of tick fauna in the Australasian region. 
Molecular and morphological analysis suggests *Ixodes* endemic to Australia, New Guinea and New Zealand belong to a unique lineage [@barkerSystematicsEvolutionTicks2004]. 
Although molecular data has not been produced from all members of this group there is a clear grouping of what appears to be the Australasian clade.

A relatively newly recognised subfamily, Bothriocrotoninae, was described and includes a group of Australiasian tick species that previously belonged to the Aponomma genus [@klompenNewSubfamilyBothriocrotoninae2002;@keiransAponommaBothriocrotonGlebopalma1994].
*Bothriocroton* is the sole genus in this subfamily, with all seven species considered endemic to Australasia (*Bt. auruginans*, *Bt. concolor*, *Bt. glebopalma*, *Bt. hydrosauri*, *B. oudemansi*, *Bt. tachyglossi* and *Bt. undatum*) (Andrews et al., 2006; Beati et al., 2008; Klompen et al., 2002). 
Further issues have since arisen, particularly surrounding polyphyly of the genus *Amblyomma* (Burger et al., 2013; Burger et al., 2012).
In an attempt to resolve these two new genera have recently been described [@barkerTwoNewGenera2018].
The genus *Robertsicus* was established and includes *Robertsicus elaphensis*, previously *Amblyomma elaphense*, known as the Trans-Pecos rat-snake tick which is present in Mexico and southeastern USA. 
The second new genus *Archaeocroton* was proposed for *Archaeocroton sphenodonti*, formerly *Amblyomma sphenodonti*, known as the tuatara tick of New Zealand.


Morphological identification remains the gold standard to differentiate tick species. 
The main method for tick identification is through the use of dichotomous keys and descriptions. 
However there are many instances were species descriptions for all life stages are not available, or the current description is not adequate for accurate differentiation [@guglielmoneIxodidaeAcariIxodoidea2020]. 
Without the comprehensive description of similar species it is difficult to conclude diagnostic features. 
Additionally, original species descriptions are generally only based on very few specimens, and as such morphological descriptions may not reflect the true variation of features within a species.

Construction of a DNA database for ticks collected in Japan: application of molecular identification based on the mitochondrial 16S rDNA gene [@takanoConstructionDNADatabase2014].

The present study provides a review of Australian ticks and in particular an extensive search of human host records. 
Molecular barcodes have also been generated for a selection of Australian ticks species and a high-throughput method to identify ticks using the Illumina platform is described.

*Amblyomma triguttatum* Koch 1844, commonly known as the ornate kangaroo tick, is endemic to Australia and widespread across the mainland. 
The first extensive study of *Am. triguttatum* was by Roberts [-@robertsStatusMorphologicallyDivergent1962] who described four distinct subspecies of this tick, namely; *Am. t. triguttatum*, *Am. t. ornatissimum*, *Am. t. queenslandensis*, and *Am. t. rosei*.

Interestingly it is noted that the original description by Koch in 1844, and additional detail later added in 1847 noted a female with 3 pale patches, perhaps indicating the type specimen may in fact be *Am. t. ornatissimum*. 
Roberts [-@robertsStatusMorphologicallyDivergent1962] noted a wide variation in characterisation among both sexes particularly with respect to ornamentation patterns. 
Roberts gave that the taxonomy of the *Amblyomma triguttatum* group a provisional status, and noted that further research may warrant a change of rank.

*Amblyomma triguttatum* is of zoonotic significance, dating back to its implication in the transmission of Q Fever [@popeCoxiellaBurnetiKangaroos1960]. While it is accepted that infection of *C. burnettii* from a tick bite is rare, and the pathogen is more commonly acquired through aerosol ticks play an important part in the life cycle of this pathogen. There is also growing support that *Am. triguttatum* is a likely vector of a zoonotic spotted fever group rickettsiae (*Rickettsia gravesii*) [@abdadRickettsiaGravesiiSp2017;@chaladaThereLymelikeDisease2016;@liHighPrevalenceRickettsia2010]. 
Additionally novel tick-borne microbes continue to be identified from this tick species [@eganBacterialCommunityProfiling2020;@goftonDetectionPhylogeneticCharacterisation2017].

The aim of this study was to provide an update on records of Australian ticks. The present study provide: (i) list of tick species in Australia and identification of records for human biting ticks; (ii) updated distribution maps for three widespread tick species and occurrence map with records of all 74 Australian tick species; (iii) review of host records for the most widespread tick species in Australia, *Am. triguttatum*; (iv) new genetic data generated from ticks, useful as reference information for identifications; and finally (v) description and validation of high-throughput assay for molecular identification of species.



(i) provide an updated distribution of the *Am. triguttatum*, including a review of host records and (ii) investigate the taxonomic stability of the *Am. triguttatum* species group using a multilocus sequence typing MLST approach.
-->
</div>
<div id="main-text" class="section level2" number="2.3">
<h2>
<span class="header-section-number">2.3</span> Main text<a class="anchor" aria-label="anchor" href="#main-text"><i class="fas fa-link"></i></a>
</h2>
<!--
This should contain the body of the research note, and may also be broken into subsections with short and informative headings. Methods should be described in sufficient detail to allow repeatability. Authors should concisely describe the data or results they present and provide a critical discussion of the findings within the context of the research field. If an observation cannot be explained or put in context of the current literature then authors are encouraged to state that.
-->
<div id="methods" class="section level3" number="2.3.1">
<h3>
<span class="header-section-number">2.3.1</span> Methods<a class="anchor" aria-label="anchor" href="#methods"><i class="fas fa-link"></i></a>
</h3>
<div id="tick-records" class="section level4" number="2.3.1.1">
<h4>
<span class="header-section-number">2.3.1.1</span> Tick records<a class="anchor" aria-label="anchor" href="#tick-records"><i class="fas fa-link"></i></a>
</h4>
<p>Records were sourced from the Australian National Insect Collection (ANIC), the Western Australia Museum (WAM), Cryptick Laboratory tick archive (Murdoch University) and the literature (including a thorough review of grey literature, e.g., government reports).
The <a href="https://www.ala.org.au/">Atlas of Living Australia</a> was also investigated to identify records of tick species.
Due to the known issues with accurate identification of records in the Living Atlas Australia database <span class="citation">(<a href="references.html#ref-belbinAtlasLivingAustralia2021" role="doc-biblioref">Belbin et al. 2021</a>)</span>, this data was further investigated.
Tick species with occurrence records outside of their known historic distribution area were verified for likelihood of accurate identification.</p>
<p>Host common names and scientific names are included and follow current taxonomic guidelines as described for mammals <span class="citation">(<a href="references.html#ref-jacksonTaxonomyAustralianMammals2015" role="doc-biblioref">S. Jackson and Groves 2015</a>)</span>, except where the taxonomic status has since been updated.</p>
<p><strong>Most prevalent species</strong><br>
The three most common species identified from the analysis above were then selected to produce individual species maps.
This also included a more detailed curation of records from the grey literature.
As the most abundant and widespread species identified, further investigation of sporadic <em>Am. triguttatum</em> was performed to confirm specimen identification.
Tick specimens that were not identified to species level in the ANIC Ixodida collection from New South Wales, Victoria and Tasmania were examined.
Historically <em>Am. triguttatum</em> records from these regions have been scarce, and a confirmed identification from these areas could vastly expand the recognised distribution of <em>Am. triguttatum</em>.
Due to the widespread nature of <em>Am. triguttatum</em> a list of host records was also synthesied for this species.</p>
</div>
<div id="tick-identification" class="section level4" number="2.3.1.2">
<h4>
<span class="header-section-number">2.3.1.2</span> Tick identification<a class="anchor" aria-label="anchor" href="#tick-identification"><i class="fas fa-link"></i></a>
</h4>
<p>Samples were visualised using an Olympus SZ61 stereomicroscope (Olympus, Centre Valley, PA, United States) with an external Schott KL 1500 LED (Schott AG Mainz, Germany) light source.
Photographs of tick specimens were taken using an Olympus SC30 digital camera (Olympus, Centre Valley, PA, United States) and analysis getIT software (Olympus, Centre Valley, PA, United States).
Instar, sex, and species was identified using a combination of available morphological keys and species descriptions <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">F. S. H. Roberts 1970</a>; <a href="references.html#ref-jacksonMorphologicalComparisonAdult2002" role="doc-biblioref">J. Jackson et al. 2002</a>; <a href="references.html#ref-laanObservationsBiologyDistribution2011" role="doc-biblioref">Laan, Handasyde, and Beveridge 2011a</a>; <a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">Stephen C. Barker and Walker 2014</a>; <a href="references.html#ref-kwakPhylogeneticAnalysisAustralian2017" role="doc-biblioref">Kwak et al. 2017</a>)</span>.
Where possible <em>Am. triguttatum</em> specimens were assigned to subspecies according to that described in <span class="citation"><a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">F. H. S. Roberts</a> (<a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">1962</a>)</span>.</p>
</div>
<div id="genetic-analysis-of-ticks" class="section level4" number="2.3.1.3">
<h4>
<span class="header-section-number">2.3.1.3</span> Genetic analysis of ticks<a class="anchor" aria-label="anchor" href="#genetic-analysis-of-ticks"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Sample collection</strong><br>
Specimens in the Cryptick Laboratory tick archive were used for molecular assays and generation of new sequence data.
These ticks were collected since 2016 from various sources such as wildlife rehabilitation centres, veterinary clinics and research projects including collection of ticks from the environment or from wildlife.
In the case of ticks collected from wildlife, samples were collected as part of the research project described in Chapters <a href="wildlife-bacteria.html#wildlife-bacteria">3</a> and <a href="wildlife-haemoprotozoa.html#wildlife-haemoprotozoa">4</a>.</p>
<p><strong>DNA extraction and sequencing</strong><br>
The complete details of the methods is described in the <a href="supplementary-material.html#ch2supp">A.1</a>.
In brief, DNA was extracted from ticks and two molecular approaches were used: (i) multi-locus sequence typing (MSLT) assays with Sanger sequencing and (ii) development of a high-throughput assay to identify ticks.</p>
<p>For the MLST assays, primers were used to amplify ticks at the following mitochondrial loci: <em>cytochrome c oxidase subunit I</em> (<em>COX1</em>) <span class="citation">(<a href="references.html#ref-songPhylogeneticPhylogeographicRelationships2011" role="doc-biblioref">Song et al. 2011</a>)</span>, <em>12S rDNA</em> <span class="citation">(<a href="references.html#ref-beatiAnalysisSystematicRelationships2001" role="doc-biblioref">Beati and Keirans 2001</a>)</span>, <em>16S rDNA</em> <span class="citation">(<a href="references.html#ref-lvDevelopmentDNABarcoding2014" role="doc-biblioref">Lv, Wu, Zhang, Zhang, et al. 2014</a>)</span> (see Table <a href="supplementary-material.html#tab:T2primers">A.1</a>).
Purified PCR products were then subjected to Sanger sequencing in the forward and reverse direction.</p>
<p>Following the results from the MLST approach, the ~370 bp product of the 12S rDNA gene was determined as providing optimal results for species delimitation.
In addition, the short fragment size made it suitable to transfer the assay onto the Illumina MiSeq platform.
In brief, extracted tick DNA was used to build libraries following the 16S Metagenomic Sequencing Library Preparation (Illumina Part # 15044223 Rev. B).
The amplicon PCR was carried out using the 12S rDNA gene pan-Ixodida primers, T1B and T2A <span class="citation">(<a href="references.html#ref-beatiAnalysisSystematicRelationships2001" role="doc-biblioref">Beati and Keirans 2001</a>)</span> (~370 bp product) with Illumina MiSeq adapters (Table <a href="supplementary-material.html#tab:T2primers">A.1</a>).
Libraries were then constructed and sequenced on the Illumina MiSeq using v2 chemistry (2 x 250 paired end).</p>
</div>
<div id="data-analysis-and-bioinformatics" class="section level4" number="2.3.1.4">
<h4>
<span class="header-section-number">2.3.1.4</span> Data analysis and bioinformatics<a class="anchor" aria-label="anchor" href="#data-analysis-and-bioinformatics"><i class="fas fa-link"></i></a>
</h4>
<p>Distribution maps were produced in RStudio <span class="citation">(<a href="references.html#ref-rstudioteamRStudioIntegratedDevelopment2015" role="doc-biblioref">RStudio Team 2015</a>)</span>.
Maps of Australia were drawn using ozmaps <span class="citation">(<a href="references.html#ref-SumnerOzmaps2021" role="doc-biblioref">Sumner 2021</a>)</span>, sf <span class="citation">(<a href="references.html#ref-PebesmaRsf2018" role="doc-biblioref">Pebesma 2018</a>)</span> and sp <span class="citation">(<a href="references.html#ref-BivandRsp2013" role="doc-biblioref">Bivand, Pebesma, and Gomez-Rubio 2013</a>)</span> R packages.</p>
<p>Sequences obtained from MLST assays were imported into Geneious 10.2.6 (<a href="https://www.geneious.com" class="uri">https://www.geneious.com</a>) for quality inspection.
Illumina MiSeq data was analysed using a bioinformatic pipeline with the program USEARCH v11 <span class="citation">(<a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>)</span> with zero radius operational taxonomic units generated (zOTUs).
All sequences were subject to BLAST analysis (BLASTN 2.11.0+ <span class="citation">(<a href="references.html#ref-zhangGreedyAlgorithmAligning2000" role="doc-biblioref">Zhang et al. 2000</a>; <a href="references.html#ref-morgulisDatabaseIndexingProduction2008" role="doc-biblioref">Morgulis et al. 2008</a>)</span>) against the NCBI nucleotide collection (nt) database to confirm identification.</p>
<p>Full details of analysis methods are available in supplementary material <a href="supplementary-material.html#ch2supp">A.1</a> and at the repository <a href="https://github.com/siobhon-egan/wildlife-ticks" class="uri">https://github.com/siobhon-egan/wildlife-ticks</a>.</p>
</div>
</div>
<div id="results-and-discussion" class="section level3" number="2.3.2">
<h3>
<span class="header-section-number">2.3.2</span> Results and Discussion<a class="anchor" aria-label="anchor" href="#results-and-discussion"><i class="fas fa-link"></i></a>
</h3>
<!-- In Research articles, the Results should be presented in a separate section from the Discussion. If you have a specific reason for writing a combined section, you should make a case for this in your covering letter for the Editor-in-Chief’s consideration. Results generally should not contain references. 

-->
<div id="australian-ticks" class="section level4" number="2.3.2.1">
<h4>
<span class="header-section-number">2.3.2.1</span> Australian ticks<a class="anchor" aria-label="anchor" href="#australian-ticks"><i class="fas fa-link"></i></a>
</h4>
<p>A list of all 74 tick species recorded in Australia is presented in Table <a href="austicks.html#tab:T2humanrecords">2.1</a> and includes 60 hard ticks and 14 soft ticks.
Five tick species are identified as introduced and are associated with domesticated livestock and companion animals; these include two soft ticks <em>Argas persicus</em> (associated with poultry) and <em>Otobius megnini</em> (imported with horses); and three hard ticks, <em>Haemaphysalis longicornis</em> (associated with cattle), <em>Rhipicephalus linnaei</em> (syn <em>Rhipicephalus sanguineus</em> tropical lineage) (associated with dogs) and <em>Rhipicephalus australis</em> (syn <em>Boophilus microplus</em>) (associated with cattle).</p>
<p><strong>Human records</strong><br>
A review of human incidents of tick bite in Australia identified 28 species reported.
In many cases records of species biting humans are confined to sporadic single identifications (Table <a href="austicks.html#tab:T2humanrecords">2.1</a>).
It is noted that reporting of a “human” record with ticks reveals little regarding the potential significance for zoonotic tick-borne diseases.
The identification of ticks from human hosts does not necessarily distinguish if the tick was attached and actively feeding, or if the tick was simply found crawling on the individual.
While this omission might seem trivial it is a key piece of information in establishing an accurate list of host records, particularly where records of the tick-host association are sparse.
In addition, it is important to note in regard to tick-borne diseases, the tick vector remains just one of three components in the cycle of tick-borne pathogens; the dynamics of microbe and reservoir host(s) are also requirements for a zoonotic pathogen to become endemic.
Identification of incidents, referred to as “tick encounters,” may prove a more useful tool to quantify the risk of tick bites to people.
While this field of research has been conducted in the northern hemisphere <span class="citation">(<a href="references.html#ref-hookHumanTickEncounters2021" role="doc-biblioref">Hook et al. 2021</a>)</span>, similar studies have not been conducted in Australia.</p>
<p>The list of human-tick records in Australia includes a diverse range of species however, the overwhelming evidence concludes that <em>Ix. holocyclus</em> and <em>Am. triguttatum</em> contribute to the vast majority (&gt;90%) of tick bites <span class="citation">(<a href="references.html#ref-goftonBacterialProfilingReveals2015" role="doc-biblioref">Gofton, Doggett, et al. 2015</a>; <a href="references.html#ref-geary30YearsSamples2020" role="doc-biblioref">Geary et al. 2020</a>)</span>.
An interesting observation in the literature was the identification of the platypus tick <em>Ixodes ornithorynchi</em> from a human in northern Victoria <span class="citation">(<a href="references.html#ref-geary30YearsSamples2020" role="doc-biblioref">Geary et al. 2020</a>)</span>. The specimen was submitted from a wildlife carer who had direct contact with platypus. In addition to the 28 species reporting biting human two tick species, <em>Argas lagenoplastis</em> and <em>Ornithodorus macmillani</em> were noted by <span class="citation"><a href="references.html#ref-geary30YearsSamples2020" role="doc-biblioref">Geary et al.</a> (<a href="references.html#ref-geary30YearsSamples2020" role="doc-biblioref">2020</a>)</span> from birds’ nests but were not directly associated with biting humans.</p>
<p>This review has provided a synthesis of tick-host records in Australia.
It is evident that some publications have reported a “new” host record, however review of extant museum collections or historic data has shown it had been previously recorded.
For example, an incidence of <em>Ix. australiensis</em> biting a human was reported as a first record by <span class="citation"><a href="references.html#ref-kwakFirstRecordsHuman2018" role="doc-biblioref">Kwak</a> (<a href="references.html#ref-kwakFirstRecordsHuman2018" role="doc-biblioref">2018</a>)</span>, occurring in October 2017.
However, <span class="citation"><a href="references.html#ref-rabyNewFociSpotted2016" role="doc-biblioref">Raby et al.</a> (<a href="references.html#ref-rabyNewFociSpotted2016" role="doc-biblioref">2016</a>)</span>, reported the tick biting a person in May 2006.
In addition, we note that <em>Ix. australiensis</em> wwas reported in close association with humans by <span class="citation"><a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">F. S. H. Roberts</a> (<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>)</span>, however it was noted the tick was “crawling.” An exhaustive review of all host records is not practical where the data is widely distributed in the literature; this serves to demonstrate the value of curated data, especially where it has the potential to be of public health significance.</p>
<!---
The identification of new human host records is of public health importance.
While much research related to human tick-borne pathogens is centred around the two most commonly encountered species, *Ix. holocyclus* and *Am. triguttatum*, it is important other tick species are not overlooked.

An early study after the initial discovery of Flinders Island spotted Fever (FISF) was conducted at sites on Flinders Island and Gippsland, Victoria [@gravesSpottedFeverGroup1993]. While the study identified SFG rickettsia from *Ix. cornuatus* and Gippsland, no ticks from Flinders Island were positive. 
Serology on wildlife blood sampled detected SFG black rats (*Rattus rattus*), unknown rat species (*Rattus* sp.), brush-tailed possum (*Trichosurus vulpecula*) and the vombat (*V. ursinus*) from Flinders Island. 
At Gippsland sites positive wildlife hosts included bush rats (*R.fuscipes*), swamp rats (*R. lutreolus*), the common house mouse (*M. musculus*) and a dusky antechinus (*Antechinus swainsonii*).

Subsequent studies later identified an association between the southern reptile tick *Bt. hydrosauri* and the causative agent of FISF, *R. honei* [@stenosAponommaHydrosauriReptileassociated2003]. 

However, the high identification of SFG in companion animals in Tasmania [@izzardSerologicalPrevalenceStudy2010] and the identification of SFG rickettsia in wildlife during earlier studies [@gravesSpottedFeverGroup1993] mean that questions surrounding the vector(s) of SFG rickettsia remain. Therefore, research and diagnosis should not be limited to associations with just the southern reptile tick *Bt. hydrosauri*. 
It is important that ticks found biting humans are accurately identified and information is disseminated in an appropriate public manner such as publication of results in scientific journals.
Accurate information relating to tick-host records is important to ensure not only a timely clinical outcome but also to assist the scientific community and assess what tick species and related pathogens may pose a zoonotic risk.

**Notes on triguttatum from [@guglielmoneTickSpeciesFound2018]**

>Australia: Roberts (1962) reported five females and three males of A. triguttatum found on people at Hughenden in Queensland, at an unknown locality in New South Wales, and at three localities in the state of Western Australia. Pearce & Grove (1987) collected more than 100 larvae, about 75 nymphs and seven adults of A. triguttatum from soldiers bivouacked in the Perth Region in Western Australia, where Owen et al. (2006) collected 32 specimens from peo- ple on Barrow Island, but tick stages were not provided, a record repeated in Abdad et al. (2017). Waudby et al. (2007) reported human parasitism by this tick at Innes National Park, South Australia, but, again, tick stages were not provided, while Gofton et al. (2015) collected 103 nymphs, 40 females and 24 males of this species from people over a three-year study in southeastern Queensland and the southwestern area of Western Australia. Barker & Walker (2014) listed new records from humans, but tick stages and collection localities were not provided. There have been some cases of human parasitism detected outside Australia, as in Heath & Hardwick (2011), who listed five persons infested with A. trigut- tatum in Australia and subsequently intercepted upon arrival in New Zealand. Merten & Durden (2000) reported a female tick on a person who had entered the USA (repeated in Keirans & Durden 2001).
Note: after an analysis of several populations of A. triguttatum, Roberts (1962) created four subspecies, an indication that more than one species may be included under this name.


>Australia: there are several records of human parasitism by H. bancrofti in the State of Queensland, where Roberts (1934a) reported this tick from people, though tick stages were not provided. Roberts (1963) found a nymph on a per- son at Brisbane, and Domrow & Derrick (1965) found a nymph attached to a human at Ravensbourne. Gofton et al. (2015) recorded seven cases of human parasitism by six nymphs and one female of H. bancrofti at two localities in southeastern Queensland and two localities in eastern New South Wales. Laan et al. (2011) published several records of this species from persons in New South Wales, Queensland and the Australian Capital Territory, but most of the ticks were not attached. Heath & Hardwick (2011) reported a case of an Australian traveler who was found infested with H. bancrofti upon arrival in New Zealand, but the tick stage was not provided.
-->
<div class="inline-table"><table class="table table" style="font-size: 8px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T2humanrecords">Table 2.1: </span>List of tick species recorded from Australia. Human records indicated by colour.
</caption>
<thead><tr>
<th style="text-align:left;">
Species
</th>
<th style="text-align:left;">
Authority
</th>
<th style="text-align:left;">
H (a)
</th>
<th style="text-align:left;">
Hosts
</th>
<th style="text-align:left;">
Notes
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 4cm; ">
 (stump-tailed lizard tick )
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1907
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards and snakes)
</td>
<td style="text-align:left;width: 6cm; ">
Similar to 
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1905
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna (possibly reptiles)
</td>
<td style="text-align:left;width: 6cm; ">
Inornate species. Very few records, description from long-beaked echidna,  (now regionally extinct) in Western Australia (historical). Most similar to .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Neuman 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Pigs, rodents
</td>
<td style="text-align:left;width: 6cm; ">
Syn. . Few records of this in Australia.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Calaby’s goanna tick)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1963
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (large monitor lizards)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1953
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna
</td>
<td style="text-align:left;width: 6cm; ">
Morphologically similar to .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (tropical reptile tick)
</td>
<td style="text-align:left;width: 3cm; ">
Koch 1844
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards and snakes)
</td>
<td style="text-align:left;width: 6cm; ">
Previously  (eye-less ). Syn. .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Keirans, King and Sharrad 1994
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (reptile tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards and snakes)
</td>
<td style="text-align:left;width: 6cm; ">
Similar to ; In addition historical identifications maybe confused with .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (seabird tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1907
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
Syn.  Roberts 1953.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1953
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Kangaroos
</td>
<td style="text-align:left;width: 6cm; ">
Very few records, only from “kangaros” in north east QLD.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (snake tick)
</td>
<td style="text-align:left;width: 3cm; ">
Koch 1867
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (snakes)
</td>
<td style="text-align:left;width: 6cm; ">
Previously thought to be subspecies of . Syn .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1953
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Hirt 1914
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Wallaby and kangaroos
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (ornate kangaroo tick)
</td>
<td style="text-align:left;width: 3cm; ">
Koch 1844
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (kangaroos)
</td>
<td style="text-align:left;width: 6cm; ">
Four subspecies as described by Roberts 1962.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Lucas 1878
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards and snakes)
</td>
<td style="text-align:left;width: 6cm; ">
Previously , (eye-less ). Syn .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Keirans, Bull and Duffield 1996
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards)
</td>
<td style="text-align:left;width: 6cm; ">
Can be confused with  and sometimes .  is ornamented with a light background coloration, little ornamentation in the scapular areas and ornamentation on all festoons.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (wombat tick)
</td>
<td style="text-align:left;width: 3cm; ">
Schulze 1936
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Wombats
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (short-beaked echidna tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna
</td>
<td style="text-align:left;width: 6cm; ">
Often confusion with  in earlier descriptions. Changes in taxonomy mean many syn. names over time e.g. , , .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Keirans, King and Sharrad 1994
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (southern reptile tick)
</td>
<td style="text-align:left;width: 3cm; ">
Denny 1843
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards and snakes)
</td>
<td style="text-align:left;width: 6cm; ">
Female holotype maybe  or . Previous indentifications remain elusive as syn. with many species in earlier descriptions (e.g. , , and  to name a few). Syn , , .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (central Queensland short-beaked echidna tick)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1953 (re-instated by Andrews et al. 2006)
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna
</td>
<td style="text-align:left;width: 6cm; ">
Although described by Roberts in 1960, later considered a subsp. of  and therefore not recognised in Roberts 1970 keys. Andrews et al. 2006 provided more detailed description and raised it to species status.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (goana tick)
</td>
<td style="text-align:left;width: 3cm; ">
Fabricius 1775
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Reptiles (lizards ), echidna
</td>
<td style="text-align:left;width: 6cm; ">
Syn. , , .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (wallaby tick)
</td>
<td style="text-align:left;width: 3cm; ">
Nuttall and Warburton 1915
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Wallabies and kangaroos
</td>
<td style="text-align:left;width: 6cm; ">
Historical records show it may have been confused with . Syn with , .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Bremner’s possum tick)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1963
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Possum
</td>
<td style="text-align:left;width: 6cm; ">
Can be confused with  or .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Ooenitz Oriental – Australian bird haemaphysalid)
</td>
<td style="text-align:left;width: 3cm; ">
Warburton and Nuttall 1909
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Birds
</td>
<td style="text-align:left;width: 6cm; ">
Presence in Australia is recognised but not well documented.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (bandicoot tick)
</td>
<td style="text-align:left;width: 3cm; ">
Warburton and Nuttall 1909
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (bandicoots, wallabies, rodents)
</td>
<td style="text-align:left;width: 6cm; ">
Can be confused with  or .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1963
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Wallabies and bandicoot
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (bush tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1901
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (kangaroos)
</td>
<td style="text-align:left;width: 6cm; ">
Introduced.Syn.  Nuttall and Warburton 1915 (record presence in Australia), .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Hirt 1914
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammal
</td>
<td style="text-align:left;width: 6cm; ">
Syn with .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (rock-wallaby tick)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1970
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammal (wallabies)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (rat tick)
</td>
<td style="text-align:left;width: 3cm; ">
Kohls 1948
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammal (rats and small marsupials)
</td>
<td style="text-align:left;width: 6cm; ">
Can be confused with  or .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Antechinus tick)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1960
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Small mammals (antechinus)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (New Zealand seabird tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1904
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
Can be confused with .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1960
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammals (kangaroos, bandicoots, possums)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Barker 2019
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Echidna
</td>
<td style="text-align:left;width: 6cm; ">
Distinct horn-like projection on palpal article 1. Some similar to  Kohls, 1960 from the long-beaked echidna of Papua New Guinea.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1960
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammals (possums, bandicoots)
</td>
<td style="text-align:left;width: 6cm; ">
Records of this tick in Australia are sparse (previous confused with ), Roberts 1970 reports just a single specimen, a female from Innisfail north Queensland.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1908
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammals (possums, rodents)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (southern paralysis tck)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1960
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (possums, wallabies, rodents)
</td>
<td style="text-align:left;width: 6cm; ">
Earlier records difficult to conclude due to similarity with .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Maskell 1885
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Warburton and Nuttall 1904
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (kangaroos, bandicoots, possums, wallabies, rodents)
</td>
<td style="text-align:left;width: 6cm; ">
Similar to .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (mountain pygmy possum tick)
</td>
<td style="text-align:left;width: 3cm; ">
Kwak, Madden and Wicker 2018
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mountain pygmy possum
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Hirst’s marsupial tick)
</td>
<td style="text-align:left;width: 3cm; ">
Hassall 1931
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (possums, wallabies, rodents) and birds
</td>
<td style="text-align:left;width: 6cm; ">
See Laan et al. 2011 for description of immature instars, and further details on adults.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (paralysis tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (kangaroos, bandicoots, possums, wallabies, rodents)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (water rat tick)
</td>
<td style="text-align:left;width: 3cm; ">
Swan 1931
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Rodents
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Andre and Colas-Belcour 1942
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
Many syns: .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Arthur 1955
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Kopstein’s bat tick)
</td>
<td style="text-align:left;width: 3cm; ">
Oudemans 1926
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Bats
</td>
<td style="text-align:left;width: 6cm; ">
Very few records of this ticks in Australia. Roberts 1970 refers to a single Australian records based on personal communication. Record from Tadarida colonicus, Nicholson River Northern Territory.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Heath and Palma 2017
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
Syn. 
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1962
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Numbat
</td>
<td style="text-align:left;width: 6cm; ">
Syn.  pre-1960. See Kwak et al. 2018 for new description of nymph and male and redescription of female.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (platypus tick)
</td>
<td style="text-align:left;width: 3cm; ">
Lucas 1846
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Platypus
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (bat tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1906
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Bats
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (common marsupial tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various mammals (kangaroos, bandicoots, possums, wallabies, rodents)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (possum tick)
</td>
<td style="text-align:left;width: 3cm; ">
Roberts 1960
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Mammals (possums, bandicoots)
</td>
<td style="text-align:left;width: 6cm; ">
See McCann et al. 2019. Reported high genetic diversity (2 distinct genotype, may have species status).
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
White 1852
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (numbat tick)
</td>
<td style="text-align:left;width: 3cm; ">
Warburton and Nuttall 1909
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Numbat
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Nuttall 1916
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Wombats and macropods
</td>
<td style="text-align:left;width: 6cm; ">
See Weaver 2016 for detailed redescription of this species.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (woylie tick)
</td>
<td style="text-align:left;width: 3cm; ">
Ash et al. 2017
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Woylie
</td>
<td style="text-align:left;width: 6cm; ">
Most similar to , however these two species can be readily differentiated by dentition and the shape of the scutum, spurs on the coxae, and palpal article 1.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Australian cattle tick)
</td>
<td style="text-align:left;width: 3cm; ">
Fuller 1899
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Cattle
</td>
<td style="text-align:left;width: 6cm; ">
Introduced. Syn. .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (brown dog tick)
</td>
<td style="text-align:left;width: 3cm; ">
Latreille 1806
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Dogs
</td>
<td style="text-align:left;width: 6cm; ">
Introduced. Recent taxonomic change in 2021, previously  tropical lineage. Since the recently described neotype of the species (See Nava et al 2018), a new name has been proposed based on molecular divergence (see Chandra et al. 2019 and Slapeta et al. 2021).
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Kaiser and Hoogstraal 1974
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Birds (kestrel)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Australian fairy martin argasid)
</td>
<td style="text-align:left;width: 3cm; ">
Froggatt 1906
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Birds (fairy martin)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Lowryae’s bird argasid)
</td>
<td style="text-align:left;width: 3cm; ">
Kaiser and Hoogstraal 1975
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Birds (kestrel)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Hoogstraal and Kaiser 1973
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Birds (crow)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (fowl tick)
</td>
<td style="text-align:left;width: 3cm; ">
Oken 1818
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Domestic poultry
</td>
<td style="text-align:left;width: 6cm; ">
Introduced.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Robert’s bird tick)
</td>
<td style="text-align:left;width: 3cm; ">
Hoogstraal, Kaiser and Kohls 1968
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Birds (egrets, domestic poultry)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">

</td>
<td style="text-align:left;width: 3cm; ">
Kohls and Hoogstraal 1962
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Bats (presumed)
</td>
<td style="text-align:left;width: 6cm; ">
Recent change of name (see Mans et al. 2019) previously .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (seabird soft tick)
</td>
<td style="text-align:left;width: 3cm; ">
Neumann 1901
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
Seabirds
</td>
<td style="text-align:left;width: 6cm; ">
Recent change of name (see Mans et al. 2019) previously .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Davies’ bat argasid)
</td>
<td style="text-align:left;width: 3cm; ">
Kaiser and Hoogstraal 1973
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Bats
</td>
<td style="text-align:left;width: 6cm; ">
Recent change of name (see Mans et al. 2019) previously .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (Dewae’s bat argasid)
</td>
<td style="text-align:left;width: 3cm; ">
Kaiser and Hoogstraal 1974
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Bats
</td>
<td style="text-align:left;width: 6cm; ">
Recent change of name (see Mans et al. 2019) previously .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (ghost bat argasid)
</td>
<td style="text-align:left;width: 3cm; ">
Hoogstraal, Moorhouse, Wolf and Wassef 1977
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Bats
</td>
<td style="text-align:left;width: 6cm; ">
Recent change of name (see Mans et al. 2019) previously .
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (kangaroo soft tick)
</td>
<td style="text-align:left;width: 3cm; ">
Warburton 1926
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #8DD3C7 !important;text-align: c;">Y</span>
</td>
<td style="text-align:left;width: 4cm; ">
associated with kangaroos
</td>
<td style="text-align:left;width: 6cm; ">
Despite its common name host records are very sparse, and no records of tick attached to host.
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (possum soft tick)
</td>
<td style="text-align:left;width: 3cm; ">
Hoogstraal and Kohls 1966
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Various (possums, birds)
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 4cm; ">
 (spinose ear tick)
</td>
<td style="text-align:left;width: 3cm; ">
Duges 1883
</td>
<td style="text-align:left;width: 1cm; ">
<span style="     color: white !important;border-radius: 4px; padding-right: 4px; padding-left: 4px; background-color: #BEBADA !important;text-align: c;">N</span>
</td>
<td style="text-align:left;width: 4cm; ">
Domestic horse
</td>
<td style="text-align:left;width: 6cm; ">
Introduced.
</td>
</tr>
</tbody>
<tfoot><tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Reference of human biting ticks sourced from published and grey literature including museum records.
</td>
</tr></tfoot>
</table></div>
<div style="page-break-after: always;"></div>
</div>
<div id="distribution-maps" class="section level4" number="2.3.2.2">
<h4>
<span class="header-section-number">2.3.2.2</span> Distribution maps<a class="anchor" aria-label="anchor" href="#distribution-maps"><i class="fas fa-link"></i></a>
</h4>
<p>An updated map of curated records for all 74 species of tick present in Australia is presented in Figure <a href="austicks.html#fig:F2mapall">2.1</a>.
Records without a species identification or those with missing location data were excluded.
A final dataset of 6,282 observation records was used to build the occurrence map (Figure <a href="austicks.html#fig:F2mapall">2.1</a>).</p>
<p>It was noted that two species were not present in any museum record searches.
<em>Argas lowryae</em> was described by <span class="citation"><a href="references.html#ref-kaiserObservationsSubgenusArgas1975" role="doc-biblioref">Kaiser and Hoogstraal</a> (<a href="references.html#ref-kaiserObservationsSubgenusArgas1975" role="doc-biblioref">1975</a>)</span>, and to the best of the authors’ knowledge it has not been recorded since these initial observations.
Another soft tick, the invasive spinose ear tick <em>Otobius mengini</em>, was also not identified from any museum records.
Instead, a single observation of the species in Perth, Western Australia, reported by the state government <span class="citation">(<a href="references.html#ref-mayberrySpinoseEarTick2003" role="doc-biblioref">Mayberry 2003</a>)</span> was used in the occurrence map.
To complete the occurrence map, data for missing tick species was sourced from the literature.
Where possible, records for species that were outside of their historical distribution were individually inspected.</p>
<p><strong>Atlas of Living Australia data</strong></p>
<p>In comparison to the curated data (Figure <a href="austicks.html#fig:F2mapall">2.1</a>), a map of records based solely on data from Atlas of Living Australia was used to build an occurrence map.
This analysis only identified 57 species (50 hard ticks and seven soft ticks).
Once records with missing data (i.e., no species level identification) were removed, a total of 2,293 records were identified and used to the build map presented in Figure <a href="supplementary-material.html#fig:FA21">A.1</a>.</p>
<p><strong>Most prevalent species</strong></p>
<p>The species with the highest number of observations were <em>Am. triguttatum</em> (n = 1,286), <em>Ix. tasmani</em> (n = 1,159), <em>Ix. holocyclus</em> (n = 982), <em>Ix. cornuatus</em> (n = 219), and <em>Bt. auruginans</em> (n = 159).
The three most common species were then selected for further analysis, including a more detailed curation of records from the grey literature and individual species maps were produced.</p>
<p>Currently <em>Am. triguttatum</em> is divided into four subspecies.
This has remained unchanged since it was established by <span class="citation"><a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">F. H. S. Roberts</a> (<a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">1962</a>)</span>.
A total of 766 records were identified with subspecies status, of these 738 records had location information.
A distribution map of these four subspecies is presented in Figure <a href="austicks.html#fig:F2atrigmapsubp">2.2</a>.
After investigation of “<em>Am. triguttatum</em>” specimens recorded in Tasmania and Victoria,
it was determined these were either incorrectly identified or incorrectly entered into databases.
In the case of records from Tasmania, these specimens were identified as <span class="corrected"><em>Bt. hydrosauri</em></span>, while records from Victoria were assigned instead to either other <em>Amblyomma</em> species or as members of the genus <em>Bothriocroton</em>.
An example of incorrect records for this species is evident in the map produced using records collected from Atlas of Living Australia (Figure <a href="supplementary-material.html#fig:FA21">A.1</a>).
<em>Amblyomma triguttatum</em> is a widespread tick present throughout Australia from the south-west coast of Western Australia up to the northeast of Queensland.
At present the species is considered absent from Tasmania and Victoria, however an invasive population has established on the Yorke Peninsula in South Australia <span class="citation">(<a href="references.html#ref-mcdiarmidRangeExpansionTick2000" role="doc-biblioref">McDiarmid et al. 2000</a>)</span>.</p>
<p>It is interesting that the distribution map (Figure <a href="austicks.html#fig:F2atrigmapsubp">2.2</a>) of <em>Am. triguttatum</em> has changed minimally since that drawn by <span class="citation"><a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">F. H. S. Roberts</a> (<a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">1962</a>)</span>.
It further enforces Roberts’ early hypothesis that these disjunct populations are possibly distinct species.
However, the authors note that despite our inspection of many thousands of <em>Am. triguttatum</em> specimens in the present study, no single morphological feature was identified that can exclusively and reliably delimit the sub-species.
These findings was outlined in the study by <span class="citation"><a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">F. H. S. Roberts</a> (<a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">1962</a>)</span> and remain unchanged today.</p>
<p>Curation of records for <em>Ix. holocyclus</em> identified the species present along the east coast of Australia (Figure <a href="austicks.html#fig:F2mapixhol">2.3</a>).
It was identified along the coastline of Queensland, New South Wales and Victoria.
<em>Ixodes tasmani</em> was identified in all states and territories except the Northern Territory (Figure <a href="austicks.html#fig:F2mapixtas">2.4</a>).
Records of the species in Western Australia were mainly confined to the Southwest corner and most records for this location were sourced from published articles and grey literature as opposed to museum collection records.</p>
<!--
Invasive populations of *Am. triguttatum* in south Australia were first reported by [@mcdiarmidRangeExpansionTick2000] further information of populations in the area have also been studied by [@andrewsDistributionDispersionAmblyomma2007;@waudbySeasonalDensityFluctuations2007;@waudbyHostsExoticOrnate2007] with an additional host record by [@petneyNewHostDisquieting2008].

The invasive status of *Am. triguttatum* on the Yorke Penninsula initiated important work into the habitat and host preferences of this tick [@andrewsDistributionDispersionAmblyomma2007;@waudbySeasonalDensityFluctuations2007] 





All records - distribution map showing all 74 species of ticks in Australia (Figure \@ref(fig:F2mapall)).

**Distribution and infection rate**

*Ix. tasmani* widespread .

Historically there have been relatively few epidemiology studies of parasites and wildlife, and even less so in ticks. Although some recent exemptions to this shown in work on woylies in south west Australia [@northoverAlteredParasiteCommunity2019], quenda [@hillmanUrbanEnvironmentsAlter2017], brush tail possum WA [@hillmanParasiticInfectionsBrushtail2018].

Low numbers in [@waudbyHostsExoticOrnate2007]

*Ix. tasmani* widespread .

Historically there have been relatively few epidemiology studies of parasites and wildlife, and even less so in ticks. Although some recent exemptions to this shown in work on woylies in south west Australia [@northoverAlteredParasiteCommunity2019], quenda [@hillmanUrbanEnvironmentsAlter2017], brush tail possum WA [@hillmanParasiticInfectionsBrushtail2018]

*Ix. holocyclus* confined to the east coast of Australia [@kelersIdentificationIxodesHolocyclus2012].

**Note on FIRST records**

**Urbanization**

Research has shown that landscape interactions strongly impact tick abundance. For example in areas of wildlife exclusion tick abundance increased markedly [@titcombInteractingEffectsWildlife2017].
Therefore ensuring adequate wildlife movement and use of habitat corridors in urban landscapes may also have flow on benefits to decrease tick abundance in areas.


Habitat impacts the abundance and network structure within tick (Acari: Ixodidae) communities on tropical small mammals [@kwakHabitatImpactsAbundance2021].

Molecular Diversity of Hard Tick Species from Selected Areas of a Wildlife-Livestock Interface Ecosystem at Mikumi National Park, Morogoro Region, Tanzania [@damianMolecularDiversityHard2021]..

Preliminary comparative analysis of the resolving power of COX1 and 16S-rDNA as molecular markers for the identification of ticks from Portugal [@filipePreliminaryComparativeAnalysis2021]s

Animal migrations and parasitism: reciprocal effects within a unified framework [@poulinAnimalMigrationsParasitism2021]

Effect of habitat and climate - for antechinus found that ectoparasite infestation was positively correlated with increased litter depth and cooler habitats [@lorchVariationEctoparasiteInfestation2007].

*Examples overseas*
Host–parasite interactions of rodent hosts and ectoparasite communities from different habitats in Germany [@obiegalaHostParasiteInteractions2021]

*Include domestic animals/livestock*

A cross-sectional study of hard ticks (acari: ixodidae) on horse farms to assess the risk factors associated with tick-borne diseases [@kamranCrossSectionalStudy2021].
-->
<div style="page-break-after: always;"></div>
<!--\begin{landscape}-->
<div class="figure">
<span style="display:block;" id="fig:F2mapall"></span>
<img src="figures/ms-figs/Ch2-mapallticks.png" alt="Occurrence map of all known 74 species of ticks (Acari: Ixodida) present in Australia using record curated data." width="95%"><p class="caption">
Figure 2.1: Occurrence map of all known 74 species of ticks (Acari: Ixodida) present in Australia using record curated data.
</p>
</div>
<!--\end{landscape}-->
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:F2atrigmapsubp"></span>
<img src="figures/ms-figs/Ch2-amtrisubsp.png" alt="Occurrence map of the four \textit{Am. triguttatum} subspecies recorded in Australia. Only records with a valid subspecies identified are included." width="95%"><p class="caption">
Figure 2.2: Occurrence map of the four  subspecies recorded in Australia. Only records with a valid subspecies identified are included.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:F2mapixhol"></span>
<img src="figures/ms-figs/Ch2-map-ixhol.png" alt="Occurrence map of \textit{Ix. holocyclus} records in Australia." width="95%"><p class="caption">
Figure 2.3: Occurrence map of  records in Australia.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:F2mapixtas"></span>
<img src="figures/ms-figs/Ch2-map-ixtas.png" alt="Occurrence map of \textit{Ix. tasmani} records in Australia." width="95%"><p class="caption">
Figure 2.4: Occurrence map of  records in Australia.
</p>
</div>
<div style="page-break-after: always;"></div>
</div>
<div id="host-records-for-amblyomma-triguttatum" class="section level4" number="2.3.2.3">
<h4>
<span class="header-section-number">2.3.2.3</span> Host records for <em>Amblyomma triguttatum</em><a class="anchor" aria-label="anchor" href="#host-records-for-amblyomma-triguttatum"><i class="fas fa-link"></i></a>
</h4>
<p>A review of host records is available in table <a href="austicks.html#tab:T2atrig">2.2</a>, and identified the majority were from mammalian hosts.
Given its wide distribution among the mainland there are still many gaps in ecology and life history of <em>Am. triguttatum</em>.
In comparison, similar tick species with a wide distribution in the northern hemisphere (e.g. <em>Ix. ricinus</em> and <em>Ix. scapularis</em>) have been extensively studied <span class="citation">(<a href="references.html#ref-mihalcaRoleRodentsEcology2013" role="doc-biblioref">Mihalca and Sándor 2013</a>; <a href="references.html#ref-tietjenComparativeEvaluationNorthern2020" role="doc-biblioref">Tietjen et al. 2020</a>)</span>.</p>
<p>A recognised issue in distribution studies more broadly is the identification of areas that truly represent absence, as opposed to a lack of investigation.
<em>Amblyomma triguttatum</em> is considered an exophilic tick and can be classified as a ‘hunter species.’
Observations by the authors note that species can be found readily in the environment where there is human activity.
In addition, <em>Am. triguttatum</em> is a relatively large tick with unfed adults reaching 3–5 mm in size <span class="citation">(<a href="references.html#ref-robertsStatusMorphologicallyDivergent1962" role="doc-biblioref">F. H. S. Roberts 1962</a>)</span> and is readily identified on hosts.
With these factors in mind the authors consider that a lack of <em>Am. triguttatum</em> identified is likely to be a true indicator of its absence, or at least of a small tick population size.
We note that in arid areas where flora and fauna studies persist, these have been good sources of records based on museum data obtained in the present study.
Therefore, until additional data is made available there is strong support that <em>Am. triguttatum</em> persists in disjunct populations throughout Australia.</p>
<p><span class="citation"><a href="references.html#ref-waudbyHostsExoticOrnate2007" role="doc-biblioref">Helen P. Waudby et al.</a> (<a href="references.html#ref-waudbyHostsExoticOrnate2007" role="doc-biblioref">2007</a>)</span> listed the domestic cat as a host and included a reference to <span class="citation"><a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">F. S. H. Roberts</a> (<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>)</span>, however after reviewing the suite of Roberts’ tick publications, we have not been able to identify such a record.
We note however that a host record from a cat was included in additional data obtained by <span class="citation"><a href="references.html#ref-waudbyHostsExoticOrnate2007" role="doc-biblioref">Helen P. Waudby et al.</a> (<a href="references.html#ref-waudbyHostsExoticOrnate2007" role="doc-biblioref">2007</a>)</span>.</p>
<!--
Invasive populations of *Am. triguttatum* in south Australia were first reported by [@mcdiarmidRangeExpansionTick2000] further information of populations in the area have also been studied by [@andrewsDistributionDispersionAmblyomma2007;@waudbySeasonalDensityFluctuations2007;@waudbyHostsExoticOrnate2007] with an additional host record by [@petneyNewHostDisquieting2008].

The invasive status of *Am. triguttatum* on the Yorke Penninsula initiated important work into the habitat and host preferences of this tick [@andrewsDistributionDispersionAmblyomma2007;@waudbySeasonalDensityFluctuations2007] within the region. While the first publication of the invasive tick was made by McDiarmid et al. [-@mcdiarmidRangeExpansionTick2000] who cited ticks the earliest collection as February 1995 from Pandalowie Bay, later Andrews et al. [-@andrewsDistributionDispersionAmblyomma2007] recorded identification of the tick as early as September 1980 from Edithburgh. Thus the exact date of arrival on Yorke Penninsula remains unknown. While a survey of residents identified that ticks have been identified within the region for 70+ years it is unknown what species were identified and if species composition has changed over time. Given the invasive nature of *Am. triguttatum* it is likely that while introductions may have occurred in the early-mid 20th century tick populations of this species became well established by 2000. Seeing as there has been a strong study of ticks within Yorke Penninsula over the past decades with good quality data on human persceptions, hosts, and habitat this system would be an excellent model to continue data collection from and provide good baseline info.
-->
<div class="inline-table"><table class="table table" style="font-size: 8.5px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T2atrig">Table 2.2: </span>List of host records for . Abbreviations: Australian National Insect Collection (ANIC); Western Australian Museum (WAM). Where host records were ambiguous (e.g., kangaroo), taxa was assigned to the most commonly recorded species in the locality.
</caption>
<thead><tr>
<th style="text-align:left;">
Host common name
</th>
<th style="text-align:left;">
Host scientific name
</th>
<th style="text-align:left;">
Reference
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Class: Mammalia
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Monotremata
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; ">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Platypus
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Ornithorhynchus anatinus
</td>
<td style="text-align:left;width: 6cm; ">
Neumann 1901 - see Roberts (1962)
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Short-beaked echidna
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Tachyglossus aculaetus
</td>
<td style="text-align:left;width: 6cm; ">
Krige et al. 2017; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Dasyuromorphia
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Numbat
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Myrmecobius fasciatus
</td>
<td style="text-align:left;width: 6cm; ">
Calaby 1960; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Peramelemorphia
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Greater Bilby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Macrotis lagotis
</td>
<td style="text-align:left;width: 6cm; ">
WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Diprotodontia
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Koala
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Phascolarctos cinereus
</td>
<td style="text-align:left;width: 6cm; ">
Barker &amp; Campelo unpublished - Barker and Walker 2014; Waudby et al. 2007; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Northern hairy-nosed wombat
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Lasiorhinus kreffti
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Western ringtail possum
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Pseudocheirus occidentalis
</td>
<td style="text-align:left;width: 6cm; ">
Clarke 2011
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Common brushtail possum
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Trichosurus vulpecula
</td>
<td style="text-align:left;width: 6cm; ">
Clarke 2011
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Eastern grey kangaroo
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Macropus giganteus
</td>
<td style="text-align:left;width: 6cm; ">
Pope et al. 1960; Roberts 1962, 1970; Guglielmone 1990; Cooper et al. 2013; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Western grey kangaroo
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Macropus fuliginosus
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; Waudby et al. 2007; Loh et al. 2018; ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Red kangaroo
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Macropus rufus
</td>
<td style="text-align:left;width: 6cm; ">
Pope et al. 1960; Roberts 1962; Loh et al. 2018; ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Tammar wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Notamacropus eugenii
</td>
<td style="text-align:left;width: 6cm; ">
Waudby et al. 2007, 2018;ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Bridled nail-tail wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Onychogalea frenata
</td>
<td style="text-align:left;width: 6cm; ">
Turni and Smales 2001; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Swamp wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Wallabia bicolor
</td>
<td style="text-align:left;width: 6cm; ">
Beveridge et al. 1985; Roberts 1962; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Agile wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Notamacropus agilis
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; Speare et al. 1983; Cooper et al. 2013; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Black-striped wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Notamacropus dorsalis
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Red-necked wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Notamacropus rufogriseus
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Northern Nail-tail wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Onychogalea unguifera
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Allied Rocky-wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Petrogale assimilis
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Western brush wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Notamacropus irma
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Unadorned Rock-wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Petrogale inornata
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Whiptailed wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Notamacropus parryi
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1970; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Spectacled Hare-wallaby
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Lagorchestes conspicillatus
</td>
<td style="text-align:left;width: 6cm; ">
ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Common wallaroo
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Osphranter robustus
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; Cooper et al. 2013; ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Antilopine wallaroo
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Osphranter antilopinus
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Northern bettong
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Bettongia tropica
</td>
<td style="text-align:left;width: 6cm; ">
Barker &amp; Campelo unpublished - Barker and Walker 2014
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Woylie
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Bettongia penicillata
</td>
<td style="text-align:left;width: 6cm; ">
Kaewmongkol et al. 2011; Northover 2019; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Rufous bettong
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Aepyprymnus rufescens
</td>
<td style="text-align:left;width: 6cm; ">
Cooper et al. 2013; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Primates
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Human
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Homo sapiens
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; Pearce and Grove 1987; McDiarmid et al. 2000; Andrews et al. 2007; Owen 2007; Gofton et al. 2015; ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Rodentia
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Black rat
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Rattus rattus
</td>
<td style="text-align:left;width: 6cm; ">
Waudby et al. 2007
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Laboratory / brown rat
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Rattus norvegicus
</td>
<td style="text-align:left;width: 6cm; ">
Guglielmone and Moorhouse 1985(a)
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Lagomorpha
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Rabbit
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Oryctolagus cuniculus
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1953(a); Guglielmone and Moorhouse 1985; Waudby et al. 2007; ANIC(a)
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Carnivora
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Dog
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Canis lupus familiaris
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; Waudby et al. 2007; Andrews et al. 2007; Greay et al. 2016; ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Cat
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Felis catus
</td>
<td style="text-align:left;width: 6cm; ">
Waudby et al. 2007; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Perissodactyla
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Horse
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Equus caballus
</td>
<td style="text-align:left;width: 6cm; ">
McCarthy 1960; Roberts 1962; Greay et al. 2016; Chalada et al. 2018; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Artiodactyla
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Pig
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Sus scrofa
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; Guglielmone 1990; Li et al. 2010; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Cattle
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Bos taurus
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962, 1970; ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Swamp buffalo
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Babalus bubalis
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Goat
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Capra hircus
</td>
<td style="text-align:left;width: 6cm; ">
Pope et al. 1960
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Domestic sheep
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Ovis aries
</td>
<td style="text-align:left;width: 6cm; ">
Roberts 1962; ANIC; WAM
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Class: Aves
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Australian raven
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Corvus coronoides
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Torresian crow
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Corvus orru
</td>
<td style="text-align:left;width: 6cm; ">
ANIC
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; font-weight: bold;">
Class: Reptilia
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;font-style: italic;">
</td>
<td style="text-align:left;width: 6cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 3cm; ">
Bobtail/sleepy lizard
</td>
<td style="text-align:left;width: 3cm; font-style: italic;">
Tiliqua rugosa
</td>
<td style="text-align:left;width: 6cm; ">
Waudby et al. 2007; Petney et al. 2008;
</td>
</tr>
</tbody>
<tfoot><tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Host record reported from laboratory animals.
</td>
</tr></tfoot>
</table></div>
<div style="page-break-after: always;"></div>
</div>
<div id="molecular-barcodes-and-systematics" class="section level4" number="2.3.2.4">
<h4>
<span class="header-section-number">2.3.2.4</span> Molecular barcodes and Systematics<a class="anchor" aria-label="anchor" href="#molecular-barcodes-and-systematics"><i class="fas fa-link"></i></a>
</h4>
<p>Molecular systematics has become the dominant method of species identification in disciplines including protozoology <span class="citation">(<a href="references.html#ref-maiaCommentsSystematicRevision2016" role="doc-biblioref">Maia, Carranza, and Harris 2016</a>; <a href="references.html#ref-maslovRecentAdvancesTrypanosomatid2019" role="doc-biblioref">Maslov et al. 2019</a>)</span> and microbiology <span class="citation">(<a href="references.html#ref-margosControversiesBacterialTaxonomy2019" role="doc-biblioref">Margos et al. 2019</a>)</span>.
However, for ectoparasites, including ticks, morphological tools remain the gold standard.
Therefore, the splitting of species based solely on molecular information without support of morphological features is unlikely to be useful to the field of tick taxonomy.
However, from an evolutionary perspective the molecular information generated here will assist in phylogenetic reconstructions and may assist in refining species boundaries.</p>
<p>New molecular barcodes generated in the present study are presented in Table <a href="supplementary-material.html#tab:T2MLSTseqs">A.2</a> which include GenBank accession numbers.
An approach to tick taxonomy incorporating both traditional and new technologies is needed and consideration needs to be given to the broader impact of species nomenclature.
Particularly, for tick species of medical and veterinary importance, splitting species or a change of name can create confusion and wider uptake can be slow.
Compelling evidence for name changes is therefore needed before considering a new taxonomic nomenclature of these species.</p>
<p>The development of molecular information provides a fundamental tool to assist in species identification and systematics.
Unlike morphological descriptions and dichotomous keys, molecular barcodes are a characteristic of a species which is shared across all life stages.
As studies progress towards adapting an integrative approach in tick taxonomy <span class="citation">(<a href="references.html#ref-dantas-torresSpeciesConceptsWhat2018" role="doc-biblioref">F. Dantas-Torres 2018</a>)</span>, the use of molecular barcodes, as generated here, will be useful to identify characteristics used to deliminate species.</p>
<!--
The development of molecular techniques provides a fundamental tool to assist in species identification and systematics. 
Unlike morphological descriptions and dichotomous keys, molecular barcodes are a characteristic of a species which is shared across all life stages.
The catalogue of sequence barcodes for tick species will assist in ensuring future morphological descriptions on additional instars are representative of the same tick species.

There are many factors which can impact species delimitation methods based soley on sequence barcodes. 
These can include small sample sizes, geographic representation,  [@phillipsIncompleteEstimatesGenetic2019], additionally the optiminal locus for species delimitation can vary between taxonomic groups. 
While the COI barcode is considered the "universal" marker, studies have shown that other loci perform better for species identification [@ritchieExaminingSensitivityMolecular2016].
Therefore the data presented in the present study is aims to provide new molecular references and expand on those currently available. 

Analysis techniques can also influence the conclusions made and result if different species models [@dellicourHitchhikerGuideSinglelocus2018].


Cox1 barcoding versus multilocus species delimitation: validation of two mite species with contrasting effective population sizes [@klimovCox1BarcodingMultilocus2019]
-->
<p><strong><span class="correction">High-throughput sequencing</span> sequencing</strong></p>
<p><span class="correction">High-throughput sequencing</span> sequencing targeting the 12S rDNA gene was successful at identifying tick species.
This included immature life stages where multiple specimens were pooled at the DNA extraction level.
Pooling of ticks (especially larvae and nymph stages) is implemented in many molecular studies of tick-borne pathogen or microbial characterisation to increase the throughput of samples <span class="citation">(<a href="references.html#ref-estrada-penaPitfallsTickTickBorne2021" role="doc-biblioref">Estrada-Pena et al. 2021</a>)</span>.
The ability to deliminate species with a short sequence length (~ 370 bp) make this high-throughput assay easily transferable to many short read sequence platforms.
The same extracted DNA or RNA used for pathogen detection and be used in the assay presented here to accurately determine species.
In particular, it is suited to the identification of immature tick life stages whose identity is ambiguous or where certain morphological features are missing or damaged, which prevent species-level identification.
An ability of the 12S rDNA gene <span class="corrected">to act</span> as a barcoding gene for ticks has been identified in similar studies <span class="citation">(<a href="references.html#ref-lvAssessmentFourDNA2014" role="doc-biblioref">Lv, Wu, Zhang, Chen, et al. 2014</a>; <a href="references.html#ref-kandumaMitochondrialNuclearMultilocus2019" role="doc-biblioref">Kanduma et al. 2019</a>)</span>, and is shown by the level of phylogenetic separation among species shown in Figure <a href="austicks.html#fig:F2NGStree">2.5</a> (see list of accession numbers and species identification in Table <a href="supplementary-material.html#tab:T2genbank">A.3</a>).
For example morphologically similar species, such as <em>Ix. holocyclus</em> (OM830732) and <em>Ix. cornuatus</em> (OM830728), can be distinguished easily as demonstrated in Figure <a href="austicks.html#fig:F2NGStree">2.5</a> (see also Table <a href="supplementary-material.html#tab:T2genbank">A.3</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F2NGStree"></span>
<img src="figures/ms-figs/Ch2-12SNGStree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of Ixodida ZOTUs based on a 377 bp alignment of the 12S rDNA gene Substitution model K3Pu +F + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequences generated in the present study represented in blue (ZOTUs), and red (reference sequences)." width="95%"><p class="caption">
Figure 2.5: Maximum likelihood (ML) phylogenetic reconstruction of Ixodida ZOTUs based on a 377 bp alignment of the 12S rDNA gene Substitution model K3Pu +F + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequences generated in the present study represented in blue (ZOTUs), and red (reference sequences).
</p>
</div>
<!--
Previous work on *Am. triguttatum* Since F.S.H. Roberts work on Australian ticks the most comprehensive studies on *Am. triguattatum* have been done by A. Gugliemone who published a collection of papers on the tick from central queensland, including laboratory and field studies which provide information on the basic biology of this tick [@guglielmoneDropoffRhythmAmblyomma1993;@guglielmoneEffectTemperatureHumidity1992;@guglielmoneDifferencesNymphsAmblyomma1985;@guglielmoneAttractionCarbonDioxide1985;@guglielmoneEffectPhotoperiodDevelopment1986;@guglielmoneReproductionAmblyommaTriguttatum1986;@guglielmoneCopulationSuccessfulInsemination1983;@guglielmoneSeasonalOccurrenceAmblyomma1994;@guglielmoneSitesAttachmentAmblyomma1990].
-->
<!--
#### Misc.

Evolutionary refugia and ecological refuges: key concepts for conserving Australian arid zone freshwater biodiversity under climate change [@davisEvolutionaryRefugiaEcological2013]



The distribution of Australian ticks remains largely undocumented, with only X number of species



The absence of this species from south eastern Australia including Victoria and Tasmania is interesting, and

Compare to overseas A study of *Amblyomma parvum* found that despite molecular findings no support for different species in distinct populations within South America [@navaDifferentLinesEvidence2016]. *Amblyomma ovale* [@uribeCharacterizationCompleteMitochondrial2020].

Amblyomma maculatum [@ladoAmblyommaMaculatumKoch2018] Amblyomma maculatum [@estrada-penaAmblyommaMaculatumKoch2005] Amblyomma exornatum and Amblyomma transversale [@hornokMolecularPhylogenyAmblyomma2020]  Amblyomma cajennense [@beatiAmblyommaCajennenseFabricius2013]

*Amblyomma* and ITS2 [@marrelliTaxonomicPhylogeneticRelationships2007]

Why *Am. triguttatum* important Research on this widespread tick species is urgently needed build on previous work highlighting the likely zoonotic significance *Am. triguttatum* [@abdadSeroepidemiologicalStudyOutdoor2014;@abdadRickettsiaGravesiiSp2017].

Future work Understanding habitat preferences useful to model distibution of ticks [@clarke-crespoEcologicalNicheModels2020]. Amblyomma americanum here [@pascoeModelingPotentialHabitat2019]. Habitat and vegetation not enough to predict tick distribution [@troutfryxellHabitatVegetationVariables2015] Phylogepgraphics of tick [@beatiPhylogeographyTicksAcari2019]

Estimating sample sizes for DNA barcoding [@zhangEstimatingSampleSizes2010].
-->
</div>
</div>
<div id="limitations" class="section level3" number="2.3.3">
<h3>
<span class="header-section-number">2.3.3</span> Limitations<a class="anchor" aria-label="anchor" href="#limitations"><i class="fas fa-link"></i></a>
</h3>
<!--
BMC Research Notes considers scientifically valid manuscripts irrespective of the interest of a study or its likely impact. In order to ensure submissions to BMC Research Notes are of maximum benefit to the research community, authors should clearly state the limitations of their work.
-->
<p>The nature of using occurrence records to map distribution does have several limitations <span class="citation">(<a href="references.html#ref-fourcadeComparingSpeciesDistributions2016" role="doc-biblioref">Fourcade 2016</a>)</span>.
However, as ticks are obligate blood feeders which require a host, the nature of their life history makes occurrence data more suitable to map distributions.
In particular, where tick species are aggressive human biting species, such as <em>Am. triguttatum</em> <span class="citation">(<a href="references.html#ref-gravesTickborneInfectiousDiseases2017" role="doc-biblioref">S. R. Graves and Stenos 2017</a>)</span>, the absence of records is a useful indicator that the species is likely absent from that area.
Alternatively, low levels of human interactions in areas where the tick is present in the environment may also be responsible for absence of records.
A comprehensive search strategy was used and attempts were made to use a diverse range of sources (museum collections, public databases, grey literature etc.).
Therefore, we accept that it is possible that our occurrence maps may not represent the complete geographical distribution of all 74 tick species present in Australia.
Tick species distribution maps included a large portion of data based on historical identifications.
While every effort was made to ensure such records were from trusted sources, such as those identified by a suitability qualified person or collections from museum records, it is not possible to verify every single observation.
Where records outside of historic distributions or unusually sporadic data points were observed, efforts were made to verify specimen identification.</p>
<p>The generation of new molecular barcodes provided in the present study represents only a portion of the 74 tick species present in Australia.
Despite this, the authors feel this information is still valuable to researchers.
By making the molecular data public it will be of assistance to future researchers working towards genetic characterisation of ticks and be of use in phylogenetic reconstruction and taxonomic studies.</p>
<!--
A limitation however in host records of tick species is accurate knowledge of tick acquisition i.e. precise location that tick attached to the host. 
In particular for *Am. triguttatum*, which is a common parasite of kangaroos they can have a relatively large range -- find ref for roo home range. 
Whilst it is likely that where kangaroo populations reside, *Am. triguttatum* populations also reside this would need to be considered in areas where a new record is made of this species.
Limitations and "in-practice" Use of whole genome sequencing of *Am. triguttatum*, in particular mitochondrial genome information, will also be useful to understand the relationship of these subspecies. However we note that in the case of tick identification emphasis is placed on morphological identification and if required, an efficient and cost effective molecular assay (i.e. amplification and sequencing of a single gene) with mininal analysis required (i.e. reconstructing genomes not practical in an ID sense). Therefore the MLST approach taken in the present study provides valuable information with regards to a practical approach to species identification. In most instances records of *Am. triguttatum* are rarely assigned to subspecies status in the literature. Given the results presented here and the historical records we conclude that assignment to species level (*Am. triguttatum*) is sufficient.
-->
</div>
</div>
<div id="conclusion" class="section level2" number="2.4">
<h2>
<span class="header-section-number">2.4</span> Conclusion<a class="anchor" aria-label="anchor" href="#conclusion"><i class="fas fa-link"></i></a>
</h2>
<!--
A continued and concerted effort needs to be made to encourage on-going research into the life cyle of Australian ticks.
-->
<p>While the biology and natural history of ticks is understood in a broad context, much of this is based on data obtained from just a handful of species.
The most frequently incriminated human biting ticks in Europe and North America, <em>Ixodes ricinus</em> and <em>Ixodes scapularis</em>, are responsible for the majority of tick-borne infections in those continents <span class="citation">(<a href="references.html#ref-eisenBlackleggedTickIxodes2018" role="doc-biblioref">Eisen and Eisen 2018</a>; <a href="references.html#ref-grayWhatWeStill2021" role="doc-biblioref">Gray, Kahl, and Zintl 2021</a>)</span>.
The life history and ecology of these tick species has been studied and provides important information needed to inform public health measures <span class="citation">(<a href="references.html#ref-grayWhatWeStill2021" role="doc-biblioref">Gray, Kahl, and Zintl 2021</a>)</span>.
In contrast few, if any, tick species in Australia have been studied to the same degree.</p>
<p>With rapid urbanization and the effects of climate change, the interface between humans and ticks is predicted to increase <span class="citation">(<a href="references.html#ref-gilbertImpactsClimateChange2021" role="doc-biblioref">L. Gilbert 2021</a>)</span>.
Despite the well documented history of the discovery of Lyme disease in North America during the 1970’s and 1980’s <span class="citation">(<a href="references.html#ref-ostfeldFunctionBiodiversityEcology2000" role="doc-biblioref">Ostfeld and Keesing 2000</a>)</span>, Australia continues to trail behind the rest of world with respect to knowledge about tick-borne diseases.
Without fundamental research into the natural history, ecology and molecular systematics of Australian ticks, the country is ill-equipped to understand the dynamics of potential tick-borne infections.</p>
<p>It is expected that the data presented from this research will provide the necessary foundations to further explore Australian tick systematics.
It is important that previous records are met with a healthy dose of criticism and work can begin towards a clearer understanding on Australian ticks.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="thesis-aims.html">Thesis aims</a></div>
<div class="next"><a href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#austicks"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="nav-link" href="#preface-1">Preface</a></li>
<li><a class="nav-link" href="#abstract-1"><span class="header-section-number">2.1</span> Abstract</a></li>
<li><a class="nav-link" href="#introduction"><span class="header-section-number">2.2</span> Introduction</a></li>
<li>
<a class="nav-link" href="#main-text"><span class="header-section-number">2.3</span> Main text</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#methods"><span class="header-section-number">2.3.1</span> Methods</a></li>
<li><a class="nav-link" href="#results-and-discussion"><span class="header-section-number">2.3.2</span> Results and Discussion</a></li>
<li><a class="nav-link" href="#limitations"><span class="header-section-number">2.3.3</span> Limitations</a></li>
</ul>
</li>
<li><a class="nav-link" href="#conclusion"><span class="header-section-number">2.4</span> Conclusion</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/ulyngs/oxforddown/blob/master/02-aus-ticks.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/ulyngs/oxforddown/edit/master/02-aus-ticks.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></strong>" was written by Siobhon Lee Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
