<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>5 Black rat trypanosome | Ecology of Ticks and Microbes in Australia Wildlife</title>
<meta name="author" content="Siobhon Lee Egan">
<meta name="generator" content="bookdown 0.24 with bs4_book()">
<meta property="og:title" content="5 Black rat trypanosome | Ecology of Ticks and Microbes in Australia Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="5 Black rat trypanosome | Ecology of Ticks and Microbes in Australia Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/header-attrs-2.11.22/header-attrs.js"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="Preface The following chapter has been drafted in accordance with the journal Parasitology Research. The following chapter has been published: Egan, S., Taylor, C., Austen, J., Banks, P.,...">
<meta property="og:description" content="Preface The following chapter has been drafted in accordance with the journal Parasitology Research. The following chapter has been published: Egan, S., Taylor, C., Austen, J., Banks, P.,...">
<meta name="twitter:description" content="Preface The following chapter has been drafted in accordance with the journal Parasitology Research. The following chapter has been published: Egan, S., Taylor, C., Austen, J., Banks, P.,...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="active" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/ulyngs/oxforddown">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="black-rat" class="section level1" number="5">
<h1>
<span class="header-section-number">5</span> Black rat trypanosome<a class="anchor" aria-label="anchor" href="#black-rat"><i class="fas fa-link"></i></a>
</h1>
<embed src="front-and-back-matter/preface/blackrat.pdf" width="95%" type="application/pdf"></embed><div style="page-break-after: always;"></div>
<div id="preface-4" class="section level2 unnumbered">
<h2>Preface<a class="anchor" aria-label="anchor" href="#preface-4"><i class="fas fa-link"></i></a>
</h2>
<p>The following chapter has been drafted in accordance with the journal <em>Parasitology Research</em>.</p>
<p>The following chapter has been published: <strong>Egan, S.</strong>, Taylor, C., Austen, J., Banks, P., Ahlstrom, L., Ryan, U., Irwin, P., and Oskam C. 2020. Molecular identification of the <em>Trypanosoma</em> (<em>Herpetosoma</em>) <em>lewisi</em> clade in black rats (<em>Rattus rattus</em>) from Australia. <em>Parasitology Research</em>, <strong>119</strong>, 1691–1696. DOI: <a href="https://doi.org/10.1007/s00436-020-06653-z">10.1007/s00436-020-06653-z</a></p>
<p>The following authors contributed to this manuscript as outlined below<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Contribution indicates the total involvement the author has had in this project. Placing an ‘X’ in the remaining boxes indicates what aspect(s) of the project each author engaged in.&lt;/p&gt;"><sup>6</sup></a>.</p>
<div class="inline-table"><table class="table table" style="font-size: 7px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Authorship order
</th>
<th style="text-align:right;">
Contribution (%)
</th>
<th style="text-align:left;">
Concept Development
</th>
<th style="text-align:left;">
Data Collection
</th>
<th style="text-align:left;">
Data Analysis
</th>
<th style="text-align:left;">
Draft
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Siobhon L. Egan
</td>
<td style="text-align:right;">
70
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
</tr>
<tr>
<td style="text-align:left;">
Casey L. Taylor
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Jill M. Austen
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter B. Banks
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Liisa A. Ahlstrom
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Una M. Ryan
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter J. Irwin
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Charlotte L. Oskam
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
<p>By signing this document, the Candidate and Principal Supervisor acknowledge that the information provided is accurate and has been agreed to by all other authors.</p>
<div class="line-block">__________________       __________________<br>
  Candidate                                 Principal Supervisor</div>
<div style="page-break-after: always;"></div>
<p><strong>Chapter linking statement:</strong>
This chapter provides further molecular insights into <em>Trypanosoma lewisi</em>-like sequences that were identified from the blood of black rats in Sydney, New South Wales. This study provided the first molecular characterisation of members of the <em>Tr. lewisi</em>-clade from Australian black rats (<em>Rattus rattus</em>). It shows that Australia <em>Tr. lewisi</em>-like sequences were distinct from <em>Tr. lewisi</em> sensu stricto clade, and were most closely related to genotypes from Europe. As detection of this blood parasite included a site managed by the Australian Wildlife Conservatory where native wildlife are being re-introduced, this finding highlights the importance of on-going surveillance and management strategies in wildlife conservation.</p>
<p><strong>Funding and acknowledgment statement:</strong> This study was part-funded by the Australian Research Council (LP160100200), Bayer HealthCare (Germany) and Bayer Australia. S.L.E. is supported by an Australian Government Research Training Program (RTP) Scholarship, C.L.T. is supported by a scholarship from the Northern Beaches Council. This project was also part supported by The Holsworth Wildlife Research Endowment &amp; The Ecological Society of Australia (awarded to S.L.E.) and the Paddy Pallin Science Grant from The Royal Zoological Society (awarded to C.L.T.). We thank Jenna Bytheway, Dr. Henry Lydecker and the Australian Wildlife Conservancy ecologists Dr. Viyanna Leo and Mareshell Wauchope for their invaluable assistance in the field. We thank two anonymous reviewers for their constructive feedback.</p>
<p><strong>Data availability:</strong>
Sequence generated in the present study has been submitted to GenBank nucleotide database under accession number MN512227.</p>
<p><strong>Author contributions::</strong>
Conceptualisation: S.L.E., U.M.R., P.J.I., C.L.O.
Data curation: S.L.E., C.L.T.
Formal Analysis: S.L.E.
Funding acquisition: S.L.E., C.L.T., P.B.B., L.A.A., U.M.R., P.J.I., C.L.O.
Investigation: S.L.E., C.L.T., J.M.A.,
Methodology: S.L.E., C.L.T., J.M.A.
Project administration: P.J.I., C.L.O.
Resources: A.S.N., P.B.B., P.J.I., C.L.O.
Software: S.L.E.
Supervision: P.B.B., U.M.R., P.J.I., C.L.O.
Visualisation: S.L.E.
Writing – original draft: S.L.E.
Writing – review &amp; editing: S.L.E, C.L.T., P.B.B., A.S.N., L.A.A., U.M.R., P.J.I., C.L.O.</p>
<p><strong>Keywords:</strong> <em>Trypanosoma lewisi</em>; <em>Rattus rattus</em>; Australia; black rats; ship rats</p>
<div style="page-break-after: always;"></div>
</div>
<div id="abstract-4" class="section level2" number="5.1">
<h2>
<span class="header-section-number">5.1</span> Abstract<a class="anchor" aria-label="anchor" href="#abstract-4"><i class="fas fa-link"></i></a>
</h2>
<p>Invasive rodent species are known hosts for a diverse range of infectious microorganisms and have long been associated with the spread of disease globally. The present study describes molecular evidence for the presence of a <em>Trypanosoma</em> sp. from black rats (<em>Rattus rattus</em>) in northern Sydney, Australia. Sequences of the 18S ribosomal RNA (rRNA) locus were obtained in two out of eleven (18%) blood samples with subsequent phylogenetic analysis confirming the identity within the <em>Trypanosoma lewisi</em> clade.</p>
</div>
<div id="introduction-3" class="section level2" number="5.2">
<h2>
<span class="header-section-number">5.2</span> Introduction<a class="anchor" aria-label="anchor" href="#introduction-3"><i class="fas fa-link"></i></a>
</h2>
<p>Black rats (<em>Rattus rattus</em>) are distributed throughout the world and considered one of the most significant invasive species. Current evidence indicates the Rattus genus originated from Southeast Asia <span class="citation">(<a href="references.html#ref-aplinMultipleGeographicOrigins2011" role="doc-biblioref">Aplin et al. 2011</a>)</span>, with black rats establishing in Australia alongside European settlement during the 1770’s, although the precise date of their first arrival on the continent is unclear <span class="citation">(<a href="references.html#ref-banksReviewEvidencePotential2012" role="doc-biblioref">Banks and Hughes 2012</a>)</span>. Black rats can act as amplifying hosts for a diverse range of pathogens that can affect humans, wildlife and domestic animals and a recent review of black rats in Europe identified at least 20 zoonotic infectious agents associated with the species <span class="citation">(<a href="references.html#ref-strandRatborneDiseasesHorizon2019" role="doc-biblioref">Strand and Lundkvist 2019</a>)</span>. However, despite the global recognition of these rodents as hosts of pathogens, there is a relatively limited understanding of the range of infectious agents present in Australian populations of black rats <span class="citation">(<a href="references.html#ref-banksReviewEvidencePotential2012" role="doc-biblioref">Banks and Hughes 2012</a>)</span>.</p>
<p>Trypanosomes are a group of flagellate protozoan parasites, the vast majority of which are transmitted by blood-feeding invertebrates. Worldwide at least 44 trypanosome species are known to infect rodents (as reviewed by Dybing et al. <span class="citation">(<a href="references.html#ref-dybingGhostsChristmasAbsence2016" role="doc-biblioref">2016</a>)</span>). Due to the morphological similarities within the <em>Trypanosoma</em> subgenus <em>Herpetosoma</em> <span class="citation">(<a href="references.html#ref-maiadasilvaPhylogeneticMorphologicalBehavioural2010" role="doc-biblioref">Maia da Silva et al. 2010</a>; <a href="references.html#ref-ortizDiagnosisGeneticAnalysis2018" role="doc-biblioref">Ortiz et al. 2018</a>)</span>, records based on microscopic observations alone may underestimate the diversity of trypanosome species infecting rodents; the number is likely to increase with more frequent application of molecular methodologies in contemporary studies. <em>Trypanosoma</em> (<em>Herpetosoma</em>) lewisi almost exclusively utilises a <em>Rattus</em> sp. host and is commonly vectored by rodent fleas, <em>Xenopsylla cheopis</em> and <em>Nosopsyllus fasciatus</em> <span class="citation">(<a href="references.html#ref-ortizDiagnosisGeneticAnalysis2018" role="doc-biblioref">Ortiz et al. 2018</a>)</span>. While members of this subgenus are largely considered non-pathogenic in their respective hosts, infections have been identified in a number of other mammalian species, including humans <span class="citation">(<a href="references.html#ref-maiadasilvaPhylogeneticMorphologicalBehavioural2010" role="doc-biblioref">Maia da Silva et al. 2010</a>; <a href="references.html#ref-ortizDiagnosisGeneticAnalysis2018" role="doc-biblioref">Ortiz et al. 2018</a>)</span>. In Australia, recent research has revealed the presence of several novel trypanosomes infecting native Australian marsupials <span class="citation">(<a href="references.html#ref-thompsonTrypanosomesAustralianMammals2014" role="doc-biblioref">C. K. Thompson, Godfrey, and Thompson 2014</a>)</span>, however investigation into the presence of trypanosomes in Australian rodents, either native or introduced, has been absent in recent years.</p>
<p>The results shared in this short communication form part of a broader investigation into vector-borne microorganisms present in Australia. To the authors’ knowledge, this study provides the first molecular identification of <em>Trypanosoma lewisi</em>-like organisms from black rats on mainland Australia.</p>
</div>
<div id="methods-3" class="section level2" number="5.3">
<h2>
<span class="header-section-number">5.3</span> Methods<a class="anchor" aria-label="anchor" href="#methods-3"><i class="fas fa-link"></i></a>
</h2>
<p>Small mammal trapping was conducted during April and May 2019 at two sites in northern Sydney, NSW, Australia; Irrawong Reserve and Warriewood Wetlands, Warriewood (-31.69, 151.28) and North Head, Manly (-33.81, 151.29). Two transects of 20 trap stations were set up at each site, with each station including one Elliot type B trap (46 x 15.5 x 15 cm) and one medium sized cage trap (72 x 32 x 31 cm) to target small and medium sized mammals. Traps were baited with peanut butter and oat balls and set for three consecutive nights. The trapping and sampling were conducted with approval of the Animal Ethics Committees of the University of Sydney (Permit number 2018/1429) and Murdoch University (Permit number R3026/18), respectively. Venous blood was collected into 1 mL EDTA tubes for the detection of haemoparasites. Thin blood smears were prepared and stained with modified Wright-Giemsa. Blood films were inspected by light microscopy (Olympus BX51) for the presence of trypanosomes at x 400 magnification and under oil immersion (x 1000). Total genomic DNA was extracted from 200 <span class="math inline">\(\mu L\)</span> of blood using a MasterPure DNA purification kit (EpicentreBiotechnologies, Madison, Wisconsin, U.S.A) following the manufacturer’s recommendations. Where 200 <span class="math inline">\(\mu L\)</span> of blood was not available, PBS was used to make samples up to 200 <span class="math inline">\(\mu L\)</span> DNA was eluted in 30 <span class="math inline">\(\mu L\)</span> of TE buffer and stored at -20 <span class="math inline">\(^\circ\)</span>C.</p>
<p>Blood samples were screened for the presence of <em>Trypanosoma</em> spp. using a nested PCR approach targeting a ~550 bp product of the 18S ribosomal RNA (rRNA) gene with external primers TRY927F / TRY927R and internal primers SSU561F / SSU561R, as previously described <span class="citation">(<a href="references.html#ref-noyesNestedPCRSsrRNA1999" role="doc-biblioref">Noyes et al. 1999</a>)</span>. Reactions were carried out in 25 <span class="math inline">\(\mu L\)</span> volumes, 2 <span class="math inline">\(\mu L\)</span> of undiluted gDNA was added to the primary PCR and 1 <span class="math inline">\(\mu L\)</span> of the primary product was used as a template for the secondary assay. PCR products were electrophoresed on a 1% agarose gel stained with SYBR safe (Invitrogen, USA), and amplicons of the correct size were excised and purified using previously described methods <span class="citation">(<a href="references.html#ref-yangSpecificQuantitativeDetection2013" role="doc-biblioref">R. Yang et al. 2013</a>)</span>. Sanger sequencing was carried out using internal primer sets in both directions and sequencing was performed at the Australian Genome Research Facility (Perth, Australia). Samples that returned a positive identification for <em>Trypanosoma lewisi</em>-like were further investigated. A near full-length fragment of the 18S rRNA locus was obtained using two nested PCR assays. Reactions were carried out in 25 <span class="math inline">\(\mu L\)</span> volumes using external primers SLF / S762 and internal primer sets S823 / S662 and S825 / SLIR as described <span class="citation">(<a href="references.html#ref-mcinnesTrypanosomaIrwiniSp2009" role="doc-biblioref">McInnes et al. 2009</a>)</span>. Gel electrophoresis and Sanger sequencing using internal primers in both directions were carried out as above. No-template and extraction controls were included throughout the laboratory processes. Extractions, pre-PCR and post-PCR procedures were performed in laboratories physically separated from each other in order to minimise the risk of contamination. In addition, no <em>Tr. lewisi</em> species have been previously isolated or amplified in the specific laboratories used.</p>
<p>Sequences were subject to BLAST analysis to identify the most similar species and genotypes. Nucleotide sequences from the <em>Trypanosoma Herpetosoma</em> subgenus were retrieved from GenBank <span class="citation">(<a href="references.html#ref-bensonGenBank2017" role="doc-biblioref">Benson et al. 2017</a>)</span> and aligned with sequences obtained in the present study using MUSCLE <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span>. The final alignments were imported into MEGA 7 <span class="citation">(<a href="references.html#ref-kumarMEGA7MolecularEvolutionary2016" role="doc-biblioref">S. Kumar, Stecher, and Tamura 2016</a>)</span>, and the most appropriate nucleotide selection model was selected using the dedicated feature based on the Bayesian Information Criterion (BIC). Phylogenetic reconstruction was conducted in MEGA7 using maximum likelihood (ML) and neighbour-joining (NJ) analyses, missing data and positions containing gaps were eliminated. Alignments of the <em>Herpetosoma</em> subgenus at the 18S v7-8 hypervariable region were also inspected for nucleotide differences. Genetic distances were calculated using the Kimura model <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span>.</p>
</div>
<div id="results-and-discussion-1" class="section level2" number="5.4">
<h2>
<span class="header-section-number">5.4</span> Results and discussion<a class="anchor" aria-label="anchor" href="#results-and-discussion-1"><i class="fas fa-link"></i></a>
</h2>
<p>A total of 47 animals were captured over the trap period. Blood samples were collected from 11 black rats from Warriewood Wetlands (n=4), and North Head (n=7). Black rats were distinguished from <em>Rattus fuscipes</em> and <em>Rattus norvegicus</em> by their slender body, elongated head, large ears and pointed nose as per Menkhorst and Knight <span class="citation">(<a href="references.html#ref-menkhorstFieldGuideMammals2011" role="doc-biblioref">2011</a>)</span>. Two rat samples from North Head were positive for <em>Trypanosoma</em> species by molecular methods, and of these a blood smear was only available in one case, however no trypomastigote stages were observed by light microscopy despite prolonged searching of the cell layer. Black rat samples that were negative for molecular evidence of trypanosomes were also screened by microscopy and, no organisms were detected. The absence of a morphological identification in this report is disappointing, however it is not unexpected as previous studies have noted that rats (<em>R. rattus</em>) experimentally infected with <em>Tr. lewisi</em> transit from an acute phase where parasites multiply rapidly, followed by a chronic phase, during which parasite numbers progressively diminish and may disappear from circulation altogether <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>)</span>.</p>
<p>Initial screening produced ~550 bp product of the 18S rRNA gene in samples BR042 and BR048, these sequences were 100% identical to each other. A near full length 18S rRNA sequence (1,928 bp) was obtained from both samples also confirming that the sequences were 100% identical and a representative sequence of the 18S rRNA gene from sample BR042 was used for phylogenetic purposes (GenBank accession MN512227). Analyses using BLAST showed sequences were highly similar (&gt;98% identity) to the <em>Herpetosoma</em> subgenus.</p>
<div class="figure">
<span style="display:block;" id="fig:F51"></span>
<img src="figures/ms-figs/Ch5-Fig1.png" alt="Maximum likelihood phylogenetic reconstruction of \textit{Trypanosoma Herpetosoma} subgenus based on the 18S rRNA locus. Evolutionary relationship inferred using the Kimura 2-parameter (K2P) + G substitution model with 1,000 bootstrap replicates. Node support values shown for ML and Neighbour-Joining (NJ) analyses respectively, values &lt;50 have been hidden. All positions containing gaps and missing data were eliminated. (a) phylogeny based on short alignment (483 bp) of V7-8 hyper-variable region of the 18S locus (b) phylogeny based on longer alignment (1,491 bp) of 18S locus, outgroup to \textit{Tr. cruzi} (AJ009150) not shown. Number of substitutions per nucleotide position is represented by the scale bar. New sequence from the present study is designated in bold. Sequences from Australia in blue \textit{Tr. nabiasi} UK isolate (AJ843896) is identical to sequences obtained from Australian wild rabbits (\textit{O. cuniculus}) and their fleas (\textit{S. cuniculi}) (see Hamilton et al., 2005)." width="95%"><p class="caption">
Figure 5.1: Maximum likelihood phylogenetic reconstruction of  subgenus based on the 18S rRNA locus. Evolutionary relationship inferred using the Kimura 2-parameter (K2P) + G substitution model with 1,000 bootstrap replicates. Node support values shown for ML and Neighbour-Joining (NJ) analyses respectively, values &lt;50 have been hidden. All positions containing gaps and missing data were eliminated. (a) phylogeny based on short alignment (483 bp) of V7-8 hyper-variable region of the 18S locus (b) phylogeny based on longer alignment (1,491 bp) of 18S locus, outgroup to  (AJ009150) not shown. Number of substitutions per nucleotide position is represented by the scale bar. New sequence from the present study is designated in bold. Sequences from Australia in blue  UK isolate (AJ843896) is identical to sequences obtained from Australian wild rabbits () and their fleas () (see Hamilton et al., 2005).
</p>
</div>
<p>Phylogenetic analysis of the shorter (483 bp) 18S rRNA gene alignment was used in order to include a greater variety of reference sequences, in particular for the context of the present study to include the only other <em>Tr. lewisi</em>-like sequences from Australia <span class="citation">(<a href="references.html#ref-hamiltonInadvertentIntroductionAustralia2005" role="doc-biblioref">P. B. Hamilton, Stevens, Holz, et al. 2005</a>; <a href="references.html#ref-averisDiversityDistributionHostparasite2009" role="doc-biblioref">Averis et al. 2009</a>)</span>. Figure <a href="black-rat.html#fig:F51">5.1</a> shows the phylogeny of the <em>Trypanosoma</em> <em>Herpetosoma</em> subgenus. As demonstrated by the polytomy present in Figure <a href="black-rat.html#fig:F51">5.1</a>a, this short region of the 18S rRNA gene is insufficient in the differentiation of members within the <em>Tr. lewisi</em> clade. Due to the speed at which the 18S rRNA locus has evolved, short regions of this locus have been reported as being unsuitable for inference of evolutionary relationships between <em>Trypanosoma</em> species <span class="citation">(<a href="references.html#ref-hamiltonResolvingRelationshipsAustralian2011" role="doc-biblioref">Patrick B. Hamilton and Stevens 2011</a>)</span>.</p>
<p>Reconstruction of phylogenetic relationships over a longer region (1,491 bp) of the 18S rRNA gene exhibited superior resolution within the <em>Tr. lewisi</em> clade Figure <a href="black-rat.html#fig:F51">5.1</a>b. In this phylogeny, sequences obtained from Australian black rats in the present study did not fall within the <em>Tr. lewisi</em> sensu stricto clade; instead they formed a distinct group of sequences that branched separately from other reference sequences. Pairwise distance analysis over a 1,491 bp alignment of the 18S rRNA gene demonstrated sequences from the black rat were 99.5% similar to <em>Trypanosoma microti</em> (AJ009158). The next most similar sequences were <em>Trypanosoma</em> sequences from voles in Japan (AB242275, AB242276) and a flea from Czech Republic (KF054111), all of which were 99.4% similar. Members of the <em>Tr. lewisi</em> sensu stricto clade, as shown in Figure <a href="black-rat.html#fig:F52">5.2</a>, were all 100% identical to each other over the 1,491 bp alignment. These were the third most similar sequences (99.3%) to the <em>Trypanosoma</em> sp. identified in the present study. Inspection of a 433 bp alignment at the V7-8 18S hyper-variable region (Figure <a href="black-rat.html#fig:F52">5.2</a>), demonstrated that the most similar trypanosome sequence to that obtained in this study were those from a native Australian mouse in the south-west of Western Australia (<em>Pseudomys albocinereus</em>) (FJ823119) with seven single nucleotide polymorphisms (SNPs).</p>
<p>The phylogeny in the present study supports previous research by Hamilton et al. <span class="citation">(<a href="references.html#ref-hamiltonInadvertentIntroductionAustralia2005" role="doc-biblioref">2005</a>)</span> that demonstrated the <em>Tr. lewisi</em> clade can be divided into two subclades, consisting of <em>Tr. lewisi</em>, <em>Tr. musculi</em>, <em>Tr. rabinowitschae</em>, <em>Tr. blanchardi</em> and <em>Tr. grosi</em> in subclade one and <em>Tr. nabiasi</em>, <em>Tr. microti</em>, and <em>Tr. otospermophili</em> in subclade two. Trypanosome sequences obtained in the present study from Australian black rats form a paraphyletic group to the two monophyletic <em>Tr. lewisi</em> clades. Despite the basal position in the phylogenetic tree, genetic distances show a high similarity to <em>Tr. lewisi</em> subclade two.</p>
<div class="figure">
<span style="display:block;" id="fig:F52"></span>
<img src="figures/ms-figs/Ch5-Fig2.png" alt="Polymorphic sites within the V7-8 hyper-variable region of the 18S rRNA locus (433 bp) for trypanosomes of the subgenus \textit{Herpetosoma}. \textit{Tr. nabiasi} UK isolate (AJ843896) is identical to sequences obtained from Australian wild rabbits (\textit{O. cuniculus}) and their fleas (\textit{S. cuniculi}) (see Hamilton et al., 2005). Country abbreviations; Australia (AU), Brazil (BR), China (CN), Czech Republic (CZ), France (FR), Indonesia (ID), Japan (JP), Spain (ES), United Kingdom (UK), United States of America (US), unknown (unk)." width="95%"><p class="caption">
Figure 5.2: Polymorphic sites within the V7-8 hyper-variable region of the 18S rRNA locus (433 bp) for trypanosomes of the subgenus .  UK isolate (AJ843896) is identical to sequences obtained from Australian wild rabbits () and their fleas () (see Hamilton et al., 2005). Country abbreviations; Australia (AU), Brazil (BR), China (CN), Czech Republic (CZ), France (FR), Indonesia (ID), Japan (JP), Spain (ES), United Kingdom (UK), United States of America (US), unknown (unk).
</p>
</div>
<p>Morphological identification of rodent trypanosomes in Australia, attributed to <em>Tr. lewisi</em>, was first made by T. L. Bancroft in 1888 in black rats in Brisbane <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>)</span>, with subsequent records by various scientists who confirmed the presence of this parasite in; Brisbane by Pound (1905), in Perth by Cleland (1906, 1908), and in Sydney by Johnston (1909) (all cited by Mackerras <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>)</span>). Trypanosomes presumed to be <em>Trypanosoma</em> <em>lewisi</em> were first identified in native Australian fauna by Mackerras <span class="citation">(<a href="references.html#ref-mackerrasCatalogueAustralianMammals1958a" role="doc-biblioref">1958</a>)</span>. Morphological detection of the parasite has been made from the bush rat (<em>Rattus fuscipes</em>; Queensland) and the water rat (<em>Hydromys chrysogaster</em>; Queensland) <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>, <a href="references.html#ref-mackerrasCatalogueAustralianMammals1958a" role="doc-biblioref">1958</a>)</span>. More recently, molecular reports of <em>Trypanosoma</em> species from the <em>Tr. lewisi</em> clade have been made from native wildlife in Western Australia, including two bush rats (<em>Rattus fuscipes</em>), a dibbler (<em>Parantechinus apicalis</em>) and an ash-grey mouse (<em>Pseudomys albocinereus</em>) <span class="citation">(<a href="references.html#ref-averisDiversityDistributionHostparasite2009" role="doc-biblioref">Averis et al. 2009</a>)</span>. Interestingly, despite sampling from 371 native mammals, 19 different species and 14 sites, detection of <em>Tr. lewisi</em>-like species was confined only to mammals from Fitzgerald River in the south-west of Australia. The identification of <em>Tr. lewisi</em>-like spp. by Averis et al. <span class="citation">(<a href="references.html#ref-averisDiversityDistributionHostparasite2009" role="doc-biblioref">2009</a>)</span> was limited by the short size of the 18S rRNA gene analysed (444 bp). As demonstrated in the present study, across a short region of the 18S rRNA gene, trypanosomes within the <em>Tr. lewisi</em> clade can share a high sequence similarity (Figure <a href="black-rat.html#fig:F51">5.1</a>), however upon more robust analysis of a longer fragment it is evident that sequences within the <em>Tr. lewisi</em> clade form distinct groups. Additional genetic information (e.g. glycosomal glyceraldehyde-3-phosphate dehydrogenase (gGAPDH)) will also assist in determining the phylogenetic relationships of these closely related species in future studies.</p>
<p>The genetic relationship of the Australian isolate BR042 to <em>Tr. microti</em>, a trypanosome isolated from a vole (<em>Microtis agrestis</em>) in England suggest that the <em>Tr. lewisi</em> isolates from this study are closely related to rodent trypanosomes in the subgenus <em>Herpetosoma</em> from Europe. This agrees with finding by Hamilton et al. <span class="citation">(<a href="references.html#ref-hamiltonEvolutionTrypanosomaCruzi2012" role="doc-biblioref">2012</a>)</span> reporting that Australian trypanosomes appear to be more closely related to trypanosomes outside of Australia than to each other. This observation is consistent with the hypothesis that some trypanosomes have been introduced into the Australian continent comparatively recently, rather than this trypanosome evolving in Australia following the breakup of Gondwana; <em>Trypanosoma lewisi</em> (known to be host specific) likely arrived in Australia with black rats when individuals colonised the mainland. Additionally, Hamilton et al. <span class="citation">(<a href="references.html#ref-hamiltonInadvertentIntroductionAustralia2005" role="doc-biblioref">2005</a>)</span> detected European <em>Tr. nabiasi</em> (from the <em>Tr. lewisi</em> clade) in Australian wild rabbits (<em>Oryctolagus cuniculus</em>) suggesting that these parasites may have been brought into Australia with the first 24 rabbits introduced from England in 1859 <span class="citation">(<a href="references.html#ref-hamiltonInadvertentIntroductionAustralia2005" role="doc-biblioref">P. B. Hamilton, Stevens, Holz, et al. 2005</a>)</span>.</p>
<p>The introduction of black rats and their associated trypanosomes to regions previously free of these species has long been considered responsible for the extinction of two native rat species <em>Rattus macleari</em> and <em>Rattus nativitatis</em>, as occurred on Christmas Island (an island Australian Territory located in the Indian Ocean, south of Indonesia) <span class="citation">(<a href="references.html#ref-wyattHistoricalMammalExtinction2008" role="doc-biblioref">Wyatt et al. 2008</a>)</span>. Recent research has since concluded that the rapid decline and extinction of the two endemic rat species was correctly attributed to infections with <em>Tr. lewisi</em> <span class="citation">(<a href="references.html#ref-wyattHistoricalMammalExtinction2008" role="doc-biblioref">Wyatt et al. 2008</a>)</span>. A review of historical records demonstrated a rapid extinction event following the arrival of black rats on the island in September 1900 and an absence of native rat sightings by October 1904 <span class="citation">(<a href="references.html#ref-greenMammalExtinctionIntroduced2014" role="doc-biblioref">Green 2014</a>)</span>. A recent study by Dybing et al. <span class="citation">(<a href="references.html#ref-dybingGhostsChristmasAbsence2016" role="doc-biblioref">2016</a>)</span> investigated the presence of <em>Trypanosoma</em> and <em>Leishmania</em> spp. from feral cats (<em>Felis catus</em>) and black rats (<em>R. rattus</em>) on Christmas Island. Through molecular analysis of spleen samples, the study did not detect any <em>Trypanosoma</em> or <em>Leishmania</em> species. In addition, the same study reported an absence of these parasites from feral cat samples from Dirk Hartog Island and sites from south-west Western Australia.</p>
<p>North Head is situated at the entrance to the Sydney harbour and is dominated by Eastern Suburbs banksia scrub, an endangered ecological community <span class="citation">(<a href="references.html#ref-perkinsEasternSuburbsBanksia2012" role="doc-biblioref">Perkins et al. 2012</a>)</span>. Following European arrival, the headland was used to quarantine arriving ship passengers. In addition to being home to endangered populations of long-nosed bandicoots (<em>Perameles nasuta</em>) and little penguins (<em>Eudyptula minor</em>), reintroductions of native fauna species, such as bush rats (<em>Rattus fuscipes</em>), eastern pygmy possums (<em>Cercartetus nanus</em>) and brown antechinus (<em>Antechinus stuartii</em>), have also occurred at this site. While, to date, there is no evidence of a spill-over of trypanosomes within the <em>Tr. lewisi</em> clade to native species, ongoing monitoring of such populations is advised given the historical significance of this parasite with respect to native animal declines <span class="citation">(<a href="references.html#ref-greenMammalExtinctionIntroduced2014" role="doc-biblioref">Green 2014</a>; <a href="references.html#ref-wyattHistoricalMammalExtinction2008" role="doc-biblioref">Wyatt et al. 2008</a>)</span>.</p>
<p>In addition to trypanosomes, black rats may act as reservoirs for many other sources of infectious agents <span class="citation">(<a href="references.html#ref-banksReviewEvidencePotential2012" role="doc-biblioref">Banks and Hughes 2012</a>)</span>. Additional information regarding the presence, distribution and diversity of pathogens harboured by black rats in Australia is critical to understanding the dynamics of pathogen spill-over <span class="citation">(<a href="references.html#ref-beckerDynamicIntegrativeApproaches2019" role="doc-biblioref">Becker et al. 2019</a>)</span>. Future research encompassing both morphological and molecular techniques is on-going by the authors. Collection of ectoparasites, blood, and tissue samples from both native and introduced wildlife will likely continue to shed light on the diversity and distribution of vector-borne microorganisms impacting wildlife, domestic animals and humans.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></div>
<div class="next"><a href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#black-rat"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="nav-link" href="#preface-4">Preface</a></li>
<li><a class="nav-link" href="#abstract-4"><span class="header-section-number">5.1</span> Abstract</a></li>
<li><a class="nav-link" href="#introduction-3"><span class="header-section-number">5.2</span> Introduction</a></li>
<li><a class="nav-link" href="#methods-3"><span class="header-section-number">5.3</span> Methods</a></li>
<li><a class="nav-link" href="#results-and-discussion-1"><span class="header-section-number">5.4</span> Results and discussion</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/ulyngs/oxforddown/blob/master/05-black-rat.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/ulyngs/oxforddown/edit/master/05-black-rat.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></strong>" was written by Siobhon Lee Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
