<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>1 Literature Review | Ecology of Ticks and Microbes in Australia Wildlife</title>
<meta name="author" content="Siobhon Lee Egan">
<meta name="generator" content="bookdown 0.24 with bs4_book()">
<meta property="og:title" content="1 Literature Review | Ecology of Ticks and Microbes in Australia Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="1 Literature Review | Ecology of Ticks and Microbes in Australia Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/header-attrs-2.11.22/header-attrs.js"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="Preface        Ticks (Acari: Ixodida) are a group of blood-feeding arthropods and represent one of the most important vector groups that affect human and animal health. Almost 900 species of tick...">
<meta property="og:description" content="Preface        Ticks (Acari: Ixodida) are a group of blood-feeding arthropods and represent one of the most important vector groups that affect human and animal health. Almost 900 species of tick...">
<meta name="twitter:description" content="Preface        Ticks (Acari: Ixodida) are a group of blood-feeding arthropods and represent one of the most important vector groups that affect human and animal health. Almost 900 species of tick...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="active" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/ulyngs/oxforddown">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="litrev" class="section level1" number="1">
<h1>
<span class="header-section-number">1</span> Literature Review<a class="anchor" aria-label="anchor" href="#litrev"><i class="fas fa-link"></i></a>
</h1>
<p><!-- this will include a mini table of contents--></p>
<div id="preface" class="section level2 unnumbered">
<h2>Preface<a class="anchor" aria-label="anchor" href="#preface"><i class="fas fa-link"></i></a>
</h2>
<p>

          
</p>
<p>Ticks (Acari: Ixodida) are a group of blood-feeding arthropods and represent one of the most important vector groups that affect human and animal health. Almost 900 species of tick have been described globally, and 10% have been documented to transmit a variety of pathogens to livestock, companion animals and humans <span class="citation">(<a href="references.html#ref-jongejanGlobalImportanceTicks2004" role="doc-biblioref">Jongejan and Uilenberg 2004</a>)</span>.
Ticks can carry a diverse range of infectious microbes such as viruses, piroplasms, spirochaetes and rickettsiales <span class="citation">(<a href="references.html#ref-pfaffleEcologyTickborneDiseases2013" role="doc-biblioref">Pfaffle et al. 2013</a>)</span>.
Tick-borne diseases globally include Lyme borreliosis (<em>Borrelia burgdorferi</em> sensu lato), ehrlichiosis (<em>Ehrlichia chaffeensis)</em>, babesiosis (<em>Babesia microti)</em>, tick-borne encephalitis (Flavivirus) and Powassan disease (Powassan virus).
A tick bite can also induce local reactions, such as irritation and erythema migrans and may induce severe syndromes such as paralysis, anaphylaxis and mammalian meat allergy<span class="citation">(<a href="references.html#ref-vannunenTickinducedAllergiesMammalian2015" role="doc-biblioref">Nunen 2015</a>; <a href="references.html#ref-beamanNoninfectiousIllnessTick2018" role="doc-biblioref">Beaman 2018</a>; <a href="references.html#ref-pienaarTickParalysisSolving2018" role="doc-biblioref">Pienaar, Neitz, and Mans 2018</a>)</span>.</p>
<p>Presently, there are 74 extant tick species in Australia that have been described <span class="citation">(<a href="references.html#ref-barkerList70Species2014" role="doc-biblioref">S. C. Barker, Walker, and Campelo 2014</a>; <a href="references.html#ref-ashMorphologicalMolecularDescription2017" role="doc-biblioref">Ash et al. 2017</a>; <a href="references.html#ref-heathNewSpeciesTick2017" role="doc-biblioref">Heath and Palma 2017</a>; <a href="references.html#ref-kwakIxodesHeathiSp2018" role="doc-biblioref">Kwak, Madden, and Wicker 2018</a>; <a href="references.html#ref-barkerIxodesBarkeriSp2019" role="doc-biblioref">D. Barker 2019</a>)</span>, with eight well known to bite humans <span class="citation">(<a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">Stephen C. Barker and Walker 2014</a>)</span>.<br>
To date only three human tick-borne pathogens (TBPs) are recognised as endemic in Australia <span class="citation">(<a href="references.html#ref-gravesTickborneInfectiousDiseases2017" role="doc-biblioref">S. R. Graves and Stenos 2017</a>)</span>; <em>Rickettsia australis</em> and <em>Rickettsia honei</em> are the causative agents for Queensland tick typhus and Australian (Flinders Island) spotted fever, respectively.
The third TBP in Australia is <em>Coxiella burnetii</em> the causative agent of Q Fever, and while it is not usually associated with a tick-bite, ticks play a critical role in the life cycle of this bacterium.
Despite endemic rickettsial infections being recognised in Australia, there has been an increasing pressure surrounding the presence of an unknown TBP causing illness in people bitten by Australian ticks.
In particular, public concern around a ‘lyme-like’ illnesses has gained considerable attention and a commonwealth senate inquiry was held in 2016 <span class="citation">(<a href="references.html#ref-radcliffeGrowingEvidenceEmerging2016" role="doc-biblioref">Radcliffe et al. 2016</a>)</span>.</p>
<p>This literature review will provide a brief overview of tick systematics and tick biology.
It will then explore some examples of tick-borne diseases recognised globally and reflect on the situation in Australia.
In particular, it will review some of the recent discoveries over the past decade that have been made on the identification of novel microbes.
The review will then explore the role of hosts in the maintenance of ticks and tick-borne pathogens.
This is given to lead to the idea of a One Health approach and the advantages of using a wildlife surveillance approach to understanding the dynamics of tick-borne diseases in Australia.
The review will then conclude with a brief insight into the technologies used to identify tick-borne microbes with a focus on new molecular approaches.
The overarching themes of this literature review include; ticks, zoonotic tick-borne pathogens, ecology of tick-borne pathogens, and use of new molecular tools and wildlife surveillance to investigate the potential the potential for a novel zoonotic tick-borne pathogen in Australia.</p>
<p>At the end of this literature review on outline of the thesis aims <a href="thesis-aims.html#thesis-aims"><strong>??</strong></a> is provided.</p>
</div>
<div id="ticks" class="section level2" number="1.1">
<h2>
<span class="header-section-number">1.1</span> Ticks<a class="anchor" aria-label="anchor" href="#ticks"><i class="fas fa-link"></i></a>
</h2>
<div id="tick-systematics" class="section level3" number="1.1.1">
<h3>
<span class="header-section-number">1.1.1</span> Tick systematics<a class="anchor" aria-label="anchor" href="#tick-systematics"><i class="fas fa-link"></i></a>
</h3>
<p>Ticks belong to order Ixodida, which is split into three extant families; Ixodidae (hard ticks), Argasidae (soft ticks) and a single species belonging to the Nuttalliellidae family <span class="citation">(<a href="references.html#ref-durdenModernTickSystematics2014" role="doc-biblioref">Durden and Beati 2014</a>)</span>.
The fourth family, Deinocrotonidae, is an extinct taxa that was described after discovery of the newly named species <em>Deinocroton draculi</em> in 99 million-year-old Cretaceous amber <span class="citation">(<a href="references.html#ref-penalverTicksParasitisedFeathered2017" role="doc-biblioref">Penalver et al. 2017</a>)</span>.
The Ixodidae, or hard tick family, has at least 742 described species <span class="citation">(<a href="references.html#ref-guglielmoneIxodidaeAcariIxodoidea2020" role="doc-biblioref">Alberto A. Guglielmone, Petney, and Robbins 2020</a>)</span> and represents the majority of species that are of veterinary and medical significance.
Ixodidae are characterised by the presence of a hard plate (scutum/conscutum) on the dorsal aspect and terminal mouth parts.
In contrast Argasidae lack a hard plate (no scutum/conscutum) and have ventral mouth parts.
The Ixodidae family can further be divided into two clades.
The prostriata, which are characterised by the anal groove anterior to the anus and consists of a single genus, <em>Ixodes</em>.
The remainding Ixodidae genera belong to the metastriata clade which the anal groove (if present) posterior to the anus.
The number of Argasidae (soft) ticks species has grown in recent years and there are over 200 species described <span class="citation">(<a href="references.html#ref-mansArgasidIxodidSystematics2019" role="doc-biblioref">Ben J. Mans et al. 2019</a>)</span>.
Many of these species are cryptic and the full diversity of this family of ticks remains to be full described.</p>
<p>While the family level classification of ticks has remain stable for many years the phylogeny and systematics within these remains a point of contention; particularly for soft ticks <span class="citation">(<a href="references.html#ref-mansArgasidIxodidSystematics2019" role="doc-biblioref">Ben J. Mans et al. 2019</a>)</span>.
Molecular tools have helped shed light on the evolutionary history of ticks and as such many changes to tick taxonomy have been made over the years.
The current working hypothesis of tick phylogeny highlights the uniqueness of tick fauna in the Australasian region.
Molecular and morphological analysis suggests <em>Ixodes</em> endemic to Australia, New Guinea and New Zealand belong to a unique <em>Ixodes</em> lineage with at least 28 extant species <span class="citation">(<a href="references.html#ref-barkerSystematicsEvolutionTicks2004" role="doc-biblioref">S. C. Barker and Murrell 2004</a>)</span>.
Although not all of these species have undergone molecular systematic analysis, there is a clear grouping of this distinct lineage (Figure <a href="litrev.html#fig:F1treeixod">1.1</a>).
The previously recognised genus <em>Aponomma</em> belonged to the hard tick family, Ixodidae, and was poorly defined.
To correct this a new genus was created called <em>Bothriocroton</em> and all indigenous Australian <em>Aponomma</em> species were moved to this new genus <span class="citation">(<a href="references.html#ref-keiransAponommaBothriocrotonGlebopalma1994" role="doc-biblioref">Keirans, King, and Sharrad 1994</a>; <a href="references.html#ref-klompenNewSubfamilyBothriocrotoninae2002" role="doc-biblioref">J. Klompen, Dobson, and Barker 2002</a>)</span>.
Since the creation of the genus <em>Bothriocroton</em> new species have been described and reinstated, bringing the total extant species to seven, all of which are endemic to the Australiasian region (Figure <a href="litrev.html#fig:F1treeixod">1.1</a>) (<em>Bt. tachyglossi</em>, <em>Bt. auruginans</em>, <em>Bt. concolor</em>, <em>Bt. hydrosauri</em>, <em>Bt. undatum</em>, <em>Bt. glebopalma</em>, and <em>Bt. oudemansi</em>) <span class="citation">(<a href="references.html#ref-keiransAponommaBothriocrotonGlebopalma1994" role="doc-biblioref">Keirans, King, and Sharrad 1994</a>; <a href="references.html#ref-klompenNewSubfamilyBothriocrotoninae2002" role="doc-biblioref">J. Klompen, Dobson, and Barker 2002</a>; <a href="references.html#ref-andrewsSystematicStatusAponomma2006" role="doc-biblioref">Andrews et al. 2006</a>; <a href="references.html#ref-beatiBothriocrotonOudemansiNeumann2008" role="doc-biblioref">Beati et al. 2008</a>)</span>.
The remainding members of the former <em>Aponomma</em> genus were transferred to <em>Amblyomma</em>, however some issues of polyphyly within the genus <em>Amblyomma</em> remained <span class="citation">(<a href="references.html#ref-burgerPhylogeneticAnalysisTicks2012" role="doc-biblioref">Burger et al. 2012</a>; <a href="references.html#ref-burgerPhylogeneticAnalysisMitochondrial2013" role="doc-biblioref">Burger, Shao, and Barker 2013</a>)</span>.
To correct this issue two new genera have since been established, <em>Robertsicus</em> and <em>Archaeocroton</em> <span class="citation">(<a href="references.html#ref-barkerTwoNewGenera2018" role="doc-biblioref">S. C. Barker and Burger 2018</a>)</span>.</p>
<div class="figure">
<span style="display:block;" id="fig:F1treeixod"></span>
<img src="figures/ms-figs/Ch1-treeixod.pdf" alt="Phylogeny of extant genera within the hard tick family (Ixodidae)." width="95%"><p class="caption">
Figure 1.1: Phylogeny of extant genera within the hard tick family (Ixodidae).
</p>
</div>
<p>To date Australia has 74 extant described tick species (Figure <a href="litrev.html#fig:F1ausixod">1.2</a>).
Five of these were introduced with domestic animals following European arrival in 1788: <em>Argas persicus</em>, <em>Otobius megnini</em>, <em>Haemaphysalis longicornis</em>, <em>Rhipicephalus sanguineus</em> (recently proposed as <em>Rh. linnaei</em>), and <em>Rhipicephalus australis</em> (<em>Boophilus microplus)</em> <span class="citation">(<a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">Stephen C. Barker and Walker 2014</a>)</span>.
The first new native Australian <em>Ixodes</em> tick was described in over 50 years, from woylies (<em>Bettongia penicillata</em>), a critically endangered marsupial in south-west Western Australia and was named <em>Ixodes woyliei</em> <span class="citation">(<a href="references.html#ref-ashMorphologicalMolecularDescription2017" role="doc-biblioref">Ash et al. 2017</a>)</span>.
The text ‘Australian Ticks’ by Roberts <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>)</span> remains the foundation reference guide on ticks in Australia, in particular the detailed specimen descriptions and taxonomic keys used for morphological identification.
The more recent guide, ‘Ticks of Australia’ by Barker and Walker <span class="citation">(<a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">2014</a>)</span>, has provided a much needed update on the most common tick species that effect humans and animals in Australia.
As raised by Barker et al. <span class="citation">(<a href="references.html#ref-barkerList70Species2014" role="doc-biblioref">2014</a>)</span> there are a number of issues surrounding the list of recognised ticks in Australia.
Since Barker et al. <span class="citation">(<a href="references.html#ref-barkerList70Species2014" role="doc-biblioref">2014</a>)</span> there have also been a number of taxonomic changes and new information has been presented showing that the full picture of tick taxonomy in Australia is not yet complete.
In many cases host records of tick species in Australia are scarce and a large amount of time has passed since some records.
This has resulted in a number of queries surrounding the complete list of tick species present in Australia.
In some cases systematic classification remains elusive with biological data incomplete.
Issues relating to classification and/or presence of Australian ticks include:
(i) <em>Amblyomma flavomaculatum</em> (yellow spotted monitor lizard tick) was listed in Roberts <span class="citation">(<a href="references.html#ref-robertsAustralianSpeciesAponomma1953" role="doc-biblioref">1953</a>, <a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>; <a href="references.html#ref-robertsFurtherObservationsAustralian1964" role="doc-biblioref">F. H. S. Roberts 1964</a>)</span> and referred to as <em>Aponomma pulchrum</em>, which is now synonymous with <em>Am. flavomaculatum</em>.
However since these records by Roberts, the presence of this tick in Australia has not been unconfirmed;
(ii) the distinction between <em>Amblyomma australiense</em> and <em>Amblyomma echidnae</em> remains unclear <span class="citation">(<a href="references.html#ref-guglielmoneCommentsControversialTick2009" role="doc-biblioref">A. A. Guglielmone et al. 2009</a>)</span>. <em>Amblyomma australiense</em> was originally described from a museum specimens of the long-beaked echidna (<em>Zaglossus bruijnii</em>) in the Kimberly.
Roberts <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>)</span> leaves open the possibility that <em>Am. echidnae</em> is a subspecies of <em>Am. australiense</em>.
(iii) the status of <em>Bt. tachyglossi</em> was resurrected by Andrews <span class="citation">(<a href="references.html#ref-andrewsSystematicStatusAponomma2006" role="doc-biblioref">2006</a>)</span>, however prior to that it was considered a synonym of <em>Bt. hydrosauri</em>, therefore records prior to 2006 should be treated with caution;
(iv) records of <em>Haempahysalis longicornis</em> were previously referred to as <em>Haemaphysalis bispinosa</em> <span class="citation">(<a href="references.html#ref-robertsSystematicStudyAustralian1963" role="doc-biblioref">F. H. S. Roberts 1963</a>)</span>, it is now recognised that <em>Ha. bispinosa</em> is a distinct species distributed across parts of Asia and early records were attributed.</p>
<p>Changes in tick nomenclature and taxonomy can cause disruption to ecologists, veterinarians and physicians.
In many instances changes in species names takes many years to filter through for use by the broader scientific community and for new names to be fully adopted.
For example, the reptile tick <em>Bt. hydrosauri</em> (formerly <em>Aponomma hydrosauri</em>) is an important vector of human disease (<em>Rickettsia honei)</em>, and its changing species name can cause confusion to the public and medical professionals (e.g. <span class="citation"><a href="references.html#ref-stenosAponommaHydrosauriReptileassociated2003" role="doc-biblioref">Stenos et al.</a> (<a href="references.html#ref-stenosAponommaHydrosauriReptileassociated2003" role="doc-biblioref">2003</a>)</span>).
Additionally the lack of accurate keys and morphological descriptions is particularly challenging in these instances.
In the case of tick-borne diseases it is vital to get an accurate identification of the tick that is responsible for the onset of symptoms.
In the case of <em>Ix. holocyclus</em> and <em>Ix. cornuatus</em>, only recently has sufficient keys and genetic data been published to allow the identification between these two morphologically very similar species <span class="citation">(<a href="references.html#ref-songPhylogeneticPhylogeographicRelationships2011" role="doc-biblioref">Song et al. 2011</a>; <a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">Stephen C. Barker and Walker 2014</a>; <a href="references.html#ref-kwakKeysMorphologicalIdentification2017" role="doc-biblioref">Kwak 2017</a>)</span>.
<em>Ixodes holocyclus</em> remains the most important vector of illness to Australians being the cause of paralysis, mammalian meat allergy, significant local reactions &amp; allergic responses and the vector of rickettsial diseases.
There are many suspected misidentifications of <em>Ix. holocyclus</em>; for example Tasmanian devils (<em>Sarcophilus harrisii</em>) in Tasmania (TAS), likely <em>Ix. cornuatus</em>, and numbats (<em>Myrmecobius fasciatus</em>), in Western Australia (WA), likely <em>Ix. myrmecobii</em>.
More recently the Australian brown dog tick has been renamed to <em>Rhipicephalus linnaei</em> <span class="citation">(<a href="references.html#ref-slapetaTropicalLineageBrown2021" role="doc-biblioref">Jan Slapeta, Chandra, and Halliday 2021</a>)</span>, previously known as <em>Rhipicephalus sanguineus</em> tropical lineage.
This name change is likely to create debate among taxonomist and the tick community, and therefore cause confusion for the general public.
While taxonomy is the foundation of biological science and must be corrected were appropriate, it does cause unavoidable consequences in the short term.</p>
<p>The unique Australian tick fauna is an important consideration in understanding the presence of tick-borne diseases, which are discussed in further detail in section three of this literature review. Tick-borne pathogens have coevolved in their vector host, and as such, the evolutionary history of Australian ticks means that it is likely that pathogens present elsewhere in the world would be different from those that have evolved with native Australian ticks.</p>
<div class="figure">
<span style="display:block;" id="fig:F1ausixod"></span>
<img src="figures/ms-figs/Ch1-ausixod.png" alt="Ticks (Ixodida) present in Australian (74 extant species)." width="95%"><p class="caption">
Figure 1.2: Ticks (Ixodida) present in Australian (74 extant species).
</p>
</div>
</div>
<div id="tick-biology-and-ecology" class="section level3" number="1.1.2">
<h3>
<span class="header-section-number">1.1.2</span> Tick biology and ecology<a class="anchor" aria-label="anchor" href="#tick-biology-and-ecology"><i class="fas fa-link"></i></a>
</h3>
<p>Ticks have a remarkably long life cycle compared to other vectors of infectious disease, such as mosquitoes (Culicidae) and lice (Phthiraptera).
The life cycle of wild hard ticks is often measured in years and consists of four major stages; eggs, larva, nymph and adult.
With the exemption of the egg stage, development requires a blood meal, which often involves several hours, or even days of engorgement <span class="citation">(<a href="references.html#ref-cuppBiologyTicks1991" role="doc-biblioref">Cupp 1991</a>)</span>.
The vast majority of hard ticks have a three-host life cycle, where each life stage feeds on a host and then detaches to spend some time in the environment before moulting to the next stage.
Adult females are able to detach and reattached multiple times to continue feeding.
In the case of generalised tick species, current literature suggests that in most cases, immature life stages (e.g. larva and nymphs) feed on small to medium sized hosts and adult life stages feed on larger hosts <span class="citation">(<a href="references.html#ref-oliverBiologySystematicsTicks1989" role="doc-biblioref">Oliver 1989</a>)</span>.
Although this has been shown to occur in many cases such as <em>Ixodes ricinus</em> <span class="citation">(<a href="references.html#ref-krasnovHostCommunityStructure2007" role="doc-biblioref">Krasnov, Stanko, and Morand 2007</a>)</span>, detailed studies on the life cycle and host dynamics of Australian ticks remains largely unknown.</p>
<p>Host specificity is the association between a tick and vertebrate species that is critical for the ongoing survival and reproduction of the tick <span class="citation">(<a href="references.html#ref-hoogstraalTickhostSpecificity1982" role="doc-biblioref">Hoogstraal and Aeschlimann 1982</a>)</span>.
It has been suggested that up to 90% of tick species are considered ‘host specific’ <span class="citation">(<a href="references.html#ref-hoogstraalTickhostSpecificity1982" role="doc-biblioref">Hoogstraal and Aeschlimann 1982</a>)</span>.
However, there has been suggestions that this finding is simply due to the incomplete sampling and reporting of tick-host records <span class="citation">(<a href="references.html#ref-klompenEvolutionTicks1996" role="doc-biblioref">J. S. H. Klompen et al. 1996</a>)</span>.
In Australia, the primary reference for tick-host associations remains Roberts <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>)</span>, although Barker and Walker <span class="citation">(<a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">2014</a>)</span> has provided an updated reference for 16 species of veterinary and medical importance.
Despite this update however, there is no single, up-to-date reference of tick-host associations.
In many cases these findings remain buried in research laboratories, special interest groups and other niches.
Since Roberts <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">1970</a>)</span> publication detailed studies on the biology, ecology &amp; distribution of native Australian ticks have only been carried out on a handful of species most notably; <em>Ornithodoros gurenyi</em> (QLD, NT, SA, WA) <span class="citation">(<a href="references.html#ref-doubeEcologyKangarooTick1972" role="doc-biblioref">B. K. Doube 1972</a>; <a href="references.html#ref-doubeTwoRacesKangaroo1975" role="doc-biblioref">B. M. Doube 1975</a>)</span>, <em>Amblyomma triguttatum</em> (SA) <span class="citation">(<a href="references.html#ref-waudbySeasonalDensityFluctuations2007" role="doc-biblioref">H. P. Waudby and Petit 2007</a>)</span>; <em>Bothriocroton hydrosauri</em> <span class="citation">(<a href="references.html#ref-bullDispersalAustralianReptile1978" role="doc-biblioref">Bull 1978</a>; <a href="references.html#ref-andrewsMatingBehaviourAustralian1980" role="doc-biblioref">Andrews and Bull 1980</a>; <a href="references.html#ref-belanHostDetectionFour1991" role="doc-biblioref">Belan and Bull 1991</a>; <a href="references.html#ref-chiltonInterspecificDifferencesMicrohabitat1993" role="doc-biblioref">Chilton and Bull 1993</a>)</span>, <em>Ixodes cornuatus</em> (VIC, TAS) <span class="citation">(<a href="references.html#ref-jacksonGeneticVariationTicks2000" role="doc-biblioref">J. Jackson et al. 2000</a>, <a href="references.html#ref-jacksonDistributionsParalysisTicks2007" role="doc-biblioref">2007</a>; <a href="references.html#ref-songPhylogeneticPhylogeographicRelationships2011" role="doc-biblioref">Song et al. 2011</a>)</span>, <em>Ix. holocyclus</em> (QLD, NSW, VIC) <span class="citation">(<a href="references.html#ref-doubeSeasonalPatternsAbundance1979" role="doc-biblioref">B. M. Doube 1979</a>; <a href="references.html#ref-jacksonGeneticVariationTicks2000" role="doc-biblioref">J. Jackson et al. 2000</a>, <a href="references.html#ref-jacksonDistributionsParalysisTicks2007" role="doc-biblioref">2007</a>; <a href="references.html#ref-songPhylogeneticPhylogeographicRelationships2011" role="doc-biblioref">Song et al. 2011</a>)</span>, <em>Ixodes tasmani</em> (NSW) <span class="citation">(<a href="references.html#ref-murdochEcologyCommonMarsupial2005" role="doc-biblioref">Murdoch and Spratt 2005</a>)</span>, <em>Ha. bancrofti</em> (VIC) <span class="citation">(<a href="references.html#ref-laanOccuranceTickHaemaphysalis2011" role="doc-biblioref">Laan, Handasyde, and Beveridge 2011b</a>)</span>.
While these studies provide a solid foundation of Australian tick biology, in many cases they are largely limited to a relatively small study site.
An experimental investigation of <em>Ixodes hirsti</em> placed 1600 larvae on rats and chickens, however they found that no larvae attached to either host species, and none survived more than 24 hours <span class="citation">(<a href="references.html#ref-laanObservationsBiologyDistribution2011" role="doc-biblioref">Laan, Handasyde, and Beveridge 2011a</a>)</span>.
The authors described hosts removing the larvae by either eating or grooming.
The conditions of the experimental animals are not described in detail, however this perhaps suggests a host specificity of <em>Ixodes hirsti</em>, although the tick-host associations reviewed in the same paper show a wide range of taxa including kangaroos (<em>Macropus</em> spp.), Honeyeaters (<em>Phylidonyris</em> spp.), rats (<em>Rattus</em> spp.), and dogs (<em>Canis familiaris</em>) <span class="citation">(<a href="references.html#ref-laanObservationsBiologyDistribution2011" role="doc-biblioref">Laan, Handasyde, and Beveridge 2011a</a>)</span>.
The life cycle of <em>Ix. tasmani</em> was shown to complete within 4 months in artificial laboratory models on rats <span class="citation">(<a href="references.html#ref-murdochEcologyCommonMarsupial2005" role="doc-biblioref">Murdoch and Spratt 2005</a>)</span>, which is comparatively short compared to other ixodids <span class="citation">(<a href="references.html#ref-oliverBiologySystematicsTicks1989" role="doc-biblioref">Oliver 1989</a>)</span>.
The nocturnal and nidicolous questing nature of <em>Ix. tasmani</em> coincides with the activity of many of its host marsupial species <span class="citation">(<a href="references.html#ref-murdochEcologyCommonMarsupial2005" role="doc-biblioref">Murdoch and Spratt 2005</a>)</span>.</p>
</div>
</div>
<div id="tick-borne-diseases" class="section level2" number="1.2">
<h2>
<span class="header-section-number">1.2</span> Tick-borne diseases<a class="anchor" aria-label="anchor" href="#tick-borne-diseases"><i class="fas fa-link"></i></a>
</h2>
<p>This section of the review will briefly discuss some of the major human tick-borne diseases recognised globally, with an emphasis on bacterial pathogens.
The summary provided is by no means exhaustive, it is given in the context to explain the types of microbes that have been associated with disease in humans.
The review will then reflect on human tick-borne pathogens currently documented and characterised in Australia.
Finally it will conclude by providing a summary of recent research that has been done in relation to novel microbes in Australian ticks and how they relate to potential causative agents of human disease.</p>
<p>This review is confined to infectious microorganisms transmitted by ticks. An in-depth and exhaustive review of each individual microbe is beyond the scope of this literature review. Instead the following is presented to provide context to microbes which are explored in more detail throughout the thesis.
In the first instance, a general introduction of tick-associated organisms are introduced within a global context focusing on zoonotic significance.
Following this, more detail is provided from an Australian context.</p>
<div id="worldwide-summary" class="section level3" number="1.2.1">
<h3>
<span class="header-section-number">1.2.1</span> Worldwide summary<a class="anchor" aria-label="anchor" href="#worldwide-summary"><i class="fas fa-link"></i></a>
</h3>
<p>Ticks are responsible for transmitting the greatest variety of pathogenic microbes of any arthropod vector.
As a consequence they are important vectors of disease that affect humans, wildlife, livestock and companion animals <span class="citation">(<a href="references.html#ref-jongejanGlobalImportanceTicks2004" role="doc-biblioref">Jongejan and Uilenberg 2004</a>)</span>.
Incidence of human tick-borne diseases are steadily increasing, with recent reports from the Centre of Disease Control and Prevention (CDC, United States of America) describing a two-fold increase in the number of tick-borne disease cases; this accounts for 77% of all vector-borne diseases <span class="citation">(<a href="references.html#ref-rosenbergVitalSignsTrends2018" role="doc-biblioref">Rosenberg et al. 2018</a>)</span>.</p>
<div id="anaplasmataceae" class="section level4" number="1.2.1.1">
<h4>
<span class="header-section-number">1.2.1.1</span> Anaplasmataceae<a class="anchor" aria-label="anchor" href="#anaplasmataceae"><i class="fas fa-link"></i></a>
</h4>
<p>The family Anaplasmataceae are a group of obligate intracellular bacteria that reside in vacuoles of eukaryotic cells.
After a major taxonomic reorganisation in 2001, the phylogeny of the Anaplasmataceae family is now well accepted and includes <em>Ehrlichia</em>, <em>Anaplasma</em>, <em>Wolbachia</em>, <em>Neorickettsia</em> and the more recently described <em>Neoehrlichia</em> <span class="citation">(<a href="references.html#ref-rarGeneticDiversityAnaplasma2021" role="doc-biblioref">Vera Rar, Tkachev, and Tikunova 2021</a>)</span>.
Well known human pathogens in this group include <em>Anaplasma phagocytophilum</em> and <em>Ehrlichia chaffeensis</em>
Anaplasmataceae are difficult to isolate and culture, and as such molecular tools are critical in the identification of members within this family.
Conserved genes such as 16S rRNA, groEL and gltA (citrate synthase gene) are used to identify and classify this group of bacteria <span class="citation">(<a href="references.html#ref-kawaharaUltrastructurePhylogeneticAnalysis2004" role="doc-biblioref">Kawahara et al. 2004</a>)</span>.</p>
<p><em>Anaplasma phagocytophilum</em> is the causative agent for granulocytic anaplasmosis in humans. The first case of human granulocytic anaplasmosis (HGA) was made in the United States in 1994 <span class="citation">(<a href="references.html#ref-chenIdentificationGranulocytotropicEhrlichia1994" role="doc-biblioref">Chen et al. 1994</a>)</span>.
However, it was not until 2001 that it reflected its current name (previously named <em>Ehrlichia phagocytophilum</em>) <span class="citation">(<a href="references.html#ref-dumlerReorganizationGeneraFamilies2001" role="doc-biblioref">Dumler et al. 2001</a>)</span>.
<em>Anaplasma phagocytophilum</em> is also responsible for tick-borne fever in ruminants, equine anaplasmosis in horses and causes severe febrile diseases in dogs and cats <span class="citation">(<a href="references.html#ref-rarAnaplasmaEhrlichiaCandidatus2011" role="doc-biblioref">V. Rar and Golovljova 2011</a>)</span>.
The vast majority of HGA cases are known from the United States, most notably in the northeastern and upper mid-western regions <span class="citation">(<a href="references.html#ref-mmwrFinal2009Reports2010" role="doc-biblioref">MMWR 2010</a>)</span>; although present in Europe, the prevalence is significantly lower.
A number of <em>A. phagocytophilum</em> vectors have been identified, including <em>Ixodes scapularis</em> <span class="citation">(<a href="references.html#ref-telfordPerpetuationAgentHuman1996" role="doc-biblioref">S. R. Telford et al. 1996</a>; <a href="references.html#ref-hodzicAcquisitionTransmissionAgent1998" role="doc-biblioref">Hodzic et al. 1998</a>)</span>, <em>Ixodes ricinus</em> <span class="citation">(<a href="references.html#ref-lizPCRDetectionGranulocytic2000" role="doc-biblioref">Liz et al. 2000</a>)</span>, <em>Ixodes persulcatus</em> <span class="citation">(<a href="references.html#ref-eremeevaPrevalenceBacterialAgents2006" role="doc-biblioref">Eremeeva et al. 2006</a>)</span>, and <em>Ixodes ovatus</em> <span class="citation">(<a href="references.html#ref-ohashiAnaplasmaPhagocytophilumInfected2005" role="doc-biblioref">Ohashi et al. 2005</a>)</span>.
The vertebrate reservoirs of <em>A. phagocytophilum</em> remain unclear due to the presence of numerous diverse strains, however animals such as white-footed mice, white-tailed deer <span class="citation">(<a href="references.html#ref-telfordPerpetuationAgentHuman1996" role="doc-biblioref">S. R. Telford et al. 1996</a>; <a href="references.html#ref-ravynIsolationEtiologicAgent2001" role="doc-biblioref">Ravyn et al. 2001</a>)</span>, dusty-footed woodrats <span class="citation">(<a href="references.html#ref-nicholsonDuskyFootedWoodRats1999" role="doc-biblioref">Nicholson et al. 1999</a>)</span> and chipmunks <span class="citation">(<a href="references.html#ref-foleyDistinctEcologicallyRelevant2009" role="doc-biblioref">Foley et al. 2009</a>)</span> have been highlighted as important reservoirs.</p>
<p>Human ehrlichiosis is most notably caused by <em>Ehrlichia chaffeensis</em>.
It was first identified in the United States as causing human monocytic ehrlichiosis (HME) <span class="citation">(<a href="references.html#ref-andersonEhrlichiaChaffeensisNew1991" role="doc-biblioref">Anderson et al. 1991</a>)</span>.
Since then 4,364 of confirmed HME have been reported to the CDC between 2003-2010 <span class="citation">(<a href="references.html#ref-mmwrFinal2009Reports2010" role="doc-biblioref">MMWR 2010</a>)</span>.
In North America the lone star tick (<em>Amblyomma americanum</em>) and white-tailed deer are regarded as the most important vector and vertebrate reservoir of the bacteria <span class="citation">(<a href="references.html#ref-rarAnaplasmaEhrlichiaCandidatus2011" role="doc-biblioref">V. Rar and Golovljova 2011</a>)</span>.
To date, pathogen isolation has only been confirmed in the Unites States, however molecular and serological reports of the pathogen have been made throughout the world including Venezuela <span class="citation">(<a href="references.html#ref-martinezEhrlichiaChaffeensisChild2008" role="doc-biblioref">Martinez et al. 2008</a>)</span>, Latin America <span class="citation">(<a href="references.html#ref-gongora-biachiFirstCaseHuman1999" role="doc-biblioref">Gongora-Biachi et al. 1999</a>; <a href="references.html#ref-dacostaMoreHumanMonocytotropic2006" role="doc-biblioref">Costa et al. 2006</a>)</span>, South Korea <span class="citation">(<a href="references.html#ref-parkDetectionAntibodiesAnaplasma2003" role="doc-biblioref">Park et al. 2003</a>)</span>, and Thailand <span class="citation">(<a href="references.html#ref-heppnerHumanEhrlichiosisThailand1997" role="doc-biblioref">Heppner et al. 1997</a>)</span>.</p>
<p>Using molecular tools, reports of an <em>Ehrlichia</em>-like organism were made from <em>Ixodes ricinus</em> ticks in the Netherlands <span class="citation">(<a href="references.html#ref-schoulsDetectionIdentificationEhrlichia1999" role="doc-biblioref">Schouls et al. 1999</a>)</span>, and from rats (<em>Rattus norvegicus</em>) in China <span class="citation">(<a href="references.html#ref-panEhrlichialikeOrganismGene2003" role="doc-biblioref">Pan et al. 2003</a>)</span>.
Further studies showed there was no cross-reactivity of this organism with members of the genera <em>Anaplasma</em>, <em>Ehrlichia</em> or <em>Neorickettsia</em>, as such it was designated to be a new genus and was formally described by <span class="citation"><a href="references.html#ref-kawaharaUltrastructurePhylogeneticAnalysis2004" role="doc-biblioref">Kawahara et al.</a> (<a href="references.html#ref-kawaharaUltrastructurePhylogeneticAnalysis2004" role="doc-biblioref">2004</a>)</span> as <em>Neoehrlichia mikurensis</em>.<br>
The first reports of <em>N. mikurensis</em> infection in humans were made in 2010 and 2011 <span class="citation">(<a href="references.html#ref-fehrSepticemiaCausedTickborne2010" role="doc-biblioref">Fehr et al. 2010</a>; <a href="references.html#ref-vonloewenichDetectionCandidatusNeoehrlichia2010" role="doc-biblioref">Loewenich et al. 2010</a>; <a href="references.html#ref-welinder-olssonFirstCaseHuman2010" role="doc-biblioref">Welinder-Olsson et al. 2010</a>; <a href="references.html#ref-pekovaCandidatusNeoehrlichiaMikurensis2011" role="doc-biblioref">Pekova et al. 2011</a>)</span> and were predominately from cases where patients were immunocompromised.
Two decades after its initial discovery, the first study was published detailing the cultivation of the organism <span class="citation">(<a href="references.html#ref-wassCultivationCausativeAgent2019" role="doc-biblioref">Wass et al. 2019</a>)</span>.
Generally a rapid and full recovery is successful after treatment with antibiotics <span class="citation">(<a href="references.html#ref-pekovaCandidatusNeoehrlichiaMikurensis2011" role="doc-biblioref">Pekova et al. 2011</a>)</span>.
<em>Neoehrlichia mikurensis</em> has been identified from ticks around the world including Sweden <span class="citation">(<a href="references.html#ref-anderssonCoinfectionCandidatusNeoehrlichia2013" role="doc-biblioref">Andersson et al. 2013</a>)</span>, Germany <span class="citation">(<a href="references.html#ref-dinizCandidatusNeoehrlichiaMikurensis2011" role="doc-biblioref">Diniz et al. 2011</a>)</span>, Austria <span class="citation">(<a href="references.html#ref-glatzDetectionCandidatusNeoehrlichia2014" role="doc-biblioref">Glatz et al. 2014</a>)</span>, and Hungary <span class="citation">(<a href="references.html#ref-hornokMolecularAnalysisIxodes2017" role="doc-biblioref">Hornok et al. 2017</a>)</span>.
Using transmission electron microscopy, <em>N. mikurensis</em> was recently identified from tick salivary glands <span class="citation">(<a href="references.html#ref-ondrusPutativeMorphologyNeoehrlichia2020" role="doc-biblioref">Ondrus et al. 2020</a>)</span> providing evidence for transmission route via a tick bite.</p>
</div>
<div id="borrelia" class="section level4" number="1.2.1.2">
<h4>
<span class="header-section-number">1.2.1.2</span> <em>Borrelia</em><a class="anchor" aria-label="anchor" href="#borrelia"><i class="fas fa-link"></i></a>
</h4>
<p>Lyme borreliosis caused by a group of spirochaete bacteria (<em>Borrelia burgdorferi</em> sensu lato) is endemic to North America and Europe.
In North America the primary vector of <em>B. burgdorferi</em> s. l. is <em>Ixodes scapularis</em>, and major reservoir hosts are well known to include white-footed mice and white-tailed deer <span class="citation">(<a href="references.html#ref-halseyRoleIxodesScapularis2018" role="doc-biblioref">Halsey, Allan, and Miller 2018</a>)</span>.
In Europe the main vector responsible for Lyme Borreliosis is <em>Ixodes ricinus</em> <span class="citation">(<a href="references.html#ref-kirsteinLocalVariationsDistribution1997" role="doc-biblioref">Kirstein et al. 1997</a>)</span> with a number of vertebrates identified as reservoir hosts throughout the continent, most notably hedgehogs (<em>Erinaceus europaeus</em>) and voles (<em>Myodes glareolus</em>) <span class="citation">(<a href="references.html#ref-jahfariMeltingPotTickborne2017" role="doc-biblioref">Jahfari et al. 2017</a>; <a href="references.html#ref-coipanGeneticDiversityBorrelia2018" role="doc-biblioref">Coipan et al. 2018</a>; <a href="references.html#ref-estrada-penaHighThroughputSequencing2018" role="doc-biblioref">Estrada-Pena et al. 2018</a>)</span>.
Lyme borreliosis is typically manifested by an erythema migrans skin lesion (60-80% of cases) <span class="citation">(<a href="references.html#ref-rizzoliLymeBorreliosisEurope2011" role="doc-biblioref">Rizzoli et al. 2011</a>)</span>, and known to develop into arthritis or various skin disorders <span class="citation">(<a href="references.html#ref-stanekLymeBorreliosis2012" role="doc-biblioref">Stanek et al. 2012</a>)</span>.
Additional early symptoms may include fever, headaches, fatigue, and body aches and pains <span class="citation">(<a href="references.html#ref-rizzoliLymeBorreliosisEurope2011" role="doc-biblioref">Rizzoli et al. 2011</a>; <a href="references.html#ref-clarkLymeBorreliosisHuman2013" role="doc-biblioref">K. L. Clark, Leydet, and Hartman 2013</a>)</span>.
Manifestation of neurological and cardiovascular symptoms have been documented, although are generally considered rare <span class="citation">(<a href="references.html#ref-clarkLymeBorreliosisHuman2013" role="doc-biblioref">K. L. Clark, Leydet, and Hartman 2013</a>)</span>.
A distinct, separate clade of <em>Borrelia</em> known as the relapsing fever (RF) group are a cause of significant disease, and be transmitted by argasid and ixodid ticks, and the human body louse <span class="citation">(<a href="references.html#ref-lopezTickBorneRelapsingFever2016" role="doc-biblioref">J. E. Lopez et al. 2016</a>)</span>.
Acute symptoms of relapsing fever in humans are generally non-specific (e.g. fever, headache and nausea); however the disease is characterize by a unique cyclic nature, where acute episodes lasting a few days are followed by afebrile periods <span class="citation">(<a href="references.html#ref-dworkinTickborneRelapsingFever2008" role="doc-biblioref">Dworkin et al. 2008</a>)</span>.</p>
</div>
<div id="coxiella" class="section level4" number="1.2.1.3">
<h4>
<span class="header-section-number">1.2.1.3</span> <em>Coxiella</em><a class="anchor" aria-label="anchor" href="#coxiella"><i class="fas fa-link"></i></a>
</h4>
<p>An obligate intracellular bacteria, <em>Coxiella</em> has been isolated from a wide range of animals, and environmental samples throughout the world.
<em>Coxiella burnetii</em> is the causative agent of Q fever that can cause disease in both humans and animals (mainly associated with livestock) <span class="citation">(<a href="references.html#ref-gonzalez-barrioCoxiellaBurnetiiWild2018" role="doc-biblioref">Gonzalez-Barrio and Ruiz-Fons 2018</a>)</span>.
First discovered in Australia in 1935 after a cluster of abattoir workers became ill <span class="citation">(<a href="references.html#ref-derrickFEVERNEWFEVER1937" role="doc-biblioref">Derrick 1937</a>)</span> the causative agent of the disease was not know, and the term Q fever was adapted meaning “query.”
Since then it has been described in multiple countries, with incidences of human infection usually associated with direct livestock contact.
Despite the wide spread prevalence and economic impacts on the agriculture industry, the natural history of <em>C. burnetii</em> is not well understood.
Q fever is usually acquired via inhalation route during close contact with infected animal material.
It has now been well established that ticks can become reservoirs and potential vectors of <em>C. burnetii</em> <span class="citation">(<a href="references.html#ref-arricau-bouveryFeverEmergingReemerging2005" role="doc-biblioref">Arricau-Bouvery and Rodolakis 2005</a>)</span>, however the significance of ticks in transmission of the disease to humans is not yet fully understood.</p>
</div>
<div id="rickettsia" class="section level4" number="1.2.1.4">
<h4>
<span class="header-section-number">1.2.1.4</span> <em>Rickettsia</em><a class="anchor" aria-label="anchor" href="#rickettsia"><i class="fas fa-link"></i></a>
</h4>
<p>The genus <em>Rickettsia</em> (Rickettsiaceae) is a group of obligate intracellular bacteria, that are among some of the oldest known vector-borne pathogens.
Members of the genus <em>Rickettsia</em> can broadly be classified into (i) spotted fever group (SFG), (ii) typhus group (iii) <em>Rickettsia bellii</em> (ancestral) group and the (iv) <em>Rickettsia canadensis</em> group <span class="citation">(<a href="references.html#ref-merhjRickettsialEvolutionLight2010" role="doc-biblioref">Merhj and Raoult 2010</a>)</span>.
The SFG is the most notable group of tick-borne disease, mainly transmitted by Ixodidae ticks <span class="citation">(<a href="references.html#ref-parolaUpdateTickBorneRickettsioses2013" role="doc-biblioref">P. Parola et al. 2013</a>)</span>.
Rocky-mountain spotted fever (RMSF), caused by <em>Rickettsia rickettsii</em> is the most well understood and studied SFG <em>Rickettsia</em>.
Important vectors of <em>R. rickettsii</em> in North and Central America include the Rocky Mountain wood tick (<em>Dermacentor andersoni</em>), the American dog tick (<em>Dermacentor variabilis</em>), the Cayenne tick (<em>Amblyomma cajennense</em>) and the brown dog tick (<em>Rhipicephalus sanguineus)</em> <span class="citation">(<a href="references.html#ref-dantas-torresRockyMountainSpotted2007" role="doc-biblioref">F. Dantas-Torres 2007</a>; <a href="references.html#ref-lopez-perezDiversityRickettsiaeDomestic2021" role="doc-biblioref">Lopez-Perez et al. 2021</a>)</span>.
Although domestic dogs and wild mammals have been known to harbor the bacteria, the role of these reservoir hosts remains unclear.
Dogs have been implicated as the main reservoir for <em>Rickettsia felis</em>, a flea-borne spotted-fever <em>Rickettsia</em> most common in companion animals <span class="citation">(<a href="references.html#ref-ng-nguyenDomesticDogsAre2020" role="doc-biblioref">Ng-Nguyen et al. 2020</a>)</span>.
In other cases, small mammals, mainly rodents, are implicated in the life cycle of many <em>Rickettsia</em> species <span class="citation">(<a href="references.html#ref-tomassoneNeglectedAspectsTickborne2018" role="doc-biblioref">Tomassone et al. 2018</a>; <a href="references.html#ref-parisBriefHistoryMajor2020" role="doc-biblioref">Paris et al. 2020</a>)</span></p>
</div>
<div id="viruses" class="section level4" number="1.2.1.5">
<h4>
<span class="header-section-number">1.2.1.5</span> Viruses<a class="anchor" aria-label="anchor" href="#viruses"><i class="fas fa-link"></i></a>
</h4>
<p>At review in 2004 identified at least 38 viruses transmitted by ticks, and many more unclassified <span class="citation">(<a href="references.html#ref-labudaTickborneViruses2004" role="doc-biblioref">Labuda and Nuttall 2004</a>)</span>.<br>
With only one exception (African swine fever virus, family <em>Asfarviridae</em>) all tick-borne viruses recognised belong the RNA virus families (<em>Reoviridae</em>, <em>Rhabdoviridae</em>, <em>Orthomyxoviridae</em>, <em>Bunyaviridae</em> and <em>Flaviviridae</em>).
Tick-borne flaviviruses represent some of the most medically important arboviruses around the world.
Tick-borne encephalitis (TBE) (<em>Flaviviridae</em>: <em>Flavivirus</em>) is a growing public health issue in parts of Europe and Asia and highlights the complexity of the dynamics involved in tick-borne diseases <span class="citation">(<a href="references.html#ref-gritsunTickborneEncephalitis2003" role="doc-biblioref">Gritsun, Lashkevich, and Gould 2003</a>)</span>.
Natural vectors of the disease involved in transmission of the virus to humans mainly include, <em>Ixodes ricinus</em> and <em>Ixodes persulcatus</em> <span class="citation">(<a href="references.html#ref-labudaTickborneViruses2004" role="doc-biblioref">Labuda and Nuttall 2004</a>; <a href="references.html#ref-sussTickborneEncephalitis20102011" role="doc-biblioref">Suss 2011</a>)</span>.
The virus has also been identified in field collected <em>Ixodes hexagonus</em> and studies have demonstrated that <em>Ixodes arboricola</em>, <em>Haemaphysalis concinna</em>, <em>Haemaphysalis inermis</em> and <em>Haemaphysalis punctate</em> are also competent vectors <span class="citation">(<a href="references.html#ref-gresikovaTickborneEncephalitis1998" role="doc-biblioref">Gresikova and Calisher 1998</a>)</span>.
Vertebrate hosts involved in the maintenance of TBE include voles (<em>M. arvalis</em>) and a variety of rodent species (<em>Apodemus</em> spp., <em>Microtus</em> spp., and <em>Myodes</em> spp.) <span class="citation">(<a href="references.html#ref-achaziRodentsSentinelsPrevalence2011" role="doc-biblioref">Achazi et al. 2011</a>)</span>.</p>
</div>
<div id="eukaryotes" class="section level4" number="1.2.1.6">
<h4>
<span class="header-section-number">1.2.1.6</span> Eukaryotes<a class="anchor" aria-label="anchor" href="#eukaryotes"><i class="fas fa-link"></i></a>
</h4>
<p>Compared to bacteria, the diversity of eukaryote organisms responsible for causing tick-borne disease is much more limited <span class="citation">(<a href="references.html#ref-tokarzDiscoverySurveillanceTickBorne2021" role="doc-biblioref">Tokarz and Lipkin 2021</a>)</span>.
Ticks have also been associated in the transmission and life of a diverse range of organisms such as protozoa, fungi and nematodes.</p>
<p>Piroplasms are a group of single-celled, intracellular parasites that belong to the Apicomplexa phylum.
Characterised by two main genera, <em>Babesia</em> and <em>Theileria</em>, they are the primary agents of eukaryote tick-borne diseases in vertebrates.
Human babesiosis is a well-known infectious disease, recognised as an emerging public health issue in many parts of the world.
Caused by members of the genus <em>Babesia</em> (Apicomplexa: Piroplasmida), disease in humans is usually attributed to <em>Babesia microti</em> (North America), and <em>Babesia divergens</em> (Europe) <span class="citation">(<a href="references.html#ref-vannierHumanBabesiosis2012" role="doc-biblioref">E. Vannier and Krause 2012</a>)</span>.
<em>Ixodes scapularis</em> is the primary vector of <em>B. microti</em> to humans, with most cases reportedly caused by nymph ticks during late spring–early summer period <span class="citation">(<a href="references.html#ref-spielmanEcologyIxodesDamminiborne1985" role="doc-biblioref">Spielman et al. 1985</a>; <a href="references.html#ref-swansonCoinfectionsAcquiredIxodes2006" role="doc-biblioref">Swanson et al. 2006</a>)</span>.
The primary vertebrate hosts identified in the transmission of the disease is the white-footed mouse (<em>Peromyscus leucopus</em>) <span class="citation">(<a href="references.html#ref-spielmanEcologyIxodesDamminiborne1985" role="doc-biblioref">Spielman et al. 1985</a>)</span>.
Manifestations of babesiosis are diverse, and can range from asymptomatic to debilitating illness that can lead to death.
Most commonly patients experience fever, fatigue, chills and headaches, with symptoms appearing gradually 1–4 weeks after tick bite <span class="citation">(<a href="references.html#ref-vannierHumanBabesiosis2008" role="doc-biblioref">Edouard Vannier, Gewurz, and Krause 2008</a>)</span>.
Only members of the <em>Babesia</em> genus have been associated with disease in humans.</p>
<p>The genus <em>Theileria</em> is distinguished from <em>Babesia</em> by the presence of stages outside the red blood cell.
There are a number of species that infect and cause disease in animals, particularly ruminants, equids, rodents, and foxes <span class="citation">(<a href="references.html#ref-almazanBabesiosisTheileriosisNorth2022" role="doc-biblioref">Almazan et al. 2022</a>)</span>.
Additionally, <em>Theileria</em> species have been described circulating in populations of wildlife and ticks worldwide <span class="citation">(<a href="references.html#ref-mansReviewTheileriaDiagnostics2015" role="doc-biblioref">B. J. Mans, Pienaar, and Latif 2015</a>; <a href="references.html#ref-wattsTheileriaOrientalisReview2016" role="doc-biblioref">Watts, Playford, and Hickey 2016</a>)</span>.
To date, there have been no reported cases of <em>Theileria</em> infecting people.</p>
<p>Trypanosomes are a group of flagellated protozoa that belong to phylum Euglenozoa.
Members of the genera <em>Leishmania</em> and <em>Trypanosoma</em> are known parasites of humans and animals and are widely distributed.
Species that can cause severe human disease include <em>Trypanosoma cruzi</em>, responsible for Chagas disease in South and Central America, <em>Trypanosoma brucei</em> which causes human African trypanosomiasis (HAT) (also known as sleeping sickness) and <em>Leishmania tropica</em> capable of causes cutaneous leishmaniasis <span class="citation">(<a href="references.html#ref-kauferReviewSystematicsSpecies2020" role="doc-biblioref">Kaufer, Stark, and Ellis 2020</a>)</span>.
Although blood-sucking insects (class Insecta) are responsible for the majority of zoonotic transmission of trypanosomes, there is growing evidence to support that ticks are involved in the life-cycle of these protozoans <span class="citation">(<a href="references.html#ref-morzariaTransmissionTrypanosomaSp1986" role="doc-biblioref">Morzaria et al. 1986</a>; <a href="references.html#ref-thekisoeTrypanosomeSpeciesIsolated2007" role="doc-biblioref">Thekisoe et al. 2007</a>)</span>.</p>
<p>Filarial nematodes have been described from a number of tick species worldwide.
Genetic characterisation has shown that similar species of filarial nematodes have recently been described from two widespread ticks in North America.
Separate genetic analysis showed that closely related <em>Monanema</em>-like filarial nematodes were identified from <em>Amblyomma americanum</em> <span class="citation">(<a href="references.html#ref-henningDiscoveryFilarialNematode2016" role="doc-biblioref">Henning et al. 2016</a>)</span> and <em>Ixodes scapularis</em> <span class="citation">(<a href="references.html#ref-tokarzCharacterizationMonanemaNematode2020" role="doc-biblioref">Tokarz et al. 2020</a>)</span>.
However, there is currently no evidence that these nematodes contribute to the burden of human tick-borne disease <span class="citation">(<a href="references.html#ref-tokarzCharacterizationMonanemaNematode2020" role="doc-biblioref">Tokarz et al. 2020</a>)</span>.</p>
</div>
</div>
<div id="infectious-human-tick-borne-pathogens-in-australia" class="section level3" number="1.2.2">
<h3>
<span class="header-section-number">1.2.2</span> Infectious human tick-borne pathogens in Australia<a class="anchor" aria-label="anchor" href="#infectious-human-tick-borne-pathogens-in-australia"><i class="fas fa-link"></i></a>
</h3>
<p>In comparison to the Northern Hemisphere, relatively few zoonotic tick-borne pathogens are recognised in Australia <span class="citation">(<a href="references.html#ref-madison-antenucciEmergingTickBorneDiseases2020" role="doc-biblioref">Madison-Antenucci et al. 2020</a>; <a href="references.html#ref-rochlinEmergingTickbornePathogens2020" role="doc-biblioref">Rochlin and Toledo 2020</a>)</span>.
Queensland Tick Typhus (QTT) was first identified from World War II soldiers training in Queensland.
After a tick bite, individuals developed an eschar, fever and vesicular rash <span class="citation">(<a href="references.html#ref-andrewTickTyphusNorth1946" role="doc-biblioref">Andrew, Bonnin, and Williams 1946</a>)</span>.
Although initial reports put it in the spotted fever group, the causative agent, <em>Rickettsia australis</em> was later shown to be genetically different.
Early experimental work showed that <em>Ix. holocyclus</em> and <em>Ix. tasmani</em> were vectors of the bacteria <span class="citation">(<a href="references.html#ref-campbellRickettsiosesAustraliaIsolation1974" role="doc-biblioref">Campbell and Domrow 1974</a>)</span>.
Since its discovery it has been identified along the east coast of Australia <span class="citation">(<a href="references.html#ref-campbellQueenslandTickTyphus1979" role="doc-biblioref">Campbell, Abeywickrema, and Fenton 1979</a>; <a href="references.html#ref-wilsonQueenslandTickTyphus2013" role="doc-biblioref">Wilson et al. 2013</a>; <a href="references.html#ref-fergieQueenslandTickTyphus2017" role="doc-biblioref">Fergie et al. 2017</a>)</span> (Table <a href="litrev.html#tab:T1ausTBD">1.1</a>).
A recent study using real-time PCR identified that <em>R. australis</em> was present in 15.4% (23/149) <em>Ix. holocyclus</em> in north-east New South Wales <span class="citation">(<a href="references.html#ref-gravesIxodesHolocyclusTicktransmitted2016" role="doc-biblioref">Stephen Graves et al. 2016</a>)</span>.
A spotted-fever-like illness was identified from a cluster of 26 patients from Flinders Island, Tasmania, a small community with a population of about 1000 <span class="citation">(<a href="references.html#ref-stewartFlindersIslandSpotted1991" role="doc-biblioref">R. S. Stewart 1991</a>)</span>.
A serological investigation found that while 46% of patients were positive to an assay for the detection of <em>Rickettsia australis</em>, evidence suggested the aetiological agent was different <span class="citation">(<a href="references.html#ref-gravesSpottedFeverGroup1993" role="doc-biblioref">S. R. Graves et al. 1993</a>)</span>.
It was later confirmed that patients were infected with <em>Rickettsia honei</em>, a pathogen originally isolated in Thailand <span class="citation">(<a href="references.html#ref-gravesRickettsiaHonei2003" role="doc-biblioref">S. Graves and Stenos 2003</a>)</span>.
An investigation into the tick reservoir on the Island found that 63% of the reptile-associated tick <em>Bothriocroton hydrosauri</em> were positive for <em>R. honei</em> <span class="citation">(<a href="references.html#ref-stenosAponommaHydrosauriReptileassociated2003" role="doc-biblioref">Stenos et al. 2003</a>)</span> (Table <a href="litrev.html#tab:T1ausTBD">1.1</a>).
Human infection of Q Fever, caused by the bacteria <em>Coxiella burnetii</em>, is usually acquired by inhalation of infectious aerosols from vertebrate hosts such as sheep, cattle and domestic pets.
<em>Coxiella burnetii</em> has been identified in a number of ticks, including the human biting species <em>Ix. holocyclus</em> <span class="citation">(<a href="references.html#ref-gravesIxodesHolocyclusTicktransmitted2016" role="doc-biblioref">Stephen Graves et al. 2016</a>)</span> and <em>Am. triguttatum</em> <span class="citation">(<a href="references.html#ref-popeCoxiellaBurnetiKangaroos1960" role="doc-biblioref">Pope, Scott, and Dwyer 1960</a>; <a href="references.html#ref-cooperSerologicalEvidenceCoxiella2012" role="doc-biblioref">A. Cooper et al. 2012</a>)</span> (Table <a href="litrev.html#tab:T1ausTBD">1.1</a>); however there is just a single published report of tick-borne Q fever in Australia (thought to be transmitted by <em>Am. triguttatum</em>) <span class="citation">(<a href="references.html#ref-beamanPericarditisAssociatedTickborne1989" role="doc-biblioref">Beaman and Hung 1989</a>)</span>.</p>
<div style="page-break-after: always;"></div>
<p><strong>Taxa of interest</strong></p>
<p>This next section will review recent research on tick-associated organisms that are related to taxa that are known to cause disease globally (i.e. family level relatedness) and tick ‘endosymbiont’ organisms.
The list of possible microbes is extensive and therefore this review will focus mainly on microbes that have been identified from ticks.
Additionally as recent research has highlighted that microbes may have a broader vector range than previously thought (e.g. <em>Bartonella</em>), where relevant this review will include other vector-related microbes.</p>
<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T1ausTBD">Table 1.1: </span>Currently recognised Australian human tick-borne diseases.
</caption>
<thead><tr>
<th style="text-align:left;">
Disease
</th>
<th style="text-align:left;">
Pathogen
</th>
<th style="text-align:left;">
Tick vector
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Queensland tick typhus
</td>
<td style="text-align:left;font-style: italic;">
Rickettsia australis
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus, Ixodes cornuatus, Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
Flinders Island Spotted Fever
</td>
<td style="text-align:left;font-style: italic;">
Rickettsia honei
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani, Bothriocroton hydrosauri
</td>
</tr>
<tr>
<td style="text-align:left;">
Australian Spotted Fever
</td>
<td style="text-align:left;font-style: italic;">
Rickettsia honei marmionii
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis novaeguinae
</td>
</tr>
<tr>
<td style="text-align:left;">
Q Fever
</td>
<td style="text-align:left;font-style: italic;">
Coxiella burnetii
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus, Amblyomma triguttatum
</td>
</tr>
</tbody>
</table></div>
<div id="anaplamataceae" class="section level4" number="1.2.2.1">
<h4>
<span class="header-section-number">1.2.2.1</span> Anaplamataceae<a class="anchor" aria-label="anchor" href="#anaplamataceae"><i class="fas fa-link"></i></a>
</h4>
<p>Through the recent use of molecular tools five novel species (or genotypes) of Anaplasmataceae have been described from native Australian ticks.
Additionally, a number of organisms are known but remain to be formally described (Table <a href="litrev.html#tab:T1anaplasm">1.2</a>).</p>
<p>Questing <em>Am. trigutattum</em> ticks were found to harbour Australia’s first endemic <em>Ehrlichia</em>, ‘<em>Ca</em>. Ehrlichia occidentalis’ and a novel genotype of <em>Anaplasma bovis</em> Y11 <span class="citation">(<a href="references.html#ref-goftonDetectionPhylogeneticCharacterisation2017" role="doc-biblioref">Gofton et al. 2017</a>)</span>.
Two <em>Neoehrlichia</em> species, ‘<em>Ca</em>. N. arcana’ and ‘<em>Ca</em>. N. australis,’ were recently characterised from <em>Ix. holocyclus</em> along the east coast of Australia at a prevalence of 11.25% (44/391; 31 females, seven males and six nymphs) <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">Gofton et al. 2016</a>)</span>.
The disease potential and transmission dynamics of these novel organisms remains unknown.
A study on the microbiome of ticks parasitsing bandicoots confirmed the presence of both <em>Neoehrlichia</em> species and expanded the tick associations to include <em>Ix. tasmani</em> <span class="citation">(<a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.
The same study also expanded the range of <em>A. bovis</em> Y11 to New South Wales and identified a novel <em>Neoehrlichia</em> and <em>Ehrlichia</em> species in ticks from quenda (<em>Isoodon fusciventer</em>) in south-west Western Australia.
‘<em>Candidatus</em> Ehrlichia ornithorhynchi’ was described from the platypus and its tick <em>Ixodes ornithorhynchi</em> <span class="citation">(<a href="references.html#ref-goftonNovelEhrlichiaSpecies2018" role="doc-biblioref">Gofton et al. 2018</a>)</span>.
A novel <em>Anaplasma</em> and <em>Ehrlichia</em> species has been identified in <em>Bothriocroton concolor</em> ticks from echidna in New South Wales and Queensland <span class="citation">(<a href="references.html#ref-lohIdentificationCharacterisationMicroorganisms2018" role="doc-biblioref">Siew-May Loh 2018</a>; <a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.
A novel <em>Neoehrlichia</em> and <em>Ehrlichia</em> were also identified from ticks (<em>Ixodes fecialis</em> and <em>Ixodes australiensis</em>) removed from quenda <em>Isoodon fusciventer</em> <span class="citation">(<a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.</p>
<div class="inline-table"><table class="table table" style="width: auto !important; margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T1anaplasm">Table 1.2: </span>Anaplasmataceae species identified from Australian ticks using molecular methods.
</caption>
<thead><tr>
<th style="text-align:left;">
Microorganism
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:left;">
Vertebrate host
</th>
<th style="text-align:left;">
Identified in host
</th>
<th style="text-align:left;">
State
</th>
<th style="text-align:left;">
Reference(s)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Anaplasma bovis (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:left;width: 8em; ">
Questing
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, WA
</td>
<td style="text-align:left;width: 6em; ">
Gofton et al. 2017; Egan et al. 2020a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Anaplasma platys (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Rh. sanguineus
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NT, QLD, SA, WA
</td>
<td style="text-align:left;width: 6em; ">
Brown et al. 2001; Greay 2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Anaplasma sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. auruginans
</td>
<td style="text-align:left;width: 8em; ">
Wombat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
VIC
</td>
<td style="text-align:left;width: 6em; ">
Beard et al. 2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Anaplasma sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. concolor
</td>
<td style="text-align:left;width: 8em; ">
Echidna
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD, NSW
</td>
<td style="text-align:left;width: 6em; ">
Loh 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Ehrlichia canis (b)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Rh. sanguineus
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NT, SA, WA
</td>
<td style="text-align:left;width: 6em; ">
<ol start="2" style="list-style-type: lower-alpha"><li></ol>
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
‘Ca. Ehrlichia occidentalis’
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:left;width: 8em; ">
Questing only
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
SA, WA
</td>
<td style="text-align:left;width: 6em; ">
Gofton et al. 2017
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
‘Ca. Ehrlichia ornithorhynchi’
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. ornithorhynchi
</td>
<td style="text-align:left;width: 8em; ">
Platypus
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD, TAS
</td>
<td style="text-align:left;width: 6em; ">
Gofton et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
‘Ca. Ehrlichia occidentalis’
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:left;width: 8em; ">
Questing only
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
SA, WA
</td>
<td style="text-align:left;width: 6em; ">
Gofton et al. 2017
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Ehrlichia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. fecialis
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
WA
</td>
<td style="text-align:left;width: 6em; ">
Egan et al. 2020a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Ehrlichia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. concolor
</td>
<td style="text-align:left;width: 8em; ">
Echidna
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD, NSW
</td>
<td style="text-align:left;width: 6em; ">
Loh 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
‘Ca Neoehrlichia arcana’
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. cornuatus, Ix. holocyclus, Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, cat, dog, horse, human
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Gofton et al. 2016; Egan et al. 2020a; Greay et al. 2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
‘Ca. Neoehrlichia australis’
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus, Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, cat, cattle, dog, horse, human
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Gofton et al. 2016; Egan et al. 2020a; Greay et al. 2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Neoehrlichia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. australiensis, Ix. fecialis
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
WA
</td>
<td style="text-align:left;width: 6em; ">
Egan et al. 2020a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Neoehrlichia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Cat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2021
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Introduced species that are now considered endemic.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>b</sup> Previously considered exotic, it was first identified in May 2020. Currently listed as a nationally notifiable disease and investigations into its origin are ongoing <a href="https://www.outbreak.gov.au/current-responses-to-outbreaks/ehrlichiosis-dogs" class="uri">https://www.outbreak.gov.au/current-responses-to-outbreaks/ehrlichiosis-dogs</a>.
</td>
</tr>
</tfoot>
</table></div>
</div>
<div id="borrelia-1" class="section level4" number="1.2.2.2">
<h4>
<span class="header-section-number">1.2.2.2</span> <em>Borrelia</em><a class="anchor" aria-label="anchor" href="#borrelia-1"><i class="fas fa-link"></i></a>
</h4>
<p>At present four species of borreliae have been identified in Australia (Table <a href="litrev.html#tab:T1borrelia">1.3</a>).
Two species were introduced <em>Borrelia theileri</em> and <em>Borrelia anserine</em> with the importation of livestock.
Avian spirochaetosis is associated with disease in poultry caused by <em>B. anserina</em> and is transmitted by the soft tick <em>Argas persicus</em>.
Bovine spirochaetosis is caused by <em>B. theileri</em> and is transmitted with the cattle tick, <em>Rhipicephalus australis</em> (formerly <em>Boophilus microplus</em>) <span class="citation">(<a href="references.html#ref-estrada-penaReinstatementRhipicephalusBoophilus2012" role="doc-biblioref">Estrada-Pena et al. 2012</a>)</span>.
The first native Australian <em>Borrelia</em> was described from long-haired rats (<em>Rattus villosissimus</em>) in north-western Queensland <span class="citation">(<a href="references.html#ref-carleyNewSpeciesBorrella1962" role="doc-biblioref">Carley and Pope 1962</a>)</span> and named <em>Borrelia queenslandica</em>.
The authors suggested the spirochaete was transmitted by the soft kangaroo tick (<em>Or. gurneyi</em>), however this was never confirmed.
To date <em>B. queensandica</em> has not been isolated since and as such no molecular data is available.
A recent discovery has presented the first molecular description of a native Australian <em>Borrelia</em> and brings the total number of (<em>Borrelia</em>) species present in Australia to four. ‘<em>Candidatus</em> Borrelia tachyglossi’ was genetically described from echidna biting ticks <em>Bothriocroton conoclor</em> <span class="citation">(<a href="references.html#ref-lohNovelBorreliaSpecies2016" role="doc-biblioref">S. M. Loh et al. 2016</a>, <a href="references.html#ref-lohMolecularCharacterizationCandidatus2017" role="doc-biblioref">2017</a>)</span>.
A novel species of <em>Borrelia</em> that falls within the reptile has also been identified in <em>Bothriocroton undatum</em> collected from the goanna in NSW <span class="citation">(<a href="references.html#ref-panettaReptileassociatedBorreliaSpecies2017" role="doc-biblioref">Panetta et al. 2017</a>)</span>.</p>
<div class="inline-table"><table class="table table" style="width: auto !important; margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T1borrelia">Table 1.3: </span> species identified from Australian ticks and animals.
</caption>
<thead><tr>
<th style="text-align:left;">
Microorganism
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:left;">
Vertebrate host
</th>
<th style="text-align:left;">
Identified in host
</th>
<th style="text-align:left;">
State
</th>
<th style="text-align:left;">
Reference(s)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Borrelia anserine (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ar. persicus
</td>
<td style="text-align:left;width: 8em; ">
Poultry
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD, NSW
</td>
<td style="text-align:left;width: 6em; ">
Petney et al. 2004
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Borrelia queenslandica (b)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Or. gurneyi (unconfirmed)
</td>
<td style="text-align:left;width: 8em; ">
Rat
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Carley and Pope 1962
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
‘Ca. Borrelia tachyglossi’
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. conolor, Ha. Humerosa
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, echidna
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Loh et al. 2016; Egan et al. 2020a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Borrelia theileri (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Rh. (Bo.) australis
</td>
<td style="text-align:left;width: 8em; ">
Cattle
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Callow and Hoyte 1961
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Borrelia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. undatum
</td>
<td style="text-align:left;width: 8em; ">
Goanna
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Panetta et al. 2017
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Borrelia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. auruginans
</td>
<td style="text-align:left;width: 8em; ">
Wombat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Beard 2021
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Introduced species with the important of livestock.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>b</sup> Identification made by culture methods, isolated from rodent host - suspected tick vector listed.
</td>
</tr>
</tfoot>
</table></div>
</div>
<div id="coxiella-1" class="section level4" number="1.2.2.3">
<h4>
<span class="header-section-number">1.2.2.3</span> <em>Coxiella</em><a class="anchor" aria-label="anchor" href="#coxiella-1"><i class="fas fa-link"></i></a>
</h4>
<p>Studies on the presence of <em>Coxiella burnetii</em> are particularly challenging due to the cross reactivity of serological assays and the conserved nature of the commonly used 16S rRNA gene (Table <a href="litrev.html#tab:T1coxiella">1.4</a>).
Therefore in this review I will only focus on molecular reports of <em>Coxiella</em> sp. from wildlife and ticks.
Sequences that were 98–99% similar to <em>C. burnetii</em> have been identified in
<em>Bothriocroton auruginans</em> from the common wombat (<em>Vombatus ursinus</em>) collected from Victoria <span class="citation">(<a href="references.html#ref-vilcinsMolecularDetectionRickettsia2009" role="doc-biblioref">Vilcins, Old, and Deane 2009a</a>; <a href="references.html#ref-beardMorphologicalIdentificationTicks2021" role="doc-biblioref">D. Beard, Stannard, and Old 2021</a>)</span>.<br>
Recent descriptions of <em>Coxiella burnetii</em> from native Australian tick species (<em>Ha. bancrofti</em>, and <em>Ix. holocyclus</em>) and the brown dog tick (<em>Rh. sanguineus</em>) are important in our understanding of the epidemiology of Q Fever in Australia <span class="citation">(<a href="references.html#ref-chaladaMolecularSurveyTickBorne2018" role="doc-biblioref">Chalada et al. 2018</a>)</span>.
The positive detection of <em>C. burnetii</em> in these samples however has limited reliably due to the lack of DNA sequence results from the study.
The authors also noted the presence of <em>Coxiella</em>-like symbionts in <em>Am. triguttatum</em>, <em>Ix. holocyclus</em>, <em>Rh. australis</em> and <em>Or. capensis</em>, which were also not sequenced.
An investigation into the presence of <em>Coxiella</em> and <em>Coxiella</em>-like symbionts in Australian brown dog ticks (<em>Rh. linnaei</em> syn <em>Rh. sanguineus</em>) revealed the presence of a <em>Coxiella</em>-like symbiont (100% prevalence) through targeted PCR of the 16S rRNA gene <span class="citation">(<a href="references.html#ref-oskamMolecularInvestigationPresence2017" role="doc-biblioref">Oskam et al. 2017</a>)</span>.
Therefore without sequence results from PCR assays, the true presence of <em>Coxiella burnetii</em> as determined by <span class="citation"><a href="references.html#ref-chaladaMolecularSurveyTickBorne2018" role="doc-biblioref">Chalada et al.</a> (<a href="references.html#ref-chaladaMolecularSurveyTickBorne2018" role="doc-biblioref">2018</a>)</span> remains questionable.<br><em>Coxiella burnetii</em> was recently detected in kangaroo meat that was intended for companion animal consumption <span class="citation">(<a href="references.html#ref-shapiroMolecularDetectionCoxiella2020" role="doc-biblioref">Shapiro et al. 2020</a>)</span> using qPCR.
One recent study stated that wildlife carers may be two times more likely to be infected with <em>C. burnetii</em> than the general public, however the study was hindered by a small sample size <span class="citation">(<a href="references.html#ref-mathewsCoxiellaBurnetiiSeroprevalence2021" role="doc-biblioref">Mathews et al. 2021</a>)</span></p>
<div class="inline-table"><table class="table table" style="width: auto !important; margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T1coxiella">Table 1.4: </span>Identification of  within Australian ticks and wildlife using molecular methods. Serological tests were excluded from this summary due to the known cross-reactivity of -like organisms with assays.
</caption>
<thead><tr>
<th style="text-align:left;">
Microorganism
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:left;">
Vertebrate host
</th>
<th style="text-align:left;">
Identified in host
</th>
<th style="text-align:left;">
State
</th>
<th style="text-align:left;">
Reference(s)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ha. bancrofti
</td>
<td style="text-align:left;width: 8em; ">
Horse
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Chalada et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Kookaburra
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Chalada et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Rh. sanguineus
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Chalada et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ha. humerosa
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
WA (islands)
</td>
<td style="text-align:left;width: 6em; ">
Bennett et al. 2011
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Cooper et al. 2013
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:left;width: 8em; ">
Kangaroo
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Cooper et al. 2013
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. auruginans
</td>
<td style="text-align:left;width: 8em; ">
Wombat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
VIC
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al. 2009b; Beard et al. 2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Coxiella burneti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Cat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2021
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Identified in a faecal sample using molecular assay.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>b</sup> Records from Islands off Western Australia coastline.
</td>
</tr>
</tfoot>
</table></div>
</div>
<div id="rickettsia-1" class="section level4" number="1.2.2.4">
<h4>
<span class="header-section-number">1.2.2.4</span> <em>Rickettsia</em><a class="anchor" aria-label="anchor" href="#rickettsia-1"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Spotted fever - Flinders Island Spotted or Australian Spotted fever?</strong></p>
<p>The identification of Flinders Island Spotted Fever (<em>R. honei</em>) was a significant breakthrough in the knowledge of tick-borne diseases in Australia.
However, ongoing research into the distribution of the disease, and more recently genetic information, has meant the label of “Flinders Island” Spotted Fever causes significant issues.
Only a few years after it was formally recognised, a new focus of <em>Rickettsia honei</em> spotted fever was identified in South Australia and Tasmania <span class="citation">(<a href="references.html#ref-dyerNewFocusRickettsia2005" role="doc-biblioref">Dyer et al. 2005</a>; <a href="references.html#ref-unsworthNotOnlyFlinders2005" role="doc-biblioref">Nathan B. Unsworth et al. 2005</a>)</span>.
Lane et al. <span class="citation">(<a href="references.html#ref-laneEvidenceSpottedFeverlike2005a" role="doc-biblioref">2005</a>)</span> identified a <em>Rickettsia honei</em>-like sequence from a <em>Haemphysalis novaeguineae</em> tick.
The patient was bitten by the tick in north-east Queensland and become acutely unwell developing signs of rickettsial disease (later report by Unsworth et al. <span class="citation">(<a href="references.html#ref-unsworthThreeRickettsiosesDarnley2007" role="doc-biblioref">2007</a>)</span>).
Although molecular analysis did show similarities with <em>R. honei</em> strain TT-118 (Thai tick typhus), and <em>R. honei</em> (Flinders Island Spotted Fever), it was not able to fully resolve the relationships among the SFG rickettsia.
A report of seven patients exhibiting similar symptoms to Flinders Island Spotted Fever was later published <span class="citation">(<a href="references.html#ref-unsworthThreeRickettsiosesDarnley2007" role="doc-biblioref">N. B. Unsworth et al. 2007</a>)</span>.
A combination of serological, molecular and culture techniques were used and <em>R. honei</em> subsp. marmionii was described in patients from Queensland, Tasmania and South Australia.
It again confirmed the presence of the novel genotype and subsequent illness caused by a <em>Haemaphysalis novaeguineae</em> bite from Cape York Peninsula in far north Queensland.
There have been no other identification of <em>R. honei</em> in a tick vector on mainland Australia.
A recent case described a negative serological result from a patient, however subsequent shot gun (whole genome) sequencing of the blood showed it was positive for <em>Rickettsia honei</em> <span class="citation">(<a href="references.html#ref-grahamDetectionSpottedFever2017" role="doc-biblioref">Graham et al. 2017</a>)</span>.
The initial naming of the disease as “Flinders Island” Spotted Fever has potentially caused havoc on the diagnosis and treatment of the disease.
Due to the non-specific acute symptoms, and at times unusual sequelae of spotted-fever, it is plausible that treating physicians may not consider FISF as a possible aetiology due to geographical restrictions.
Scientifically sound and timely case reports are fundamental to ensure information is disseminated to the medical community.
A number of case reports of a <em>Rickettsia</em>-like illness have also been identified from patients in Western Australia.
Molecular screening of a punch biopsy (taken at edge of eschar) sample taken from a patient bitten by <em>Ixodes australiensis</em> in Walpole, south-west Western Australia showed the presence of <em>Rickettsia</em> sp. (unable to identify to species).
Using a PCR assay the whole blood was negative, however serological testing showed evidence of acute infection with SFG <em>Rickettsia</em>; culture and PCR from the tick were negative <span class="citation">(<a href="references.html#ref-rabyNewFociSpotted2016" role="doc-biblioref">Raby et al. 2016</a>)</span>.
A female patient was bitten by a tick 150 km east of Esperance and DNA extracted from acute phase serum underwent PCR for the rickettsial 17kD antigen gene which generated a 429 bp sequence showed 100% similarity to <em>R. honei</em>, and 99.7% to <em>R. gravesii</em>.
It is unclear if <em>R. honei</em> has a larger geographical and vector range than previously thought or if in fact this was <em>R. gravesii</em> sequence showing a 1 bp mis-match to reference sequence <span class="citation">(<a href="references.html#ref-rabyNewFociSpotted2016" role="doc-biblioref">Raby et al. 2016</a>)</span>.
A serological study in Western Australia showed that those who frequented bushland had a higher risk of exposure to spotted fever group rickettsia compared to the reference population <span class="citation">(<a href="references.html#ref-abdadSeroepidemiologicalStudyOutdoor2014" role="doc-biblioref">Abdad et al. 2014</a>)</span>.
In addition, there have also been a number of novel <em>Rickettsia</em> species recently described from native Australian ticks (Table <a href="litrev.html#tab:T1rickettsia">1.5</a>).
Importantly many of these novel findings highlight the difficultly and ambiguity in species delimitation of the genus.<br>
A novel Australian <em>Rickettsia</em> was identified from the soft tick <em>Argas dewae</em> from bat roosting boxes in Victoria <span class="citation">(<a href="references.html#ref-izzardRickettsialesRickettsialDiseases2010" role="doc-biblioref">L. H. Izzard 2010</a>)</span>.
Gene sequences from five genes (<em>gltA</em>, <em>rOmpB</em>, <em>rOmpA</em>, <em>rrs</em> and <em>sca4</em>) totalling over 10 kb demonstrated that it fit the criteria for the designation of a novel species as per <span class="citation"><a href="references.html#ref-fournierGeneSequenceBasedCriteria2003" role="doc-biblioref">Fournier et al.</a> (<a href="references.html#ref-fournierGeneSequenceBasedCriteria2003" role="doc-biblioref">2003</a>)</span> and was tentatively named <em>Rickettsia dewae</em>.<br>
However, a recent study published by the same authors illustrates how whole genome sequence revealed that it is actually a divergent strain of <em>Rickettsia japonica</em> <span class="citation">(<a href="references.html#ref-izzardIsolationDivergentStrain2018" role="doc-biblioref">L. Izzard et al. 2018</a>)</span>.
In that same study the authors also raise important questions on the classification of <em>Rickettsia</em> species as outlined in <span class="citation"><a href="references.html#ref-fournierGeneSequenceBasedCriteria2003" role="doc-biblioref">Fournier et al.</a> (<a href="references.html#ref-fournierGeneSequenceBasedCriteria2003" role="doc-biblioref">2003</a>)</span>.
With the growing trend towards whole genome sequencing it may in fact raise questions around descriptions of other <em>Rickettsia</em> species.</p>
<div class="inline-table"><table class="table table" style="width: auto !important; margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T1rickettsia">Table 1.5: </span> species identified from Australian ticks using molecular methods.
</caption>
<thead><tr>
<th style="text-align:left;">
Microorganism
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:left;">
Vertebrate host
</th>
<th style="text-align:left;">
Identified in host
</th>
<th style="text-align:left;">
State
</th>
<th style="text-align:left;">
Reference(s)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia australis (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus, Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Bandicoots, reptiles
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD, VIC
</td>
<td style="text-align:left;width: 6em; ">
Andrew et al. 1946; Brody 1946; Pope 1955; Campbell and Domrow 1974; Campbell et al. 1979
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia honei (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. hydrosauri
</td>
<td style="text-align:left;width: 8em; ">
Reptiles
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Graves et al. 1991; Stewart 1991; Graves and Stenos 2003; Stenos et al. 2003; Dyer et al. 2005
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia honei marmionii (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ha. novaeguineae
</td>
<td style="text-align:left;width: 8em; ">
Unknown
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Lane et al. 2005; Unsworth et al. 2007a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia gravesii
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:left;width: 8em; ">
Feral pigs, horses, macropods
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD, WA
</td>
<td style="text-align:left;width: 6em; ">
Owen et al. 2006b; Li et al. 2010; Sentausa et al. 2013; Abdad et al. 2017; Chalada et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia antechini
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. antechini
</td>
<td style="text-align:left;width: 8em; ">
Antechinus
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
WA
</td>
<td style="text-align:left;width: 6em; ">
Owen et al. 2006a; Owen 2007
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia tasmanensis
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Tasmanian devil
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Izzard 2010
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia massillae-like
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Tasmanian devil
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al. 2009c
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia bellii-like
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. concolor
</td>
<td style="text-align:left;width: 8em; ">
Echidna
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
VIC
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al. 2009d
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia massillae-like (koala genotype)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Koala
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, VIC
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al. 2008
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia massillae-like
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. auruginans
</td>
<td style="text-align:left;width: 8em; ">
Wombat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
VIC
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al. 2009d
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia tamurae-like
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. fimbriatum, Bt concolor
</td>
<td style="text-align:left;width: 8em; ">
Echidna, reptiles
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NT, QLD
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al. 2009a; Chalada et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia japonica (str. argasii)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ar. (Ca.) dewae, Ha. bancrofti
</td>
<td style="text-align:left;width: 8em; ">
Bats, horse
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD, VIC
</td>
<td style="text-align:left;width: 6em; ">
Parola et al. 2013; Chalada et al. 2018; Izzard et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia felis
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ha. bancrofti
</td>
<td style="text-align:left;width: 8em; ">
Horse
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Chalada et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Rickettsia sp. ARRL2016-156
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
A,. Albolimbatum
</td>
<td style="text-align:left;width: 8em; ">
Bobtail
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
WA
</td>
<td style="text-align:left;width: 6em; ">
Tadepalli et al. 2021
</td>
</tr>
</tbody>
<tfoot><tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Recognised human tick-borne pathogen.
</td>
</tr></tfoot>
</table></div>
</div>
<div id="viruses-1" class="section level4" number="1.2.2.5">
<h4>
<span class="header-section-number">1.2.2.5</span> Viruses<a class="anchor" aria-label="anchor" href="#viruses-1"><i class="fas fa-link"></i></a>
</h4>
<p>Despite the presence and pathogenicity of tick-borne viruses being well described overseas, Australia does not currently recognise the presence of an endemic tick-borne virus.
Virus-tick-vertebrate host relationships are highly specific <span class="citation">(<a href="references.html#ref-labudaTickborneViruses2004" role="doc-biblioref">Labuda and Nuttall 2004</a>)</span>.
A review of neglected arboviruses in Australia highlighted that despite a surge of research pioneered by the Commonwealth Scientific and Industrial Research Organisation (CSIRO), tick-borne virus research has largely remained undocumented <span class="citation">(<a href="references.html#ref-gyawaliNeglectedAustralianArboviruses2017" role="doc-biblioref">Gyawali et al. 2017</a>)</span>.
A summary of past and recent virus discoveries from Australian ticks is provided in Table <a href="litrev.html#tab:T1virus">1.6</a>.</p>
<p>During the 1960’s and 1970’s there was a strong research presence around novel tick-borne viruses in Australia led by CSIRO team.
Gadgets Gully virus was described in 1976 from <em>Ixodes uriae</em> ticks at Macquarie Island <span class="citation">(<a href="references.html#ref-st.georgeIsolationArbovirusesIncluding1985" role="doc-biblioref">St. George et al. 1985</a>)</span>.
The virus was isolated by intra-cerebral inoculation of ground tick suspensions into neonatal mice, which resulted in mice developing neurological symptoms and died 5 days post infection.
Serological characterisation of the isolate demonstrated that it was an unknown flavivirus however, there has been no further research and as such the ecology, transmission dynamics and its potential to cause disease in humans remains unknown.
Saumerez Reef virus was described in 1974 from <em>Ornithorodoros capensis</em> ticks collected from nests of sooty terns (<em>Sterna fuscata</em>), Queensland <span class="citation">(<a href="references.html#ref-st.georgeIsolationSaumarezReef1977" role="doc-biblioref">St. George et al. 1977</a>)</span>.
Several other viral isolates were obtained from the hard tick <em>Ixodes eudyptidis</em>, collected in Tasmania <span class="citation">(<a href="references.html#ref-st.georgeIsolationSaumarezReef1977" role="doc-biblioref">St. George et al. 1977</a>)</span>.
Intra-cerebral inoculation of neonatal mice caused death while intra-cerebral inoculation of weanling mice did induce antibody formation and no clinical illness.
No data on potential human infection or human illness are available.
Serological studies in silver gulls (<em>Larus novaehollandiae</em>) in Tasmania showed they had an antibody seroprevalence rates of 25%. Ticks and birds are thought to contribute to the natural transmission of Saumarez Reef virus, however, further studies are needed to elucidate the natural cycle of the virus.
Vinegar Hill virus (VINHV) is a member of the Bunyavirales order (tentative member of the genus <em>Orthonairovirus</em>) and was isolated from the soft tick <em>Argas robertsi</em> collected from cattle egrets <span class="citation">(<a href="references.html#ref-gauciGenomicCharacterisationVinegar2017" role="doc-biblioref">Gauci et al. 2017</a>)</span>.
Avian and human sera from local residents in the Coral Sea and Great Barrier Reef was used for testing against 19 known arboviruses.
It found that antibodies were detected in 4% of avian and human sera and included Gadget’s Gully virus (flavivirus) and Murray Valley Encephalitis.
It was noted however, that a number of antibodies were restricted to sea birds only.
Novel Phlebovirus with zoonotic potential was identified from a colony of shy albatross (<em>Thalassarche cauta</em>) on Albatross Island, northwest of Tasmania in the Hunter Island Group <span class="citation">(<a href="references.html#ref-wangNovelPhlebovirusZoonotic2014" role="doc-biblioref">Wang et al. 2014</a>)</span>.
Both ticks (<em>Ixodes eudyptidis</em>) and serum samples were collected, and sequences were obtaining via RNAseq using the 454 platform, however subsequent testing by ESLISA or qPCR was unsuccessful.</p>
<p>A unique iflavivirus has recently been identified from <em>Ix. holocyclus</em> in Queensland and New South Wales <span class="citation">(<a href="references.html#ref-obrienDiscoveryNovelIflavirus2018" role="doc-biblioref">O’Brien et al. 2018</a>)</span>.
Designed <em>Ix. holocyclus</em> iflavirus (IhIV), it represents the first virus sequence identified in <em>Ix. holocyclus</em> ticks.
Members of the <em>Iflaviridae</em> family are considered ‘arthropod-only’ viruses, and to date have not been implicated in human disease.
More recently metatranscriptomic applications have characterised a number of novel viral sequences.
Using this approach 19 novel RNA viruses were characterised which included members of the <em>Flaviviridae</em> and <em>Reoviridae</em> families <span class="citation">(<a href="references.html#ref-harveyExtensiveDiversityRNA2019" role="doc-biblioref">Harvey et al. 2019</a>)</span>.
Members of these viral families include known human pathogens described in the northern hemisphere such as tick-borne encephalitis virus and Powassan virus (both belong to genus <em>Flavivirus</em>) and Colorado tick fever virus (genus <em>Coltivirus</em>).
Their isolation in the common human biting tick <em>Ix. holocyclus</em> makes them important candidates for future research into possible links to cases of human disease.</p>
<div class="inline-table"><table class="table table table" style="width: auto !important; margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T1virus">Table 1.6: </span>Viruses identified from Australian ticks using molecular and culture based methods. Name of virus and genus or family in parentheses.
</caption>
<thead><tr>
<th style="text-align:left;">
Microorganism
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:left;">
Vertebrate host
</th>
<th style="text-align:left;">
Identified in host
</th>
<th style="text-align:left;">
State
</th>
<th style="text-align:left;">
Reference(s)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Nugget virus (Orbivirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
Doherty et al. 1975
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Taggert virus (Orthonairovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
Doherty et al. 1975
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Lake Clarendon virus (Reoviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ar. robertsi
</td>
<td style="text-align:left;width: 8em; ">
Cattle egrets
</td>
<td style="text-align:left;width: 6em; ">
<ol style="list-style-type: lower-alpha"><li></ol>
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
St. George et al. 1984
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Precarious Point virus (Phlebovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
St. George et al. 1985
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Upolu virus (Thogotovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Or. capensis
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Doherty et al. 1968; Briese et al. 2014
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Gadgets Gully virus (Flavivirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
St. George, 1991
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Saumarez Reef virus (Flavivirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Or. capensis, Ix. eudyptidis
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD, TAS
</td>
<td style="text-align:left;width: 6em; ">
St. George et al. 1977
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Johnston Atoll virus (Quarjavirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Or. capensis
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD (c)
</td>
<td style="text-align:left;width: 6em; ">
Doherty et al. 1968
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Vinegar Hills virus (Orthonairovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ar. robertsi
</td>
<td style="text-align:left;width: 8em; ">
Cattle egrets
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
St. George, 2011; Gauci et al. 2017
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hunter Island Group virus (syn. Albatross Island virus) (Phlebovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. eudyptidis
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (d)
</td>
<td style="text-align:left;width: 6em; ">
Wang et al. 2014; Gauci et al. 2015
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Catch-me-Cave virus (Phlebovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
Major et al. 2009
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Finch Creek virus (Orthonairovirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
Major et al. 2009
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Sandy Bay virus (Orbivirus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. uriae
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (b)
</td>
<td style="text-align:left;width: 6em; ">
Major et al. 2009
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Little Diamond Island virus group (Uknown)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. kohlsi
</td>
<td style="text-align:left;width: 8em; ">
Seabirds
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS (e)
</td>
<td style="text-align:left;width: 6em; ">
Major et al. 2009
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Iflavivirus (iFlaviviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Mammals
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
O’Brien et al. 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Manly virus (Rhabdoviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. moreliae
</td>
<td style="text-align:left;width: 8em; ">
Blue-tongue
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Fairlight virus (Mononegavirales)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. moreliae
</td>
<td style="text-align:left;width: 8em; ">
Blue-tongue
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Cannae Point virus (Chuviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. moreliae
</td>
<td style="text-align:left;width: 8em; ">
Blue-tongue
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Store Beach virus (Luteo-like virus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. moreliae
</td>
<td style="text-align:left;width: 8em; ">
Blue-tongue
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Quarantine Head virus (Mononegavirales)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. moreliae
</td>
<td style="text-align:left;width: 8em; ">
Blue-tongue
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
North Shore virus (Partitiviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, human, questing
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Blue Fish Point virus (Luteo-like virus)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, black rat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Shelly Headland virus (Reoviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Jump Rock virus (Picornaviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Ingleside virus (Virgaviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, questing
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Collins Beach virus (Flaviviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Fairfax Lookout virus (Flaviviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. trichosuri
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, bush rat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Timbillica virus (Phenuiviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Genoa virus (Chuviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Nadgee virus (Narnaviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Wangarabell virus (Narnaviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Yambulla virus (Narnaviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Old Quarry Swamp virus (Orthomyxoviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, black rat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Shelly Beach virus (Reoviridae)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot, black rat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Antibodies identified from host blood samples.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>b</sup> Records from Macquarie Island off Tasmania coastline.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>c</sup> Records from Heron Island off Queensland coastline.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>d</sup> Records from Hunter Island Group off Tasmania coastline.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>e</sup> Records from Diamond Island off Tasmania coastline.
</td>
</tr>
</tfoot>
</table></div>
</div>
<div id="eukaryotes-1" class="section level4" number="1.2.2.6">
<h4>
<span class="header-section-number">1.2.2.6</span> Eukaryotes<a class="anchor" aria-label="anchor" href="#eukaryotes-1"><i class="fas fa-link"></i></a>
</h4>
<p>Eukaryote organisms reviewed here were chosen based on their relationship with known agents responsible for tick-borne diseases globally.
Taxa included within this section include piroplasms (e.g. <em>Babesia</em> and <em>Theileria</em>), <em>Hepatozoon</em>, and <em>Trypanosoma</em>.
These organisms can broadly be classified as haemoprotozoa.
In Australia there is currently no recognised human eukaryote tick-borne pathogens.
So far studies have failed to provide evidence for agents of human tick-borne diseases that have been described in the northern hemisphere to be present in Australia.</p>
<p>Due to the recognised issues associated with morphological identification of haemoprotozoa <span class="citation">(<a href="references.html#ref-zhuLooksCanDeceive2009" role="doc-biblioref">Zhu et al. 2009</a>; <a href="references.html#ref-lackPhylogenyEvolutionPiroplasmida2012" role="doc-biblioref">Lack, Reichard, and Van Den Bussche 2012</a>; <a href="references.html#ref-kostygovEuglenozoaTaxonomyDiversity2021" role="doc-biblioref">Kostygov et al. 2021</a>)</span> this review will focus on molecular identifications of eukaryotes from Australian ticks.
A summary of selected eukaryote organisms that have been identified from Australian ticks are available in Table <a href="litrev.html#tab:T1eukaryotes">1.7</a>.</p>
<div class="inline-table"><table class="table table table" style="width: auto !important; margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T1eukaryotes">Table 1.7: </span>A selection of eukaryotes identified from Australian ticks using molecular based methods. Eukaryotes groups presented here represent organisms related to taxa associated with tick-borne diseases globally.
</caption>
<thead><tr>
<th style="text-align:left;">
Microorganism
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:left;">
Vertebrate host
</th>
<th style="text-align:left;">
Identified in host
</th>
<th style="text-align:left;">
State
</th>
<th style="text-align:left;">
Reference(s)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia canis vogeli (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Rh. sanguineus
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
Yes/No
</td>
<td style="text-align:left;width: 6em; ">
NT, Torres Strait
</td>
<td style="text-align:left;width: 6em; ">
Jefferies et al. 2003; Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia lohae
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Cat
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia sp. (lohae-like)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus, Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Brushtail possum
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Loh et al. 2018a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia sp. (lohae-like)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ixodes sp., Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Eastern grey kangaroo, red-necked wallaby
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Storey-Lewis 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia mackerrasorum
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Haemaphysalis sp.
</td>
<td style="text-align:left;width: 8em; ">
Horse
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ixodes sp., Ix. holocyclus, Ha. Petrogalis
</td>
<td style="text-align:left;width: 8em; ">
Red-necked wallaby, unknown
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Storey-Lewis 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Babesia sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Haemaphysalis sp., Ha. petrogalis
</td>
<td style="text-align:left;width: 8em; ">
Eastern grey kangaroo, red-necked wallaby
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Storey-Lewis 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria apogeana
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria fuliginosa-like
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. australiensis
</td>
<td style="text-align:left;width: 8em; ">
Western grey kangaroo
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
WA
</td>
<td style="text-align:left;width: 6em; ">
Loh et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria orientalis (a)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ha. longicornis, Ha. bancrofti
</td>
<td style="text-align:left;width: 8em; ">
Cattle, dog, questing, red fox
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Hammer et al. 2015; Greay et al. 2018b; Loh et al. 2018a; Marendy et al. 2019; Emery et al. 2021; Lakew et al. 2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria ornithorhynchi
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. ornithorhynchi
</td>
<td style="text-align:left;width: 8em; ">
Platypus
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Paparini et al. 2015
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria palermi
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria paparinii
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria worthingtonorum
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD, TAS
</td>
<td style="text-align:left;width: 6em; ">
Loh et al. 2018a
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ixodes sp., Haemaphysalis sp., Ha. petrogalis, Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Red-necked wallaby
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Storey-Lewis 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Horse
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Storey-Lewis 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Theileria sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Bt. concolor
</td>
<td style="text-align:left;width: 8em; ">
Echidna
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Storey-Lewis 2018
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hepatozoon banethi
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hepatozoon canis (b)
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Dog
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
QLD
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018c
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hepatozoon ewingi
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ha. bancrofti
</td>
<td style="text-align:left;width: 8em; ">
Horse
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Greay et al. 2018b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hepatozoon sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. fimbriatum
</td>
<td style="text-align:left;width: 8em; ">
Lizard, snakes
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NT
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al 2009e
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hepatozoon sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. fimbriatum, Am. moreliae
</td>
<td style="text-align:left;width: 8em; ">
Lizard, snakes
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NT
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al 2009e
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Hepatozoon sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Tasmanian devil
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
TAS
</td>
<td style="text-align:left;width: 6em; ">
Vilcins et al 2009c
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Trypanosoma copemani
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. australiensis, Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Gilberts potoroo, koala, quokka
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Austen et al. 2011
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Trypanosoma gilletti
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus, Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Koala
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Barbosa et al. 2017b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Trypanosoma irwini
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Koala
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Austen et al. 2011; Barbosa et al. 2017b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Trypanosoma noyesi
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:left;width: 8em; ">
Questing
</td>
<td style="text-align:left;width: 6em; ">
NA
</td>
<td style="text-align:left;width: 6em; ">
WA
</td>
<td style="text-align:left;width: 6em; ">
Krige et al. 2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Trypanosoma vegrandis
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:left;width: 8em; ">
Koala
</td>
<td style="text-align:left;width: 6em; ">
Yes
</td>
<td style="text-align:left;width: 6em; ">
NSW, QLD
</td>
<td style="text-align:left;width: 6em; ">
Barbosa et al. 2017b
</td>
</tr>
<tr>
<td style="text-align:left;width: 10em; font-style: italic;">
Trypanosoma sp.
</td>
<td style="text-align:left;width: 10em; font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:left;width: 8em; ">
Bandicoot
</td>
<td style="text-align:left;width: 6em; ">
No
</td>
<td style="text-align:left;width: 6em; ">
NSW
</td>
<td style="text-align:left;width: 6em; ">
Harvey et al. 2019
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Imported species that are now considered endemic.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>b</sup> Previously considered an exotic species to Australia, first identification made in 2018.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>c</sup> Retrospective sequence analysis showed that sequences are more similar to the genus Hemolivia.
</td>
</tr>
</tfoot>
</table></div>
<p>There are currently no recognised endemic human protozoal tick-borne diseases in Australia.
To date there has been a single case of human babesiosis caused by <em>Babesia microti</em> in Australia from a patient without a travel history <span class="citation">(<a href="references.html#ref-senanayakeFirstReportHuman2012" role="doc-biblioref">Senanayake et al. 2012</a>)</span>.
Subsequent molecular investigation confirmed that the identification of <em>B. microti</em> and phylogenetic analysis showed that it grouped closely with other <em>B. microti</em> genotypes identified from the northern hemisphere <span class="citation">(<a href="references.html#ref-papariniMolecularConfirmationFirst2014" role="doc-biblioref">Andrea Paparini et al. 2014</a>)</span>.
An epidemiological investigation was carried out at the time of the case and testing was conducted from a close relative and pet dog which all yielded negative results <span class="citation">(<a href="references.html#ref-senanayakeFirstReportHuman2012" role="doc-biblioref">Senanayake et al. 2012</a>)</span>.
Subsequently a widespread serological investigation was carried out on Australian blood donor samples for the presence of <em>B. microti</em> which included 7000 patients and did not identify any positives samples <span class="citation">(<a href="references.html#ref-faddyNoEvidenceWidespread2019" role="doc-biblioref">Faddy et al. 2019</a>)</span>.</p>
<p>A number of piroplasms have been introduced to Australia, however they all related to livestock and domestic animals.
<em>Babesia canis</em> and <em>Theileria orientalis</em> are both associated with introduced tick species, the brown dog ticks (<em>Rh. sanguinus</em>) and the Asian longhorned ticks (<em>Ha. longicornis</em>).
While <em>Th. orientalis</em> has been identified in native Australian tick species such as <em>Ha. bancrofti</em> <span class="citation">(<a href="references.html#ref-lakewEndemicInfectionCattle2021" role="doc-biblioref">Lakew et al. 2021</a>)</span>, the overwhelming evidence suggests the main vector is <em>Ha. longicornis</em> <span class="citation">(<a href="references.html#ref-marendyHaemaphysalisLongicornisLifecycle2019" role="doc-biblioref">Marendy et al. 2019</a>)</span>.
However, there may be some variation of vector competence between the difference genotypes of <em>Th. orientalis</em>, widespread detection of this <em>Theileria</em> in native Australian ticks or animals has not yet been shown.</p>
<p>Genetic analysis of native species of piroplasms described from Australia has shown that they form a distinct clade <span class="citation">(<a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>)</span>.
Of particular note is the group of <em>Babesia</em> species identified from Australian ticks and marsupials is phylogenticaly different from the human pathogen <em>B. microti</em> (belonging to the <em>Babesia microti</em> group.</p>
<p>While traditionally not associated with ticks, an early study identified that ticks may be involved in the life cycle of native Australian trypanosomes.
Early studies using morphological tools identified trypanosomes within the Australian paralysis tick (<em>Ix. holocyclus</em>) collected from bandicoots infected with <em>Trypanosoma thylacis</em> <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>)</span>.
Molecular evidence of the relationship between Australian trypanosomes and ticks followed 50 years later from the opposite side of the country in south-west Australia.
Austen et al. <span class="citation">(<a href="references.html#ref-austenVectorTrypanosomaCopemani2011" role="doc-biblioref">2011</a>)</span> identified <em>Trypanosoma copemani</em> from <em>Ixodes</em> sp. ticks collected from infected marsupials.
Motile trypanosomes were observed in <em>Ixodes australiensis</em> collected from quokkas and the Gilbert’s potoroo on Bald Island and the nature reserve Two Peoples Bay.</p>
<p>Australia was once considered free of <em>Leishmania</em>, however an endemic species has since been described after it was noted red kangaroos were showing signs of cutaneous leishmaniasis <span class="citation">(<a href="references.html#ref-roseCutaneousLeishmaniasisRed2004" role="doc-biblioref">Rose et al. 2004</a>)</span>.
The species has formally been described as <em>Leishmania macropodum</em> <span class="citation">(<a href="references.html#ref-barrattIsolationNovelTrypanosomatid2017" role="doc-biblioref">Barratt et al. 2017</a>)</span> and although studies are limited, current evidence suggests day-biting midges, <em>Forcipomyia</em> (<em>Lasiohelea</em>), as a likely vector <span class="citation">(<a href="references.html#ref-dougallEvidenceIncriminatingMidges2011" role="doc-biblioref">A. M. Dougall et al. 2011</a>; <a href="references.html#ref-panahiUtilisingNovelSurveillance2020" role="doc-biblioref">Panahi et al. 2020</a>)</span>.
To date it has not been identified from any Australian ticks and or any other Australian state/territory outside the NT <span class="citation">(<a href="references.html#ref-cleareRemainingVigilantExotic2014" role="doc-biblioref">Cleare et al. 2014</a>; <a href="references.html#ref-dybingGhostsChristmasAbsence2016" role="doc-biblioref">Dybing et al. 2016</a>; <a href="references.html#ref-thompsonExoticParasiteThreats2018" role="doc-biblioref">R. C. A. Thompson 2018</a>)</span>.</p>
<p>Haemogregarines are a group of blood parasites that belong to the apicomplexa phylum, and are capable of infecting a range of vertebrates.
The classifications of haemogregarines is problematic, with the lack of a monophyletic group within the family Haemogregarinida <span class="citation">(<a href="references.html#ref-al-quraishyHaemogregarinesCriteriaIdentification2021" role="doc-biblioref">Al-Quraishy et al. 2021</a>)</span>. The group is likely to consist of at least three families.
Recent molecular discoveries have shown that morphological identifications at the genus level may not be reliable for many haemoprotozoa.
For example previous studies have shown that organisms considered to be <em>Hepatozoon</em> (-like) were actually members of distantly related groups <span class="citation">(<a href="references.html#ref-merinoSarcocystidMisidentifiedHepatozoon2008" role="doc-biblioref">Santiago Merino et al. 2008</a>; <a href="references.html#ref-zhuLooksCanDeceive2009" role="doc-biblioref">Zhu et al. 2009</a>)</span>.
Not only do these findings challenge taxonomic classification but they also show that microbes may not be as tissue-specific (e.g. blood, liver etc) as previously thought raising additional questions about the true life cycle of organisms.</p>
<p>The genus <em>Hepatozoon</em> was first described following the identification of <em>Hepatozoon muris</em> (syn. <em>Hepatozoon perniciosum</em>) in laboratory rats and mites (<em>Laelaps echidninus</em>).
The genus is considered distinct from other haemogregarines due to two features of its life cycle; (i) the production of polysporocystic oocysts in hematophagous invertebrate definitive hosts, and (ii) transmission to vertebrate intermediate hosts via the ingestion of definitive hosts carrying sporulated oocysts <span class="citation">(<a href="references.html#ref-smithGenusHepatozoonApicomplexa1996" role="doc-biblioref">T. G. Smith 1996</a>; <a href="references.html#ref-mathewPHYLOGENETICRELATIONSHIPSHEPATOZOON2000" role="doc-biblioref">Mathew et al. 2000</a>)</span>.
A review of <em>Hepatozoon</em> reported from Australian ticks is presented in Table <a href="litrev.html#tab:T1eukaryotes">1.7</a>.
Upon retrospective analysis of sequences generated from reptile ticks attributed to <em>Hepatozoon</em> <span class="citation">(<a href="references.html#ref-vilcinsMolecularMorphologicalDescription2009" role="doc-biblioref">Vilcins, Ujvari, et al. 2009</a>)</span> showed they were misclassified.
A BLAST analysis revealed the sequences were more closely related to <em>Hemoliva</em>, and only 94% similar to <em>Hepatozoon</em> sequences.
<em>Hemoliva</em> has previously been reported from reptile ticks in Australia and to date has not been associated with disease in mammals.</p>
<p>Reports of associations between nematodes and Australian ticks are sparse in the literature. The most well documented is the parasitic filarial nematode <em>Cercopithifilaria johnstoni</em> association with the tick <em>Ixodes trichosuri</em> from Australian mammals, usually from bush rats <span class="citation">(<a href="references.html#ref-sprattAspectsLifeHistory1988a" role="doc-biblioref">Spratt and Haycock 1988</a>; <a href="references.html#ref-mccannGenomeSequenceAustralian2021" role="doc-biblioref">McCann, Grant, and Doyle 2021</a>)</span>.</p>
</div>
</div>
</div>
<div id="molecular-tools" class="section level2" number="1.3">
<h2>
<span class="header-section-number">1.3</span> Molecular tools<a class="anchor" aria-label="anchor" href="#molecular-tools"><i class="fas fa-link"></i></a>
</h2>
<p>Tick microbiome studies began by using microscopy and cell culture techniques, these studies set the ground for work being done on the tick microbiome today.
Early studies by Cowdry <span class="citation">(<a href="references.html#ref-cowdryGroupMicroorganismsTransmitted1925" role="doc-biblioref">1925</a>)</span> were limited by the technology at the time and as a result, were restricted by; (i) lack of distinguishing features between bacterial species and often pleomorphic taxa, and (ii) the presence of unculturable bacteria.
The advent of the polymerase chain reaction (PCR) <span class="citation">(<a href="references.html#ref-mullisSpecificSynthesisDNA1987" role="doc-biblioref">Mullis and Faloona 1987</a>)</span> and DNA sequencing technologies <span class="citation">(<a href="references.html#ref-sangerDNASequencingChaintermination1977" role="doc-biblioref">Sanger, Nicklen, and Coulson 1977</a>)</span>, overcame these early challenges and meant a definitive identification of microbes was now possible.
The implementation of Sanger sequencing transformed the understanding bacterial pathogens with a vast increase in the sensitivity of detection methods <span class="citation">(<a href="references.html#ref-chakravortyDetailedAnalysis16S2007" role="doc-biblioref">Chakravorty et al. 2007</a>; <a href="references.html#ref-macdonaldFrameworkDevelopingValidating2016" role="doc-biblioref">MacDonald and Sarre 2016</a>)</span>.
In the context of tick-borne pathogens, PCR and Sanger sequencing shifted thinking from a single pathogen, to the idea that ticks can harbour a range of different microbes, which may co-occur in some tick species <span class="citation">(<a href="references.html#ref-brouquiGuidelinesDiagnosisTickborne2004" role="doc-biblioref">Brouqui et al. 2004</a>; <a href="references.html#ref-scolesPhylogeneticAnalysisFrancisellalike2004" role="doc-biblioref">Scoles 2004</a>; <a href="references.html#ref-parolaTickFleaborneRickettsial2005" role="doc-biblioref">Philippe Parola, Davoust, and Raoult 2005</a>)</span>.
Further advancements over time, such as quantitative PCR (qPCR), sophisticated methods for nucleic acid extraction, optimisation of PCR conditions (including inhibition and low copy number) have continued to change the way researchers investigate the microbiome <span class="citation">(<a href="references.html#ref-hoffmannAnalysisTickSurface2020" role="doc-biblioref">Hoffmann, Fingerle, and Noll 2020</a>; <a href="references.html#ref-koloAnaplasmaPhagocytophilumOther2020" role="doc-biblioref">Kolo et al. 2020</a>)</span>.
The advancement of molecular tools has highlighted the diversity of microbes harboured within ticks.
Ticks removed from hosts contain nucleic acid material that consists of three main sources: (i) the tick itself; (ii) from the host (vertebrate) it was parasitising; and (iii) microbial organisms (e.g. bacteria, protozoa, viruses and helminths).
The complex nature of the material from ticks lends itself well to platforms used in the characterisation of environment DNA.</p>
<p>The introduction of high throughput sequencing has revolutionised the analysis of complex environmental samples.
Microbial diversity studies have adapted to high-throughput sequencing techniques using two broad appoaches by; (i) amplicon based - through targeting the hypervariable regions in the 16S rRNA gene; and (ii) shotgun sequencing - which sequences the entire suite of nucleic acid material in the extracted sample (variations can be made during library preparation) <span class="citation">(<a href="references.html#ref-liuPracticalGuideAmplicon2020" role="doc-biblioref">Liu et al. 2020</a>; <a href="references.html#ref-bhartiCurrentChallengesBestpractice2021" role="doc-biblioref">Bharti and Grimm 2021</a>)</span>.
Despite the presumption that shotgun sequencing remains superior to amplicon metabarcoding approaches, studies have shown targeted 16S rRNA metabarcoding can yield up to 50% more phyla than shotgun based methods <span class="citation">(<a href="references.html#ref-tesslerLargescaleDifferencesMicrobial2017" role="doc-biblioref">Tessler et al. 2017</a>)</span>.
The use of targeted 16S rRNA metabarcoding is particularly favourable where the level of detection is low or where there is a large amount of host material.
The first application of high throughput sequencing methods to study the tick microbiome was in <em>Rhipicephalus</em> (<em>Boophilus</em>) <em>microplus</em> <span class="citation">(<a href="references.html#ref-andreottiAssessmentBacterialDiversity2011" role="doc-biblioref">Andreotti et al. 2011</a>)</span>.
Since then, high-throughput methods have shed light on the diversity of the tick microbiome <span class="citation">(<a href="references.html#ref-greayRecentInsightsTick2018" role="doc-biblioref">T. L. Greay, Gofton, et al. 2018</a>)</span> and shift the on tick - one pathogen thinking towards the understanding of the bacterial community prresent in ticks <span class="citation">(<a href="references.html#ref-moutaillerCoinfectionTicksRule2016" role="doc-biblioref">Moutailler et al. 2016</a>)</span>.
Bacterial metabarcoding approaches target one (or more) of the nine available hypervariable regions on the 16S rRNA gene; common regions sequenced include the V1-2 and V3-4 <span class="citation">(<a href="references.html#ref-barbDevelopmentAnalysisPipeline2016" role="doc-biblioref">Barb et al. 2016</a>; <a href="references.html#ref-yangSensitivityCorrelationHypervariable2016" role="doc-biblioref">B. Yang, Wang, and Qian 2016</a>; <a href="references.html#ref-sperlingComparisonBacterial16S2017" role="doc-biblioref">J. L. Sperling et al. 2017</a>)</span>.
The short amplicons hinders accurate taxonomic and follow up molecular analysis is needed to provide greater phylogenetic resolution.
However, despite this, high throughput sequencing technologies is superior to traditional cloning and Sanger sequencing methods in characterising a diverse community of organisms.
Studies of tick-borne diseases are often restricted in scope due to their narrow focus on known pathogens, by the use of species-specific or genus-specific primers, and as a result are at risk of overlooking potentially pathogenic agents or novel organisms.
Unbiased high-throughput sequencing provides an ideal method to identify new microbes that have the potential to cause disease in animals or humans.</p>
<p>Whilst the use of high-throughput sequencing has been increasingly adapted by researchers worldwide, much less attention is given to its caveats and limitations. Methodological challenges of NGS include; (i) sequencing depth and short amplicon sequencing <span class="citation">(<a href="references.html#ref-gihringMassivelyParallelRRNA2012" role="doc-biblioref">Gihring, Green, and Schadt 2012</a>; <a href="references.html#ref-houImpactNextgenerationSequencing2013" role="doc-biblioref">Hou et al. 2013</a>; <a href="references.html#ref-simsSequencingDepthCoverage2014" role="doc-biblioref">Sims et al. 2014</a>)</span>; (ii) sequencing artefacts (errors and chimeric sequences) <span class="citation">(<a href="references.html#ref-kuninWrinklesRareBiosphere2010" role="doc-biblioref">Kunin et al. 2010</a>; <a href="references.html#ref-haasChimeric16SRRNA2011" role="doc-biblioref">Haas et al. 2011</a>)</span>; and (iii) PCR amplification bias through the effect of 16S copy number <span class="citation">(<a href="references.html#ref-ahnEffectsPCRCycle2012" role="doc-biblioref">Ahn et al. 2012</a>)</span> and annealing temperature <span class="citation">(<a href="references.html#ref-suzukiBiasCausedTemplate1996" role="doc-biblioref">Suzuki and Giovannoni 1996</a>)</span>.
To overcome some of the limitations associated with high-throughput sequencing a number of bioinformatic steps are introduced to provide quality control and assist in the interpretation of the large amount of data produced.
The steps can include primer trimming, removal of low quality reads, chimera detection (and removal), removal of low abundance reads and denoising <span class="citation">(<a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>; <a href="references.html#ref-kuninWrinklesRareBiosphere2010" role="doc-biblioref">Kunin et al. 2010</a>; <a href="references.html#ref-haasChimeric16SRRNA2011" role="doc-biblioref">Haas et al. 2011</a>; <a href="references.html#ref-edgarUNOISE2ImprovedErrorcorrection2016" role="doc-biblioref">Robert C. Edgar 2016</a>)</span>.
There a number of pipelines and programs that have been developed to analyze 16S metabarcoding data, however they largely rely on similar underlying algorithms.
Widely used programs include mothur <span class="citation">(<a href="references.html#ref-schlossIntroducingMothurOpensource2009" role="doc-biblioref">Schloss et al. 2009</a>)</span>, Quantitative Insights into Microbial Ecology (QIIME) <span class="citation">(<a href="references.html#ref-caporasoQIIMEAllowsAnalysis2010" role="doc-biblioref">Caporaso et al. 2010</a>)</span>, dada2 <span class="citation">(<a href="references.html#ref-callahanDADA2HighresolutionSample2016" role="doc-biblioref">Callahan et al. 2016</a>)</span>, USEARCH <span class="citation">(<a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>)</span> and vsearch <span class="citation">(<a href="references.html#ref-rognesVSEARCHVersatileOpen2016" role="doc-biblioref">Rognes et al. 2016</a>)</span>.
All programs listed are free, with the exception of 64-bit version of USEARCH (32-bit version is freely available).
Generally, comparisons between these, and other pipelines, conclude that they remain more-or-less comparable – with the emphasis on customising the parameters to best suit each unique dataset <span class="citation">(<a href="references.html#ref-nilakantaReviewSoftwareAnalyzing2014" role="doc-biblioref">Nilakanta et al. 2014</a>; <a href="references.html#ref-plummerComparisonThreeBioinformatics2015" role="doc-biblioref">Plummer and Twin 2015</a>; <a href="references.html#ref-forsterComparisonThreeClustering2016" role="doc-biblioref">Forster et al. 2016</a>)</span>.</p>
<p>Large curated reference 16S datasets include GreenGenes <span class="citation">(<a href="references.html#ref-desantisGreengenesChimerachecked16S2006" role="doc-biblioref">DeSantis et al. 2006</a>)</span>, the Ribosomal Database project <span class="citation">(<a href="references.html#ref-coleRibosomalDatabaseProject2009" role="doc-biblioref">Cole et al. 2009</a>)</span>, SILVA <span class="citation">(<a href="references.html#ref-pruesseSILVAComprehensiveOnline2007" role="doc-biblioref">Pruesse et al. 2007</a>)</span> and the EZ-Taxon <span class="citation">(<a href="references.html#ref-chunEzTaxonWebbasedTool2007" role="doc-biblioref">Chun et al. 2007</a>)</span>.
These datasets provide an additional point of stability between studies and generate more comparable results.
Bioinformatic pipelines and reference databases present an additional source of variation in analysis of NGS data, whereby a ‘one-size-fits-all’ approach is not appropriate.
The development of freely available applications and pipelines has provided a fundamental basis for 16S bioinformatic analyses <span class="citation">(<a href="references.html#ref-schlossIntroducingMothurOpensource2009" role="doc-biblioref">Schloss et al. 2009</a>; <a href="references.html#ref-caporasoQIIMEAllowsAnalysis2010" role="doc-biblioref">Caporaso et al. 2010</a>; <a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>)</span>; however care must still be taken, particularly when forming comparisons between data sets analysed with different pipelines.</p>
<p>Estimates of abundance in microbiology are widely used to describe microbial community composition and diversity.
The genomic copy number of the 16S gene varies considerably, from 1- 15 copies in some bacteria <span class="citation">(<a href="references.html#ref-loucaCorrecting16SRRNA2018" role="doc-biblioref">Louca, Doebeli, and Parfrey 2018</a>)</span>.
Therefore, the variation in abundance of 16S genes is due to both actual relative abundance differences in samples and variation in genomic 16S copy number among bacteria present.
In part this can be overcome through ecological models, such as an assessment of beta-diversity, which can be divided into two components; (i) turnover: difference between communities based on species presence/absence; and (ii) nestedness: differences in the abundance of species composition between communities <span class="citation">(<a href="references.html#ref-baselgaPartitioningTurnoverNestedness2010" role="doc-biblioref">Andres Baselga 2010</a>; <a href="references.html#ref-baselgaBetapartPartioningBeta2017" role="doc-biblioref">A. Baselga et al. 2017</a>)</span>.</p>
</div>
<div id="one-health-wildlife-surveillance" class="section level2" number="1.4">
<h2>
<span class="header-section-number">1.4</span> One Health &amp; Wildlife Surveillance<a class="anchor" aria-label="anchor" href="#one-health-wildlife-surveillance"><i class="fas fa-link"></i></a>
</h2>
<p>The final section of the review will provide some context to the One Health framework.
This is essential to understanding the rational for searching in wildlife for the causative agent(s) of human tick-borne disease.
The complex nature of tick-borne diseases means that they require a number of factors to overlap spatially and temporally.
Finding the pathogenic microbe in humans is only one part of the puzzle, and often the microbe(s) are more readily identified in the tick vector or wildlife reservoir host.
This section will introduce the concept of One Health, it will then briefly review the history of Lyme borreliosis in the United States and provide some insights into the potential cycle of similar tick-borne diseases in an Australian context.</p>
<div id="the-one-health-concept" class="section level3" number="1.4.1">
<h3>
<span class="header-section-number">1.4.1</span> The One Health Concept<a class="anchor" aria-label="anchor" href="#the-one-health-concept"><i class="fas fa-link"></i></a>
</h3>
<p>The One Health concept is based around a seamless interaction between veterinary and human medicine, bringing together clinicians, researchers, agencies and governments.
The modern One Health trend usually focuses on zoonotic pathogens emerging from wildlife and production animal species <span class="citation">(<a href="references.html#ref-dayOneHealthImportance2011" role="doc-biblioref">Day 2011</a>)</span>.
Understanding of the role wildlife reservoirs have in the maintenance of pathogenic hosts has significantly shifted our view on human health.
The use of wildlife surveillance as a tool for the detection of emerging zoonotic infectious disease is well established in some contexts.
For example, the sylvatic life cycle of <em>Trypanosoma cruzi</em> in South America <span class="citation">(<a href="references.html#ref-denoyaEcologicalOverviewFactors2015" role="doc-biblioref">Noya and Gonzalez 2015</a>)</span>, Ebola virus in Africa <span class="citation">(<a href="references.html#ref-osterholmTransmissionEbolaViruses2015" role="doc-biblioref">Osterholm et al. 2015</a>)</span>, Australian bat lyssavirus <span class="citation">(<a href="references.html#ref-mayIdentificationFocusAreas2020" role="doc-biblioref">May et al. 2020</a>)</span>.
Research has shown that compared to human infection, these zoonotic microbes can be readily identified in their respective wildlife reservoirs.
The mosquito-borne West Nile virus life cycle has been well documented and the sensitivity of detection is well understood.
It shows that surveillance of sentinel hosts and vectors provide a quicker and more sensitive method detection method than waiting for signs of infected people <span class="citation">(<a href="references.html#ref-lemonGlobalInfectiousDisease2007" role="doc-biblioref">Lemon et al. 2007</a>)</span>.</p>
<p>Tick-borne diseases present a particularly challenging system to study, with a variety of complex factors influencing the epidemiology.
The presence of tick-borne diseases involves the interplay between the pathogen, host, vector and the environment; with major contributors including climatic change, globalization, increased global travel and trade, urbanization and drug resistance <span class="citation">(<a href="references.html#ref-dantas-torresClimateChangeBiodiversity2015" role="doc-biblioref">F. Dantas-Torres 2015</a>; <a href="references.html#ref-kulesChallengesAdvancesDiagnosis2017" role="doc-biblioref">Kules et al. 2017</a>; <a href="references.html#ref-gilbertImpactsClimateChange2021" role="doc-biblioref">L. Gilbert 2021</a>)</span>.
Examples of established monitoring programs include; tick surveillance in Great Britian <span class="citation">(<a href="references.html#ref-jamesonTickSurveillanceGreat2011" role="doc-biblioref">Jameson and Medlock 2011</a>; <a href="references.html#ref-cullSurveillanceBritishTicks2018" role="doc-biblioref">Cull et al. 2018</a>)</span> and use of hunter-killed white-tailed deer (<em>Odocoileus virginianus</em>) in Canada <span class="citation">(<a href="references.html#ref-bouchardHarvestedWhitetailedDeer2013" role="doc-biblioref">Bouchard et al. 2013</a>)</span>.</p>
<p>Currently in Australia there are a number of arbovrius monitoring programs that are operated by governments or partner organisations.
For example sentinel ruminant herds are used to monitor the distribution of arboviruses and their vectors in Australia such as Bluetongue, Akabane and Bovine ephemeral fever (BEF) viruses <span class="citation">(<a href="references.html#ref-nationalarbovirusmonitoringprogramNationalArbovirusMonitoring2019" role="doc-biblioref">Program 2019</a>)</span>.
In Western Australia the Sentinel Chicken Surveillance Program is used as an early warning system to detect increases in flavivirus activity, mainly focused on the mosquito-borne Murray Valley encephalitis (MVE) and Kunjin (KUN) viruses <span class="citation">(<a href="references.html#ref-departmentofhealthMedicalEntomologyAnnual2020" role="doc-biblioref">Health 2020</a>)</span>.
However there is currently no state or national level surveillance of tick-borne pathogens in Australia.</p>
</div>
<div id="lyme-borreliosis" class="section level3" number="1.4.2">
<h3>
<span class="header-section-number">1.4.2</span> Lyme Borreliosis<a class="anchor" aria-label="anchor" href="#lyme-borreliosis"><i class="fas fa-link"></i></a>
</h3>
<p>Lyme arthritis was first described in the scientific literature in 1977 from a cluster of cases in eastern Connecticut <span class="citation">(<a href="references.html#ref-steereLymeArthritisEpidemic1977" role="doc-biblioref">Steere et al. 1977</a>)</span>.
The illness was characterized by recurrent attack of asymmetric swelling and joint paint.
The authors described a geographical clustering of patients in sparsely settled areas with woody surrounds and peak disease occurrence in the summer months.
As a result doctors concluded the disease was best explained by transmission of an infectious agent by an arthropod vector.
In 1982 a study demonstrated that <em>Ixodes dammini</em> (now recognised as <em>Ixodes scapularis</em> <span class="citation">(<a href="references.html#ref-sandersIxodesDamminiJunior1998" role="doc-biblioref">Sanders 1998</a>)</span>), was responsible for the transmission of a treponema-like spirochete causing long-lasting cutaneous lesions on rabbits 10-12 weeks after tick attachment <span class="citation">(<a href="references.html#ref-burgdorferLymeDiseaseTickborne1982" role="doc-biblioref">W. Burgdorfer et al. 1982</a>)</span>.
It was the first of its study to investigate to the role of vectors in the transmission of Lyme disease in North America.
A formal description of <em>Borrelia burgdorferi</em> followed <span class="citation">(<a href="references.html#ref-johnsonBorreliaBurgdorferiSp1984" role="doc-biblioref">Johnson et al. 1984</a>)</span> and in 1997 <span class="citation">(<a href="references.html#ref-fraserGenomicSequenceLyme1997" role="doc-biblioref">Fraser et al. 1997</a>)</span> was the third bacteria to have its genome sequenced.</p>
<div id="the-ecology-of-borrelia-in-north-america" class="section level4" number="1.4.2.1">
<h4>
<span class="header-section-number">1.4.2.1</span> The ecology of <em>Borrelia</em> in North America<a class="anchor" aria-label="anchor" href="#the-ecology-of-borrelia-in-north-america"><i class="fas fa-link"></i></a>
</h4>
<p>In most areas of North America the primary vector of <em>B. burgdorferi</em> s. l. is <em>Ixodes scapularis</em>, and major reservoir hosts are well known to include white-footed mice and white-tailed deer <span class="citation">(<a href="references.html#ref-halseyRoleIxodesScapularis2018" role="doc-biblioref">Halsey, Allan, and Miller 2018</a>)</span>.
However pine squirrels and deer mice have been identified as host sentinel hosts for <em>Borrelia hermsii</em> in western North America <span class="citation">(<a href="references.html#ref-cadenasIdentificationHostBloodmeal2007" role="doc-biblioref">Cadenas et al. 2007</a>)</span>.
Studies have identified significant difference in species diversity among sites, noting that islands have fewer species than mainland.
Body mass has also been shown to have an effect on transmission dynamics.
A positive correlation between shown between host body mass and tick burdens for the different stages of <em>Ix. ricinus</em> <span class="citation">(<a href="references.html#ref-hofmeesterFewVertebrateSpecies2016" role="doc-biblioref">Hofmeester et al. 2016</a>)</span>.
Nymphal burdens was positively correlated with increased infection of <em>B. burgdorgferi</em>.
The study also demonstrated that only a few hosts feed the majority of <em>Ix. ricinus</em> and they usually include the most widespread species in the environment.</p>
</div>
</div>
<div id="an-australian-framework-to-an-endemic-zoonotic-tick-borne-disease" class="section level3" number="1.4.3">
<h3>
<span class="header-section-number">1.4.3</span> An Australian Framework to an endemic zoonotic tick-borne disease<a class="anchor" aria-label="anchor" href="#an-australian-framework-to-an-endemic-zoonotic-tick-borne-disease"><i class="fas fa-link"></i></a>
</h3>
<p>The search for a novel zoonotic tick-borne disease in Australia faces a number of challenges.
In order to robustly investigate potential novel pathogens it is important that detection methods are sufficiently broad, to limit biases.
Using 16S rRNA next-generation sequencing it has revealed that Australian ticks have a unique suite of bacteria <span class="citation">(<a href="references.html#ref-goftonBacterialProfilingReveals2015" role="doc-biblioref">Gofton, Doggett, et al. 2015</a>; <a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">Gofton, Oskam, et al. 2015</a>)</span>.
In particular a number of bacterial organisms, that are similar to, yet unique from global TBPs have since been described from Australian ticks <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">Gofton et al. 2016</a>, <a href="references.html#ref-goftonDetectionPhylogeneticCharacterisation2017" role="doc-biblioref">2017</a>, <a href="references.html#ref-goftonNovelEhrlichiaSpecies2018" role="doc-biblioref">2018</a>; <a href="references.html#ref-lohNovelBorreliaSpecies2016" role="doc-biblioref">S. M. Loh et al. 2016</a>)</span>.<br>
Even in the case of rickettsial diseases, which are recognised human tick-borne pathogens in Australia, the distribution and public health impact is poorly defined <span class="citation">(<a href="references.html#ref-stewartRickettsiaAustralisQueensland2017" role="doc-biblioref">A. Stewart et al. 2017</a>)</span>.
There is currently no compulsory reporting of rickettsial infections in Australia.
Additionally acute infections can be misdiagnosed or treated ineffectively.
The genetic similarity of rickettsial organisms also makes accurate diagnosis difficult.
Currently serological tests remain the ‘gold standard’ for the diagnosis of a rickettsial infection in Australia, however serological results often result in cross-reactivity with both other rickettsial species and nonrickettsial infectious diseases.
In the search for a human disease(s) related to tick bites, records of human-tick associations are critical.
Much of the research to date has centered around the paralysis tick (<em>Ix. holocyclus</em>), which is present along the east coast of Australia.
However increasingly reports of tick-human associations have expanded geographically.
A tissue punch biopsy or swab either at site of tick attachment or site of eschar has been shown to have a higher chance of identifying potential tick-borne pathogens than blood samples <span class="citation">(<a href="references.html#ref-portilloGuidelinesDetectionRickettsia2017" role="doc-biblioref">Portillo et al. 2017</a>)</span>.</p>
</div>
</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="preface-material.html">Preface material</a></div>
<div class="next"><a href="thesis-aims.html">Thesis aims</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#litrev"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="nav-link" href="#preface">Preface</a></li>
<li>
<a class="nav-link" href="#ticks"><span class="header-section-number">1.1</span> Ticks</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#tick-systematics"><span class="header-section-number">1.1.1</span> Tick systematics</a></li>
<li><a class="nav-link" href="#tick-biology-and-ecology"><span class="header-section-number">1.1.2</span> Tick biology and ecology</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#tick-borne-diseases"><span class="header-section-number">1.2</span> Tick-borne diseases</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#worldwide-summary"><span class="header-section-number">1.2.1</span> Worldwide summary</a></li>
<li><a class="nav-link" href="#infectious-human-tick-borne-pathogens-in-australia"><span class="header-section-number">1.2.2</span> Infectious human tick-borne pathogens in Australia</a></li>
</ul>
</li>
<li><a class="nav-link" href="#molecular-tools"><span class="header-section-number">1.3</span> Molecular tools</a></li>
<li>
<a class="nav-link" href="#one-health-wildlife-surveillance"><span class="header-section-number">1.4</span> One Health &amp; Wildlife Surveillance</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#the-one-health-concept"><span class="header-section-number">1.4.1</span> The One Health Concept</a></li>
<li><a class="nav-link" href="#lyme-borreliosis"><span class="header-section-number">1.4.2</span> Lyme Borreliosis</a></li>
<li><a class="nav-link" href="#an-australian-framework-to-an-endemic-zoonotic-tick-borne-disease"><span class="header-section-number">1.4.3</span> An Australian Framework to an endemic zoonotic tick-borne disease</a></li>
</ul>
</li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/ulyngs/oxforddown/blob/master/01-literature-review.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/ulyngs/oxforddown/edit/master/01-literature-review.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></strong>" was written by Siobhon Lee Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
