<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>A Supplementary material | Ecology of Ticks and Microbes in Australia Wildlife</title>
<meta name="author" content="Siobhon Lee Egan">
<meta name="generator" content="bookdown 0.24 with bs4_book()">
<meta property="og:title" content="A Supplementary material | Ecology of Ticks and Microbes in Australia Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="A Supplementary material | Ecology of Ticks and Microbes in Australia Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/header-attrs-2.11.22/header-attrs.js"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="A.1 Chapter 2 Supplementary material for Chapter 2 on Australian Ticks.  A.1.1 Molecular methods for analysis of ticks  A.1.1.1 DNA extraction Once ticks were identified they were...">
<meta property="og:description" content="A.1 Chapter 2 Supplementary material for Chapter 2 on Australian Ticks.  A.1.1 Molecular methods for analysis of ticks  A.1.1.1 DNA extraction Once ticks were identified they were...">
<meta name="twitter:description" content="A.1 Chapter 2 Supplementary material for Chapter 2 on Australian Ticks.  A.1.1 Molecular methods for analysis of ticks  A.1.1.1 DNA extraction Once ticks were identified they were...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="active" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/ulyngs/oxforddown">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="supplementary-material" class="section level1" number="8">
<h1>
<span class="header-section-number">A</span> Supplementary material<a class="anchor" aria-label="anchor" href="#supplementary-material"><i class="fas fa-link"></i></a>
</h1>
<div id="ch2supp" class="section level2" number="8.1">
<h2>
<span class="header-section-number">A.1</span> Chapter 2<a class="anchor" aria-label="anchor" href="#ch2supp"><i class="fas fa-link"></i></a>
</h2>
<p>Supplementary material for Chapter <a href="austicks.html#austicks">2</a> on Australian Ticks.</p>
<div id="molecular-methods-for-analysis-of-ticks" class="section level3" number="8.1.1">
<h3>
<span class="header-section-number">A.1.1</span> Molecular methods for analysis of ticks<a class="anchor" aria-label="anchor" href="#molecular-methods-for-analysis-of-ticks"><i class="fas fa-link"></i></a>
</h3>
<div id="dna-extraction-1" class="section level4" number="8.1.1.1">
<h4>
<span class="header-section-number">A.1.1.1</span> DNA extraction<a class="anchor" aria-label="anchor" href="#dna-extraction-1"><i class="fas fa-link"></i></a>
</h4>
<p>Once ticks were identified they were surface-sterilised by washing in 10 % hypochlorite solution, rinsed in 70 % ethanol and DNA-free PBS, and then air-dried. Genomic DNA was extracted using the DNeasy Blood and Tissue kit (QIAGEN, Germany) for adults or the QIAamp DNA Mini Kit (QIAGEN, Germany) for nymphs and larvae.
Due to the large number of larvae collected from some animals and the expected low DNA yeild, in comes cases up to 5 larvae were pooled based on host, species and engorgement status.
Ticks were placed in a 2 mL safe lock Eppendorf tube with a 5mm steel bead, frozen in liquid nitrogen for 1 min and homogenised by shaking at 40 Hz in a Tissue Lyser LT (QIAGEN, Germany). Final elution of DNA was adjusted according to tick size, between 30-150 <span class="math inline">\(\mu L\)</span> of AE buffer was added to the silcon membrane.
Extraction controls (EXBs) consisting of 200 <span class="math inline">\(\mu L\)</span> sterile DNA free PBS, were included randomly in each extraction batch.</p>
<p>Prior to DNA extraction ticks were washed in 10% bleach solution, followed by 80% ethanol and rinsed in sterile-DNA free PBS. For museum specimens ticks were extracted in a separate dedicated laboratory, away from modern DNA. Ticks were soaked in sterile 1X PBS solution for 24-48 hours to remove ethanol from specimens. For museum and reference samples non-destructive sampling was performed, where ticks were punctured using a sterile pin and soaked in proteinase K and lysis buffer overnight (16-18 hours) at 56<span class="math inline">\(^\circ C\)</span>, 700 rpm.
DNA was then extracted using QIAamp DNA mini kit (Qiagen, Germany) following manusctures recommendations, except for final elution which was reduced to 30 <span class="math inline">\(\mu L\)</span> of buffer AE and incubated for 30 mins, before final centrifugation.
In the case of modern specimens adults ticks were bisected and half was placed in proteinase K and lysis buffer overnight (16-18 hours) at 56<span class="math inline">\(^\circ C\)</span>, 700 rpm, the other half was retained as a voucher and placed back into 90% ethanol.
The supernatant was then used for subsequent extraction using DNeasy Blood and Tissue kit (Qiagen, Germany) following manufactures recommendations. Strict laboratory hygiene was maintained throughout, equipment and surafces were cleaned with DNAaway in between each sample.</p>
</div>
<div id="targeted-assays-and-sanger-sequencing" class="section level4" number="8.1.1.2">
<h4>
<span class="header-section-number">A.1.1.2</span> Targeted assays and Sanger sequencing<a class="anchor" aria-label="anchor" href="#targeted-assays-and-sanger-sequencing"><i class="fas fa-link"></i></a>
</h4>
<p>A multilocus sequence typing (MSLT) approach was used to obtain sequence information from 3 mitochondrial genetic markers. These were; cytochrome c oxidase subunit I (COX1), 12S rDNA, 16S rDNA (primers outlined in table <a href="supplementary-material.html#tab:T2primers">A.1</a>.
All reactions were carried out in 25 <span class="math inline">\(\mu L\)</span> volumes, each containing; 1X buffer (KAPA Biosystems, South Africa), 2.5 mM <span class="math inline">\(MgCl_{2}\)</span>, 0.4<span class="math inline">\(\mu M\)</span> of primer, 200 mM of each dNTP, 1.0 U of Taq (KAPA Biosystems, South Africa), and between 1<span class="math inline">\(\mu L\)</span> (modern specimens) 5 <span class="math inline">\(\mu L\)</span> (museum specimens) of gDNA was added. MLST assays on museum DNA were performed in a separate dedicated facility away from modern DNA.</p>
<p>The COX1 locus was amplified using a hemi-nested assay with primers HCO2064/HCO1215 in the primary assay and HCO2064/HCO1240 from Song et al. <span class="citation">(<a href="references.html#ref-songPhylogeneticPhylogeographicRelationships2011" role="doc-biblioref">2011</a>)</span>.
Despite repeated attempts for many specimens of <em>Amblyomma</em> amplification using a single round PCR was not successful. Previous research has demonstrated the primer pair HCO2064/HCO1240 is successful at amplifying COX1 gene from tick specimens of the genera <em>Ixodes</em>, <em>Haemaphysalis</em> and <em>Rhipicephalus</em> <span class="citation">(<a href="references.html#ref-evansAutomaticBarcodeGap2019" role="doc-biblioref">Evans et al. 2019</a>)</span>.
However this assay did not achieve reliable repeatable results for <em>Am. triguttatum</em>, therefore a hemi-nested PCR assay was used.
Reactions were carried out in 25<span class="math inline">\(\mu L\)</span> reactions, each containing; 1X buffer (KAPA Biosystems, South Africa), 2.5 mM MgCl<sub>2</sub>, 0.4<span class="math inline">\(\mu M\)</span> of each primer (HCO2064/HCO1215 and HCO2064/HCO1240), 0.25 mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1<span class="math inline">\(\mu L\)</span> of either tick DNA or primary product. Thermal cycling conditions for both assays were as followed: 95<span class="math inline">\(^\circ C\)</span> for 2 mins, followed by 35 (modern specimens) or 50 (museum specimens) cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 48<span class="math inline">\(^\circ C\)</span> for 30 secs, 72<span class="math inline">\(^\circ C\)</span> for 50 secs, and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 2 mins.</p>
<p>A 450 bp product of the 16S rDNA gene was amplified <span class="citation">(<a href="references.html#ref-lvDevelopmentDNABarcoding2014" role="doc-biblioref">Lv, Wu, Zhang, Zhang, et al. 2014</a>)</span>. Reactions were carried out in 25<span class="math inline">\(\mu L\)</span> reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5mM MgCl<sub>2</sub>, 0.4<span class="math inline">\(\mu M\)</span> of each primer (16S-F/16S-R1), 0.25 mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1<span class="math inline">\(\mu L\)</span> of tick gDNA. A touchdown approach was used for the 16S assays were as followed; 95<span class="math inline">\(^\circ C\)</span> for 5 mins, followed by 5 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 49<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; 5 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 47<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; 5 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 45<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; 25 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 43<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; and a final extension of 68<span class="math inline">\(^\circ C\)</span> for 5 mins.</p>
<p>A 370bp product of the 12S rDNA gene was amplified. Reactions were carried out in 25<span class="math inline">\(\mu L\)</span> reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5mM MgCl<sub>2</sub>, 0.4<span class="math inline">\(\mu M\)</span> of each primer (T1B/T2A), 0.25mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1<span class="math inline">\(\mu L\)</span> of tick gDNA.Thermal cycling conditions were as followed; 95<span class="math inline">\(^\circ C\)</span> for 5 mins, followed by 5 cycles of 95<span class="math inline">\(^\circ C\)</span> for 15 secs, 51<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; 25 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 53<span class="math inline">\(^\circ C\)</span> for 30 secs, 72<span class="math inline">\(^\circ C\)</span> for 1 min; and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 5 mins.</p>
<p>Amplified products were visualised on a 1% agarose gel stained with SYBR Safe (Invitrogen, USA) and bands of the correct size were excised using a sterile scalpel blade and purified using previously described methods <span class="citation">(<a href="references.html#ref-yangSpecificQuantitativeDetection2013" role="doc-biblioref">R. Yang et al. 2013</a>)</span>. Sanger sequencing was carried out in both directions and sequencing was performed at the Australian Genome Research Facility (Perth, Western Australia) on an Applied Biosystems 3730xl DNA Analyzer using BigDye(TM) Terminator v3.1 Cycle Sequencing Kit.</p>

<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T2primers">Table A.1: </span>Primers used to amplify mitochondrial genes of ticks (Ixodida). The following loci were targeted; cytochrome c oxidase subunit 1 (COX1), 12S rDNA, 16S rDNA. References <strong>1.</strong> Song et al. <span class="citation">(<a href="references.html#ref-songPhylogeneticPhylogeographicRelationships2011" role="doc-biblioref">2011</a>)</span>, <strong>2.</strong> Beati et al. <span class="citation">(<a href="references.html#ref-beatiAnalysisSystematicRelationships2001" role="doc-biblioref">2001</a>)</span>, <strong>3.</strong> Lv et al. <span class="citation">(<a href="references.html#ref-lvDevelopmentDNABarcoding2014" role="doc-biblioref">2014</a>)</span>.
</caption>
<thead><tr>
<th style="text-align:left;">
Primer
</th>
<th style="text-align:left;">
Sequence (5’-3’)
</th>
<th style="text-align:left;">
Reference
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;">
Cytochrome c oxidase subunit 1
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
HCO2064
</td>
<td style="text-align:left;">
GGTGGGCTCATACAATAAATCC
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
HCO1215
</td>
<td style="text-align:left;">
GCCATTTTACCGCGATGA
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
HCO1240
</td>
<td style="text-align:left;">
CCACAAATCATAAAGACATTGG
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
12S rDNA
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
T2A
</td>
<td style="text-align:left;">
AAACTAGGATTAGATACCCT
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
T1B
</td>
<td style="text-align:left;">
AATGAGAGCGACGGGCGATGT
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
16S rDNA
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
16S-F
</td>
<td style="text-align:left;">
TTAAATTGCTGTRGTATT
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
16S-R1
</td>
<td style="text-align:left;">
CCGGTCTGAACTCASAWC
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Illumina MiSeq adapters
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Forward overhand
</td>
<td style="text-align:left;">
TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG
</td>
<td style="text-align:left;">
Illumina
</td>
</tr>
<tr>
<td style="text-align:left;">
Reverse overhand
</td>
<td style="text-align:left;">
GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
</div>
<div id="library-prepation-and-amplicon-next-generation-sequencing" class="section level4" number="8.1.1.3">
<h4>
<span class="header-section-number">A.1.1.3</span> Library prepation and amplicon next-generation sequencing<a class="anchor" aria-label="anchor" href="#library-prepation-and-amplicon-next-generation-sequencing"><i class="fas fa-link"></i></a>
</h4>
<p>Based on results from the MLST approached used above a a high-throughput metabarcoding approach was developed to identify tick species.
In addition the short fragment size (~370 bp) and ability to deliminate between species made the 12rDNA loci most suitable for transferring onto the Illumina MiSeq.
The pan-Ixodida primers T1B and T2A <span class="citation">(<a href="references.html#ref-beatiAnalysisSystematicRelationships2001" role="doc-biblioref">Beati and Keirans 2001</a>)</span> were ordered with Illumina MiSeq adapter overhangs.
Amplicon PCRs were carried out in 25 <span class="math inline">\(\mu L\)</span> reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu M\)</span> of each primer, 0.25 mM of each dNTP, 1 U of Taq (KAPA Biosystems, South Africa) and 1 <span class="math inline">\(\mu L\)</span> of tick gDNA.Thermal cycling conditions were as followed; 95<span class="math inline">\(^\circ C\)</span> for 5 mins, followed by 5 cycles of 95<span class="math inline">\(^\circ C\)</span> for 15 secs, 51<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; 25 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 53<span class="math inline">\(^\circ C\)</span> for 30 secs, 72<span class="math inline">\(^\circ C\)</span> for 1 min; and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 5 mins.</p>
<p>Amplicon PCR products were then indexed using the Nextera XT DNA library preparation kit in 25 <span class="math inline">\(\mu L\)</span> volumes following manufacturer’s recommendations.
All PCRs included no template controls (NTC) during each reaction set up and PCRs were performed under strict laboratory conditions. Amplicons were then dual-indexed using the Nextera XT index kit.
Reactions were performed in 25 <span class="math inline">\(\mu L\)</span> volumes using following manufacturers recommendations.
Libraries were purified with Axygen PCR clean up beads, pooled in equimolar amounts and quantified using Qubit High Sensitivity dsDNA assay kit (Thermo Fisher Scientific, Waltham, MA, USA) and amplicons were pooled in equimolar amounts.
Libraries were shipped to Australian Genomic Research Facility (Melbourne, Australia) for final QC and sequenced on the Illumina MiSeq using v2 chemistry (2 x 250 paired end).</p>
</div>
<div id="bioinformatics-1" class="section level4" number="8.1.1.4">
<h4>
<span class="header-section-number">A.1.1.4</span> Bioinformatics<a class="anchor" aria-label="anchor" href="#bioinformatics-1"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Multilocus sequence typing data produced by Sanger Sequencing</strong></p>
<p>Sequences were imported into Geneious 10.2.6 (<a href="https://www.geneious.com" class="uri">https://www.geneious.com</a>) for quality inspection.
Forward and reverse sequenced were aligned using the default parameters in Geneious and primers were trimmed.
Reference sequences were retrieved from GenBank <span class="citation">(<a href="references.html#ref-bensonGenBank2017" role="doc-biblioref">Benson et al. 2017</a>)</span> and aligned with sequences obtained in the present study using MUSCLE <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span>.</p>
<p><strong>High-throughout sequence data produced by Illumina MiSeq</strong></p>
<p>Illumina MiSeq fastq data was analysed using bioinformatic pipeline with the program USEARCH v11 <span class="citation">(<a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>)</span>.
Paired end reads were merged and sequences matching forward and reverse primers were retrieved (max number of mismatches = 2).
Sequences were then quality filtered and singletons were removed.
The unoise3 <span class="citation">(<a href="references.html#ref-edgarUNOISE2ImprovedErrorcorrection2016" role="doc-biblioref">Robert C. Edgar 2016</a>)</span> algorithm was used to perform denoising (error-correction) and generate zero-radius taxonomic units (zOTUs) equivalent to amplicon sequence variants (ASVs).
Resulting zOTUs were subject to BLAST analysis (BLASTN 2.11.0+ <span class="citation">(<a href="references.html#ref-zhangGreedyAlgorithmAligning2000" role="doc-biblioref">Zhang et al. 2000</a>; <a href="references.html#ref-morgulisDatabaseIndexingProduction2008" role="doc-biblioref">Morgulis et al. 2008</a>)</span>) against NCBI nucleotide collection (nt) database.
Taxonomic lineage for each top hit (based on e-value score) was retrieved from NCBI taxonomy database using TaxonKit <span class="citation">(<a href="references.html#ref-weissHostReproductiveCycle2021" role="doc-biblioref">Weiss et al. 2021</a>)</span>.
Final taxonomy assignment was then refined to lowest common ancestor based on BLAST results (percentage identity, e-value and query coverage) and sequence alignment method.</p>
<p>Sequences generated were aligned with reference sequences were retrieved from GenBank <span class="citation">(<a href="references.html#ref-bensonGenBank2017" role="doc-biblioref">Benson et al. 2017</a>)</span> using MAFFT <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span>.
Phylogenies were inferred using the maximum likelihood (ML) method.
The optimal evolutionary model was selected using ModelFinder <span class="citation">(<a href="references.html#ref-kalyaanamoorthyModelFinderFastModel2017" role="doc-biblioref">Kalyaanamoorthy et al. 2017</a>)</span> based on bayesian information criterion.
Phylogenetic analysis was performed in IQ-TREE v1.6.11 <span class="citation">(<a href="references.html#ref-nguyenIQTREEFastEffective2015" role="doc-biblioref">Nguyen et al. 2015</a>)</span> and bootstrap support was calculated using ultrafast (UFBoot2) method with 10,000 replicates <span class="citation">(<a href="references.html#ref-hoangUFBoot2ImprovingUltrafast2018" role="doc-biblioref">Hoang et al. 2018</a>)</span>.
Genetic sequence similarity was calculated using the Kimura Two-Parameter method <span class="citation">(<a href="references.html#ref-tamuraEstimationNumberNucleotide1993" role="doc-biblioref">Tamura and Nei 1993</a>)</span>.</p>
<div style="page-break-after: always;"></div>
<!--\begin{landscape}-->
<div class="figure">
<span style="display:block;" id="fig:FA21"></span>
<img src="figures/ms-figs-appendix/FigA-2.1.png" alt="Occurrence map of ticks (Acari: Ixodida) records sourced from the Atlas of Living Australia (www.ala.org.au), data has not been curated." width="95%"><p class="caption">
Figure A.1: Occurrence map of ticks (Acari: Ixodida) records sourced from the Atlas of Living Australia (www.ala.org.au), data has not been curated.
</p>
</div>
<!--\end{landscape}-->
<div style="page-break-after: always;"></div>
<div class="inline-table"><table class="table table table" style="font-size: 7.5px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T2MLSTseqs">Table A.2: </span>Genetic sequences generated from Australian ticks using a multi-locus sequence typing approach. Nucleotide data has been deposited in GenBank with accession numbers of specimens included in the table.
</caption>
<thead><tr>
<th style="text-align:left;">
Target gene
</th>
<th style="text-align:left;">
Organism
</th>
<th style="text-align:left;">
isolate
</th>
<th style="text-align:left;">
Host
</th>
<th style="text-align:left;">
Country
</th>
<th style="text-align:left;">
Note
</th>
<th style="text-align:left;">
Dev_stage
</th>
<th style="text-align:left;">
Genbank
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
ACF1
</td>
<td style="text-align:left;">
Short-beaked echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI1268
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840098
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
ACF3
</td>
<td style="text-align:left;">
Short-beaked echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI1268
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840099
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes ornithorhynchi
</td>
<td style="text-align:left;">
IOF3
</td>
<td style="text-align:left;">
Platypus (Ornithorhynchus anatinus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI419
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840100
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes ornithorhynchi
</td>
<td style="text-align:left;">
IOF4
</td>
<td style="text-align:left;">
Platypus (Ornithorhynchus anatinus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI419
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840101
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes ornithorhynchi
</td>
<td style="text-align:left;">
IOF5
</td>
<td style="text-align:left;">
Platypus (Ornithorhynchus anatinus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI419
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840102
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BAF5
</td>
<td style="text-align:left;">
Wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Victoria
</td>
<td style="text-align:left;">
PI1382
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840103
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BAF6
</td>
<td style="text-align:left;">
Wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Victoria
</td>
<td style="text-align:left;">
PI1382
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840104
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HHM1
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Northern Territory
</td>
<td style="text-align:left;">
PI797
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
OM840105
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
ITF6
</td>
<td style="text-align:left;">
Eastern barred bandicoot (Perameles gunnii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI321
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840106
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
ITF10
</td>
<td style="text-align:left;">
Eastern barred bandicoot (Perameles gunnii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI321
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840107
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
ITN6
</td>
<td style="text-align:left;">
Eastern barred bandicoot (Perameles gunnii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI321
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840108
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
B1
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
PI387
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840109
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
B2
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
PI1592
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840110
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B8
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
PI1132
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840111
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B16
</td>
<td style="text-align:left;">
Bandicoot sp.
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI319
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840112
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
B57
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
PI1129
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840113
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B62
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI1260
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840114
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B63
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI1260
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840115
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B64
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI1260
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840116
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
B77
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
PI1388
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840117
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B87
</td>
<td style="text-align:left;">
Eastern barred bandicoot (Perameles gunnii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI321
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840118
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B88
</td>
<td style="text-align:left;">
Eastern barred bandicoot (Perameles gunnii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI321
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840119
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
B89
</td>
<td style="text-align:left;">
Eastern barred bandicoot (Perameles gunnii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI321
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840120
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
B110
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
PI771
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840121
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD1
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2105
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840122
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD2
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2105
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840123
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD3
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2105
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840124
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD5
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2105
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840125
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD6
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2106
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840126
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD7
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2107
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840127
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD8
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2108
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840128
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD9
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2109
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840129
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD10
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2110
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840130
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes fecialis
</td>
<td style="text-align:left;">
TD12
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2112
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840131
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes fecialis
</td>
<td style="text-align:left;">
TD13
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2113
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
OM840132
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes fecialis
</td>
<td style="text-align:left;">
TD14
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2114
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840133
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
TD15
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
PI2115
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM840134
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT2F
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106717
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT3M
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
MN106719
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT4M
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
MN106718
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BA1F
</td>
<td style="text-align:left;">
Common wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Trafalgar, Victoria
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106720
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BA2F
</td>
<td style="text-align:left;">
Common wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Trafalgar, Victoria
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106721
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
HB2F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Stony Chute, New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106722
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
HB3N
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Stony Chute, New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
MN106723
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH1N
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Beerwah, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Nymph
</td>
<td style="text-align:left;">
MN106726
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH3M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Palmerston, Northern Territory
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
MN106724
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH4M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Palmerston, Northern Territory
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
MN106725
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes cornuatus
</td>
<td style="text-align:left;">
IC1F
</td>
<td style="text-align:left;">
Dog (Canis lupus familiaris)
</td>
<td style="text-align:left;">
Australia: Port Sorrell, Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106727
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes hirsti
</td>
<td style="text-align:left;">
IHi1F
</td>
<td style="text-align:left;">
Cat (Felis catus)
</td>
<td style="text-align:left;">
Australia: Port Sorrell, Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106728
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
Q92
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Springbrook, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106729
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
Q93
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Springbrook, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
MN106730
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
C4IT
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
MN106731
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
WL1
</td>
<td style="text-align:left;">
Possum (Pseudocheirus peregrinus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791407
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
WL2
</td>
<td style="text-align:left;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791408
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
WL11
</td>
<td style="text-align:left;">
Kangaroo (Macropus giganteus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791409
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL18
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791410
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL19
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791411
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL20
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791412
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL30
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791413
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL31
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791414
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL32
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791415
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL33
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791416
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL34
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791417
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL35
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791418
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL36
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791419
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL37
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791420
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL38
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791421
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL39
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791422
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL40
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791423
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL41
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791424
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL42
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791425
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL44
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791426
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL45
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791427
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL46
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791428
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL51
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791429
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL52
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791430
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL53
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791431
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL54
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791432
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL55
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791433
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL56
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791434
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL57
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791435
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL58
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791436
</td>
</tr>
<tr>
<td style="text-align:left;">
COX1
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL60
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM791437
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT1F
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830384
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT2F
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830385
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT3M
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830386
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT4M
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830387
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT5N
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830388
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BA1F
</td>
<td style="text-align:left;">
Common wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Trafalgar, Victoria
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830389
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BA2F
</td>
<td style="text-align:left;">
Common wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Trafalgar, Victoria
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830390
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
HB2F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Stony Chute, New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830391
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
HB3F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Stony Chute, New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830392
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH1M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Beerwah, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830393
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH3M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Palmerston, Northern Territory
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830394
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH4M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Palmerston, Northern Territory
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830395
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes cornuatus
</td>
<td style="text-align:left;">
IC1F
</td>
<td style="text-align:left;">
Dog (Canis lupus familiaris)
</td>
<td style="text-align:left;">
Australia: Port Sorrell, Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830396
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes hirsti
</td>
<td style="text-align:left;">
IHi1F
</td>
<td style="text-align:left;">
Cat (Felis catus)
</td>
<td style="text-align:left;">
Australia: Port Sorrell, Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830397
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
IH1F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830398
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
Q92
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Springbrook, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830399
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
Q93
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Springbrook, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830400
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
IT3
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830401
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
WL1
</td>
<td style="text-align:left;">
Possum (Pseudocheirus peregrinus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830402
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
WL2
</td>
<td style="text-align:left;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830403
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
WL11
</td>
<td style="text-align:left;">
Kangaroo (Macropus giganteus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830404
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL19
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830405
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL20
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830406
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL30
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830407
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL31
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830408
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL32
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830409
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL33
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830410
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL34
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830411
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL35
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830412
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL36
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830413
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL37
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830414
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL38
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830415
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL39
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830416
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL41
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830417
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL42
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830418
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL44
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830419
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL45
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830420
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL46
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830421
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL51
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830422
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL52
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830423
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL53
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830424
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL54
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830425
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL55
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830426
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL56
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830427
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL57
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830428
</td>
</tr>
<tr>
<td style="text-align:left;">
16S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL58
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830429
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT1F
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830716
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT2F
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830717
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT3M
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830718
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT4M
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830719
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
AT5N
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Yanchep, Western Australia
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830720
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BA1F
</td>
<td style="text-align:left;">
Common wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Trafalgar, Victoria
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830721
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
<td style="text-align:left;">
BA2F
</td>
<td style="text-align:left;">
Common wombat (Vombatus ursinus)
</td>
<td style="text-align:left;">
Australia: Trafalgar, Victoria
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830722
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
HB2F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Stony Chute, New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830723
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
HB3F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: Stony Chute, New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830724
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH1M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Beerwah, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830725
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH3M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Palmerston, Northern Territory
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830726
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
HH4M
</td>
<td style="text-align:left;">
Northern brown bandicoot (Isoodon macrourus)
</td>
<td style="text-align:left;">
Australia: Palmerston, Northern Territory
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830727
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes cornuatus
</td>
<td style="text-align:left;">
IC1F
</td>
<td style="text-align:left;">
Dog (Canis lupus familiaris)
</td>
<td style="text-align:left;">
Australia: Port Sorrell, Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830728
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes hirsti
</td>
<td style="text-align:left;">
IHi1F
</td>
<td style="text-align:left;">
Cat (Felis catus)
</td>
<td style="text-align:left;">
Australia: Port Sorrell, Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830729
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
IH1F
</td>
<td style="text-align:left;">
Long-nosed bandicoot (Perameles nasuta)
</td>
<td style="text-align:left;">
Australia: New South Wales
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830730
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
Q92
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Springbrook, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830731
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
Q93
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Springbrook, Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830732
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
IT3
</td>
<td style="text-align:left;">
Tasmanian Devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;">
Australia: Tasmania
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830733
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
WL1
</td>
<td style="text-align:left;">
Possum (Pseudocheirus peregrinus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830734
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
<td style="text-align:left;">
WL2
</td>
<td style="text-align:left;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830735
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
WL11
</td>
<td style="text-align:left;">
Kangaroo (Macropus giganteus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830736
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL18
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830737
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL19
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830738
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL20
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830739
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL30
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830740
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL31
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830741
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL32
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830742
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL33
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830743
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL34
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830744
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL35
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830745
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL36
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830746
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL37
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830747
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL38
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830748
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL39
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830749
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL40
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830750
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL41
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830751
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL42
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830752
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL44
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830753
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL45
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830754
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
<td style="text-align:left;">
WL46
</td>
<td style="text-align:left;">
Echidna (Tachyglossus aculeatus)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830755
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL51
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830756
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL52
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830757
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL53
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830758
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL54
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830759
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL55
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830760
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL56
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830761
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL57
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830762
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL58
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830763
</td>
</tr>
<tr>
<td style="text-align:left;">
12S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
WL60
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM830764
</td>
</tr>
<tr>
<td style="text-align:left;">
18S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma albolimbatum
</td>
<td style="text-align:left;">
R1
</td>
<td style="text-align:left;">
Bobtail (Tiliqua rugosa)
</td>
<td style="text-align:left;">
Australia: Western Australia
</td>
<td style="text-align:left;">
PI982
</td>
<td style="text-align:left;">
Female
</td>
<td style="text-align:left;">
OM756762
</td>
</tr>
<tr>
<td style="text-align:left;">
18S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma albolimbatum
</td>
<td style="text-align:left;">
R2
</td>
<td style="text-align:left;">
Bobtail (Tiliqua rugosa)
</td>
<td style="text-align:left;">
Australia: Western Australia
</td>
<td style="text-align:left;">
PI982
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
OM756763
</td>
</tr>
<tr>
<td style="text-align:left;">
18S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma albolimbatum
</td>
<td style="text-align:left;">
R21
</td>
<td style="text-align:left;">
Bobtail (Tiliqua rugosa)
</td>
<td style="text-align:left;">
Australia: Western Australia
</td>
<td style="text-align:left;">
PI1733
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
OM756764
</td>
</tr>
<tr>
<td style="text-align:left;">
18S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma albolimbatum
</td>
<td style="text-align:left;">
R22
</td>
<td style="text-align:left;">
Bobtail (Tiliqua rugosa)
</td>
<td style="text-align:left;">
Australia: Western Australia
</td>
<td style="text-align:left;">
PI1732
</td>
<td style="text-align:left;">
Male
</td>
<td style="text-align:left;">
OM756765
</td>
</tr>
<tr>
<td style="text-align:left;">
18S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
WL52
</td>
<td style="text-align:left;">
Lace Monitor (Varanus varius)
</td>
<td style="text-align:left;">
Australia: Queensland
</td>
<td style="text-align:left;">
PI1732
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM756761
</td>
</tr>
<tr>
<td style="text-align:left;">
18S rRNA
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
ATT51
</td>
<td style="text-align:left;">
Questing
</td>
<td style="text-align:left;">
Australia: Western Australia
</td>
<td style="text-align:left;">
ATT51
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
OM756760
</td>
</tr>
</tbody>
</table></div>
<div style="page-break-after: always;"></div>
<div class="inline-table"><table class="table table table" style="font-size: 7.5px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T2genbank">Table A.3: </span>GenBank accession numbers and species identification of sequences used to produce phylogenetic tree (Figure 2.5).
</caption>
<thead><tr>
<th style="text-align:left;">
GenBank (reference)
</th>
<th style="text-align:left;">
Species (Reference)
</th>
<th style="text-align:left;">
GenBank (new)
</th>
<th style="text-align:left;">
Species (new)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
AB073725
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes persulcatus
</td>
<td style="text-align:left;">
MW665133
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
</tr>
<tr>
<td style="text-align:left;">
AB075955
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
MW665134
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
AB087746
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes uriae
</td>
<td style="text-align:left;">
MW665135
</td>
<td style="text-align:left;font-style: italic;">
Ixodes trichosuri
</td>
</tr>
<tr>
<td style="text-align:left;">
AB113317
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Amblyomma triguttatum
</td>
<td style="text-align:left;">
MW665136
</td>
<td style="text-align:left;font-style: italic;">
Ixodes trichosuri
</td>
</tr>
<tr>
<td style="text-align:left;">
AF031851
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Dermacentor variabilis
</td>
<td style="text-align:left;">
MW665137
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
AF031852
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Haemaphysalis humerosa
</td>
<td style="text-align:left;">
MW665138
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
AF031853
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Haemaphysalis longicornis
</td>
<td style="text-align:left;">
MW665139
</td>
<td style="text-align:left;font-style: italic;">
Ixodes australiensis
</td>
</tr>
<tr>
<td style="text-align:left;">
AM410581
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes uriae
</td>
<td style="text-align:left;">
MW665140
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
EF173724
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton glebopalma
</td>
<td style="text-align:left;">
MW665141
</td>
<td style="text-align:left;font-style: italic;">
Ixodes antechini
</td>
</tr>
<tr>
<td style="text-align:left;">
EF173725
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton concolor
</td>
<td style="text-align:left;">
MW665142
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
EF173726
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
MW665143
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
HG918113
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes scapularis
</td>
<td style="text-align:left;">
MW665144
</td>
<td style="text-align:left;font-style: italic;">
Ixodes antechini
</td>
</tr>
<tr>
<td style="text-align:left;">
IUU95907
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes uriae
</td>
<td style="text-align:left;">
MW665145
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
JN863727
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton concolor
</td>
<td style="text-align:left;">
MW665146
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
JN863728
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton undatum
</td>
<td style="text-align:left;">
MW665147
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
JN863729
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Robertsicus elaphensis
</td>
<td style="text-align:left;">
MW665148
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
JN863730
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Amblyomma fimbriatum
</td>
<td style="text-align:left;">
MW665149
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
JN863731
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Archaeocroton sphenodonti
</td>
<td style="text-align:left;">
MW665150
</td>
<td style="text-align:left;font-style: italic;">
Ixodes trichosuri
</td>
</tr>
<tr>
<td style="text-align:left;">
JQ346679
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Haemaphysalis doenitzi
</td>
<td style="text-align:left;">
OM830716
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
KC503255
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Rhipicephalus australis
</td>
<td style="text-align:left;">
OM830717
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
KF197132
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes ricinus
</td>
<td style="text-align:left;">
OM830718
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
KF583608
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Haemaphysalis longicornis
</td>
<td style="text-align:left;">
OM830719
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
KJ000060
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes pavlovskyi
</td>
<td style="text-align:left;">
OM830720
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma triguttatum
</td>
</tr>
<tr>
<td style="text-align:left;">
KM455964
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes simplex
</td>
<td style="text-align:left;">
OM830721
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043264
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
OM830722
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton auruginans
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043265
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
OM830723
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043266
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
OM830724
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043267
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes holocyclus
</td>
<td style="text-align:left;">
OM830725
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043268
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Haemaphysalis bancrofti
</td>
<td style="text-align:left;">
OM830726
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043269
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes tasmani
</td>
<td style="text-align:left;">
OM830727
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis humerosa
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043270
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes tasmani
</td>
<td style="text-align:left;">
OM830728
</td>
<td style="text-align:left;font-style: italic;">
Ixodes cornuatus
</td>
</tr>
<tr>
<td style="text-align:left;">
MH043271
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes tasmani
</td>
<td style="text-align:left;">
OM830729
</td>
<td style="text-align:left;font-style: italic;">
Ixodes hirsti
</td>
</tr>
<tr>
<td style="text-align:left;">
NC_002010
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Ixodes hexagonus
</td>
<td style="text-align:left;">
OM830730
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
</tr>
<tr>
<td style="text-align:left;">
U95858
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton glebopalma
</td>
<td style="text-align:left;">
OM830732
</td>
<td style="text-align:left;font-style: italic;">
Ixodes holocyclus
</td>
</tr>
<tr>
<td style="text-align:left;">
U95860
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
Bothriocroton hydrosauri
</td>
<td style="text-align:left;">
OM830733
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830734
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830735
</td>
<td style="text-align:left;font-style: italic;">
Ixodes tasmani
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830736
</td>
<td style="text-align:left;font-style: italic;">
Haemaphysalis bancrofti
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830746
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton tachyglossi
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830756
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830757
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830761
</td>
<td style="text-align:left;font-style: italic;">
Amblyomma fimbriatum
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;border-right:1px solid;">
</td>
<td style="text-align:left;">
OM830762
</td>
<td style="text-align:left;font-style: italic;">
Bothriocroton undatum
</td>
</tr>
</tbody>
</table></div>
<div style="page-break-after: always;"></div>
</div>
</div>
</div>
<div id="ch3supp" class="section level2" number="8.2">
<h2>
<span class="header-section-number">A.2</span> Chapter 3<a class="anchor" aria-label="anchor" href="#ch3supp"><i class="fas fa-link"></i></a>
</h2>
<p>Supplementary material for Chapter <a href="wildlife-bacteria.html#wildlife-bacteria">3</a> on the Microbiome of Ticks and Wildlife.</p>
<div class="inline-table"><table class="table table" style="font-size: 8.5px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:TA31">Table A.4: </span>Summary of sequences obtained from bacterial next-generation sequencin.g
</caption>
<thead><tr>
<th style="text-align:left;">
Bioinformatics step
</th>
<th style="text-align:left;">
Control (totals)
</th>
<th style="text-align:left;">
Sample (totals)
</th>
<th style="text-align:left;">
Blood
</th>
<th style="text-align:left;">
Tick
</th>
<th style="text-align:left;">
Tissue
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Sum input
</td>
<td style="text-align:left;">
5534288
</td>
<td style="text-align:left;">
96969355
</td>
<td style="text-align:left;">
35160160
</td>
<td style="text-align:left;">
24844688
</td>
<td style="text-align:left;">
36964507
</td>
</tr>
<tr>
<td style="text-align:left;">
Mean input
</td>
<td style="text-align:left;">
85143
</td>
<td style="text-align:left;">
152228
</td>
<td style="text-align:left;">
185053
</td>
<td style="text-align:left;">
95556
</td>
<td style="text-align:left;">
197671
</td>
</tr>
<tr>
<td style="text-align:left;">
Sum filtered
</td>
<td style="text-align:left;">
3922421
</td>
<td style="text-align:left;">
70048790
</td>
<td style="text-align:left;">
24929182
</td>
<td style="text-align:left;">
19809370
</td>
<td style="text-align:left;">
25310238
</td>
</tr>
<tr>
<td style="text-align:left;">
Mean filtered
</td>
<td style="text-align:left;">
60345
</td>
<td style="text-align:left;">
109967
</td>
<td style="text-align:left;">
131206
</td>
<td style="text-align:left;">
76190
</td>
<td style="text-align:left;">
135349
</td>
</tr>
<tr>
<td style="text-align:left;">
Filtered %
</td>
<td style="text-align:left;">
70.90%
</td>
<td style="text-align:left;">
72.20%
</td>
<td style="text-align:left;">
70.90%
</td>
<td style="text-align:left;">
79.70%
</td>
<td style="text-align:left;">
68.50%
</td>
</tr>
<tr>
<td style="text-align:left;">
Sum denoised
</td>
<td style="text-align:left;">
3906305
</td>
<td style="text-align:left;">
69477650
</td>
<td style="text-align:left;">
24758558
</td>
<td style="text-align:left;">
19753200
</td>
<td style="text-align:left;">
24965892
</td>
</tr>
<tr>
<td style="text-align:left;">
Mean denoised
</td>
<td style="text-align:left;">
60097
</td>
<td style="text-align:left;">
109070
</td>
<td style="text-align:left;">
130308
</td>
<td style="text-align:left;">
75974
</td>
<td style="text-align:left;">
133507
</td>
</tr>
<tr>
<td style="text-align:left;">
Sum merged
</td>
<td style="text-align:left;">
3815914
</td>
<td style="text-align:left;">
67183779
</td>
<td style="text-align:left;">
24028959
</td>
<td style="text-align:left;">
19127172
</td>
<td style="text-align:left;">
24027648
</td>
</tr>
<tr>
<td style="text-align:left;">
Mean merged
</td>
<td style="text-align:left;">
58706
</td>
<td style="text-align:left;">
105469
</td>
<td style="text-align:left;">
126468
</td>
<td style="text-align:left;">
73566
</td>
<td style="text-align:left;">
128490
</td>
</tr>
<tr>
<td style="text-align:left;">
Merged %
</td>
<td style="text-align:left;">
69.00%
</td>
<td style="text-align:left;">
69.30%
</td>
<td style="text-align:left;">
68.30%
</td>
<td style="text-align:left;">
77.00%
</td>
<td style="text-align:left;">
65.00%
</td>
</tr>
<tr>
<td style="text-align:left;">
Sum non-chimeric
</td>
<td style="text-align:left;">
3719634
</td>
<td style="text-align:left;">
64186307
</td>
<td style="text-align:left;">
23061496
</td>
<td style="text-align:left;">
18368918
</td>
<td style="text-align:left;">
22755893
</td>
</tr>
<tr>
<td style="text-align:left;">
Mean non-chimeric
</td>
<td style="text-align:left;">
57225
</td>
<td style="text-align:left;">
100763
</td>
<td style="text-align:left;">
121376
</td>
<td style="text-align:left;">
70650
</td>
<td style="text-align:left;">
121689
</td>
</tr>
<tr>
<td style="text-align:left;">
Input non-chimeric %
</td>
<td style="text-align:left;">
67.20%
</td>
<td style="text-align:left;">
66.20%
</td>
<td style="text-align:left;">
65.60%
</td>
<td style="text-align:left;">
73.90%
</td>
<td style="text-align:left;">
61.60%
</td>
</tr>
</tbody>
</table></div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA31"></span>
<img src="figures/ms-figs-appendix/FigA-3.1.png" alt="Summary of sequences obtained form bacterial 16S rRNA next-generation sequencing (Illumina MiSeq). (A) Library size (i.e. number of sequences) obtained from samples. (B) Prevalence of contaminant ASVs as identified by `decontam` analysis in controls and samples. (C) Distribution of the number of reads in sample categories blood, tick and tissue." width="95%"><p class="caption">
Figure A.2: Summary of sequences obtained form bacterial 16S rRNA next-generation sequencing (Illumina MiSeq). (A) Library size (i.e. number of sequences) obtained from samples. (B) Prevalence of contaminant ASVs as identified by <code>decontam</code> analysis in controls and samples. (C) Distribution of the number of reads in sample categories blood, tick and tissue.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA32"></span>
<img src="figures/ms-figs-appendix/FigA-3.2.png" alt="Rarefaction curves (number of ASVs) of wildlife blood, tick and tissue samples from bacterial 16S rRNA profiling (step size = 100). Rarefaction curves were used to determine how adequate sequencing depth was in detecting the complete theoretical suite of bacterial organisms present; of note, rarefaction plots excluded OTUs considered environmental contaminants (described methods)." width="85%"><p class="caption">
Figure A.3: Rarefaction curves (number of ASVs) of wildlife blood, tick and tissue samples from bacterial 16S rRNA profiling (step size = 100). Rarefaction curves were used to determine how adequate sequencing depth was in detecting the complete theoretical suite of bacterial organisms present; of note, rarefaction plots excluded OTUs considered environmental contaminants (described methods).
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA33"></span>
<img src="figures/ms-figs-appendix/FigA-3.3.png" alt="Boxplot of Alpha-diversity indices for tick samples. Diversity indexes (A) Observed number of ASVs, (B) Chao1 index, (C) Shannon index and (D) inverse Simpson index. Boxplots and violin plots represent the distribution of diversity among tick species. Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test. Pools of different tick species excluded from analysis." width="95%"><p class="caption">
Figure A.4: Boxplot of Alpha-diversity indices for tick samples. Diversity indexes (A) Observed number of ASVs, (B) Chao1 index, (C) Shannon index and (D) inverse Simpson index. Boxplots and violin plots represent the distribution of diversity among tick species. Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test. Pools of different tick species excluded from analysis.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA34"></span>
<img src="figures/ms-figs-appendix/FigA-3.4.png" alt="Phylum level distribution plots of bacterial composition in wildlife samples." width="95%"><p class="caption">
Figure A.5: Phylum level distribution plots of bacterial composition in wildlife samples.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA35"></span>
<img src="figures/ms-figs-appendix/FigA-3.5.png" alt="Abundance (no. of sequences) and prevalence in samples of bacterial phyla identified from wildlife samples (A) overall, (B) blood, (C) tick and (D) tissue." width="95%"><p class="caption">
Figure A.6: Abundance (no. of sequences) and prevalence in samples of bacterial phyla identified from wildlife samples (A) overall, (B) blood, (C) tick and (D) tissue.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA36"></span>
<img src="figures/ms-figs-appendix/FigA-3.6.png" alt="Bacterial family taxa (top 6 most abundant) identified in wildlife blood samples." width="80%"><p class="caption">
Figure A.7: Bacterial family taxa (top 6 most abundant) identified in wildlife blood samples.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA37"></span>
<img src="figures/ms-figs-appendix/FigA-3.7.png" alt="Bacterial family taxa (top 6 most abundant) identified in wildlife tick samples." width="80%"><p class="caption">
Figure A.8: Bacterial family taxa (top 6 most abundant) identified in wildlife tick samples.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA38"></span>
<img src="figures/ms-figs-appendix/FigA-3.8.png" alt="Bacterial family taxa (top 6 most abundant) identified in wildlife tissue samples." width="80%"><p class="caption">
Figure A.9: Bacterial family taxa (top 6 most abundant) identified in wildlife tissue samples.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA39"></span>
<img src="figures/ms-figs-appendix/FigA-3.9.png" alt="Hierarchical cluster analysis of bacterial communities from wildlife samples. Data points coloured by sample type; blood, tick and tissue. Cluster analysis was performed using euclidean distance measure (average), data was transformed using Hellinger transformation." width="95%"><p class="caption">
Figure A.10: Hierarchical cluster analysis of bacterial communities from wildlife samples. Data points coloured by sample type; blood, tick and tissue. Cluster analysis was performed using euclidean distance measure (average), data was transformed using Hellinger transformation.
</p>
</div>
<div style="page-break-after: always;"></div>
<div class="figure">
<span style="display:block;" id="fig:FA310"></span>
<img src="figures/ms-figs-appendix/FigA-3.10.png" alt="Hierarchical cluster analysis of bacterial communities from wildlife samples, separated by sample types. (A) Blood, (B) Tick, (C) Tissue. Data points coloured by host species (and tick species (B)). Cluster analysis was performed using euclidean distance measure (average), data was transformed using Hellinger transformation." width="95%"><p class="caption">
Figure A.11: Hierarchical cluster analysis of bacterial communities from wildlife samples, separated by sample types. (A) Blood, (B) Tick, (C) Tissue. Data points coloured by host species (and tick species (B)). Cluster analysis was performed using euclidean distance measure (average), data was transformed using Hellinger transformation.
</p>
</div>
<div id="ch3stats" class="section level3" number="8.2.1">
<h3>
<span class="header-section-number">A.2.1</span> Statistical analysis<a class="anchor" aria-label="anchor" href="#ch3stats"><i class="fas fa-link"></i></a>
</h3>
<p>Beta diversity statistical analysis of bacterial microbiome composition using analysis of variance (ANOVA) and permutational multivariate analysis of variance (PERMANOVA) methods.
Statistical analysis calculated using the vegan R package.
Data cleaning and transformation: removed samples with &lt; 1000 sequences, transformed data using Hellinger transformation method. Distance calculated using Bray-Curtis distance measure.</p>
<div id="ch3stats1" class="section level4" number="8.2.1.1">
<h4>
<span class="header-section-number">A.2.1.1</span> Sample Type<a class="anchor" aria-label="anchor" href="#ch3stats1"><i class="fas fa-link"></i></a>
</h4>
<p>Statistical analysis comparing the bacterial microbiome of blood, tick and tissue samples collected form wildlife.</p>
<p><em>A. Analysis of Variance (ANOVA)</em></p>
<pre><code>Analysis of Variance Table

Response: Distances
           Df Sum Sq  Mean Sq F value    Pr(&gt;F)    
Groups      2 0.4466 0.223289  36.209 1.364e-15 ***
Residuals 614 3.7863 0.006167                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1</code></pre>
<p><em>B. Permutational multivariate analysis of variance (PERMANOVA)</em></p>
<pre><code>Permutation test for homogeneity of multivariate dispersions
Permutation: free
Number of permutations: 999

Response: Distances
           Df Sum Sq  Mean Sq      F N.Perm Pr(&gt;F)    
Groups      2 0.4466 0.223289 36.209    999  0.001 ***
Residuals 614 3.7863 0.006167                         
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Pairwise comparisons:
(Observed p-value below diagonal, permuted p-value above diagonal)
            Blood       Tick Tissue
Blood             4.1600e-01  0.001
Tick   4.0839e-01             0.001
Tissue 8.8347e-17 3.5464e-15       </code></pre>
<div style="page-break-after: always;"></div>
</div>
<div id="ch3stats2" class="section level4" number="8.2.1.2">
<h4>
<span class="header-section-number">A.2.1.2</span> Tick species<a class="anchor" aria-label="anchor" href="#ch3stats2"><i class="fas fa-link"></i></a>
</h4>
<p>Statistical analysis comparing bacterial microbiome of tick collected from wildlife (all tick species).</p>
<p><em>A. Analysis of Variance (ANOVA)</em></p>
<pre><code>Analysis of Variance Table

Response: Distances
           Df Sum Sq Mean Sq F value    Pr(&gt;F)    
Groups      6 2.5929 0.43215   14.01 9.594e-14 ***
Residuals 251 7.7423 0.03085         </code></pre>
<p><em>B. Permutational multivariate analysis of variance (PERMANOVA)</em></p>
<pre><code>
Permutation test for homogeneity of multivariate dispersions
Permutation: free
Number of permutations: 999

Response: Distances
           Df Sum Sq Mean Sq     F N.Perm Pr(&gt;F)    
Groups      6 2.5929 0.43215 14.01    999  0.001 ***
Residuals 251 7.7423 0.03085                        
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Pairwise comparisons:
(Observed p-value below diagonal, permuted p-value above diagonal)
                 Amtri Ixant      Ixaus      Ixhol Ixhol;Ixtri      Ixtas Ixtri
Amtri                        9.4400e-01 3.6500e-01  2.5700e-01 7.8500e-01 0.001
Ixant                                                                          
Ixaus       9.3797e-01                  5.9300e-01  3.8000e-02 7.3300e-01 0.001
Ixhol       3.4424e-01       5.7323e-01             5.3700e-01 1.0000e-03 0.001
Ixhol;Ixtri 2.4249e-01       4.5601e-02 5.3705e-01             3.8000e-02 0.001
Ixtas       7.8073e-01       7.4761e-01 3.5985e-05  3.8405e-02            0.001
Ixtri       1.0858e-06       4.6854e-09 2.0461e-06  1.2151e-05 5.1524e-15   </code></pre>
<div style="page-break-after: always;"></div>
<p>Statistical analysis comparing bacterial microbiome of cohabiting tick species collected from wildlife in Sydney Northern Beaches area, New South Wales (<em>Ix. holocyclus</em>, <em>Ix. tasmani</em>, and <em>Ix. trichosuri</em>).</p>
<p><em>A. Analysis of Variance (ANOVA)</em></p>
<pre><code>Analysis of Variance Table

Response: Distances
           Df Sum Sq Mean Sq F value    Pr(&gt;F)    
Groups      3 2.3202 0.77341  25.465 2.177e-14 ***
Residuals 247 7.5019 0.03037                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1</code></pre>
<p><em>B. Permutational multivariate analysis of variance (PERMANOVA)</em></p>
<pre><code>Permutation test for homogeneity of multivariate dispersions
Permutation: free
Number of permutations: 999

Response: Distances
           Df Sum Sq Mean Sq      F N.Perm Pr(&gt;F)    
Groups      3 2.3202 0.77341 25.465    999  0.001 ***
Residuals 247 7.5019 0.03037                         
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Pairwise comparisons:
(Observed p-value below diagonal, permuted p-value above diagonal)
                 Ixhol Ixhol;Ixtri      Ixtas Ixtri
Ixhol                   5.3000e-01 1.0000e-03 0.001
Ixhol;Ixtri 5.3687e-01             3.9000e-02 0.001
Ixtas       3.5862e-05  3.8383e-02            0.001
Ixtri       2.0485e-06  1.2237e-05 5.1829e-15     </code></pre>
</div>
</div>
<div id="ch3-esupp" class="section level3" number="8.2.2">
<h3>
<span class="header-section-number">A.2.2</span> Supplementary Electronic Tables<a class="anchor" aria-label="anchor" href="#ch3-esupp"><i class="fas fa-link"></i></a>
</h3>
<p>The following electronic supplementary files are available for download on <a href="https://figshare.com/s/0f5495957bce881caef8">FigShare</a>, repository doi: 10.6084/m9.figshare.14363627.</p>
<div id="supplementary-table-e1.1" class="section level4" number="8.2.2.1">
<h4>
<span class="header-section-number">A.2.2.1</span> Supplementary table E1.1<a class="anchor" aria-label="anchor" href="#supplementary-table-e1.1"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E1.1:</strong> Detail for taxa of interest with sequence information and sample metadata, including results from NCBI BLAST analysis.
<a href="https://ndownloader.figshare.com/files/27452714">Taxa of interest sample data and BLAST results</a></p>
</div>
<div id="supplementary-table-e1.2" class="section level4" number="8.2.2.2">
<h4>
<span class="header-section-number">A.2.2.2</span> Supplementary table E1.2<a class="anchor" aria-label="anchor" href="#supplementary-table-e1.2"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E1.2:</strong> Genetic similarity of Anaplasmataceae sequences based on a 1,244 bp alignment of the 16S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data.
<a href="https://ndownloader.figshare.com/files/27452720">Anaplasmataceae genetic diversity table</a></p>
</div>
<div id="supplementary-table-e1.3" class="section level4" number="8.2.2.3">
<h4>
<span class="header-section-number">A.2.2.3</span> Supplementary table E1.3<a class="anchor" aria-label="anchor" href="#supplementary-table-e1.3"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E1.3:</strong> Genetic similarity of <em>Borrelia</em> sequences based on a 431 bp alignment of the 16S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data.
<a href="https://ndownloader.figshare.com/files/27452726"><em>Borrelia</em> genetic diversity table</a></p>
</div>
<div id="supplementary-table-e1.4" class="section level4" number="8.2.2.4">
<h4>
<span class="header-section-number">A.2.2.4</span> Supplementary table E1.4<a class="anchor" aria-label="anchor" href="#supplementary-table-e1.4"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E1.4:</strong> Genetic similarity of <em>Bartonella</em> sequences based on a 522 bp alignment of the 16S rRNA-ITS locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data.
<a href="https://ndownloader.figshare.com/files/27452723"><em>Bartonella</em> genetic diversity table</a></p>
<hr>
</div>
</div>
</div>
<div id="ch4-supp" class="section level2" number="8.3">
<h2>
<span class="header-section-number">A.3</span> Chapter 4<a class="anchor" aria-label="anchor" href="#ch4-supp"><i class="fas fa-link"></i></a>
</h2>
<p>Supplementary material for Chapter <a href="wildlife-haemoprotozoa.html#wildlife-haemoprotozoa">4</a> on Haemoprotozoa Surveillance.</p>
<div class="figure">
<span style="display:block;" id="fig:FA41"></span>
<img src="figures/ms-figs-appendix/FigA-4.1.png" alt="Summary of sequences obtained from trypanosome 18S rRNA metabatcoding (Illumina MiSeq). (A) Library size (i.e. number of sequences) obtained from samples. (B) Distribution of the number of reads in sample categories blood, tick and tissue." width="95%"><p class="caption">
Figure A.12: Summary of sequences obtained from trypanosome 18S rRNA metabatcoding (Illumina MiSeq). (A) Library size (i.e. number of sequences) obtained from samples. (B) Distribution of the number of reads in sample categories blood, tick and tissue.
</p>
</div>
<div id="ch4-esupp" class="section level3" number="8.3.1">
<h3>
<span class="header-section-number">A.3.1</span> Supplementary Electronic Tables<a class="anchor" aria-label="anchor" href="#ch4-esupp"><i class="fas fa-link"></i></a>
</h3>
<p>The following electronic supplementary files are available for download on <a href="https://figshare.com/s/9eb3dba4096470af9904">FigShare</a>, repository doi: 10.6084/m9.figshare.14370965.</p>
<div id="supplementary-table-e2.1" class="section level4" number="8.3.1.1">
<h4>
<span class="header-section-number">A.3.1.1</span> Supplementary table E2.1<a class="anchor" aria-label="anchor" href="#supplementary-table-e2.1"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E2.1:</strong> <em>Trypanosoma</em> amplicon sequence data from blood, tick and tissue samples. Data showing zOTUs belonging to the Trypanosomatidae family with results from NCBI BLAST analysis. <a href="https://figshare.com/ndownloader/files/30578460">Trypanosome zOTU taxonomy and sample data</a></p>
</div>
<div id="supplementary-table-e2.2" class="section level4" number="8.3.1.2">
<h4>
<span class="header-section-number">A.3.1.2</span> Supplementary table E2.2<a class="anchor" aria-label="anchor" href="#supplementary-table-e2.2"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E2.2:</strong> Genetic similarity of piroplasm sequences based on an 852 bp alignment of the 18S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data.
<a href="https://figshare.com/ndownloader/files/30578457">Piroplasm genetic diversity table</a></p>
</div>
<div id="supplementary-table-e2.3" class="section level4" number="8.3.1.3">
<h4>
<span class="header-section-number">A.3.1.3</span> Supplementary table E2.3<a class="anchor" aria-label="anchor" href="#supplementary-table-e2.3"><i class="fas fa-link"></i></a>
</h4>
<p><strong>Supplementary table E2.3:</strong> Genetic similarity of <em>Hepatozoon</em> sequences based on an 819 bp alignment of the 18S rRNA locus. Sequences generated from the present study in bold. Bottom half of matrix showing sequence similarity and top half containing genetic distance data.
<a href="https://figshare.com/ndownloader/files/30578463"><em>Hepatozoon</em> genetic diversity table</a></p>
<hr>
</div>
</div>
</div>
<div id="ch6-supp" class="section level2" number="8.4">
<h2>
<span class="header-section-number">A.4</span> Chapter 6<a class="anchor" aria-label="anchor" href="#ch6-supp"><i class="fas fa-link"></i></a>
</h2>
<p>Supplementary material for Chapter <a href="tas-devil.html#tas-devil">6</a> on Tasmanian Devil Haemoprotozoa.</p>
<div class="inline-table"><table class="table table table" style="margin-left: auto; margin-right: auto; font-size: 8px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:TA61">Table A.5: </span> sequences used in phylogenetic analysis in the present study. Sequences produced in the present study indicated in bold.
</caption>
<thead><tr>
<th style="text-align:left;">
Species
</th>
<th style="text-align:left;">
Isolate
</th>
<th style="text-align:left;">
Host
</th>
<th style="text-align:left;">
Location (a)
</th>
<th style="text-align:left;">
18S rRNA
</th>
<th style="text-align:left;">
gGAPDH
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
JD-2008a Mika
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (NSW)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
<td style="text-align:left;width: 3cm; ">
GU966585
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
JD-2008a Charlton
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD)
</td>
<td style="text-align:left;width: 3cm; ">
GU966588
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
TD-BRI115
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tasmanian devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;width: 3cm; ">
AU (TAS)
</td>
<td style="text-align:left;width: 3cm; ">
MT883297
</td>
<td style="text-align:left;width: 3cm; ">
MT514664
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; font-weight: bold;">
H26
</td>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Wombat (Vombatus ursinus)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AJ009169
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
G1
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753530
</td>
<td style="text-align:left;width: 3cm; ">
KC812982
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
APP
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Wombat (Vombatus ursinus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620558
</td>
<td style="text-align:left;width: 3cm; ">
AJ620277
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
G2
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753531
</td>
<td style="text-align:left;width: 3cm; ">
KC812983
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. copemani
</td>
<td style="text-align:left;width: 4cm; ">
Q2088
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Quokka (Setonix brachyurus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
HQ267094
</td>
<td style="text-align:left;width: 3cm; ">
HQ267095
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. vegrandis
</td>
<td style="text-align:left;width: 4cm; ">
G4
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753532
</td>
<td style="text-align:left;width: 3cm; ">
KC812985
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. vegrandis
</td>
<td style="text-align:left;width: 4cm; ">
G7
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753536
</td>
<td style="text-align:left;width: 3cm; ">
KC812987
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. vegrandis
</td>
<td style="text-align:left;width: 4cm; ">
G5
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753534
</td>
<td style="text-align:left;width: 3cm; ">
KC812986
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. vegrandis
</td>
<td style="text-align:left;width: 4cm; ">
G3
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753533
</td>
<td style="text-align:left;width: 3cm; ">
KC812984
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. vegrandis
</td>
<td style="text-align:left;width: 4cm; ">
G6
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KC753535
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. gilletti
</td>
<td style="text-align:left;width: 4cm; ">
Lanie
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD)
</td>
<td style="text-align:left;width: 3cm; ">
GU966589
</td>
<td style="text-align:left;width: 3cm; ">
GU966587
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. cruzi
</td>
<td style="text-align:left;width: 4cm; ">
G
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Opossum (Didelphis marsupialis)
</td>
<td style="text-align:left;width: 3cm; ">
BR
</td>
<td style="text-align:left;width: 3cm; ">
AF239981
</td>
<td style="text-align:left;width: 3cm; ">
GQ140351
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. cruzi marinkellei
</td>
<td style="text-align:left;width: 4cm; ">
TryCC 344
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Bat (Carollia perspicillata)
</td>
<td style="text-align:left;width: 3cm; ">
BR
</td>
<td style="text-align:left;width: 3cm; ">
FJ001664
</td>
<td style="text-align:left;width: 3cm; ">
GQ140360
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. erneyi
</td>
<td style="text-align:left;width: 4cm; ">
TCC1294
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Bat (Tadarida sp.)
</td>
<td style="text-align:left;width: 3cm; ">
MZ
</td>
<td style="text-align:left;width: 3cm; ">
JN040988
</td>
<td style="text-align:left;width: 3cm; ">
JN040965
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. dionisii
</td>
<td style="text-align:left;width: 4cm; ">
TryCC 211
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Bat (Eptesicus brasiliensis)
</td>
<td style="text-align:left;width: 3cm; ">
BR
</td>
<td style="text-align:left;width: 3cm; ">
FJ001666
</td>
<td style="text-align:left;width: 3cm; ">
GQ140362
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. rangeli
</td>
<td style="text-align:left;width: 4cm; ">
AM80
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Human (Homo sapiens)
</td>
<td style="text-align:left;width: 3cm; ">
BR
</td>
<td style="text-align:left;width: 3cm; ">
AY491766
</td>
<td style="text-align:left;width: 3cm; ">
JN040973
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. noyesi
</td>
<td style="text-align:left;width: 4cm; ">
H25
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Kangaroo (Macropus giganteus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ009168
</td>
<td style="text-align:left;width: 3cm; ">
AJ620276
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. noyesi
</td>
<td style="text-align:left;width: 4cm; ">
WC6218
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Woylie (Bettongia penicillata)
</td>
<td style="text-align:left;width: 3cm; ">
AU (WA)
</td>
<td style="text-align:left;width: 3cm; ">
KU354263
</td>
<td style="text-align:left;width: 3cm; ">
KU354264
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. livingstonei
</td>
<td style="text-align:left;width: 4cm; ">
TCC1270
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Bat (Rhinolophus landeri)
</td>
<td style="text-align:left;width: 3cm; ">
MZ
</td>
<td style="text-align:left;width: 3cm; ">
KF192979
</td>
<td style="text-align:left;width: 3cm; ">
KF192958
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. lewisi
</td>
<td style="text-align:left;width: 4cm; ">
Molteno B3
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Rat (Rattus sp.)
</td>
<td style="text-align:left;width: 3cm; ">
UK
</td>
<td style="text-align:left;width: 3cm; ">
AJ009156
</td>
<td style="text-align:left;width: 3cm; ">
AJ629272
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. microti
</td>
<td style="text-align:left;width: 4cm; ">
TRL132
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Vole (Microtis agrestis)
</td>
<td style="text-align:left;width: 3cm; ">
UK
</td>
<td style="text-align:left;width: 3cm; ">
AJ009158
</td>
<td style="text-align:left;width: 3cm; ">
AJ620273
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TD-WPP601 (genotype A)
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tasmanian devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;width: 3cm; ">
AU (TAS)
</td>
<td style="text-align:left;width: 3cm; ">
MT883326
</td>
<td style="text-align:left;width: 3cm; ">
MT514665
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; font-weight: bold;">
TD-WPP602 (genotype A)
</td>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tasmanian devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AU (TAS)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
<ul><li></ul>
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
MT514666
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; font-weight: bold;">
TD-WPP585 (genotype A)
</td>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tasmanian devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AU (TAS)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
MT883324
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; font-weight: bold;">
TD-BRI111 (genotype B)
</td>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tasmanian devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AU (TAS)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
MT883296
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; font-weight: bold;">
TD-TKN211 (genotype B)
</td>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tasmanian devil (Sarcophilus harrisii)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AU (TAS)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
MT883322
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; font-weight: bold;">
ABF (wallaby)
</td>
<td style="text-align:left;width: 6cm; font-style: italic;font-weight: bold;">
Swamp wallaby (Wallabia bicolor)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AJ620564
</td>
<td style="text-align:left;width: 3cm; font-weight: bold;">
AJ620278
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. cyclops
</td>
<td style="text-align:left;width: 4cm; ">
LV492
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Macaque (Macaca sp.)
</td>
<td style="text-align:left;width: 3cm; ">
MY
</td>
<td style="text-align:left;width: 3cm; ">
AJ131958
</td>
<td style="text-align:left;width: 3cm; ">
FJ649493
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AQ.22
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech(Philaemon)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620574
</td>
<td style="text-align:left;width: 3cm; ">
AJ620280
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AQ.45
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech(Philaemon)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620575
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AV.44. cl157
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech (Micobdella)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620573
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AV.44 cl. 156B
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech (Micobdella)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620572
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AV.43 cl100B
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech (Micobdella)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620570
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AV.43 cl.101E
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech (Micobdella)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620571
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AQ.40
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech(Philaemon)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620576(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.AQ.48
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech(Philaemon)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620577(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
Frog ADE
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Frog (Mixophyes flaeyi)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620569(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.SL.1
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech (Haemadipsa zeylanica)
</td>
<td style="text-align:left;width: 3cm; ">
SL
</td>
<td style="text-align:left;width: 3cm; ">
AJ620578(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
Wallaby 10
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Brush-tailed rock-wallaby (Petrogale penicillate)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620563(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
TL.NG.1
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Leech (Leiobdella jawarerensis)
</td>
<td style="text-align:left;width: 3cm; ">
PNG
</td>
<td style="text-align:left;width: 3cm; ">
AJ620581(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. theileri
</td>
<td style="text-align:left;width: 4cm; ">
K127
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Cattle (Bos taurus)
</td>
<td style="text-align:left;width: 3cm; ">
DE
</td>
<td style="text-align:left;width: 3cm; ">
AJ009164
</td>
<td style="text-align:left;width: 3cm; ">
AJ620282
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. theileri
</td>
<td style="text-align:left;width: 4cm; ">
KM
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Cattle (Bos taurus)
</td>
<td style="text-align:left;width: 3cm; ">
JP
</td>
<td style="text-align:left;width: 3cm; ">
AB007814(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. theileri
</td>
<td style="text-align:left;width: 4cm; ">
Bb756
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
European bison(Bison bonasus)
</td>
<td style="text-align:left;width: 3cm; ">
PL
</td>
<td style="text-align:left;width: 3cm; ">
KF765801(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. theileri
</td>
<td style="text-align:left;width: 4cm; ">
Tthc26
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Cattle (Bos sp.)
</td>
<td style="text-align:left;width: 3cm; ">
BR
</td>
<td style="text-align:left;width: 3cm; ">
GQ176153(b)
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. irwini
</td>
<td style="text-align:left;width: 4cm; ">
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Koala (Phascolarctos cinereus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (QLD, NSW)
</td>
<td style="text-align:left;width: 3cm; ">
FJ649479
</td>
<td style="text-align:left;width: 3cm; ">
FJ649485
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. bennetti
</td>
<td style="text-align:left;width: 4cm; ">
KT-2
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Kestrel (Falco sparverius)
</td>
<td style="text-align:left;width: 3cm; ">
US
</td>
<td style="text-align:left;width: 3cm; ">
AJ223562
</td>
<td style="text-align:left;width: 3cm; ">
FJ649486
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. sp.
</td>
<td style="text-align:left;width: 4cm; ">
ATT
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Currawong (Strepera sp.)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620557
</td>
<td style="text-align:left;width: 3cm; ">
AJ620264
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. mega
</td>
<td style="text-align:left;width: 4cm; ">
ATCC 30038
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Toad (Bufo regularis)
</td>
<td style="text-align:left;width: 3cm; ">
Africa
</td>
<td style="text-align:left;width: 3cm; ">
AJ009157
</td>
<td style="text-align:left;width: 3cm; ">
AJ620253
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. rotatorium
</td>
<td style="text-align:left;width: 4cm; ">
B2-II
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Bullfrog (Rana catesbeiana)
</td>
<td style="text-align:left;width: 3cm; ">
CA
</td>
<td style="text-align:left;width: 3cm; ">
AJ009161
</td>
<td style="text-align:left;width: 3cm; ">
AJ620256
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. binneyi
</td>
<td style="text-align:left;width: 4cm; ">
AWW
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Platypus (Ornithorhynchus anatinus)
</td>
<td style="text-align:left;width: 3cm; ">
AU (VIC)
</td>
<td style="text-align:left;width: 3cm; ">
AJ620565
</td>
<td style="text-align:left;width: 3cm; ">
AJ620266
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Tr. granulosum
</td>
<td style="text-align:left;width: 4cm; ">
UK
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
Eel (Anguilla anguilla)
</td>
<td style="text-align:left;width: 3cm; ">
PT
</td>
<td style="text-align:left;width: 3cm; ">
AJ620551
</td>
<td style="text-align:left;width: 3cm; ">
<ul><li></ul>
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Phytomonas serpens
</td>
<td style="text-align:left;width: 4cm; ">
N/A
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
N/A
</td>
<td style="text-align:left;width: 3cm; ">
N/A
</td>
<td style="text-align:left;width: 3cm; ">
U39577
</td>
<td style="text-align:left;width: 3cm; ">
EU084892
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-style: italic;">
Herpetomonas muscarum
</td>
<td style="text-align:left;width: 4cm; ">
N/A
</td>
<td style="text-align:left;width: 6cm; font-style: italic;">
N/A
</td>
<td style="text-align:left;width: 3cm; ">
N/A
</td>
<td style="text-align:left;width: 3cm; ">
L18872
</td>
<td style="text-align:left;width: 3cm; ">
DQ092548
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Country abbreviations: AU - Australia, MZ - Mozambique, UK - United Kingdom, MY - Malaysia, SL - Sri Lanka, PNG - Papua New Guinea, DE - Germany, JP- Japan, PL- Poland, BR - Brazil, CA- Canada, PT - Portugal, US - United States of America. Australian states: QLD - Queensland, TAS - Tasmania, WA - Western Australia.
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup>b</sup> Short 18S rRNA sequences only, included for comparison within the Trypanosoma cyclops clade.
</td>
</tr>
</tfoot>
</table></div>
<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:TA62">Table A.6: </span>Pairwise genetic similarity matrix of the  clade at the 18S rRNA locus. Analysis conducted over at 559 bp alignment of the V7-8 hypervariable using the Kimura Two-Parameter (K2P) method.
</caption>
<thead><tr>
<th style="text-align:right;">
…1
</th>
<th style="text-align:left;">
Genotype
</th>
<th style="text-align:right;">
1
</th>
<th style="text-align:right;">
2
</th>
<th style="text-align:right;">
3
</th>
<th style="text-align:right;">
4
</th>
<th style="text-align:right;">
5
</th>
<th style="text-align:right;">
6
</th>
<th style="text-align:right;">
7
</th>
<th style="text-align:right;">
8
</th>
<th style="text-align:right;">
9
</th>
<th style="text-align:right;">
10
</th>
<th style="text-align:right;">
11
</th>
<th style="text-align:right;">
12
</th>
<th style="text-align:right;">
13
</th>
<th style="text-align:right;">
14
</th>
<th style="text-align:right;">
15
</th>
<th style="text-align:right;">
16
</th>
<th style="text-align:right;">
17
</th>
<th style="text-align:right;">
18
</th>
<th style="text-align:right;">
19
</th>
<th style="text-align:right;">
20
</th>
<th style="text-align:left;">
21
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:right;font-style: italic;">
1
</td>
<td style="text-align:left;">
T. theileri KM AB007814
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
2
</td>
<td style="text-align:left;">
T. theileri Bb756 KF765801
</td>
<td style="text-align:right;">
100.0
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
3
</td>
<td style="text-align:left;">
T. theileri K127 AJ009164
</td>
<td style="text-align:right;">
97.5
</td>
<td style="text-align:right;">
97.5
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
4
</td>
<td style="text-align:left;">
T. theileri Tthc26 GQ176153
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
99.8
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
5
</td>
<td style="text-align:left;">
T. sp. TL.AQ.40 AJ620576
</td>
<td style="text-align:right;">
87.3
</td>
<td style="text-align:right;">
87.3
</td>
<td style="text-align:right;">
87.2
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
6
</td>
<td style="text-align:left;">
T. sp. TL.AQ.48 AJ620577
</td>
<td style="text-align:right;">
87.3
</td>
<td style="text-align:right;">
87.3
</td>
<td style="text-align:right;">
87.2
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
99.8
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
7
</td>
<td style="text-align:left;">
T. sp. Frog ADE AJ620569
</td>
<td style="text-align:right;">
87.4
</td>
<td style="text-align:right;">
87.4
</td>
<td style="text-align:right;">
87.3
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
95.3
</td>
<td style="text-align:right;">
95.1
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
8
</td>
<td style="text-align:left;">
T. cyclops LV492 AJ131958
</td>
<td style="text-align:right;">
82.2
</td>
<td style="text-align:right;">
82.2
</td>
<td style="text-align:right;">
82.2
</td>
<td style="text-align:right;">
82.4
</td>
<td style="text-align:right;">
89.6
</td>
<td style="text-align:right;">
89.3
</td>
<td style="text-align:right;">
88.1
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
9
</td>
<td style="text-align:left;">
T. sp. TL.AQ.45 AJ620575
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
88.4
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
97.0
</td>
<td style="text-align:right;">
98.1
</td>
<td style="text-align:right;">
89.8
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
10
</td>
<td style="text-align:left;">
WPP585 MT883324
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.2
</td>
<td style="text-align:right;">
90.7
</td>
<td style="text-align:right;">
90.4
</td>
<td style="text-align:right;">
89.4
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
91.1
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
11
</td>
<td style="text-align:left;">
WPP601 MT883326
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.2
</td>
<td style="text-align:right;">
90.7
</td>
<td style="text-align:right;">
90.4
</td>
<td style="text-align:right;">
89.4
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
91.1
</td>
<td style="text-align:right;">
100.0
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
12
</td>
<td style="text-align:left;">
BRI111 MT883296
</td>
<td style="text-align:right;">
87.0
</td>
<td style="text-align:right;">
87.0
</td>
<td style="text-align:right;">
87.0
</td>
<td style="text-align:right;">
87.2
</td>
<td style="text-align:right;">
96.9
</td>
<td style="text-align:right;">
96.7
</td>
<td style="text-align:right;">
97.0
</td>
<td style="text-align:right;">
89.5
</td>
<td style="text-align:right;">
98.4
</td>
<td style="text-align:right;">
90.9
</td>
<td style="text-align:right;">
90.9
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
13
</td>
<td style="text-align:left;">
T. sp. TL.AQ.22 AJ620574
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.0
</td>
<td style="text-align:right;">
82.2
</td>
<td style="text-align:right;">
89.8
</td>
<td style="text-align:right;">
89.6
</td>
<td style="text-align:right;">
88.8
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
90.5
</td>
<td style="text-align:right;">
98.5
</td>
<td style="text-align:right;">
98.5
</td>
<td style="text-align:right;">
90.2
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
14
</td>
<td style="text-align:left;">
T. sp. TL.AV.44 cl157 AJ620573
</td>
<td style="text-align:right;">
87.2
</td>
<td style="text-align:right;">
87.2
</td>
<td style="text-align:right;">
87.1
</td>
<td style="text-align:right;">
87.4
</td>
<td style="text-align:right;">
95.6
</td>
<td style="text-align:right;">
95.4
</td>
<td style="text-align:right;">
98.4
</td>
<td style="text-align:right;">
88.8
</td>
<td style="text-align:right;">
98.4
</td>
<td style="text-align:right;">
89.7
</td>
<td style="text-align:right;">
89.7
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
89.1
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
15
</td>
<td style="text-align:left;">
T. sp. TL.SL.1 AJ620578
</td>
<td style="text-align:right;">
86.8
</td>
<td style="text-align:right;">
86.8
</td>
<td style="text-align:right;">
86.7
</td>
<td style="text-align:right;">
86.9
</td>
<td style="text-align:right;">
95.4
</td>
<td style="text-align:right;">
95.2
</td>
<td style="text-align:right;">
98.1
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
98.1
</td>
<td style="text-align:right;">
89.5
</td>
<td style="text-align:right;">
89.5
</td>
<td style="text-align:right;">
97.0
</td>
<td style="text-align:right;">
88.9
</td>
<td style="text-align:right;">
99.3
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
16
</td>
<td style="text-align:left;">
T. sp. ABF AJ620564
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
87.7
</td>
<td style="text-align:right;">
97.7
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
90.4
</td>
<td style="text-align:right;">
98.8
</td>
<td style="text-align:right;">
90.9
</td>
<td style="text-align:right;">
90.9
</td>
<td style="text-align:right;">
98.1
</td>
<td style="text-align:right;">
90.7
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
97.0
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
17
</td>
<td style="text-align:left;">
T. sp. TL.AV.44 cl158b AJ620572
</td>
<td style="text-align:right;">
87.1
</td>
<td style="text-align:right;">
87.1
</td>
<td style="text-align:right;">
87.0
</td>
<td style="text-align:right;">
87.2
</td>
<td style="text-align:right;">
95.4
</td>
<td style="text-align:right;">
95.2
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
88.3
</td>
<td style="text-align:right;">
98.2
</td>
<td style="text-align:right;">
89.6
</td>
<td style="text-align:right;">
89.6
</td>
<td style="text-align:right;">
96.5
</td>
<td style="text-align:right;">
89.0
</td>
<td style="text-align:right;">
98.4
</td>
<td style="text-align:right;">
98.2
</td>
<td style="text-align:right;">
97.0
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
18
</td>
<td style="text-align:left;">
T. sp. TL.AV.43 cl100b AJ620570
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
88.2
</td>
<td style="text-align:right;">
88.4
</td>
<td style="text-align:right;">
95.6
</td>
<td style="text-align:right;">
95.3
</td>
<td style="text-align:right;">
96.7
</td>
<td style="text-align:right;">
88.7
</td>
<td style="text-align:right;">
98.4
</td>
<td style="text-align:right;">
89.6
</td>
<td style="text-align:right;">
89.6
</td>
<td style="text-align:right;">
96.7
</td>
<td style="text-align:right;">
89.0
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
96.8
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
97.0
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
19
</td>
<td style="text-align:left;">
T. sp. TL.AV.43 cl100e AJ620571
</td>
<td style="text-align:right;">
88.7
</td>
<td style="text-align:right;">
88.7
</td>
<td style="text-align:right;">
88.6
</td>
<td style="text-align:right;">
88.8
</td>
<td style="text-align:right;">
96.3
</td>
<td style="text-align:right;">
96.0
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
89.0
</td>
<td style="text-align:right;">
99.1
</td>
<td style="text-align:right;">
90.3
</td>
<td style="text-align:right;">
90.3
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
89.7
</td>
<td style="text-align:right;">
97.7
</td>
<td style="text-align:right;">
97.5
</td>
<td style="text-align:right;">
97.9
</td>
<td style="text-align:right;">
97.7
</td>
<td style="text-align:right;">
99.3
</td>
<td style="text-align:right;">
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
20
</td>
<td style="text-align:left;">
T. sp. wallaby 10 AJ620563
</td>
<td style="text-align:right;">
88.4
</td>
<td style="text-align:right;">
88.4
</td>
<td style="text-align:right;">
88.4
</td>
<td style="text-align:right;">
88.6
</td>
<td style="text-align:right;">
96.0
</td>
<td style="text-align:right;">
95.8
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
88.7
</td>
<td style="text-align:right;">
98.8
</td>
<td style="text-align:right;">
90.0
</td>
<td style="text-align:right;">
90.0
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
89.4
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
97.2
</td>
<td style="text-align:right;">
97.7
</td>
<td style="text-align:right;">
97.5
</td>
<td style="text-align:right;">
99.1
</td>
<td style="text-align:right;">
99.8
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:right;font-style: italic;">
21
</td>
<td style="text-align:left;">
T. sp. TL.NG.1 AJ620581
</td>
<td style="text-align:right;">
90.5
</td>
<td style="text-align:right;">
90.5
</td>
<td style="text-align:right;">
90.0
</td>
<td style="text-align:right;">
90.0
</td>
<td style="text-align:right;">
96.0
</td>
<td style="text-align:right;">
95.7
</td>
<td style="text-align:right;">
96.8
</td>
<td style="text-align:right;">
87.5
</td>
<td style="text-align:right;">
98.7
</td>
<td style="text-align:right;">
88.7
</td>
<td style="text-align:right;">
88.7
</td>
<td style="text-align:right;">
96.8
</td>
<td style="text-align:right;">
88.0
</td>
<td style="text-align:right;">
97.1
</td>
<td style="text-align:right;">
96.8
</td>
<td style="text-align:right;">
97.6
</td>
<td style="text-align:right;">
97.4
</td>
<td style="text-align:right;">
98.9
</td>
<td style="text-align:right;">
99.7
</td>
<td style="text-align:right;">
100
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></div>
<div class="next"><a href="references.html">References</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#supplementary-material"><span class="header-section-number">A</span> Supplementary material</a></li>
<li>
<a class="nav-link" href="#ch2supp"><span class="header-section-number">A.1</span> Chapter 2</a><ul class="nav navbar-nav"><li><a class="nav-link" href="#molecular-methods-for-analysis-of-ticks"><span class="header-section-number">A.1.1</span> Molecular methods for analysis of ticks</a></li></ul>
</li>
<li>
<a class="nav-link" href="#ch3supp"><span class="header-section-number">A.2</span> Chapter 3</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#ch3stats"><span class="header-section-number">A.2.1</span> Statistical analysis</a></li>
<li><a class="nav-link" href="#ch3-esupp"><span class="header-section-number">A.2.2</span> Supplementary Electronic Tables</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#ch4-supp"><span class="header-section-number">A.3</span> Chapter 4</a><ul class="nav navbar-nav"><li><a class="nav-link" href="#ch4-esupp"><span class="header-section-number">A.3.1</span> Supplementary Electronic Tables</a></li></ul>
</li>
<li><a class="nav-link" href="#ch6-supp"><span class="header-section-number">A.4</span> Chapter 6</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/ulyngs/oxforddown/blob/master/front-and-back-matter/98-appendices.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/ulyngs/oxforddown/edit/master/front-and-back-matter/98-appendices.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></strong>" was written by Siobhon Lee Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
