<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>6 Tasmanian devil haemoprotozoa | Ecology of Ticks and Microbes in Australian Wildlife</title>
<meta name="author" content="Siobhon Lee Egan">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="6 Tasmanian devil haemoprotozoa | Ecology of Ticks and Microbes in Australian Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="6 Tasmanian devil haemoprotozoa | Ecology of Ticks and Microbes in Australian Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Pathogens. The following manuscript has been published: Egan, S., Ruiz-Aravena, M., Austen, J.,...">
<meta property="og:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Pathogens. The following manuscript has been published: Egan, S., Ruiz-Aravena, M., Austen, J.,...">
<meta name="twitter:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Pathogens. The following manuscript has been published: Egan, S., Ruiz-Aravena, M., Austen, J.,...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australian Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="active" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/siobhon-egan/PhD">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="tas-devil" class="section level1" number="6">
<h1>
<span class="header-section-number">6</span> Tasmanian devil haemoprotozoa<a class="anchor" aria-label="anchor" href="#tas-devil"><i class="fas fa-link"></i></a>
</h1>
<div class="inline-figure"><img src="front-and-back-matter/preface/tasdevil.jpg" width="100%"></div>
<div style="page-break-after: always;"></div>
<div id="preface-5" class="section level2 unnumbered">
<h2>Preface<a class="anchor" aria-label="anchor" href="#preface-5"><i class="fas fa-link"></i></a>
</h2>
<p><strong>Attribution Statement</strong></p>
<p>The following chapter has been drafted in accordance with the journal <em>Pathogens</em>.</p>
<p>The following manuscript has been published: <strong>Egan, S.</strong>, Ruiz-Aravena, M., Austen, J., Barton, X., Comte, S., Hamilton, D., Hamede, R., Ryan, U., Irwin, P., Jones, M., and Oskam, C. <em>2020</em>. Blood parasites in endangered wildlife: trypanosomes discovered during a survey of haemoprotozoa from the Tasmanian devil. <em>Pathogens</em>, <strong>9</strong>(11), 873. DOI: <a href="https://doi.org/10.3390/pathogens9110873">10.3390/pathogens9110873</a></p>
<p>The following authors contributed to this manuscript as outlined below<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Contribution indicates the total involvement the author has had in this project. Placing an ‘X’ in the remaining boxes indicates what aspect(s) of the project each author engaged in.&lt;/p&gt;"><sup>7</sup></a>.</p>
<div class="inline-table"><table class="table table" style="font-size: 7px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Authorship order
</th>
<th style="text-align:right;">
Contribution (%)
</th>
<th style="text-align:left;">
Concept Development
</th>
<th style="text-align:left;">
Data Collection
</th>
<th style="text-align:left;">
Data Analysis
</th>
<th style="text-align:left;">
Drafting of manuscript
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Siobhon L. Egan
</td>
<td style="text-align:right;">
70
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
</tr>
<tr>
<td style="text-align:left;">
Manuel Ruiz-Aravena
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Jill M. Austen
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Xavier Barton
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Sebastien Comte
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
David G. Hamilton
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Robrigo K. Hamede
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Una M. Ryan
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter J. Irwin
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Menna E. Jones
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Charlotte L. Oskam
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
<p>By signing this document, the Candidate and Principal Supervisor acknowledge that the information provided is accurate and has been agreed to by all other authors.</p>
<div class="line-block">__________________       __________________<br>
  Candidate                                 Principal Supervisor</div>
<div style="page-break-after: always;"></div>
<p><strong>Chapter linking statement:</strong>
This chapter comprises a collaborative research project with the University of Tasmania. The Tasmanian devil is the largest extant carnivore and experience devastating declines due to a transmissible cancer known as devil facial tumour disease (DFTD). In this study we provide the first molecular survey of blood parasites in the Tasmanian devil. This research provide important important insights into the diversity of haemoprotozoa and ground work for future research to further understand the clinical impacts and interactions with DFTD. In addition in the broader context of this thesis this research is also of interest into zoonoses, and builds on the theme of wildlife surveillance. As as large carnivore devils fill a unique niche and could serve as an important reservoir for zoonotic pathogens. These findings further add to research that shows Australian wildlife harbor unique parasites that are endemic to Australia and surrounds, adding the growing research that Australia is free of many known pathogens described over seas.</p>
<p><strong>Acknowledgement statement:</strong>
We thank Sarah Munns for extraction of DNA and A/Prof Chris Peacock (The University of Western Australia) for provision of <em>Leishmania macropodum</em> control. We acknowledge the generous support of the large number of volunteers who assisted with fieldwork and data collection.</p>
<p><strong>Funding statement:</strong> This study was part-funded by the Australian Research Council (LP160100200), Bayer HealthCare (Germany) and Bayer Australia. S.L.E. was supported by an Australian Government Research Training Program (RTP) Scholarship. This project was also part supported by The Holsworth Wildlife Research Endowment &amp; The Ecological Society of Australia (awarded to S.C. and S.L.E). Fieldwork and data collection were funded by the US National Institute of Health (R01-GM126563-01), the Save the Tasmanian Devil Appeal of the University of Tasmania Foundation (Eric Guiler Grant Scheme) and the Australian Research Council (DE170101116).</p>
<p><strong>Data availability:</strong> Sequences generated in the present study have been submitted to GenBank nucleotide database under accession numbers MT883295 – MT883326 (trypanosome <em>18S rRNA</em>), MT514664 – MT514666 (trypanosome <em>gGAPDH</em>) and MW084364 (<em>Babesia</em> <em>18S rRNA</em>).</p>
<p><strong>Keywords:</strong> Tasmanian devil; <em>Sarcophilus harrisii</em>; Haemoprotozoa; <em>Trypanosoma</em>; Marsupial; Devil facial tumour disease (DFTD)</p>
<div style="page-break-after: always;"></div>
</div>
<div id="abstract-5" class="section level2" number="6.1">
<h2>
<span class="header-section-number">6.1</span> Abstract<a class="anchor" aria-label="anchor" href="#abstract-5"><i class="fas fa-link"></i></a>
</h2>
<p>The impact of emerging infectious diseases is increasingly recognised as a major threat to wildlife. Wild populations of the endangered Tasmanian devil, <em>Sarcophilus harrisii</em>, are experiencing devastating losses from a novel transmissible cancer, devil facial tumour disease (DFTD), however, despite the rapid decline of this species there is currently no information on the presence of haemoprotozoan parasites. In the present study, 95 Tasmanian devil blood samples were collected from four populations in Tasmania, Australia, which underwent molecular screening to detect four major groups of haemoprotozoa: (i) trypanosomes, (ii) piroplasms, (iii) <em>Hepatozoon</em> and (iv) haemosporidia. Sequence results revealed <em>Trypanosoma</em> infections in 32/95 individuals. <em>Trypanosoma copemani</em> was identified in 10 Tasmanian devils from three sites and a second <em>Trypanosoma</em> sp. was identified in 22 individuals that grouped within the poorly described <em>Tr. cyclops</em> clade. A single blood sample was positive for <em>Babesia</em> sp. that most closely matched <em>Babesia lohae</em>. No other blood protozoan parasite DNA was detected. This study provides the first insight into haemoprotozoa from the Tasmanian devil and the first identification of <em>Trypanosoma</em> and <em>Babesia</em> in this carnivorous marsupial.</p>
</div>
<div id="introduction-4" class="section level2" number="6.2">
<h2>
<span class="header-section-number">6.2</span> Introduction<a class="anchor" aria-label="anchor" href="#introduction-4"><i class="fas fa-link"></i></a>
</h2>
<p>Haemoprotozoan parasites are unicellular eukaryotic organisms with complex lifecycles involving an invertebrate vector and often alternate in their tropism between the tissues and the blood of their vertebrate hosts. Four major haemoprotozoan assemblages infect mammals <span class="citation">(<a href="references.html#ref-odonoghueHaemoprotozoaMakingBiological2017" role="doc-biblioref">P. O’Donoghue 2017</a>)</span>; (i) Trypanosomatids (Kinetoplastea), which are flagellated protists characterised by the presence of a unique organelle, the kinetoplast; (ii) haemogregarines (Adeleorina); (iii) haemosporidia (Haemosporidia); and (iv) piroplasms (Piroplasmida). Broadly haemoprotozoans are considered either host specific (e.g. <em>Theileria ornithorhynchi</em> infecting the platypus (<em>Ornithorhynchus anatinus</em>) host <span class="citation">(<a href="references.html#ref-papariniFirstMolecularCharacterization2015" role="doc-biblioref">A. Paparini et al. 2015</a>)</span>) or generalists (e.g. <em>Trypanosoma cruzi</em> which infects a wide range of mammals <span class="citation">(<a href="references.html#ref-clementOutAfricaOrigins2020" role="doc-biblioref">Clement et al. 2020</a>)</span>). However, the true diversity and epidemiology of most of these species remains unknown.</p>
<p>The Tasmanian devil, <em>Sarcophilus harrisii</em>, is the largest extant marsupial carnivore. Once present across mainland Australia (~3,000 years ago), the distribution of wild populations of Tasmanian devils became restricted to the island of Tasmania <span class="citation">(<a href="references.html#ref-bruniche-olsenAncientDNATracks2018" role="doc-biblioref">Bruniche-Olsen et al. 2018</a>)</span>, off the southern coast of the Australian mainland. Biogeographical events have influenced declines in the effective size of devil populations and their distribution, with consequential genetic bottleneck resulting in low genetic diversity <span class="citation">(<a href="references.html#ref-jonesGeneticDiversityPopulation2004" role="doc-biblioref">M. E. Jones et al. 2004</a>; <a href="references.html#ref-bruniche-olsenAncientDNATracks2018" role="doc-biblioref">Bruniche-Olsen et al. 2018</a>; <a href="references.html#ref-pattonContemporaryDemographicReconstruction2019" role="doc-biblioref">Patton et al. 2019</a>)</span>. Since 1996, a transmissible cancer (devil tumour facial disease, DFTD hereafter) gradually spread across most of the distributional range of the species and caused local population declines of upwards of 80% <span class="citation">(<a href="references.html#ref-mccallumTransmissionDynamicsTasmanian2009" role="doc-biblioref">McCallum et al. 2009</a>; <a href="references.html#ref-lazenbyDensityTrendsDemographic2018" role="doc-biblioref">Lazenby et al. 2018</a>)</span>. Transmission of DFTD occurs through direct transfer of live tumour cells between individuals when devils bite each other during social interactions <span class="citation">(<a href="references.html#ref-pearseAllograftTheoryTransmission2006" role="doc-biblioref">Pearse and Swift 2006</a>; <a href="references.html#ref-hamedeBitingInjuriesTransmission2013" role="doc-biblioref">Hamede, McCallum, and Jones 2013</a>; <a href="references.html#ref-hamiltonRateIntersexualInteractions2019" role="doc-biblioref">D. G. Hamilton et al. 2019</a>)</span>. Survival in DFTD infected individuals is usually 6-12 months after clinical signs appear; the disease is lethal in almost 100% of cases <span class="citation">(<a href="references.html#ref-pyeDevilFacialTumor2016" role="doc-biblioref">Pye, Woods, and Kreiss 2016</a>)</span>. As a consequence of transmission being driven by social interactions, DFTD is still present even in largely depleted populations of hosts <span class="citation">(<a href="references.html#ref-mccallumTransmissionDynamicsTasmanian2009" role="doc-biblioref">McCallum et al. 2009</a>)</span>. Therefore, any additional pressure(s) on the host health, such as co-infections with haemoparasites, could further threaten imperilled populations, yet, little is known about the devil’s parasite community <span class="citation">(<a href="references.html#ref-waitReviewParasitesTasmanian2017" role="doc-biblioref">Wait, Peck, et al. 2017</a>)</span>.</p>
<p>To date, only four protozoan parasite groups have been identified from wild devil populations; <em>Cryptosporidium</em> spp., <em>Giardia</em> spp., <em>Toxoplasma gondii</em> and <em>Sarcocystis</em> sp. A previous study identified sporulated sporocysts consistent with <em>Sarcocystis</em> (family: Sarcocystidae) species from the intestinal mucosa using microscopic examination <span class="citation">(<a href="references.html#ref-mundaySarcocystisRelatedOrganisms1978" role="doc-biblioref">Munday et al. 1978</a>)</span>. In north-west Tasmania, <em>T. gondii</em> prevalence was higher in carnivorous marsupials such as the Tasmanian devil, spotted-tail quoll (<em>Dasyurus maculatus</em>) and eastern quoll (<em>D. viverrinus</em>), compared to sympatric prey marsupials like the brushtail possum (<em>Trichosurus vulpecula</em>), Bennett’s wallaby (<em>Notamacropus rufogriseus</em>, syn. <em>Macropus rufogriseus</em>) and Tasmanian pademelon (<em>Thylogale billardierii</em>) <span class="citation">(<a href="references.html#ref-hollingsWildlifeDiseaseEcology2013" role="doc-biblioref">Hollings et al. 2013</a>)</span>. Recently, <em>Cryptosporidium</em> and <em>Giardia</em> spp. were also identified from faecal samples of wild devils using molecular techniques <span class="citation">(<a href="references.html#ref-waitMolecularCharacterizationCryptosporidium2017" role="doc-biblioref">Wait, Fox, et al. 2017</a>)</span>.</p>
<p>Over the past two decades the use of molecular techniques has significantly extended our understanding of haemoprotozoan parasites. As a result, it has revealed that Australian marsupials and monotremes can host a high diversity of haemoprotozoa species, most of which are unique to Australia <span class="citation">(<a href="references.html#ref-thompsonTrypanosomesAustralianMammals2014" role="doc-biblioref">C. K. Thompson, Godfrey, and Thompson 2014</a>; <a href="references.html#ref-austenInvestigationMorphologicalDiversity2015" role="doc-biblioref">Austen et al. 2015</a>; <a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>; <a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">Amy S. Northover et al. 2019</a>)</span>. The clinical effects of co-infections in wildlife are often difficult to evaluate. One well documented case is the investigation into a canine distemper virus epidemic in Serengeti lions (<em>Panthera leo</em>), which identified that joint infection by haemoprotozoa was a major contributing factor to fatal outcomes <span class="citation">(<a href="references.html#ref-munsonClimateExtremesPromote2008" role="doc-biblioref">Munson et al. 2008</a>)</span>. While there are limited studies on the clinical and pathological consequences of such interactions in native Australian wildlife, there is evidence that co-infection and co-morbidities can place species at an increased risk of disease when challenged with infection of haemoprotozoa <span class="citation">(<a href="references.html#ref-mcinnesPotentialImpactNative2011" role="doc-biblioref">McInnes, Gillett, et al. 2011</a>; <a href="references.html#ref-boteroTrypanosomesGeneticDiversity2013" role="doc-biblioref">A. Botero et al. 2013</a>; <a href="references.html#ref-thompsonTemporalSpatialDynamics2014" role="doc-biblioref">C. K. Thompson et al. 2014</a>)</span>.</p>
<p>The present study aimed to screen blood samples from Tasmanian devils for the presence of haemoprotozoan parasites. To the authors’ knowledge the present study provides the first survey of haemoprotozoa from Tasmanian devils.</p>
</div>
<div id="materials-and-methods" class="section level2" number="6.3">
<h2>
<span class="header-section-number">6.3</span> Materials and Methods<a class="anchor" aria-label="anchor" href="#materials-and-methods"><i class="fas fa-link"></i></a>
</h2>
<div id="study-sites-and-sampling" class="section level3" number="6.3.1">
<h3>
<span class="header-section-number">6.3.1</span> Study sites and sampling<a class="anchor" aria-label="anchor" href="#study-sites-and-sampling"><i class="fas fa-link"></i></a>
</h3>
<p>Animal use was approved by the University of Tasmania Animal Ethics Committee permit numbers A0015835 and A0016789) and the Department of Primary Industries, Parks, Water and Environment Animal Ethics Committee (permit numbers TFA19144 and TFA18028).</p>
<p>A total of 95 blood samples were collected from wild Tasmanian devils captured during the austral autumn (May 2018) from four sites across Tasmania: Black River, Takone, West Pencil Pine and Freycinet (Figure <a href="tas-devil.html#fig:F61">6.1</a>). The sampling effort at each site involved deployment of 40 PVC pipe traps during 7 to 10 consecutive nights. Individual devils were identified via subcutaneously implanted microchips (AllFlex<span class="math inline">\(\copyright\)</span> ISO FDX-B).</p>
<p>Blood was collected (between 0.3-1 mL) from either the jugular vein (Takone) or marginal ear vein (Black River, Freycinet and West Pencil Pine). The puncture site was disinfected with sterile alcohol swabs for at least 15 seconds prior to collection and blood was stored in ethylenediaminetetraacetic acid (EDTA) vacutainers. Samples were kept refrigerated at 4<span class="math inline">\(^\circ\)</span>C and shipped to Murdoch University, Western Australia where they were stored at -20<span class="math inline">\(^\circ\)</span>C until analysis.</p>
<div class="figure">
<span style="display:block;" id="fig:F61"></span>
<img src="figures/ms-figs/Ch6-Fig1.png" alt="Map of study sites for field collection of Tasmanian Devil \textit{Sarcophilus harrisii}." width="95%"><p class="caption">
Figure 6.1: Map of study sites for field collection of Tasmanian Devil .
</p>
</div>
</div>
<div id="molecular-screening" class="section level3" number="6.3.2">
<h3>
<span class="header-section-number">6.3.2</span> Molecular screening<a class="anchor" aria-label="anchor" href="#molecular-screening"><i class="fas fa-link"></i></a>
</h3>
<div id="dna-extraction" class="section level4" number="6.3.2.1">
<h4>
<span class="header-section-number">6.3.2.1</span> DNA extraction<a class="anchor" aria-label="anchor" href="#dna-extraction"><i class="fas fa-link"></i></a>
</h4>
<p>Total genomic DNA was extracted from 200 <span class="math inline">\(\mu\)</span>L of blood using a MasterPure DNA purification kit (EpicentreBiotechnologies, Madison, Wisconsin, U.S.A) following the manufacturer’s recommendations. Where 200 <span class="math inline">\(\mu\)</span>L of blood was not available, sterile DNA free phosphate-buffered saline (PBS) was used to make samples up to 200 <span class="math inline">\(\mu\)</span>L. Genomic DNA (gDNA) was eluted in 30 <span class="math inline">\(\mu\)</span>L of TE buffer and stored at -20<span class="math inline">\(^\circ\)</span>C. Extraction controls (EXBs) consisting of 200 <span class="math inline">\(\mu\)</span>L sterile DNA free PBS, were included randomly in each extraction batch (N=7).</p>
</div>
<div id="pcr-assays" class="section level4" number="6.3.2.2">
<h4>
<span class="header-section-number">6.3.2.2</span> PCR assays<a class="anchor" aria-label="anchor" href="#pcr-assays"><i class="fas fa-link"></i></a>
</h4>
<p>Molecular screening was carried out for haemoprotozoa on gDNA from blood samples (N=95) and EXBs (N=7). Reactions were carried out in 25 <span class="math inline">\(\mu\)</span>L volumes containing 2 <span class="math inline">\(\mu\)</span>L of gDNA or 1 <span class="math inline">\(\mu\)</span>L of the primary product for nested and semi-nested PCR assays. An overview of assays, including primer sequences, is available in Table <a href="tas-devil.html#tab:T61">6.1</a>.</p>

<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T61">Table 6.1: </span>List of primers used for screening of haemoprotozoa from Tasmanian Devils (<em>Sarcophilus harrisii</em>). References: <strong>1.</strong> McInnes et al. <span class="citation">(<a href="references.html#ref-mcinnesTrypanosomaIrwiniSp2009" role="doc-biblioref">2009</a>)</span>, <strong>2.</strong> Maslov et al. <span class="citation">(<a href="references.html#ref-maslovPhylogenyTrypanosomesInferred1996" role="doc-biblioref">1996</a>)</span>, <strong>3.</strong> Jefferies et al. <span class="citation">(<a href="references.html#ref-jefferiesPCRRFLPDetectionDifferentiation2007" role="doc-biblioref">2007</a>)</span>, <strong>4.</strong> Ujvari et al. <span class="citation">(<a href="references.html#ref-ujvariHighPrevalenceHepatozoon2004" role="doc-biblioref">2004</a>)</span>, <strong>5.</strong> Waldenstrom et al. <span class="citation">(<a href="references.html#ref-waldenstromNewNestedPolymerase2004" role="doc-biblioref">2004</a>)</span>, <strong>6.</strong> Bensch et al. <span class="citation">(<a href="references.html#ref-benschHostSpecificityAvian2000" role="doc-biblioref">2000</a>)</span>.
</caption>
<thead><tr>
<th style="text-align:left;">
Target
</th>
<th style="text-align:left;">
Primer
</th>
<th style="text-align:left;">
Sequence
</th>
<th style="text-align:left;">
Size
</th>
<th style="text-align:right;">
Reference
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;">
Trypanosomes
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:right;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
18S primary SSU
</td>
<td style="text-align:left;">
SLF
</td>
<td style="text-align:left;">
GCTTGTTTCAAGGACTTAGC
</td>
<td style="text-align:left;">
~1.5 kb
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
18S primary SSU
</td>
<td style="text-align:left;">
S762R
</td>
<td style="text-align:left;">
GACTTTTGCTTCCTCTAATG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
18S secondary SSU2
</td>
<td style="text-align:left;">
S823F
</td>
<td style="text-align:left;">
CGAACAACTGCCCTATCAGC
</td>
<td style="text-align:left;">
~904 bp
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
18S secondary SSU2
</td>
<td style="text-align:left;">
662R
</td>
<td style="text-align:left;">
GACTACAATGGTCTCTAATC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
18S secondary SSU1
</td>
<td style="text-align:left;">
S825F
</td>
<td style="text-align:left;">
ACCGTTTCGGCTTTTGTTGG
</td>
<td style="text-align:left;">
~959 bp
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
18S secondary SSU1
</td>
<td style="text-align:left;">
SLIR
</td>
<td style="text-align:left;">
ACATTGTAGTGCGCGTGTC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
GAPDH Primer
</td>
<td style="text-align:left;">
GAPDHF
</td>
<td style="text-align:left;">
CTYMTCGGNAMKGAGATYGAY
</td>
<td style="text-align:left;">
~ 900 bp
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
GAPDH Primer
</td>
<td style="text-align:left;">
GAPDHR
</td>
<td style="text-align:left;">
GRTKSGARTADCCCCACTCG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
GAPDH Secondary
</td>
<td style="text-align:left;">
GAPDHF
</td>
<td style="text-align:left;">
CTYMTCGGNAMKGAGATYGAY
</td>
<td style="text-align:left;">
~ 880 bp
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
GAPDH Secondary
</td>
<td style="text-align:left;">
Ga4
</td>
<td style="text-align:left;">
GTTYTGCAGSGTCGCCTTGG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
1
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Piroplasms
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:right;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
18S primary
</td>
<td style="text-align:left;">
BT1F
</td>
<td style="text-align:left;">
GGCTCATTACAACAGTTATAG
</td>
<td style="text-align:left;">
~ 900 bp
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
18S primary
</td>
<td style="text-align:left;">
BT1R
</td>
<td style="text-align:left;">
CCCAAAGACTTTGATTTCTCTC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
18S secondary
</td>
<td style="text-align:left;">
BT2F
</td>
<td style="text-align:left;">
CCGTGCTAATTGTAGGGCTAAT
</td>
<td style="text-align:left;">
~ 800 bp
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
18S secondary
</td>
<td style="text-align:left;">
BT2R
</td>
<td style="text-align:left;">
GGACTACGACGGTATCTGATCG
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
3
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Hepatozoon
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:right;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
18S
</td>
<td style="text-align:left;">
HepF300
</td>
<td style="text-align:left;">
GTTTCTGACCTATCAGCTTTCGACG
</td>
<td style="text-align:left;">
~600 bp
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;">
18S
</td>
<td style="text-align:left;">
Hep900
</td>
<td style="text-align:left;">
CAAATCTAAGAATTTCACCTCTGAC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
4
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Haemosporidia
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:right;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Cytb primary
</td>
<td style="text-align:left;">
HaemNF
</td>
<td style="text-align:left;">
CATATATTAAGAGAATTATGGAG
</td>
<td style="text-align:left;">
~580 bp
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
Cytb primary
</td>
<td style="text-align:left;">
HaemNR2
</td>
<td style="text-align:left;">
AGAGGTGTAGCATATCTATCTAC
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
Cytb secondary
</td>
<td style="text-align:left;">
HaemF
</td>
<td style="text-align:left;">
ATGGTGCTTCGATATATGCATG
</td>
<td style="text-align:left;">
~520 bp
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
Cytb secondary
</td>
<td style="text-align:left;">
HaemR2
</td>
<td style="text-align:left;">
GCATTATCTGGATGTGATAATGGT
</td>
<td style="text-align:left;">
</td>
<td style="text-align:right;">
6
</td>
</tr>
</tbody>
</table></div>
<p>For identification of trypanosomes (<em>Trypanosoma</em> and <em>Leishmania</em> spp.) samples were screened using a nested PCR assay targeting an ~959 bp product of the second half of the 18S ribosomal RNA (<em>18S rRNA</em>) gene. This assay amplifies most species of <em>Trypanosoma</em> <span class="citation">(<a href="references.html#ref-mcinnesTrypanosomaIrwiniSp2009" role="doc-biblioref">McInnes et al. 2009</a>)</span> and was also validated in the present study to amplify <em>Leishmania</em> spp. using control gDNA of <em>Leishmania infantum</em> from a canine bone marrow sample and <em>Leishmania macropodum</em> culture isolate. External primers SLF/S762R and internal primers S825F/SLIR were used as an initial screen and sequenced as per details below. Samples representative of the different genotypes identified then underwent an additional secondary assay with a second set of internal primers S823/S662R to yield a near full length <em>18S rRNA</em> sequence. Reactions were carried out in 25 <span class="math inline">\(\mu\)</span>L volumes containing 0.4 <span class="math inline">\(\mu\)</span>M of each primer and 12.5 <span class="math inline">\(\mu\)</span>L GoTaqGreen master mix (Promega, U.S.A.). Thermal cycling conditions were as follows; initial cycle of 95<span class="math inline">\(^\circ\)</span>C for 5 mins, 50<span class="math inline">\(^\circ\)</span>C for 2 mins, 72<span class="math inline">\(^\circ\)</span>C for 4 mins, followed by 35 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 52<span class="math inline">\(^\circ\)</span>C (primary) or 55<span class="math inline">\(^\circ\)</span>C (secondary) for 30 secs, 72<span class="math inline">\(^\circ\)</span>C for 2 mins 20 secs (primary) or 1 min (secondary), and a final extension of 72<span class="math inline">\(^\circ\)</span>C for 5 mins. To obtain further phylogenetic information from the genotypes identified by <em>18S rRNA</em> screening, samples underwent amplification of the glycosomal Glyceraldehyde Phosphate Dehydrogenase (<em>gGAPDH</em>) gene using a semi-nested PCR assay with external primers GAPDHF/GAPDHR and internal primers GAPDHF/Ga4 <span class="citation">(<a href="references.html#ref-mcinnesTrypanosomaIrwiniSp2009" role="doc-biblioref">McInnes et al. 2009</a>)</span>. Reactions contained 1X buffer (KAPA Biosystems, South Africa), 2.0 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu\)</span>M of each primer, 0.25 mM of each dNTP, and 0.5 U of Taq (KAPA Biosystems, South Africa). Thermal cycling conditions were as follows; initial cycle of 95<span class="math inline">\(^\circ\)</span>C for 5 mins, 50<span class="math inline">\(^\circ\)</span>C for 2 mins, 72<span class="math inline">\(^\circ\)</span>C for 4 mins, followed by 35 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 52<span class="math inline">\(^\circ\)</span>C (primary) or 55<span class="math inline">\(^\circ\)</span>C (secondary) for 30 secs, 72<span class="math inline">\(^\circ\)</span>C for 2 mins 20 secs, and a final extension of 72<span class="math inline">\(^\circ\)</span>C for 5 mins.</p>
<p>Screening for piroplasms (<em>Babesia</em> and <em>Theileria</em> spp.) utilised a previously described nested PCR assay. Primers were used targeting an ~800 bp product of the <em>18S rRNA</em> with external primers BT1F/BT1R and internal primers BT2F/BT2R <span class="citation">(<a href="references.html#ref-jefferiesPCRRFLPDetectionDifferentiation2007" role="doc-biblioref">Jefferies, Ryan, and Irwin 2007</a>)</span>. Reactions contained 1X buffer (KAPA Biosystems, South Africa), 2.0 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu\)</span>M of each primer, 0.25 mM of each dNTP, and 0.5 U of Taq (KAPA Biosystems, South Africa). Thermal cycling conditions were as follows; initial cycle of 95<span class="math inline">\(^\circ\)</span>C for 2 mins, 58<span class="math inline">\(^\circ\)</span>C for 1 min, 72<span class="math inline">\(^\circ\)</span>C for 2 mins, followed by 35 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 62<span class="math inline">\(^\circ\)</span>C for 20 secs, 72<span class="math inline">\(^\circ\)</span>C for 45 secs, and a final extension of 72<span class="math inline">\(^\circ\)</span>C for 7 mins.</p>
<p>A genus specific assay for <em>Hepatozoon</em> targeting the <em>18S rRNA</em> was used with primers HepF300/Hep900 to amplify an ~600 bp product <span class="citation">(<a href="references.html#ref-ujvariHighPrevalenceHepatozoon2004" role="doc-biblioref">Ujvari, Madsen, and Olsson 2004</a>)</span>. Reactions contained 1X buffer (KAPA Biosystems, South Africa), 1.5mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu\)</span>M of each primer, 0.25 mM of each dNTP, and 0.5 U of Taq (KAPA Biosystems, South Africa). Thermal cycling conditions were as follows; initial denaturation of 95<span class="math inline">\(^\circ\)</span>C for 3 mins, followed by 40 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 60<span class="math inline">\(^\circ\)</span>C for 30 secs, 72<span class="math inline">\(^\circ\)</span>C for 1 min, and a final extension of 72<span class="math inline">\(^\circ\)</span>C for 10 mins.</p>
<p>Screening for haemosporidia (<em>Plasmodium</em> spp.) utilised a previously described nested PCR assay <span class="citation">(<a href="references.html#ref-waldenstromNewNestedPolymerase2004" role="doc-biblioref">Waldenstrom et al. 2004</a>)</span>. Primers targeting an ~420 bp product of the <em>cytochrome b</em> (<em>cytb</em>) gene with external primers HAEMNF/HAEMNR2 and internal primers HAEMF/HAEMR2. Reactions contained 1X buffer (KAPA Biosystems, South Africa), 1.5 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu\)</span>M of each primer, 0.25 mM of each dNTP, and 1.0 U of Taq (KAPA Biosystems, South Africa). Thermal cycling conditions were as follows; initial denaturation of 95<span class="math inline">\(^\circ\)</span>C for 8 mins, followed by 35 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 50<span class="math inline">\(^\circ\)</span>C (primary) or 52<span class="math inline">\(^\circ\)</span>C (secondary) for 30 secs, 72<span class="math inline">\(^\circ\)</span>C for 45 secs, and a final extension of 72<span class="math inline">\(^\circ\)</span>C for 10 mins.</p>
<p>Suitable positive controls, extraction reagent blanks, and no-template PCR controls were included throughout the laboratory processes. Extractions, pre-PCR and post-PCR procedures were performed in laboratories physically separated from each other.</p>
</div>
<div id="gel-electrophoresis-and-sanger-sequencing" class="section level4" number="6.3.2.3">
<h4>
<span class="header-section-number">6.3.2.3</span> Gel electrophoresis and Sanger sequencing<a class="anchor" aria-label="anchor" href="#gel-electrophoresis-and-sanger-sequencing"><i class="fas fa-link"></i></a>
</h4>
<p>Amplicons were electrophoresed on a 1% agarose gel stained with SYBR safe (Invitrogen, U.S.A.). Products of the correct size (see Table <a href="tas-devil.html#tab:T61">6.1</a>) were excised from the gel and purified using previously described methods <span class="citation">(<a href="references.html#ref-yangSpecificQuantitativeDetection2013" role="doc-biblioref">R. Yang et al. 2013</a>)</span>. Sanger sequencing was carried out in forward and reverse directions on all positive amplicons. Sequencing was performed at the Australian Genome Research Facility (Perth, Western Australia) on an Applied Biosystems 3730 using Big Dye Terminator chemistry version 3.1.</p>
</div>
<div id="phylogenetic-analysis" class="section level4" number="6.3.2.4">
<h4>
<span class="header-section-number">6.3.2.4</span> Phylogenetic analysis<a class="anchor" aria-label="anchor" href="#phylogenetic-analysis"><i class="fas fa-link"></i></a>
</h4>
<p>Sequences were imported and trimmed in Geneious 10.2.6 (<a href="https://www.geneious.com" class="uri">https://www.geneious.com</a>) and then subjected to BLAST analysis using BLASTN 2.10.0+ <span class="citation">(<a href="references.html#ref-zhangGreedyAlgorithmAligning2000" role="doc-biblioref">Zhang et al. 2000</a>)</span> against nucleotide collection (nt) database <span class="citation">(<a href="references.html#ref-morgulisDatabaseIndexingProduction2008" role="doc-biblioref">Morgulis et al. 2008</a>)</span> to identify the most similar species and genotypes. Reference sequences were retrieved from GenBank <span class="citation">(<a href="references.html#ref-bensonGenBank2017" role="doc-biblioref">Benson et al. 2017</a>)</span> (details available in Table <a href="supplementary-material.html#tab:TA61">A.5</a>) and aligned with sequences obtained in the present study using MUSCLE <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span>. Alignments of the <em>18S rRNA</em> and the <em>gGAPDH</em> sequences were then used for phylogenetic purposes. Phylogenies were inferred using the maximum likelihood (ML) method. The optimal evolutionary model was selected using ModelFinder <span class="citation">(<a href="references.html#ref-kalyaanamoorthyModelFinderFastModel2017" role="doc-biblioref">Kalyaanamoorthy et al. 2017</a>)</span> based on bayesian information criterion. Phylogenetic analysis was performed in IQ-TREE v1.6.11 <span class="citation">(<a href="references.html#ref-nguyenIQTREEFastEffective2015" role="doc-biblioref">Nguyen et al. 2015</a>)</span> and bootstrap support was calculated using ultrafast (UFBoot2) method with 10,000 replicates <span class="citation">(<a href="references.html#ref-hoangUFBoot2ImprovingUltrafast2018" role="doc-biblioref">Hoang et al. 2018</a>)</span>.</p>
<p>Maximum likelihood phylogeny based on the 18S rRNA locus was performed using a 1,505 bp alignment based on the transition (AC=CG, AT=GT with equal base frequencies) (TIM3e) substitution model <span class="citation">(<a href="references.html#ref-posadaUsingMODELTESTPAUP2003" role="doc-biblioref">Posada 2003</a>)</span>, with invariable sites (I=0.408) and a discrete gamma distribution (four categories) (G4=0.438) <span class="citation">(<a href="references.html#ref-guMaximumLikelihoodEstimation1995" role="doc-biblioref">Gu, Fu, and Li 1995</a>)</span>. To include a wider range of neighbouring reference sequences, a second phylogenetic analysis was performed on the V7-8 hypervariable region of the 18S rRNA gene; this region is useful to differentiate between closely related trypanosome sequences <span class="citation">(<a href="references.html#ref-hamiltonResolvingRelationshipsAustralian2011" role="doc-biblioref">Patrick B. Hamilton and Stevens 2011</a>)</span>. A phylogeny was produced using a 559 bp alignment based on the Kimura Two-Parameter (K2P) substitution model <span class="citation">(<a href="references.html#ref-kimuraSimpleMethodEstimating1980" role="doc-biblioref">Kimura 1980</a>)</span> with discrete gamma distribution (four categories) (G4=0.227) <span class="citation">(<a href="references.html#ref-guMaximumLikelihoodEstimation1995" role="doc-biblioref">Gu, Fu, and Li 1995</a>)</span>. A 767 bp alignment of the <em>gGAPDH</em> gene was used for phylogenetic reconstruction based on the Tamura-Nei (TN) substitution model <span class="citation">(<a href="references.html#ref-tamuraEstimationNumberNucleotide1993" role="doc-biblioref">Tamura and Nei 1993</a>)</span>, with empirical base frequencies (F), invariable sites (I=0.387) and discrete gamma distribution (four categories) (G4=0.893) <span class="citation">(<a href="references.html#ref-guMaximumLikelihoodEstimation1995" role="doc-biblioref">Gu, Fu, and Li 1995</a>)</span>. Genetic sequence similarity was calculated using the Kimura Two-Parameter method <span class="citation">(<a href="references.html#ref-tamuraEstimationNumberNucleotide1993" role="doc-biblioref">Tamura and Nei 1993</a>)</span>.</p>
<p>Sequences generated in the present study have been submitted to GenBank nucleotide database under accession numbers MT883295–MT883326 (<em>Trypanosoma</em> <em>18S rRNA</em>), MT514664–MT514666 (<em>Trypanosoma</em> <em>gGAPDH</em>) and MW084364 (<em>Babesia</em> <em>18S rRNA</em>).</p>
</div>
</div>
<div id="statisical-analysis" class="section level3" number="6.3.3">
<h3>
<span class="header-section-number">6.3.3</span> Statisical analysis<a class="anchor" aria-label="anchor" href="#statisical-analysis"><i class="fas fa-link"></i></a>
</h3>
<p>The overall prevalence of <em>Trypanosoma</em> species was compared between males and females using two-tailed Fisher Exact test.</p>
</div>
<div id="microscopy-of-blood-smears" class="section level3" number="6.3.4">
<h3>
<span class="header-section-number">6.3.4</span> Microscopy of blood smears<a class="anchor" aria-label="anchor" href="#microscopy-of-blood-smears"><i class="fas fa-link"></i></a>
</h3>
<p>Thin blood smears were prepared from animals sampled at Takone site only (three per individual), within 4 hours of collection, and then air dried and fixed in methanol. Blood smears were then stained with modified Wright-Giemsa (HematekStain Pak) using a Hema-Tek Slide Stainer (Ames Company Division, Miles Laboratories Pty Ltd., Victoria, Australia) and a coverslip was mounted using DPX neutral mounting medium (LabChem, Victoria, Australia). Smears were inspected by light microscopy (Olympus BX51) for the presence of haemoprotozoa at x 400 magnification and under oil immersion (x 1000).</p>
</div>
</div>
<div id="results-2" class="section level2" number="6.4">
<h2>
<span class="header-section-number">6.4</span> Results<a class="anchor" aria-label="anchor" href="#results-2"><i class="fas fa-link"></i></a>
</h2>
<div id="molecular-screening-1" class="section level3" number="6.4.1">
<h3>
<span class="header-section-number">6.4.1</span> Molecular screening<a class="anchor" aria-label="anchor" href="#molecular-screening-1"><i class="fas fa-link"></i></a>
</h3>
<p>Molecular screening identified 33.7% (32/95) of the blood samples were positive for <em>Trypanosoma</em> DNA. Sequence identity from BLAST results identified; <em>Tr. copemani</em> and <em>Tr. cyclops</em>-like in 10.5% (n=10) and 23.2% (n=22) of the individuals respectively (Table <a href="tas-devil.html#tab:T62">6.2</a>). <em>Trypanosoma</em> <em>copemani</em> was identified almost exclusively in males (90.0%, P=0.0159), while <em>Tr. cyclops</em>-like genotypes were more commonly found in females (68.2%, P=0.0432). All samples tested negative for presence of <em>Leishmania</em>, <em>Hepatozoon</em>, <em>Plasmodium</em> and <em>Theileria</em> species.</p>
<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T62">Table 6.2: </span>Overview of Tasmanian devil sampling across survey sites showing number of individuals samples (n) and number of individuals positive for species.
</caption>
<thead><tr>
<th style="text-align:left;">
Site
</th>
<th style="text-align:right;">
n
</th>
<th style="text-align:right;">
Trypanosoma spp.
</th>
<th style="text-align:right;">
T. copemani
</th>
<th style="text-align:right;">
T. cyclops-like
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Black River
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
Takone
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
West Pencil Pine
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
Freycinet
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Overall
</td>
<td style="text-align:right;font-weight: bold;">
95
</td>
<td style="text-align:right;font-weight: bold;">
32
</td>
<td style="text-align:right;font-weight: bold;">
10
</td>
<td style="text-align:right;font-weight: bold;">
22
</td>
</tr>
</tbody>
</table></div>
<p>Comparison at the <em>18S rRNA</em> V7-8 hypervariable region of the ten <em>Tr. copemani</em> sequences showed they were all &gt;99% similar to each other, and as such a representative sample (BRI115) was used for subsequent phylogeny.
Analysis of long <em>18S rRNA</em> alignment concluded that BRI115 sample shared 99.1% similarity to <em>Tr. copemani</em> Charlton (GU966588) and 98.5% with <em>Tr. copemani</em> H26 (AJ009169) (Figure <a href="tas-devil.html#fig:F62">6.2</a>).
At the <em>gGAPDH</em> gene BRI115 sample was identical to <em>Tr. copemani</em> Mika (GU966585), and 99.7% similar to <em>Tr. copemani</em> AAP (AJ620277) (Figure <a href="tas-devil.html#fig:F63">6.3</a>).
There is no <em>18S rRNA</em> sequence data for <em>Tr. copemani</em> Mika available.
Comparison of <em>Tr. copemani</em> Charlton (GU966584) at the <em>gGAPDH</em> gene showed it was 99.9% similar to the BRI115 sample with one single nucleotide polymorphism (SNP) (data not shown).
At the <em>18S rRNA</em> V7–8 hypervariable region two genotypes of the <em>Tr. cyclops</em> clade were identified, referred to as Tasmanian devil genotype A (n=13) and genotype B (n=9).
Representatives of each genotype were used for phylogenetic reconstruction based on a longer alignment of <em>18S rRNA</em> sequences (Figure <a href="tas-devil.html#fig:F63">6.3</a>).
Analysis of near full length <em>18S rRNA</em> alignment showed sequences were identical within each genotype (genotype A samples WPP585 and WPP601, and genotype B samples TKN211 and BRI111).
The most similar <em>18S rRNA</em> sequences to genotype A were <em>Trypanosoma</em> sp. TL.AQ.22 (AJ620574; 98.8% similar) and <em>Tr. cyclops</em> (AJ131958; 96.9% similar).
Genotype B <em>18S rRNA</em> sequences were most similar to <em>Trypanosoma</em> sp. ABF (AJ620564; 99.2% similar) and <em>Trypanosoma</em> sp. TL.AV.43 cl.101E (AJ620571; 99.1% similar).
The <em>gGAPDH</em> sequences obtained from genotype A samples WPP601 and WPP602, were 99.9% similar to each other (Figure <a href="tas-devil.html#fig:F63">6.3</a>), with just one SNP.
The most similar sequence was <em>Trypanosoma</em> sp. ABF (AJ620278), which was 97.2% and 97.4% similar to WPP601 and WPP602 respectively.
The next most similar sequence was <em>Tr. cyclops</em> (FJ649493) which was 95.7% and 95.8% similar to WPP601 and WPP602 respectively.
Unfortunately, <em>gGAPDH</em> sequences from genotype B were not successfully amplified.
Phylogenetic reconstruction utilising the V7–8 hypervariable region of the 18S rRNA gene showed support for a monophyletic <em>Tr. cyclops</em> clade (Figure <a href="tas-devil.html#fig:F64">6.4</a>).
Sequences from the present study grouped with genotypes detected from Malaysia, Sri Lanka and Australia (Queensland and Victoria) with the clear distinction of two different genotypes of <em>Tr. cyclops</em> from Tasmanian devils (Table <a href="supplementary-material.html#tab:TA62">A.6</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F62"></span>
<img src="figures/ms-figs/Ch6-Fig2.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of the \textit{Trypanosoma} genus based on a 1,505 bp alignment of the 18S rRNA gene using TIM3e + I + G4 substitution model. Node values correspond to bootstrap support / aBayes support, values less than 60 are hidden. Number of substitutions per nucleotide position is represented by the scale bar. Lineages that have been described from native Australian marsupials are denoted by orange lines. Sequences generated in the present study in bold. GenBank accession numbers for sequences are available in Supplementary Table A.5." width="95%"><p class="caption">
Figure 6.2: Maximum likelihood (ML) phylogenetic reconstruction of the genus based on a 1,505 bp alignment of the 18S rRNA gene using TIM3e + I + G4 substitution model. Node values correspond to bootstrap support / aBayes support, values less than 60 are hidden. Number of substitutions per nucleotide position is represented by the scale bar. Lineages that have been described from native Australian marsupials are denoted by orange lines. Sequences generated in the present study in bold. GenBank accession numbers for sequences are available in Supplementary Table A.5.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:F63"></span>
<img src="figures/ms-figs/Ch6-Fig3.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of the \textit{Trypanosoma} genus based on a 767 bp alignment of glycosomal Glyceraldehyde Phosphate Dehydrogenase (\textit{gGAPDH}) using TN93 + F + I + G4 substitution model. Node values correspond to bootstrap support / aBayes support, values less than 60 are hidden. Number of substitutions per nucleotide position is represented by the scale bar. Lineages that have been described from native Australian marsupials are denoted by orange lines. Sequences generated in the present study in bold. GenBank accession numbers for sequences are available in Supplementary Table A.5." width="95%"><p class="caption">
Figure 6.3: Maximum likelihood (ML) phylogenetic reconstruction of the genus based on a 767 bp alignment of glycosomal Glyceraldehyde Phosphate Dehydrogenase () using TN93 + F + I + G4 substitution model. Node values correspond to bootstrap support / aBayes support, values less than 60 are hidden. Number of substitutions per nucleotide position is represented by the scale bar. Lineages that have been described from native Australian marsupials are denoted by orange lines. Sequences generated in the present study in bold. GenBank accession numbers for sequences are available in Supplementary Table A.5.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:F64"></span>
<img src="figures/ms-figs/Ch6-Fig4.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of sequences from the \textit{Trypanosoma cyclops} clade based on a 559 bp alignment of the 18S rRNA gene across the V7-8 hypervariable region using K2P + G4 substitution model. Node values correspond to bootstrap support / aBayes support, values less than 60 are hidden. Number of substitutions per nucleotide position is represented by the scale bar. Australian states abbreviated to QLD (Queensland), TAS (Tasmania) and VIC (Victoria). GenBank accession numbers for sequences are available in Supplementary Table A.5 and pairwise distances in Supplementary Table A.6." width="95%"><p class="caption">
Figure 6.4: Maximum likelihood (ML) phylogenetic reconstruction of sequences from the clade based on a 559 bp alignment of the 18S rRNA gene across the V7-8 hypervariable region using K2P + G4 substitution model. Node values correspond to bootstrap support / aBayes support, values less than 60 are hidden. Number of substitutions per nucleotide position is represented by the scale bar. Australian states abbreviated to QLD (Queensland), TAS (Tasmania) and VIC (Victoria). GenBank accession numbers for sequences are available in Supplementary Table A.5 and pairwise distances in Supplementary Table A.6.
</p>
</div>
<p>A single sample (BRI115) obtained from Black River site was positive for a <em>Babesia</em> sp. and this individual was also positive for <em>Tr. copemani</em> infection. Analysis of an 800 bp fragment of the <em>18S rRNA</em> revealed it was similar to <em>Babesia</em> sp. identified from <em>Ixodes tasmani</em> in Queensland (MG251436; 96.0% similarity) and <em>Babesia lohae</em> from <em>Ixodes holocyclus</em> in Queensland (MG593272; 95.5% similarity).</p>
</div>
<div id="microscopy" class="section level3" number="6.4.2">
<h3>
<span class="header-section-number">6.4.2</span> Microscopy<a class="anchor" aria-label="anchor" href="#microscopy"><i class="fas fa-link"></i></a>
</h3>
<p>Microscopy of blood smears from Takone site did not yield any positive detection of haemoparasites.</p>
</div>
</div>
<div id="discussion-2" class="section level2" number="6.5">
<h2>
<span class="header-section-number">6.5</span> Discussion<a class="anchor" aria-label="anchor" href="#discussion-2"><i class="fas fa-link"></i></a>
</h2>
<p>The present study represents the first survey of haemoprotozoa from wild populations of the endangered Tasmanian devil. Our findings of <em>Trypanosoma</em> infections across all four sites suggests that this infection is widespread and potentially endemic within Tasmanian devil populations. This wide distribution of <em>Trypanosoma</em> across populations contrasts with the absence of <em>Leishmania</em>, <em>Theileria</em>, <em>Hepatozoon</em>, or <em>Plasmodium</em> species, and the low detection of <em>Babesia</em> which was only detected in a single individual.</p>
<p>The extension of the host range of <em>Tr. copemani</em> is notable and supports recent research identifying this parasite in a wide range of marsupial hosts from across Australia. To date, <em>Tr. copemani</em> has been identified in all Australian states and territories except the Northern Territory and South Australia. The identification of this parasite in Tasmanian devils is potentially significant from a health perspective as previous reports have associated it with pathological changes in the woylie (<em>Bettongia penicillata</em>) <span class="citation">(<a href="references.html#ref-boteroTrypanosomesGeneticDiversity2013" role="doc-biblioref">A. Botero et al. 2013</a>; <a href="references.html#ref-thompsonTemporalSpatialDynamics2014" role="doc-biblioref">C. K. Thompson et al. 2014</a>)</span> and in koalas (<em>Phascolarctos cinereus</em>) with co-morbidities <span class="citation">(<a href="references.html#ref-mcinnesPotentialImpactNative2011" role="doc-biblioref">McInnes, Gillett, et al. 2011</a>)</span>. With this report, there are now at least ten Australian vertebrate host records for <em>Tr. copemani</em>, therefore this <em>Trypanosoma</em> species appears to be a marsupial generalist and capable of infecting a diverse range of mammals. Further studies to map the complete distribution and host range of <em>Tr. copemani</em> will help provide insights into the co-evolution of this native trypanosome in its marsupial hosts.</p>
<p>The identification of genotypes from the <em>Trypanosoma cyclops</em> clade was unexpected. <em>Trypanosoma cyclops</em> was described from wild caught southern pig-tail macaques (<em>Macaca nemestrina</em>) in jungle areas of West Malaysia <span class="citation">(<a href="references.html#ref-weinmanTrypanosomaCyclopsSp1972" role="doc-biblioref">Weinman 1972</a>)</span> and <span class="citation">P. B. Hamilton, Stevens, Gidley, et al. (<a href="references.html#ref-hamiltonNewLineageTrypanosomes2005" role="doc-biblioref">2005</a>)</span> identified novel genotypes of this clade from Australia, Papua New Guinea, and Sri Lanka. However, since its identification, there have been very few published reports of <em>Tr. cyclops</em>. Phylogeny within the <em>Tr. cyclops</em> clade was not well resolved, as demonstrated by the polytomy in Figure <a href="tas-devil.html#fig:F64">6.4</a>. The identification of two distinct genotypes in devils that do not form a monophyletic group, and show different level of relatedness to sequences overseas highlights the uncovered diversity within this clade. The closest named species to <em>Tr. cyclops</em> is <em>Trypanosoma</em> (<em>Megatrypanum</em>) <em>theileri</em>, which has a cosmopolitan distribution and predominantly infects cattle. The most important vector of <em>Tr. theileri</em> is thought to be tabanid flies (Tabanidae) <span class="citation">(<a href="references.html#ref-hoareTrypanosomesMammalsZoological1972" role="doc-biblioref">Hoare 1972</a>; <a href="references.html#ref-werszkoMolecularDetectionMegatrypanum2020" role="doc-biblioref">Werszko et al. 2020</a>)</span>. While this trypanosome is generally considered non-pathogenic, chronic infection has been associated with the development of secondary diseases in cattle <span class="citation">(<a href="references.html#ref-rodriguesCharacterizationSplicedLeader2010" role="doc-biblioref">A. C. Rodrigues et al. 2010</a>)</span>.</p>
<p>The <em>gGAPDH</em> has been shown to be a more suitable marker than the 18S rRNA gene for determining the phylogeny within the <em>Trypanosoma</em> genus <span class="citation">(<a href="references.html#ref-hamiltonTrypanosomesAreMonophyletic2004" role="doc-biblioref">Patrick B. Hamilton et al. 2004</a>)</span>. A comparative analysis of <em>Tr. copemani</em> shows intra-specific sequence similarity within the clade is 97.0–100% and inter-specific similarity to the nearest named species, <em>Tr. gilletti</em>, is 91.3–92.3%. Intra-specific sequence similarity within the <em>Tr. cyclops</em> clade was 94.8–99.9%, while inter-specific sequence similarity to nearest named species, <em>Tr. theileri</em>, is 90.4%. Therefore, there is sufficient support that the sequences generated for devils can be attributed to <em>Tr. cyclops</em>. Differentiation of genotypes within this clade are best identified by analysis of the V7–8 hypervariable region of the <em>18S rRNA</em> (Figure <a href="tas-devil.html#fig:F64">6.4</a>, Table <a href="supplementary-material.html#tab:TA62">A.6</a>). This study further supports that <em>Tr. cyclops</em> is more closely related to sequences obtained from Australia, Papua New Guinea, and Sri Lanka than it is to <em>Tr. theileri</em>. <span class="citation">(<a href="references.html#ref-hamiltonNewLineageTrypanosomes2005" role="doc-biblioref">P. B. Hamilton, Stevens, Gidley, et al. 2005</a>)</span>. As shown in the phylogeny within the <em>Tr. cyclops</em> clade (Figure <a href="tas-devil.html#fig:F64">6.4</a>) there is no geographical or host distinction between the different genotypes, further supporting the taxonomy as a single species. Research on functional traits including growth dynamics in culture coupled with morphological analysis is needed to better understand the taxonomy of this clade. The geographical and host extensions reported here will be useful for the future work aimed at understanding the evolutionary history of the <em>Tr. cyclops</em> and <em>Tr. theileri</em> clades and the <em>Trypanosoma</em> genus more broadly <span class="citation">(<a href="references.html#ref-hamiltonEvolutionTrypanosomaCruzi2012" role="doc-biblioref">Patrick B. Hamilton, Teixeira, and Stevens 2012</a>)</span>.</p>
<p>The description of a <em>Tr. cyclops</em>-like species in this work expands the number of <em>Trypanosoma</em> species in Tasmanian mammals to four. The platypus-specific <em>Tr. binneyi</em> has been observed from blood samples and subsequently described phylogenetically <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>; <a href="references.html#ref-jakesPhylogeneticRelationshipsTrypanosoma2001" role="doc-biblioref">Jakes, O’Donoghue, and Adlard 2001</a>; <a href="references.html#ref-papariniNovelGenotypesTrypanosoma2014" role="doc-biblioref">A. Paparini et al. 2014</a>)</span> with suggestions that leeches are the likely vector <span class="citation">(<a href="references.html#ref-papariniNovelGenotypesTrypanosoma2014" role="doc-biblioref">A. Paparini et al. 2014</a>)</span>. Trypanosomes have also been identified in populations of the eastern barred bandicoot (<em>Perameles gunnii</em>) and southern brown bandicoot (<em>Isoodon obesulus</em>) <span class="citation">(<a href="references.html#ref-bettiolFirstRecordTrypanosomes1998" role="doc-biblioref">S. S. Bettiol et al. 1998</a>)</span>, however due to well-described limitations of morphological identification <span class="citation">(<a href="references.html#ref-lukesEvolutionParasitismKinetoplastid2014" role="doc-biblioref">Lukes et al. 2014</a>)</span>, species-level identification has not been achieved. Additionally, <em>Tr. copemani</em> was identified for the first time in Tasmania in 2020 from wild populations of the eastern quoll (<em>Dasyurus viverrinus</em>) <span class="citation">(<a href="references.html#ref-portasBaselineHealthDisease2020" role="doc-biblioref">T. J. Portas et al. 2020</a>)</span>. The current study now brings the number of known trypanosome hosts in Tasmania to five, with the addition of Tasmanian devils, and confirmation of a new species from the <em>Tr. cyclops</em> clade. It is interesting to note that while previous studies have described only one species of <em>Trypanosoma</em> in the sampled hosts from Tasmania, in the current study we have described at least two species circulating in devils. The higher diversity of <em>Trypanosoma</em> species found in devils might be a consequence of our larger sample size or spatial coverage in comparison to previous studies.</p>
<p>Despite molecular identification of <em>Trypanosoma</em> infection in devils from Takone, no trypomastigotes were observed in corresponding blood smears. This finding is consistent with other studies which have found molecular tools are more sensitive than microscopy for trypanosome detection <span class="citation">(<a href="references.html#ref-paguemWidespreadCoendemicityTrypanosoma2019" role="doc-biblioref">Paguem et al. 2019</a>; <a href="references.html#ref-rodriguesUncoveringTrypanosomaSpp2019" role="doc-biblioref">M. S. Rodrigues et al. 2019</a>)</span>. The absence of patent parasitaemia (i.e. no flagellates observed in fresh blood smears) could be the result of low-level infections; however it may also indicate that devils are not a competent host of the <em>Trypanosoma</em> species <span class="citation">(<a href="references.html#ref-brandaoTrypanosomatidsSmallMammals2019" role="doc-biblioref">Brandao et al. 2019</a>)</span>.</p>
<p>The trypanosomatids are obligatory parasites with a single flagellum and include several genera that are pathogenic to humans, animals and plants <span class="citation">(<a href="references.html#ref-davila-levyExploringEnvironmentalDiversity2015" role="doc-biblioref">d’Avila-Levy et al. 2015</a>; <a href="references.html#ref-maslovRecentAdvancesTrypanosomatid2019" role="doc-biblioref">Maslov et al. 2019</a>)</span>. Due to their medical importance, trypanosomatids have been studied more intensively with trypanosome and <em>Leishmania</em> parasites the causative agents of sleeping sickness (<em>Trypanosoma brucei gambiense</em> and <em>Trypanosoma brucei rhodesiense</em>), Chagas disease (<em>Trypanosoma cruzi</em>) and leishmaniases <span class="citation">(<a href="references.html#ref-davila-levyExploringEnvironmentalDiversity2015" role="doc-biblioref">d’Avila-Levy et al. 2015</a>; <a href="references.html#ref-lukesTrypanosomatidsAreMuch2018" role="doc-biblioref">Lukes et al. 2018</a>)</span>. Broadly, <em>Trypanosoma</em> species can be divided into two groups based on their transmission route; the salivarian trypanosomes which are transmitted via the saliva of the tsetse fly (<em>Glossina</em> spp.), and stercorarian trypanosomes which are passed to their host via the faeces of the arthropod intermediate host <span class="citation">(<a href="references.html#ref-jacksonGenomeEvolutionTrypanosomatid2015" role="doc-biblioref">A. P. Jackson 2015</a>)</span>. The vectors of Australian trypanosomes are still largely unknown, with few studies investigating the presence of trypanosomes in invertebrates. Previous studies have implicated ticks (<em>Ixodes</em> spp.) <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>; <a href="references.html#ref-austenVectorTrypanosomaCopemani2011" role="doc-biblioref">Austen et al. 2011</a>)</span> and tabanid flies <span class="citation">(<a href="references.html#ref-boteroMorphologicalPhylogeneticDescription2016" role="doc-biblioref">Adriana Botero et al. 2016</a>)</span> as potential vectors. While it has been hypothesised that carnivory is important in the maintenance of trypanosomes, experimental studies do not support this <span class="citation">(<a href="references.html#ref-roelligOralTransmissionTrypanosoma2009" role="doc-biblioref">Roellig, Ellis, and Yabsley 2009</a>)</span>, and instead suggested that increased infection is attributed to insectivores who consume infected arthropod vectors <span class="citation">(<a href="references.html#ref-rochaTrypanosomaCruziInfection2013" role="doc-biblioref">Rocha et al. 2013</a>)</span>. The phylogenetic position of the <em>Tr. cyclops</em> clade within the stercorarian group means that its potential vector(s), could be a range of arthropod(s) with transmission occurring via contact with vector faeces. It is interesting however that although members of this clade have been identified from leeches and frogs <span class="citation">(<a href="references.html#ref-hamiltonNewLineageTrypanosomes2005" role="doc-biblioref">P. B. Hamilton, Stevens, Gidley, et al. 2005</a>)</span>, their phylogenetic position is distinct from other aquatic and leech-associated <em>Trypanosoma</em> species <span class="citation">(<a href="references.html#ref-hamiltonTrypanosomesAreMonophyletic2004" role="doc-biblioref">Patrick B. Hamilton et al. 2004</a>; <a href="references.html#ref-hamiltonPatternsCoevolutionTrypanosomes2007" role="doc-biblioref">P. B. Hamilton, Gibson, and Stevens 2007</a>; <a href="references.html#ref-papariniNovelGenotypesTrypanosoma2014" role="doc-biblioref">A. Paparini et al. 2014</a>; <a href="references.html#ref-lemosPhylogeneticMorphologicalCharacterization2015" role="doc-biblioref">Lemos et al. 2015</a>)</span>. The range extensions of both <em>Tr. copemani</em> and <em>Tr. cyclops</em> clades recorded in the present study provide additional data to aid vector identification. The large geographical range of both these <em>Trypanosoma</em> species suggests that any vector(s) should be equally ubiquitous and capable of biting or coming into close contact with a wide range of vertebrate hosts.</p>
<p>In the present study, we validated that a previously published <em>Trypanosoma</em> PCR assay <span class="citation">(<a href="references.html#ref-mcinnesTrypanosomaIrwiniSp2009" role="doc-biblioref">McInnes et al. 2009</a>)</span> is also capable of detecting/amplifying <em>Leishmania</em> DNA. Two controls of genetically diverse species of <em>Leishmania</em>, <em>L. infantum</em> and <em>L. macropodum</em>, <span class="citation">(<a href="references.html#ref-barrattIsolationNovelTrypanosomatid2017" role="doc-biblioref">Barratt et al. 2017</a>)</span> were used. In the context of this study, we determined that generic trypanosome primers are able to amplify <em>L. macropodum</em>, which was identified as a cause of cutaneous leishmaniasis in kangaroos <span class="citation">(<a href="references.html#ref-roseCutaneousLeishmaniasisRed2004" role="doc-biblioref">Rose et al. 2004</a>)</span>. The preferred diagnostic sample for the detection of <em>Leishmania</em> is generally bone marrow aspirate or tissue samples, however previous studies have recorded detection via blood samples <span class="citation">(<a href="references.html#ref-humbergLeishmaniaChagasiOpossums2012" role="doc-biblioref">Humberg et al. 2012</a>; <a href="references.html#ref-medkourCanineVectorborneProtozoa2020" role="doc-biblioref">Medkour et al. 2020</a>)</span>. Obtaining bone marrow and internal tissue samples from free ranging wildlife is not practical or ethical, especially where populations are experiencing significant declines. While the present study reported the absence of <em>Leishmania</em> via PCR of blood samples, it is not possible to discount occult infection and future studies utilising additional sample types would be recommended. Although co-infections of <em>Leishmania</em> and <em>Trypanosoma</em> cannot be ruled out completely in the individuals tested here, the absence of <em>Leishmania</em> DNA is consistent with previous studies that have utilised the same assay on other Australian mammals <span class="citation">(<a href="references.html#ref-papariniIdentificationNovelTrypanosome2011" role="doc-biblioref">A. Paparini et al. 2011</a>; <a href="references.html#ref-austenInvestigationMorphologicalDiversity2015" role="doc-biblioref">Austen et al. 2015</a>; <a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>)</span>. Therefore, reports of Australia’s only endemic <em>Leishmania</em> species remain confined to kangaroos and biting midges (Ceratopogonidae) in the Northern Territory <span class="citation">(<a href="references.html#ref-roseCutaneousLeishmaniasisRed2004" role="doc-biblioref">Rose et al. 2004</a>; <a href="references.html#ref-dougallNewReportsAustralian2009" role="doc-biblioref">A. Dougall et al. 2009</a>; <a href="references.html#ref-dougallEvidenceIncriminatingMidges2011" role="doc-biblioref">A. M. Dougall et al. 2011</a>)</span>.</p>
<p>An unexpected outcome was the low detection of <em>Babesia</em> species and the lack of detection of <em>Theileria</em> and <em>Hepatozoon</em> species.
Recent molecular studies in Australian wildlife have demonstrated a range of unique endemic piroplasms, with reports of up to 80–90% prevalence in some marsupial host populations <span class="citation">(<a href="references.html#ref-rongHighPrevalenceTheileria2012" role="doc-biblioref">Rong et al. 2012</a>; <a href="references.html#ref-papariniFirstMolecularCharacterization2015" role="doc-biblioref">A. Paparini et al. 2015</a>; <a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">Amy S. Northover et al. 2019</a>)</span>.
Piroplasms and <em>Hepatozoon</em> spp. are transmitted by ticks and therefore dependent on the density and interactions between host and vector(s) to continue their lifecycle.
Recent investigations have revealed the presence of numerous novel <em>Babesia</em>, <em>Theileria</em> and <em>Hepatozoon</em> species in native Australian ticks <span class="citation">(<a href="references.html#ref-greayEndemicExoticNovel2018" role="doc-biblioref">T. L. Greay, Zahedi, et al. 2018</a>; <a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>; <a href="references.html#ref-storey-lewisMolecularDetectionCharacterisation2018" role="doc-biblioref">Storey-Lewis, Mitrovic, and McParland 2018</a>)</span>.
With respect to the single identification of <em>Babesia</em> at the Black River site, the genetic relationship shows it is most closely related to <em>Babesia lohae</em> sequences identified in <em>Ixodes holocyclus</em> and <em>Ixodes tasmani</em> ticks from the east coast of Queensland <span class="citation">(<a href="references.html#ref-greayEndemicExoticNovel2018" role="doc-biblioref">T. L. Greay, Zahedi, et al. 2018</a>; <a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span>.
The low prevalence of <em>Babesia</em> might be attributed to seasonality of tick infestations, as has been described in relationship between <em>Ix. tasmani</em> ticks and brushtail possums hosts <span class="citation">(<a href="references.html#ref-murdochEcologyCommonMarsupial2005" role="doc-biblioref">Murdoch and Spratt 2005</a>)</span>.
Future longitudinal sampling including multiple seasons are valuable to better understand the phylogeny and epidemiology of this <em>Babesia</em> species circulating in devil populations.</p>
<p>In Tasmania, piroplasms and <em>Hepatozoon</em> species have been identified from three marsupial species and more recently from ticks (<em>Ixodes</em> spp.). A <em>Hepatozoon</em> has been identified in eastern barred bandicoots (<em>Perameles gunnii</em>) <span class="citation">(<a href="references.html#ref-bettiolFirstRecordMember1996" role="doc-biblioref">Silvana S. Bettiol et al. 1996</a>)</span> by morphological examination of blood smears and the eastern quoll (<em>Dasyurus viverrinus</em>) through molecular tools <span class="citation">(<a href="references.html#ref-portasBaselineHealthDisease2020" role="doc-biblioref">T. J. Portas et al. 2020</a>)</span>. <em>Theileria</em> species have been identified from eastern bettongs (<em>Bettongia gaimardi</em>) <span class="citation">(<a href="references.html#ref-portasHealthEvaluationFreeranging2014" role="doc-biblioref">T. Portas et al. 2014</a>)</span> and eastern quolls <span class="citation">(<a href="references.html#ref-portasBaselineHealthDisease2020" role="doc-biblioref">T. J. Portas et al. 2020</a>)</span>. A molecular survey of ticks (<em>Ix. tasmani</em>) from Tasmanian devils demonstrated 34.1% (15/44) of sample pools were positive for <em>Hepatozoon</em> sp. <span class="citation">(<a href="references.html#ref-vilcinsDetectionHepatozoonSpotted2009" role="doc-biblioref">Vilcins, Old, and Deane 2009b</a>)</span>. More recently, novel species of <em>Theileria</em> and <em>Hepatozoon</em> were identified from <em>Ix. tasmani</em> collected in Tasmania <span class="citation">(<a href="references.html#ref-greayEndemicExoticNovel2018" role="doc-biblioref">T. L. Greay, Zahedi, et al. 2018</a>; <a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span>. Given the high prevalence of <em>Ix. tasmani</em> parasitising sampled Tasmanian devils (Ruiz-Aravena pers. comms.), the low prevalence of piroplasms and <em>Hepatozoon</em> species is therefore unexpected. This raises important questions about the sylvatic lifecycle and reservoir hosts of these piroplasm and <em>Hepatozoon</em> species, suggesting potential and contrasting explanations: (i) Tasmanian devils are not competent reservoir hosts or (ii) they are natural hosts which are able to mount a sufficient immune response against the infection. In both cases, parasitaemia might be low or absent. Additional explanations could include (iii) infections by haemoprotozoa are acute and fatal to Tasmanian devils, however the absence of <em>Theileria</em> and <em>Hepatozoon</em> in DFTD-free individuals makes this hypothesis less likely; or (iv) the detection of haemoprotozoa in ticks is an opportunistic/accidental finding and they are not competent vectors of these organisms. Additional sampling from DFTD-free populations and capturing seasonal variation will be important to assess the prevalence and susceptibility of Tasmanian devils to these haemoprotozoans.</p>
<p>The inclusion of morphological data obtained from blood smears and the application of additional molecular tools, such as <span class="correction">high-throughput sequencing</span> to identify co-infections <span class="citation">(<a href="references.html#ref-barbosaIncreasedGeneticDiversity2017" role="doc-biblioref">A. D. Barbosa et al. 2017</a>)</span> and genome level information <span class="citation">(<a href="references.html#ref-reis-cunhaWholeGenomeSequencing2018" role="doc-biblioref">Reis-Cunha et al. 2018</a>)</span>, are vital for further work. The collection of additional tissue samples may also prove useful for developing diagnostic tools, as well as better understanding the lifecycle of haemoprotozoa in the host. For example, bone marrow, skeletal muscle, tongue, brain and liver samples have been tested positive for <em>Trypanosoma</em> DNA, while blood (including blood smears) were negative <span class="citation">(<a href="references.html#ref-northoverDebilitatingDiseasePolyparasitised2018" role="doc-biblioref">A. S. Northover, Elliot, et al. 2018</a>)</span>. While there is a body of work demonstrating Australian mammals are hosts to a range of endemic species of <em>Trypanosoma</em>, <em>Leishmania</em>, <em>Babesia</em>, <em>Theileria</em> and <em>Hepatozoon</em> <span class="citation">(<a href="references.html#ref-austenInvestigationMorphologicalDiversity2015" role="doc-biblioref">Austen et al. 2015</a>; <a href="references.html#ref-boteroMorphologicalPhylogeneticDescription2016" role="doc-biblioref">Adriana Botero et al. 2016</a>; <a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>; <a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">Amy S. Northover et al. 2019</a>)</span>, the clinical impact of these haemoprotozoa is largely unknown. This is particularly important in the context of endangered species such as the Tasmanian devil.</p>
<p>The results presented here highlight the need for additional studies on the haemoprotozoa and broader parasite community of Tasmanian devils. In order to provide meaningful information for management and conservation strategies, it is vital that the complete biology of this species be understood. Conservation strategies including translocation, captive breeding and establishment of insurance populations are management strategies that have been implemented in the case of Tasmanian devils. Parasite communities are an important consideration in species recovery, highlighting the need to further study host-parasite relationships <span class="citation">(<a href="references.html#ref-northoverHiddenConsequencesAltering2018" role="doc-biblioref">A. S. Northover, Lymbery, et al. 2018</a>)</span>. For example, while translocation of animals could facilitate introduction of parasites into new host populations, the same translocation could dilute adaptations of the receiving populations to the local parasites. This last case is particularly relevant when the local individuals in supplemented populations are outnumbered by the translocated individuals. Immediate future work is now needed to document the prevalence and diversity of haemoprotozoa in devils with a focus towards understanding the clinical impacts of infection and importantly impact of co-infection with DFTD.</p>
</div>
<div id="conclusion-1" class="section level2" number="6.6">
<h2>
<span class="header-section-number">6.6</span> Conclusion<a class="anchor" aria-label="anchor" href="#conclusion-1"><i class="fas fa-link"></i></a>
</h2>
<p>Here, we have provided the first insights into haemoprotozoa infecting the endangered Tasmanian devil. Further research is urgently needed to document the full haemoprotozoan diversity in the Tasmanian devil, with a focus towards understanding the clinical significance of these infections and co-infections with DFTD. The identification of <em>Tr. copemani</em> and novel members of the <em>Tr. cyclops</em> clade provide further insights into the co-evolution of trypanosomes in marsupials, showing Australian native fauna harbour genetically unique parasitic species. Finally, the lack of detection of major haemoprotozoa genera; <em>Hepatozoon</em>, <em>Leishmania</em>, <em>Plasmodium</em> and <em>Theileria</em> in our study opens questions about the overall status of these parasites in the island ecosystem of Tasmania and whether their absence might reflect specific host-parasite interactions.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></div>
<div class="next"><a href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#tas-devil"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="nav-link" href="#preface-5">Preface</a></li>
<li><a class="nav-link" href="#abstract-5"><span class="header-section-number">6.1</span> Abstract</a></li>
<li><a class="nav-link" href="#introduction-4"><span class="header-section-number">6.2</span> Introduction</a></li>
<li>
<a class="nav-link" href="#materials-and-methods"><span class="header-section-number">6.3</span> Materials and Methods</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#study-sites-and-sampling"><span class="header-section-number">6.3.1</span> Study sites and sampling</a></li>
<li><a class="nav-link" href="#molecular-screening"><span class="header-section-number">6.3.2</span> Molecular screening</a></li>
<li><a class="nav-link" href="#statisical-analysis"><span class="header-section-number">6.3.3</span> Statisical analysis</a></li>
<li><a class="nav-link" href="#microscopy-of-blood-smears"><span class="header-section-number">6.3.4</span> Microscopy of blood smears</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#results-2"><span class="header-section-number">6.4</span> Results</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#molecular-screening-1"><span class="header-section-number">6.4.1</span> Molecular screening</a></li>
<li><a class="nav-link" href="#microscopy"><span class="header-section-number">6.4.2</span> Microscopy</a></li>
</ul>
</li>
<li><a class="nav-link" href="#discussion-2"><span class="header-section-number">6.5</span> Discussion</a></li>
<li><a class="nav-link" href="#conclusion-1"><span class="header-section-number">6.6</span> Conclusion</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/siobhon-egan/PhD/blob/master/06-tas-devil.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/siobhon-egan/PhD/edit/master/06-tas-devil.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australian Wildlife</p></strong>" was written by Siobhon Lee Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
