<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>4 Haemoprotozoa surveillance | Ecology of Ticks and Microbes in Australia Wildlife</title>
<meta name="author" content="Siobhon Lee Egan">
<meta name="generator" content="bookdown 0.26 with bs4_book()">
<meta property="og:title" content="4 Haemoprotozoa surveillance | Ecology of Ticks and Microbes in Australia Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="4 Haemoprotozoa surveillance | Ecology of Ticks and Microbes in Australia Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
<link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Current Research in Parasitology and Vector-Borne Diseases. The following manuscript has been...">
<meta property="og:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Current Research in Parasitology and Vector-Borne Diseases. The following manuscript has been...">
<meta name="twitter:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Current Research in Parasitology and Vector-Borne Diseases. The following manuscript has been...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="active" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/ulyngs/oxforddown">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="wildlife-haemoprotozoa" class="section level1" number="4">
<h1>
<span class="header-section-number">4</span> Haemoprotozoa surveillance<a class="anchor" aria-label="anchor" href="#wildlife-haemoprotozoa"><i class="fas fa-link"></i></a>
</h1>
<div class="inline-figure"><img src="front-and-back-matter/preface/haemoprotozoa.jpg" width="100%"></div>
<div style="page-break-after: always;"></div>
<div id="preface-3" class="section level2 unnumbered">
<h2>Preface<a class="anchor" aria-label="anchor" href="#preface-3"><i class="fas fa-link"></i></a>
</h2>
<p><strong>Attribution Statement</strong></p>
<p>The following chapter has been drafted in accordance with the journal <em>Current Research in Parasitology and Vector-Borne Diseases</em>.</p>
<p>The following manuscript has been published: <strong>Egan, S.</strong>, Taylor, C., Austen, J., Banks, P., Northover, A., Ahlstrom, L., Ryan, U., Irwin, P., and Oskam C. Haemoprotozoan surveillance in peri-urban native and introduced wildlife from Australia. <em>Current Research in Parasitology and Vector-Borne Diseases</em>, <strong>1</strong>, 100052. DOI: <a href="https://doi.org/10.1016/j.crpvbd.2021.100052">10.1016/j.crpvbd.2021.100052</a></p>
<p>The following authors contributed to this manuscript as outlined below<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Contribution indicates the total involvement the author has had in this project. Placing an ‘X’ in the remaining boxes indicates what aspect(s) of the project each author engaged in.&lt;/p&gt;"><sup>5</sup></a>.</p>
<div class="inline-table"><table class="table table" style="font-size: 7px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Authorship order
</th>
<th style="text-align:right;">
Contribution (%)
</th>
<th style="text-align:left;">
Concept Development
</th>
<th style="text-align:left;">
Data Collection
</th>
<th style="text-align:left;">
Data Analysis
</th>
<th style="text-align:left;">
Draft
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Siobhon L. Egan
</td>
<td style="text-align:right;">
70.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
</tr>
<tr>
<td style="text-align:left;">
Casey L. Taylor
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Jill M. Austen
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter B. Banks
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Amy S. Northover
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Liisa A. Ahlstrom
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Una M. Ryan
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter J. Irwin
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Charlotte L. Oskam
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
<p>By signing this document, the Candidate and Principal Supervisor acknowledge that the information provided is accurate and has been agreed to by all other authors.</p>
<div class="line-block">__________________       __________________<br>
  Candidate                                 Principal Supervisor</div>
<div style="page-break-after: always;"></div>
<p><strong>Chapter linking statement:</strong>
This chapter explore the haemoprotozoa present in samples from Australian wildlife. In particular the focus was on blood-borne protozoa; trypanosomes (<em>Leishmania</em> and <em>Trypanosoma</em>), piroplasms (<em>Babesia</em> and <em>Theileria</em>) and <em>Hepatozoon</em>. Using a combination of amplicon metabarcoding, Sanger sequencing and microscopy a total of eight species of <em>Babesia</em>, <em>Hepatozoon</em>, <em>Theileria</em> and <em>Trypanosoma</em> were identified. For this chapter the focus was on the blood samples from wildlife, however in the case of trypanosomes tissue and tick samples were also sequenced. The aim of this study was to identify what haemoprotozoa species are present in wildlife at the urban interface.</p>
<p><strong>Acknowledgment statement:</strong> This work was supported by resources provided by the Pawsey Supercomputing Centre with funding from the Australian Government and the Government of Western Australia.
We acknowledge the Department of Biodiversity, Conservation and Attractions staff for their invaluable assistance in Western Australian fieldwork, in particular Rebecca Kay, Douglas Giles and Hannah Kilian.
We also thank Jade Kelly for assistance in fieldwork and sample collection.
We acknowledge Drs Amanda Barbosa and Andrea Paparini, and A/Prof. Christopher Peacock, for the provision of control isolates used for validation of assays.
We thank Drs David Chandler, Christopher Noune and Matthew Stevens at the Australian Genomics Research Facility for their assistance in sequencing.</p>
<p><strong>Funding statement:</strong> This study was part-funded by the Australian Research Council (LP160100200), Bayer HealthCare (Germany) and Bayer Australia. SLE was supported by an Australian Government Research Training Program (RTP) Scholarship. CLT was supported by a scholarship from the Northern Beaches Council. This project was also part supported by The Holsworth Wildlife Research Endowment from The Ecological Society of Australia (awarded to SLE) and the Paddy Pallin Science Grant from The Royal Zoological Society (awarded to CLT).</p>
<p><strong>Data availability:</strong></p>
<p>Raw Illumina MiSeq data has been deposited in the European Nucleotide Archive under project number <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB46031">PRJEB46031</a> (sample accession numbers ERS6633767–ERS6634241). Nucleotide data produced has been submitted to GenBank under accession numbers MW664957–MW664997 and MW676261–MW676287. Scripts and code used for bioinformatics and statistical analysis are available on <a href="https://github.com/siobhon-egan/wildlife-haemoprotozoa">GitHub</a> and <a href="https://siobhonlegan.com/wildlife-haemoprotozoa">the project website</a>.</p>
<p><strong>Animal ethics:</strong>
Ethical approval for this study was granted by the Murdoch University Animal Ethics Committee (permit number R3026/18), the Department of Biodiversity Conservation and Attractions (permit numbers 2018/54B, 2018/55B, 2018/57D and 2017/30), and the University of Sydney Animal Ethics Committee (permit number 2018/1429).</p>
<p><strong>Author contributions::</strong>
Conceptualisation: S.L.E., U.M.R., P.J.I., C.L.O.
Data curation: S.L.E., C.L.T.
Formal Analysis: S.L.E.
Funding acquisition: S.L.E., C.L.T., P.B.B., L.A.A., U.M.R., P.J.I., C.L.O.
Investigation: S.L.E., C.L.T., J.M.A.,
Methodology: S.L.E., C.L.T., J.M.A., A.S.N.
Project administration: P.J.I., C.L.O.
Resources: A.S.N., P.B.B., P.J.I., C.L.O.
Software: S.L.E.
Supervision: P.B.B., U.M.R., P.J.I., C.L.O.
Visualisation: S.L.E.
Writing – original draft: S.L.E.
Writing – review &amp; editing: S.L.E, C.L.T., P.B.B., A.S.N., L.A.A., U.M.R., P.J.I., C.L.O.</p>
<p><strong>Keywords:</strong> Haemoprotozoa, Wildlife, Marsupial, <em>Trypanosoma</em>, <em>Babesia</em>, <em>Hepatozoon</em>, <em>Theileria</em></p>
<div style="page-break-after: always;"></div>
</div>
<div id="abstract-3" class="section level2" number="4.1">
<h2>
<span class="header-section-number">4.1</span> Abstract<a class="anchor" aria-label="anchor" href="#abstract-3"><i class="fas fa-link"></i></a>
</h2>
<p>Vector-borne haemoprotozoans comprise a diverse group of eukaryote single-celled organisms transmitted by haematophagous (blood-feeding) invertebrates.
They can cause debilitating diseases that impact wildlife, livestock, companion animals and humans.
Recent research has shown that Australian wildlife host a diverse range of haemoprotozoan species; however, to date this work has primarily been confined to a few host species or isolated populations in rural habitats.
There has been little investigation into the presence of these blood parasites in wildlife inhabiting urban and peri-urban areas.
In this study, blood and tissue samples and ticks were collected from wildlife in New South Wales and Western Australia.
Extracted DNA samples were screened with pan-specific molecular assays to determine the presence of haemoprotozoans using amplicon metabarcoding and Sanger sequencing approaches. In addition, light microscopy was performed on blood films.
Eight haemoprotozoans were identified in the present study, which included species of <em>Babesia</em>, <em>Hepatozoon</em>, <em>Theileria</em> and <em>Trypanosoma.</em> Blood samples were collected from 134 animals; 70 black rats (<em>Rattus</em> <em>rattus</em>), 18 common brush-tailed possums (<em>Trichosurus</em> vulpecula), two bush rats (<em>Rattus fuscipes</em>), 22 chuditch (<em>Dasyurus geoffroii</em>), 20 long-nosed bandicoots (<em>Perameles nasuta</em>), one quenda (<em>Isoodon fusciventer</em>) and one swamp rat (<em>Rattus lutreolus</em>).
Molecular screening of DNA extracted from blood samples identified 52.2% (95% CI: 43.8–60.5%) of individuals were positive for at least one haemoprotozoan species, with 19.4% (95% CI: 13.4–26.7%) positive for more than one species.
The present study provides the first sequences of <em>Theileria</em> cf. <em>peramelis</em> from black rats and long-nosed bandicoots.
<em>Babesia lohae</em> was identified from brush-tailed possums.
Two <em>Hepatozoon</em> genotypes were identified from black rats and bush rats. Black rats showed the highest haemoprotozoan diversity, with five species identified.
No known human pathogens that have been described in the northern hemisphere were identified in the present study, and future work is required to understand the zoonotic potential of these microbes in Australia.
This work represents the first large-scale body of research using molecular tools to investigate haemoprotozoans in animals at the urban-wildland interface.
Further research is needed to investigate potential consequences of infection in wildlife, particularly effects of pathogen spillover from invasive black rats to native wildlife.</p>
</div>
<div id="introduction-2" class="section level2" number="4.2">
<h2>
<span class="header-section-number">4.2</span> Introduction<a class="anchor" aria-label="anchor" href="#introduction-2"><i class="fas fa-link"></i></a>
</h2>
<p>Urbanisation is increasing globally with far reaching effects outside the city boundaries, leading to biodiversity loss and changes in species interactions <span class="citation">(<a href="references.html#ref-bradleyUrbanizationEcologyWildlife2007" role="doc-biblioref">C. A. Bradley and Altizer 2007</a>)</span>.
Some wildlife can successfully persist in these urban environments <span class="citation">(<a href="references.html#ref-banksEcologicalImpactsCommensal2015" role="doc-biblioref">Banks and Smith 2015</a>; <a href="references.html#ref-hillmanUrbanEnvironmentsAlter2017" role="doc-biblioref">Hillman et al. 2017</a>; <a href="references.html#ref-rothenburgerEnvironmentalFactorsZoonotic2017" role="doc-biblioref">Rothenburger et al. 2017</a>)</span>.
The increased interaction between wildlife, people and domestic animals can result in a higher risk of potential spillover events where pathogens jump from wildlife to humans <span class="citation">(<a href="references.html#ref-hosseiniDoesImpactBiodiversity2017" role="doc-biblioref">Hosseini et al. 2017</a>)</span>.
The influences of human disturbance in these urban areas can have complex flow-on effects that impact ecosystem health.
Anthropogenic effects such as food sources, shelter and habitat fragmentation can alter wildlife abundance, behaviour and interactions, which may result in a heightened risk of infection and disease transmission <span class="citation">(<a href="references.html#ref-daszakEmergingInfectiousDiseases2000" role="doc-biblioref">Daszak 2000</a>; <a href="references.html#ref-bradleyUrbanizationEcologyWildlife2007" role="doc-biblioref">C. A. Bradley and Altizer 2007</a>)</span>.</p>
<p>Haemoprotozoans are blood-borne unicellular eukaryotes that are capable of infecting all terrestrial vertebrate groups <span class="citation">(<a href="references.html#ref-mcadamInfectiousDiseasesChapter2010" role="doc-biblioref">McAdam and Sharpe 2010</a>)</span>.
A diverse range of vectors have been implicated in the life-cycle of haemoprotozoans, including flies (Diptera), ticks (Ixodida), assassin bugs (Reduviidae) and leeches (Hirudinea) <span class="citation">(<a href="references.html#ref-odonoghueHaemoprotozoaMakingBiological2017" role="doc-biblioref">P. O’Donoghue 2017</a>)</span>.
Globally there are numerous species of veterinary and medical importance, many of which naturally circulate through domestic animals or urban wildlife and may pose a zoonotic risk.
Human and animal diseases from haemoprotozoan infections can include trypanosomiasis (<em>Trypanosoma</em>), babesiosis (<em>Babesia</em>), leishmaniasis (<em>Leishmania</em>) and theileriosis (<em>Theileria</em>) <span class="citation">(<a href="references.html#ref-schreegMitochondrialGenomeSequences2016" role="doc-biblioref">Schreeg et al. 2016</a>; <a href="references.html#ref-kostygovEuglenozoaTaxonomyDiversity2021" role="doc-biblioref">Kostygov et al. 2021</a>)</span>.
Human infection with haemoprotozoans is a consequence of zoonotic spillover events, where pathogens jump from animals to humans.
In the case of <em>Babesia</em>, wildlife such as deer and small rodents have been described as an important part of the enzootic cycles in parts of Europe <span class="citation">(<a href="references.html#ref-silaghiAnaplasmaPhagocytophilumBabesia2020" role="doc-biblioref">Silaghi et al. 2020</a>; <a href="references.html#ref-fanelliHistoricalReviewBabesia2021" role="doc-biblioref">Fanelli 2021</a>)</span> and the USA <span class="citation">(<a href="references.html#ref-yabsleyNaturalHistoryZoonotic2013" role="doc-biblioref">Yabsley and Shock 2013</a>; <a href="references.html#ref-azagiCirculationBabesiaSpecies2021" role="doc-biblioref">Azagi et al. 2021</a>)</span>.
Changes in land use and encroachment of areas where wildlife inhabit are the main drivers of these spillover events <span class="citation">(<a href="references.html#ref-diuk-wasserImpactLandUse2020" role="doc-biblioref">Diuk-Wasser, VanAcker, and Fernandez 2020</a>; <a href="references.html#ref-plowrightLandUseinducedSpillover2021" role="doc-biblioref">Plowright et al. 2021</a>)</span>.
Therefore, surveillance of microbes circulating in wildlife at this urban-wildland interface is critical to untangle zoonotic disease risk.</p>
<p>In recent years there have been increasing reports of tick-associated illnesses in Australians <span class="citation">(<a href="references.html#ref-chaladaThereLymelikeDisease2016" role="doc-biblioref">Chalada, Stenos, and Bradbury 2016</a>)</span>.
A single case of babesiosis, caused by <em>Babesia microti</em>, from an Australian patient with no recent travel history has also been reported <span class="citation">(<a href="references.html#ref-senanayakeFirstReportHuman2012" role="doc-biblioref">Senanayake et al. 2012</a>; <a href="references.html#ref-papariniMolecularConfirmationFirst2014" role="doc-biblioref">Andrea Paparini et al. 2014</a>)</span>.
An epidemiological investigation, including screening of relatives and pets, did not identify any additional cases.
Since then, a large-scale screening of blood donor samples (<em>n</em> = 7,000) further supported the absence of widespread <em>Babesia microti</em> in Australia <span class="citation">(<a href="references.html#ref-faddyNoEvidenceWidespread2019" role="doc-biblioref">Faddy et al. 2019</a>)</span>.
As a result, the origin of the infection remains unknown. However, one hypothesis of its origin is that a local tick may have been responsible for an autochthonous infection, presumably from introduced rodent(s) <span class="citation">(<a href="references.html#ref-senanayakeFirstReportHuman2012" role="doc-biblioref">Senanayake et al. 2012</a>)</span>.</p>
<p>Pioneered by Mackerras, microscopic insights revealed that Australia’s unique wildlife are hosting an equally diverse collection of haemoprotozoans <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>, <a href="references.html#ref-mackerrasHaematozoaAustralianReptiles1961" role="doc-biblioref">1961</a>; <a href="references.html#ref-mackerrasHaematozoaAustralianBirds1960" role="doc-biblioref">Mackerras and Mackerras 1960</a>)</span>.
The advent of molecular tools has helped to provide further insight into the taxonomy and diversity of blood-borne protozoans from Australian wildlife <span class="citation">(<a href="references.html#ref-adlardPerspectivesBiodiversityParasitic1998" role="doc-biblioref">Adlard and O’Donoghue 1998</a>; <a href="references.html#ref-sprattWildlifeParasitologyAustralia2019" role="doc-biblioref">Spratt and Beveridge 2019</a>)</span>.
Several novel haemoprotozoans have been described recently, including <em>Leishmania</em> and <em>Babesia</em> from kangaroos <span class="citation">(<a href="references.html#ref-roseCutaneousLeishmaniasisRed2004" role="doc-biblioref">Rose et al. 2004</a>; <a href="references.html#ref-dawoodObservationNovelBabesia2013" role="doc-biblioref">Dawood et al. 2013</a>)</span>, <em>Trypanosoma</em> from koalas, potoroos and quokkas <span class="citation">(<a href="references.html#ref-austenMorphologicalMolecularCharacterization2009" role="doc-biblioref">Austen et al. 2009</a>; <a href="references.html#ref-mcinnesPotentialImpactNative2011" role="doc-biblioref">McInnes, Gillett, et al. 2011</a>)</span> and <em>Theileria</em> from bettongs and potoroos <span class="citation">(<a href="references.html#ref-leeTheileriaGilbertiSp2009" role="doc-biblioref">Lee et al. 2009</a>; <a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">Amy S. Northover et al. 2019</a>)</span>.
However to date, most of this research in Australia has been ‘vertebrate host-centric,’ and there have been few holistic studies on wildlife haemoprotozoans at the urban-wildland interface.
Furthermore, very few studies have incorporated eutherian hosts despite this clade accounting for about half of the mammal species extant in Australia <span class="citation">(<a href="references.html#ref-flemingGoodBadUgly2016" role="doc-biblioref">Fleming and Bateman 2016</a>)</span>.
As a result, relatively little data has been collected on eutherian haemoprotozoans in recent decades compared to marsupials (metatherians) and monotremes.
It is possible that such endemic haemoprotozoans could be involved in spillover event(s); thus, from a public health perspective, it is essential to collect baseline data on the health and parasite communities of all Australian wildlife, particularly in urban and peri-urban areas.
In addition, further surveillance may also shed light on the status of <em>Babesia microti</em> in Australia, given the events described above.</p>
<p>This study aimed to document three haemoprotozoan groups (i) trypanosomes (<em>Leishmania</em> spp. and <em>Trypanosoma</em> spp.), (ii) piroplasms (<em>Babesia</em> spp. and <em>Theileria</em> spp.) and (iii) <em>Hepatozoon</em> spp., from Australian wildlife in urban and peri-urban areas.
Using amplicon metabarcoding and Sanger sequencing, a semi-targeted approach was used with assays targeting conserved regions of the <em>18S rRNA</em> gene.
The focus of this study was to screen wildlife blood and tissue biopsy samples and their ticks for haemoprotozoans.</p>
</div>
<div id="methods-2" class="section level2" number="4.3">
<h2>
<span class="header-section-number">4.3</span> Methods<a class="anchor" aria-label="anchor" href="#methods-2"><i class="fas fa-link"></i></a>
</h2>
<div id="sample-collection-and-microscopy" class="section level3" number="4.3.1">
<h3>
<span class="header-section-number">4.3.1</span> Sample collection and microscopy<a class="anchor" aria-label="anchor" href="#sample-collection-and-microscopy"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Small mammal trapping</strong></p>
<p>Small mammal trapping was performed at sites in Perth, Western Australia and Sydney, New South Wales (Figure <a href="wildlife-haemoprotozoa.html#fig:Ch4map">4.1</a>).
Sites were targeted in urban and peri-urban areas to reflect the human-wildland interface.
Elliot and cage traps were set and baited with universal bait (rolled oats, peanut butter and sardines).
Traps were set at dusk and cleared at sunrise over three or four consecutive nights.
To ensure a comprehensive assessment and sampling, target mammals were briefly anaesthetised in the field using isoflurane vaporised in medical oxygen (1 litre/minute).
Animals were weighed and examined systematically for ectoparasites.
Up to 1 mL of blood was collected from either the caudal (tail) vein, femoral vein or ear capillary and stored in Mini Collect EDTA tubes.
If practical, a 2 mm punch biopsy was taken at the tick bite site; where possible, a biopsy was taken from the ear and stored in RNAlater or 80% ethanol.
Animals were systematically examined for ectoparasites which were removed and stored in 80% ethanol.
Recovery from anaesthesia was achieved by providing medical oxygen, and once fully alert, animals were released at trap point.
An individual mark was applied to identify animals using either a microchip administered sub-cutaneously or a unique patch of hair was removed.</p>
<p><strong>Opportunistic collection</strong></p>
<p>In a small number of cases, animal carcasses were also obtained opportunistically (<em>n</em> = 10).
These specimens were collected through incidental findings during fieldwork or from situations where animals were humanely euthanised in accordance with animal ethics and state and federal guidelines.
In the case of carcasses spleen samples were collected in replacement for blood samples, this
128 occurred for the following hosts; black rat (<em>n</em> = 2), long-nosed bandicoot (<em>n</em> = 1) and rabbit (<em>n</em> = 7).</p>
<p><strong>Tick identification</strong></p>
<p>Ticks were collected and stored in tubes containing 70–90% ethanol. Samples were visualised using an Olympus SZ61 stereomicroscope (Olympus, Centre Valley, PA, USA) with an external Schott KL 1500 LED (Schott AG Mainz, Germany) light source. Photographs of tick specimens were taken using an Olympus SC30 digital camera (Olympus, Centre Valley, PA, USA) and analysis getIT software (Olympus, Centre Valley, PA, USA).
Instar, sex and species were identified using a combination of available morphological keys and species descriptions <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">F. S. H. Roberts 1970</a>; <a href="references.html#ref-jacksonMorphologicalComparisonAdult2002" role="doc-biblioref">J. Jackson et al. 2002</a>; <a href="references.html#ref-laanObservationsBiologyDistribution2011" role="doc-biblioref">Laan, Handasyde, and Beveridge 2011a</a>; <a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">Stephen C. Barker and Walker 2014</a>; <a href="references.html#ref-kwakPhylogeneticAnalysisAustralian2017" role="doc-biblioref">Kwak et al. 2017</a>)</span>.</p>
<div class="figure">
<span style="display:block;" id="fig:Ch4map"></span>
<img src="figures/ms-figs/Ch4-map.png" alt="Map of study sites for collection of wildlife sample used in haemoprotozoan surveillance (denoted by blue; capital cities denoted for reference). Insert maps of sites in (1) Perth, Western Australia and (2) Sydney, New South Wales." width="95%"><p class="caption">
Figure 4.1: Map of study sites for collection of wildlife sample used in haemoprotozoan surveillance (denoted by blue; capital cities denoted for reference). Insert maps of sites in (1) Perth, Western Australia and (2) Sydney, New South Wales.
</p>
</div>
<p><strong>Haemoparasite microscopy</strong></p>
<p>Thin blood smears were made from whole blood, allowed to air dry and then fixed with 100% methanol.
Slides were stained with Diff Quick (Siemens, Germany) and a coverslip was mounted using DPX neutral mounting medium (LabChem, Australia).
The smears were inspected by light microscopy (Olympus BX51) for the presence of haemoprotozoans at magnifications of ×400 and under oil immersion at ×1000.</p>
</div>
<div id="dna-extractions-1" class="section level3" number="4.3.2">
<h3>
<span class="header-section-number">4.3.2</span> DNA extractions<a class="anchor" aria-label="anchor" href="#dna-extractions-1"><i class="fas fa-link"></i></a>
</h3>
<p>Total genomic DNA (gDNA) was extracted from 200 <span class="math inline">\(\mu\)</span>L of blood using a MasterPure DNA purification kit (EpicentreBiotechnologies, Madison, Wisconsin, U.S.A) following the manufacturer’s recommendations.
Where 200 <span class="math inline">\(\mu\)</span>L of blood was not available, sterile DNA free phosphate-buffered saline (PBS) was used to make samples up to 200 <span class="math inline">\(\mu\)</span>L. Genomic DNA was eluted in 30–40 <span class="math inline">\(\mu\)</span>L of TE buffer and stored at -20<span class="math inline">\(^\circ\)</span>C until further processing.</p>
<p>Tissue samples (skin and spleen) were first rinsed in sterile, DNA-free PBS and cut into small pieces using a sterile scalpel blade. Samples were homogenised in 180 <span class="math inline">\(\mu\)</span>L of buffer ATL and 20 <span class="math inline">\(\mu\)</span>L of proteinase K was added prior to incubation at 56<span class="math inline">\(^\circ\)</span>C for ~ 16 hours. gDNA was extracted using the QIAamp DNA Mini Kit (QIAGEN, Germany) following the manufacturer’s protocols and final elution volume was decreased to 40–50 <span class="math inline">\(\mu\)</span>L to optimise gDNA yield.</p>
<p>Once ticks were identified they were surface-sterilised by washing in 10 % hypochlorite solution, rinsed in 70 % ethanol and DNA-free PBS, and then air-dried.
Genomic DNA was extracted using the DNeasy Blood and Tissue kit (QIAGEN, Germany) for adults or the QIAamp DNA Mini Kit (QIAGEN, Germany) for nymphs and larvae.
Due to the large number of larvae collected from some animals and the expected low DNA yield, in some cases up to 5 larvae were pooled based on host, species and engorgement status.
Ticks were placed in a 2 mL safe lock Eppendorf tube with a 5 mm steel bead, frozen in liquid nitrogen for 1 min and homogenised by shaking at 40 Hz in a Tissue Lyser LT (QIAGEN, Germany).
Final elution of DNA was adjusted according to tick size, between 30-150 <span class="math inline">\(\mu\)</span>L of AE buffer was added to the silicon membrane.</p>
<p>Extraction blank controls (EXBs) consisting of 200 <span class="math inline">\(\mu\)</span>L sterile DNA free PBS, were included randomly in each extraction batch. Blood and tissue samples were extracted in a separate laboratory, away from where ticks were processed to avoid cross-contamination between sample types. Sterile procedures were followed throughout the laboratory process.</p>
</div>
<div id="trypanosomes" class="section level3" number="4.3.3">
<h3>
<span class="header-section-number">4.3.3</span> Trypanosomes<a class="anchor" aria-label="anchor" href="#trypanosomes"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Validation</strong></p>
<p>Positive controls of <em>Leishmania infantum</em> (bone marrow sample obtained from a domestic dog), <em>Leishmania macropodum</em> (culture isolate), <em>Trypanosoma binneyi</em> (blood sample obtained from a platypus, <em>Ornithorhynchus anatinus</em>), <em>Trypanosoma cyclops</em>-like (blood sample obtained from a Tasmanian devil, <em>Sarcophilus harrisii</em>) and <em>Trypanosoma teixeirae</em> (blood sample obtained from a flying fox, <em>Pteropus scapulatus</em>) were used for assay validation.</p>
<p><strong>Sequencing and bioinformatics</strong></p>
<p>A high-throughput metabarcoding approach was used to sequence samples (blood, ticks and tissue) using the Illumina MiSeq platform. Sample libraries were built following the 16S Metagenomic Sequencing Library Preparation (Illumina Part # 15044223 Rev. B), with amplicon PCR primers containing an Illumina MiSeq adapter sequences (Table <a href="wildlife-haemoprotozoa.html#tab:T4primers">4.1</a>).</p>
<p>Kinetoplastid libraries were built targeting the <em>18S rRNA</em> hypervariable region v7-8. Samples were amplified using a semi-nested PCR with primary primers S825F and TryAllR1 <span class="citation">(<a href="references.html#ref-maslovPhylogenyTrypanosomesInferred1996" role="doc-biblioref">Maslov et al. 1996</a>; <a href="references.html#ref-barbosaIncreasedGeneticDiversity2017" role="doc-biblioref">A. D. Barbosa et al. 2017</a>)</span> and the secondary primers S825F and 662R <span class="citation">(<a href="references.html#ref-maslovPhylogenyTrypanosomesInferred1996" role="doc-biblioref">Maslov et al. 1996</a>)</span> containing MiSeq sequence adapters.
Reactions were carried out in 25 <span class="math inline">\(\mu\)</span>L volumes each containing: 1X buffer (KAPA Biosystems, South Africa), 2.0 mM MgCl<sub>2</sub>, 0.4 mg/ml of bovine serum albumin (Fisher Biotech), 0.4 <span class="math inline">\(\mu\)</span>M of each primer, 0.25 mM of each dNTP, 0.5 U of Taq (KAPA Biosystems, South Africa) and 2<span class="math inline">\(\mu\)</span>L of gDNA or 1 <span class="math inline">\(\mu\)</span>L of primary PCR product.
Thermal cycling conditions were as follows; 95<span class="math inline">\(^\circ\)</span>C for 3 mins, followed by 40 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 53<span class="math inline">\(^\circ\)</span>C (primary) or 55<span class="math inline">\(^\circ\)</span>C (secondary) for 30 secs, 72<span class="math inline">\(^\circ\)</span>C for 30 secs; and a final extension step of 72<span class="math inline">\(^\circ\)</span>C for 5 mins.</p>
<p>Amplicon PCR products were then indexed using the Nextera XT DNA library preparation kit in 25 <span class="math inline">\(\mu\)</span>L volumes following the manufacturer’s recommendations.
All PCRs included no template controls (NTC) during each reaction set up and PCRs were performed under strict laboratory conditions.
Amplicons were then dual-indexed using the Nextera XT index kit.
Reactions were performed in 25 <span class="math inline">\(\mu\)</span>L volumes following the manufacturer’s recommendations.
Libraries were purified with Axygen PCR clean up beads and quantified using Qubit High Sensitivity dsDNA assay kit (Thermo Fisher Scientific, Waltham, MA, USA).
Concentration of trypanosome libraries was highly variable among samples.
To ensure samples were analysed uniformly, all libraries were included for sequencing regardless of apparent negative result after PCR (as indicated by gel electrophoresis or Qubit concentration).
This decision was made to ensure that even low levels of haemoprotozoans would be detected.
Libraries were shipped to the Australian Genomic Research Facility (Melbourne, Australia) for final QC and sequenced on an Illumina MiSeq using v2 chemistry (2 x 250 paired-end).</p>
<p>Trypanosome MiSeq data was analysed using a bioinformatic pipeline with the program USEARCH v11 <span class="citation">(<a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>)</span>.
Briefly paired-end reads were merged and sequences matching forward and reverse primers were retrieved (max number of mismatches = 2).
Sequences were then quality-filtered and singletons were removed.
The unoise3 <span class="citation">(<a href="references.html#ref-edgarUNOISE2ImprovedErrorcorrection2016" role="doc-biblioref">Robert C. Edgar 2016</a>)</span> algorithm was used to perform denoising (error-correction) and generate zero-radius taxonomic units (zOTUs).</p>

<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T4primers">Table 4.1: </span>List of primers used for the amplification of haemoprotozoans from Australian wildlife samples. References <strong>1.</strong> Maslov et al. <span class="citation">(<a href="references.html#ref-maslovPhylogenyTrypanosomesInferred1996" role="doc-biblioref">1996</a>)</span>, <strong>2.</strong> Barbosa et al. <span class="citation">(<a href="references.html#ref-barbosaIncreasedGeneticDiversity2017" role="doc-biblioref">2017</a>)</span>, <strong>3.</strong> Jefferies et al. <span class="citation">(<a href="references.html#ref-jefferiesPCRRFLPDetectionDifferentiation2007" role="doc-biblioref">2007</a>)</span>.
</caption>
<thead><tr>
<th style="text-align:left;">
Primer
</th>
<th style="text-align:left;">
Sequence (5’-3’)
</th>
<th style="text-align:left;">
Reference
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;">
Kinetoplastid 18S rRNA
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
S825F
</td>
<td style="text-align:left;">
ACCGTTTCGGCTTTTGTTGG
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
TryAllR1
</td>
<td style="text-align:left;">
GACTGTAACCTCAAAGCTTTCGC
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
662R
</td>
<td style="text-align:left;">
GACTACAATGGTCTCTAATC
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Piroplasm 18S rRNA
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
BTF1
</td>
<td style="text-align:left;">
GGCTCATTACAACAGTTATAG
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
BTF2
</td>
<td style="text-align:left;">
CCCAAAGACTTTGATTTCTCTC
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
BTR1
</td>
<td style="text-align:left;">
CCGTGCTAATTGTAGGGCTAATAC
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
BTR2
</td>
<td style="text-align:left;">
GGACTACGACGGTATCTGATCG
</td>
<td style="text-align:left;">
3
</td>
</tr>
</tbody>
</table></div>
</div>
<div id="piroplasm-and-hepatozoon" class="section level3" number="4.3.4">
<h3>
<span class="header-section-number">4.3.4</span> Piroplasm and <em>Hepatozoon</em><a class="anchor" aria-label="anchor" href="#piroplasm-and-hepatozoon"><i class="fas fa-link"></i></a>
</h3>
<p>Blood samples were screened for piroplasms using a nested PCR targeting an ~ 800 bp product of the 18S rRNA gene (Table <a href="wildlife-haemoprotozoa.html#tab:T4primers">4.1</a>) <span class="citation">(<a href="references.html#ref-jefferiesPCRRFLPDetectionDifferentiation2007" role="doc-biblioref">Jefferies, Ryan, and Irwin 2007</a>)</span>.
Amplicon PCRs were carried out in 25 <span class="math inline">\(\mu\)</span>L reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu\)</span>M of each primer, 0.25 mM of each dNTP, 0.5 U of Taq (KAPA Biosystems, South Africa) and 2 <span class="math inline">\(\mu\)</span>L of gDNA or 1 <span class="math inline">\(\mu\)</span>L of primary product.
Thermal cycling conditions were as follows; 95<span class="math inline">\(^\circ\)</span>C for 2 mins, 58<span class="math inline">\(^\circ\)</span>C for 1 min, 72<span class="math inline">\(^\circ\)</span>C for 2 mins, followed by 40 cycles of 95<span class="math inline">\(^\circ\)</span>C for 30 secs, 58<span class="math inline">\(^\circ\)</span>C (primary) or 62<span class="math inline">\(^\circ\)</span>C (secondary) for 20 secs, 72<span class="math inline">\(^\circ\)</span>C for 1 min (primary) or 45 secs (secondary); and a final extension step of 72<span class="math inline">\(^\circ\)</span>C for 5 mins.</p>
<p>Amplicons were visualised on agarose gel and products of the expected size were excised with a sterile scalpel blade and purified <span class="citation">(<a href="references.html#ref-yangSpecificQuantitativeDetection2013" role="doc-biblioref">R. Yang et al. 2013</a>)</span>.
Sanger sequencing was performed at the Australian Genome Research Facility (Perth, Western Australia) on an Applied Biosystems 3730XL DNA Analyzer using BigDye<sup>TM</sup> Terminator v3.1 Cycle Sequencing Kit.
Nucleotide sequences were inspected and quality-filtered using Geneious 10.2.6 (<a href="https://www.geneious.com" class="uri">https://www.geneious.com</a>).</p>
</div>
<div id="sequence-identification-and-phylogenetics" class="section level3" number="4.3.5">
<h3>
<span class="header-section-number">4.3.5</span> Sequence identification and phylogenetics<a class="anchor" aria-label="anchor" href="#sequence-identification-and-phylogenetics"><i class="fas fa-link"></i></a>
</h3>
<p>Nucleotide sequences were subject to BLAST analysis (BLASTN 2.11.0+ <span class="citation">(<a href="references.html#ref-zhangGreedyAlgorithmAligning2000" role="doc-biblioref">Zhang et al. 2000</a>; <a href="references.html#ref-morgulisDatabaseIndexingProduction2008" role="doc-biblioref">Morgulis et al. 2008</a>)</span>) against NCBI nucleotide collection (nt) database.
Taxonomic lineage for each top hit (based on e-value score) was retrieved from NCBI taxonomy database using TaxonKit <span class="citation">(<a href="references.html#ref-weissHostReproductiveCycle2021" role="doc-biblioref">Weiss et al. 2021</a>)</span>.
Final taxonomy assignment was then refined to the lowest common ancestor based on BLAST results (percentage identity, e-value and query coverage) and sequence alignment method.
For trypanosome metabarcoding positive and negative controls were inspected, a read cut-off threshold was set with sequence values &lt;150 removed.
Generated sequences were then aligned with reference sequences retrieved from GenBank <span class="citation">(<a href="references.html#ref-bensonGenBank2017" role="doc-biblioref">Benson et al. 2017</a>)</span> using MUSCLE <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span>.
Phylogenies were inferred using the maximum likelihood (ML) method.
The optimal evolutionary model was selected using ModelFinder <span class="citation">(<a href="references.html#ref-kalyaanamoorthyModelFinderFastModel2017" role="doc-biblioref">Kalyaanamoorthy et al. 2017</a>)</span> based on Bayesian information criterion.
Phylogenetic analysis was performed in IQ-TREE v1.6.11 <span class="citation">(<a href="references.html#ref-nguyenIQTREEFastEffective2015" role="doc-biblioref">Nguyen et al. 2015</a>)</span> and bootstrap support was calculated using the ultrafast (UFBoot2) method with 10,000 replicates <span class="citation">(<a href="references.html#ref-hoangUFBoot2ImprovingUltrafast2018" role="doc-biblioref">Hoang et al. 2018</a>)</span>.
Genetic sequence similarity was calculated using the Kimura 2-Parameter method <span class="citation">(<a href="references.html#ref-tamuraEstimationNumberNucleotide1993" role="doc-biblioref">Tamura and Nei 1993</a>)</span>.</p>
<p>Scripts for data analysis are available at <a href="https://github.com/siobhon-egan/wildlife-haemoprotozoa" class="uri">https://github.com/siobhon-egan/wildlife-haemoprotozoa</a>.
Raw Illumina MiSeq data has been deposited in the European Nucleotide Archive under project accession number PRJEB46031 (sample accession numbers ERS6633767–ERS6634241) and nucleotide data has been deposited in GenBank nucleotide database under accession numbers MW664957–MW664997, and MW676261–MW676287.</p>
</div>
<div id="prevalence" class="section level3" number="4.3.6">
<h3>
<span class="header-section-number">4.3.6</span> Prevalence<a class="anchor" aria-label="anchor" href="#prevalence"><i class="fas fa-link"></i></a>
</h3>
<p>Confidence intervals for haemoprotozoan prevalence were calculated based on positive identification from blood samples only.
This is due to the inherent difficulties in calculating prevalence based on blood-fed ticks <span class="citation">(<a href="references.html#ref-estrada-penaPitfallsTickTickBorne2021" role="doc-biblioref">Estrada-Pena et al. 2021</a>)</span>.
Additionally, the focus of the present study was the identification of haemoprotozoa from a wildlife host perspective.
Confidence intervals (95% CIs) were calculated based on the Jeffreys interval <span class="citation">(<a href="references.html#ref-brownIntervalEstimationBinomial2001" role="doc-biblioref">L. D. Brown, Cai, and DasGupta 2001</a>)</span>.</p>
</div>
</div>
<div id="results-1" class="section level2" number="4.4">
<h2>
<span class="header-section-number">4.4</span> Results<a class="anchor" aria-label="anchor" href="#results-1"><i class="fas fa-link"></i></a>
</h2>
<div id="sample-information" class="section level3" number="4.4.1">
<h3>
<span class="header-section-number">4.4.1</span> Sample information<a class="anchor" aria-label="anchor" href="#sample-information"><i class="fas fa-link"></i></a>
</h3>
<p>Blood samples were collected from 134 individuals: 70 black rats (<em>Rattus rattus</em>), 18 common brush-tailed possums (<em>Trichosurus vulpecula</em>), two bush rats (<em>Rattus fuscipes</em>), 22 chuditch (<em>Dasyurus geoffroii</em>), 20 long-nosed bandicoots (<em>Perameles nasuta</em>), one quenda (<em>Isoodon fusciventer</em>, previously <em>Isoodon obesulus fusciventer</em>), and one swamp rat (<em>Rattus lutreolus</em>).
A total of 52.2% (95% CI: 43.8–60.5%) of animals were positive for at least one haemoprotozoan, with 19.4% (95% CI: 13.4–26.7%) of animals positive for more than one species (Figure <a href="wildlife-haemoprotozoa.html#fig:Ch4upset">4.2</a>).</p>
<p>A total of 205 tick DNA pools which consisted of 257 ticks were extracted.
A total of six tick species were identified, the following numbers refer to DNA extracts analysed and total number of specimens for each species respectively: <em>Amblyomma triguttatum</em> (4/5), <em>Ixodes antechini</em> (1/1), <em>Ixodes australiensis</em> (1/1), <em>Ixodes holocyclus</em> (67/81), <em>Ixodes tasmani</em> (90/103) and <em>Ixodes trichosuri</em> (37/48). An additional 5 DNA pools which included 18 specimens were identified as a mix of <em>Ix. holocyclus</em> and <em>Ix. trichousri</em>.</p>
<div class="figure">
<span style="display:block;" id="fig:Ch4upset"></span>
<img src="figures/ms-figs/Ch4-haem-blood-prev.png" alt="Prevalence of haemoprotozoans identified from wildlife blood samples presented as number infected of individuals. Data are visualised using UpSet plot showing set intersections of \textit{Trypanosoma}, \textit{Theileria}, \textit{Babesia} and \textit{Hepatozoon} species identified. Total number of individuals sampled: \textit{n} = 134." width="95%"><p class="caption">
Figure 4.2: Prevalence of haemoprotozoans identified from wildlife blood samples presented as number infected of individuals. Data are visualised using UpSet plot showing set intersections of , ,  and  species identified. Total number of individuals sampled:  = 134.
</p>
</div>
</div>
<div id="trypanosomes-1" class="section level3" number="4.4.2">
<h3>
<span class="header-section-number">4.4.2</span> Trypanosomes<a class="anchor" aria-label="anchor" href="#trypanosomes-1"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Bioinformatic results</strong></p>
<p>A total of 51,719,606 raw sequences were obtained from Illumina MiSeq, with 24,138,102 sequences after merging, 22,716,805 sequences after matching primers and then 12,047,863 sequences post-quality filtering (representing ~23.3% of raw sequences).
After denoising, 10,283,530 sequences were obtained from samples; 1,355,654 sequences from positive controls and 305,910 negative controls (Figure <a href="supplementary-material.html#fig:FA41">A.12</a>).
The average length of sequences post-filtering was 298 bp for positive controls (see validation) and 271 bp for samples.
Denoising via the unoise3 algorithm produced 1,630 zOTUs (avg length 233 bp). Taxonomic assignment showed 607 zOTUs were assigned to Eukaryota, with 346 zOTUs belonging to Kinetoplastea (Phylum: Euglenozoa) and 129 zOTUs identified as family Trypanosomatidae.
A final 65 Trypanosomatidae zOTUs remained after removing low abundant sequences (i.e. &lt; 150 sequences) with a total of 6,959,707 sequences (samples = 5,811,323; positive controls = 1,148,384) (see <a href="supplementary-material.html#supplementary-table-e2.1">Supplementary Table E2.1</a> for sequence data and taxonomy including BLAST results).
For phylogenetic reconstruction, a representative subset of 27 zOTUs were used (Figure <a href="wildlife-haemoprotozoa.html#fig:Ch4tryptree">4.3</a>); of which 22 zOTUs were from samples and an additional five zOTUs that were generated from positive control samples.</p>
<p><strong>Validation</strong></p>
<p>All positive controls of <em>L. infantum</em>, <em>L. macropodum</em>, <em>Tr. binneyi</em>, <em>Tr. cyclops</em>-like and <em>Tr. teixeirae</em> were successfully amplified, confirming the ability of this assay to detect a phylogenetically diverse range of trypanosome sequences (<a href="supplementary-material.html#supplementary-table-e2.1">Supplementary Table E2.1</a>). The resulting zOTUs were included in phylogenetic reconstruction (Figure <a href="wildlife-haemoprotozoa.html#fig:Ch4tryptree">4.3</a>).</p>
<p><strong>Samples</strong></p>
<p>Amplicon sequencing results from blood samples showed that 51 animals were positive for at least one species of <em>Trypanosoma</em>.
Mixed trypanosome infections were identified in 5.2% (95% CI: 2.4–10.0%) of blood samples from 4.3% (95% CI: 1.2–11.0%) of black rats, 16.7% (95% CI: 4.9–38.1%) of brush-tailed possums, and a single chuditch (4.6%, 95% CI: 0.5–19.3%).
One black rat (1.4%, 95% CI: 0.2–6.9%) was infected with three species of <em>Trypanosoma</em>.</p>
<p><em>Trypanosoma gilletti</em> was the most abundant trypanosome species identified.
A total number of 69 zOTUs had a top match to <em>Tr. gilletti</em>, eleven zOTUs had greater than 1000 reads, of which seven zOTUs were used for phylogenetic purposes in Figure <a href="wildlife-haemoprotozoa.html#fig:Ch4tryptree">4.3</a>.
This was the only species of <em>Trypanosoma</em> identified in all three sample types (blood, ticks and tissue).
A total of 103 samples were positive (43 blood samples, 23 tick samples and 37 tissue samples); these samples were identified from 65 individuals.
Hosts identified as positive in blood and/or tissue samples for <em>Tr. gilletti</em> were black rats (<em>n</em> = 14), brush-tailed possums (<em>n</em> = 12), chuditch (<em>n</em> = 7) and long-nosed bandicoots (<em>n</em> = 28).
Ticks positive for <em>Tr. gilletti</em> were <em>Ix. holocyclus</em> (larvae, nymphs and females), <em>Ix. tasmani</em> (females) and <em>Ix. trichosuri</em> (nymphs and females).
A thin trypomastigote stage was identified in the blood film of a long-nosed bandicoot (LNB113) (Figure <a href="wildlife-haemoprotozoa.html#fig:microscopy">4.4</a>A).
Amplicon metabarcoding of duplicate blood DNA extracts from this individual showed a single infection with <em>Tr. gilletti</em>.</p>
<p>Fourteen zOTUs were most similar to members of the <em>Trypanosoma cyclops</em> clade, matching sequences from Australian marsupials and leeches (GenBank: AJ620571, AJ620574, MT898517 and MT898518; 97.5–99.7% similarity).
Representatives of the 12 most abundant zOTUs were used for the phylogenetic tree in Figure <a href="wildlife-haemoprotozoa.html#fig:Ch4tryptree">4.3</a>.
<em>Trypanosoma cyclops</em>-like sequences were identified in 13 blood samples from 13 individuals (7 black rats, 5 brush-tailed possums and 1 chuditch were positive).
Only one tissue sample, from a brush-tailed possum (blood sample negative), was positive and all ticks were negative for <em>Tr. cyclops</em>-like sequences.
Unfortunately, morphological verification of this trypanosome was difficult.
A potential amastigote stage of <em>Tr. cyclops</em> was identified in a black rat (BR022) (Figure <a href="wildlife-haemoprotozoa.html#fig:microscopy">4.4</a>B) and a trypomastigote like stage from a second black rat (BR025) (Figure <a href="wildlife-haemoprotozoa.html#fig:microscopy">4.4</a>C), amplicon metabarcoding of both these animals identified a single trypanosome infection with <em>Tr. cyclops</em>.</p>
<p>One sequence (zOTU 5) was identical to <em>Trypanosoma</em> sp. BR042 (MN512227, 100% similarity), which was characterised in Egan et al. <span class="citation">(<a href="references.html#ref-eganMolecularIdentificationTrypanosoma2020" role="doc-biblioref">2020</a>)</span> to explore the phylogeny of this <em>Tr. lewisi</em>-like sequence.
This genotype was identified in three black rats and a single bush rat from two sites, North Head and Manly Dam in New South Wales.
Two black rats (sample IDs BR042 and BRAH1) were positive for <em>Tr. lewisi</em>-like sequences in both tissue and blood samples.
No tick samples were positive for <em>Tr. lewisi</em>-like sequences.
<em>Trypanosoma noyesi</em> was identified in a single brush-tailed possum blood sample from Western Australia.
<em>Trypanosoma vegrandis</em> (zOTU 47; GenBank: KC753534, 100% similarity) was identified in an <em>Am. triguttatum</em> (nymph) (ex quenda) from Western Australia.</p>
<div class="figure">
<span style="display:block;" id="fig:Ch4tryptree"></span>
<img src="figures/ms-figs/Ch4-trypNGStree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of trypanosome zOTUs based on a 426 bp alignment of the 18S rRNA locus. Substitution model TIM3e + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by filled circles. Number of substitutions per nucleotide position is represented by the scale-bar. Sequences generated in the present study are in bold." width="95%"><p class="caption">
Figure 4.3: Maximum likelihood (ML) phylogenetic reconstruction of trypanosome zOTUs based on a 426 bp alignment of the 18S rRNA locus. Substitution model TIM3e + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by filled circles. Number of substitutions per nucleotide position is represented by the scale-bar. Sequences generated in the present study are in bold.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:microscopy"></span>
<img src="figures/ms-figs/Ch4-microscopy-images.png" alt="Haemoprotozoa identified in wildlife blood films. (A) Trypomastigote of \textit{Trypanosoma gilletti} from long-nosed bandicoot (LNB113). (B) Suspected amastigote of \textit{Trypanosoma cyclops} from black rat (BR022). (C) Trypomastigote of \textit{Trypanosoma cyclops} from black rat (BR025). (D-E) \textit{Hepatozoon} gametocytes within leukocytes of a black rat (BR039). All scale-bars: 10 $\mu M$." width="95%"><p class="caption">
Figure 4.4: Haemoprotozoa identified in wildlife blood films. (A) Trypomastigote of  from long-nosed bandicoot (LNB113). (B) Suspected amastigote of  from black rat (BR022). (C) Trypomastigote of  from black rat (BR025). (D-E)  gametocytes within leukocytes of a black rat (BR039). All scale-bars: 10 <span class="math inline">\(\mu M\)</span>.
</p>
</div>
</div>
<div id="piroplasms" class="section level3" number="4.4.3">
<h3>
<span class="header-section-number">4.4.3</span> Piroplasms<a class="anchor" aria-label="anchor" href="#piroplasms"><i class="fas fa-link"></i></a>
</h3>
<p><em>Theileria</em> was identified in blood samples from 12.9% (95% CI: 6.6–22.2%) of black rats and 75.0% (95% CI: 53.6–89.8%) of long-nosed bandicoots from New South Wales.
All the sequences were 100% identical to each other, and therefore one representative was used for phylogenetic analysis (Figure <a href="wildlife-haemoprotozoa.html#fig:F4pirotree">4.5</a>).
Sequences in the present study were identical to <em>Theileria</em> sp. B16 and B60 (GenBank: MG251437 and MG251439) from <em>Ix. tasmani</em> ticks ex. eastern barred bandicoot (<em>Perameles gunnii</em>, Tasmania) and long-nosed bandicoot (Queensland), respectively.
The most similar named species was <em>Theileria penicillata</em> (GenBank: EF554395; 97.4%) from the long-nosed potoroo (<em>Potorous tridactylus</em>) <span class="citation">(<a href="references.html#ref-leeTheileriaGilbertiSp2009" role="doc-biblioref">Lee et al. 2009</a>)</span>.
Phylogenetic reconstruction showed that the sequences in the present study formed a unique clade with those by Loh et al. <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">2018</a>)</span> (Figure <a href="wildlife-haemoprotozoa.html#fig:F4pirotree">4.5</a>) and demonstrated a divergence of 2.6% to the nearest named species (see genetic distances in <a href="supplementary-material.html#supplementary-table-e2.2">Supplementary Table E2.2</a>).</p>
<p>All <em>Babesia</em> sequences were obtained from 72.2% (95% CI: 49.4–88.5%) of brush-tailed possum blood samples in New South Wales. Three genotypes were identified with 99.7–99.9% similarity to each other (Figure <a href="wildlife-haemoprotozoa.html#fig:F4pirotree">4.5</a>).
The most similar sequence to those obtained in the present study was <em>Babesia</em> sp. BP1 obtained from <em>Ix. tasmani</em> (GenBank: MG251435, 99.7–100% similar) ex. brush-tailed possum (NSW) <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span>.
The next most similar was <em>Babesia lohae</em> sequences identified from <em>Ix. tasmani</em> ex brush-tailed possum (GenBank: MG251436, 98.3–98.6% similar) <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span> and <em>Ix. holocyclus</em> ex. domestic cat (GenBank: MG593272, 98.2–98.5% similar) <span class="citation">(<a href="references.html#ref-greayEndemicExoticNovel2018" role="doc-biblioref">T. L. Greay, Zahedi, et al. 2018</a>)</span> (see genetic distances in <a href="supplementary-material.html#supplementary-table-e2.2">Supplementary Table E2.2</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F4pirotree"></span>
<img src="figures/ms-figs/Ch4-pirotree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of piroplasm species based on an 852 bp alignment of the 18S rRNA locus. Substitution model TIM3 + F + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by filled circles. Number of substitutions per nucleotide position is represented by the scale-bar. Sequences generated in the present study are in bold." width="95%"><p class="caption">
Figure 4.5: Maximum likelihood (ML) phylogenetic reconstruction of piroplasm species based on an 852 bp alignment of the 18S rRNA locus. Substitution model TIM3 + F + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by filled circles. Number of substitutions per nucleotide position is represented by the scale-bar. Sequences generated in the present study are in bold.
</p>
</div>
</div>
<div id="hepatozoon" class="section level3" number="4.4.4">
<h3>
<span class="header-section-number">4.4.4</span> Hepatozoon<a class="anchor" aria-label="anchor" href="#hepatozoon"><i class="fas fa-link"></i></a>
</h3>
<p><em>Hepatozoon</em> sequences were identified in three blood samples, two from bush rats and one from a black rat.
The two sequences from bush rats were 100% identical to each other and the single sequence from a black rat was 98.8% similar to the bush rat genotype (Figure <a href="wildlife-haemoprotozoa.html#fig:F4hepattree">4.6</a>).
The black rat genotype was most similar to <em>Hepatozoon</em> sp. HepBiCM001 (GenBank: AB181504, 99.0% similar) from a greater bandicoot rat (<em>Bandicota indica</em>) in Thailand (Dantrakool et al. unpublished), the next most similar were sequences from the bush rat, followed by <em>Hepatozoon ayorgbor</em> (GenBank: EF157822, 98.63% similar) from a ball python (<em>Python regius</em>) in Ghana <span class="citation">(<a href="references.html#ref-slobodaNEWSPECIESHEPATOZOON2007" role="doc-biblioref">Sloboda et al. 2007</a>)</span> (see genetic distances in <a href="supplementary-material.html#supplementary-table-e2.3">Supplementary Table E2.3</a>).
The bush rat genotype was most similar to sequences from olive grass mice (<em>Abrothrix olivaceus</em>) in Chile <span class="citation">(<a href="references.html#ref-merinoMolecularCharacterizationAncient2009" role="doc-biblioref">S. Merino et al. 2009</a>)</span>, genotypes <em>Hepatozoon</em> sp. AO5 and A06 (GenBank: FJ719818 and FJ719817; both 98.8% similar).
The next most similar sequence to the bush rat genotype was <em>Hepatozoon ophisauri</em> (host unknown) from Iran (Zechmeisterova, unpublished) (GenBank: MN723845, 98.8% similar).
Microscopy of blood films from positive individuals identified oval shaped <em>Hepatozoon</em> gametocytes within leukocytes (Figure <a href="wildlife-haemoprotozoa.html#fig:microscopy">4.4</a>D-E).</p>
<div class="figure">
<span style="display:block;" id="fig:F4hepattree"></span>
<img src="figures/ms-figs/Ch4-hepattree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of \textit{Hepatozoon} based on an 829 bp alignment of the 18S rRNA locus. Substitution model HKY + F + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by filled circles. Number of substitutions per nucleotide position is represented by the scale-bar. Sequences generated in the present study are in bold." width="95%"><p class="caption">
Figure 4.6: Maximum likelihood (ML) phylogenetic reconstruction of  based on an 829 bp alignment of the 18S rRNA locus. Substitution model HKY + F + I + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by filled circles. Number of substitutions per nucleotide position is represented by the scale-bar. Sequences generated in the present study are in bold.
</p>
</div>
</div>
</div>
<div id="discussion-1" class="section level2" number="4.5">
<h2>
<span class="header-section-number">4.5</span> Discussion<a class="anchor" aria-label="anchor" href="#discussion-1"><i class="fas fa-link"></i></a>
</h2>
<p>This study identified a total of eight species of haemoprotozoans from Australian wildlife; seven species were identified from hosts directly (blood and tissue samples), and an additional species, <em>Tr. vegrandis</em>, was identified from a single tick (<em>Am. triguttatum</em>). Black rats had the highest diversity with five haemoprotozoan species.</p>
<p>To the best of the authors’ knowledge this is the first report of <em>Tr. cyclops</em>-like genotypes from black rats, brush-tailed possums, and a chuditch.
In addition, <em>Tr. gilletti</em> from black rats, brush-tailed possums, chuditch and long-nosed bandicoots were also identified in these hosts for the first time. This is the first identification of <em>Babesia</em> from brush-tailed possums. We also report the first molecular sequences obtained from presumed <em>Theileria</em> cf. <em>peramelis</em> and a new host record of this species from black rats. A putative novel <em>Hepatozoon</em> species was identified from black rats and bush rats.</p>
<div id="trypanosomes-2" class="section level3" number="4.5.1">
<h3>
<span class="header-section-number">4.5.1</span> Trypanosomes<a class="anchor" aria-label="anchor" href="#trypanosomes-2"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Validation</strong></p>
<p>To identify trypanosome infection a previously developed high-throughput assay was used. This assay was previously used to identify <em>Trypanosoma</em> from koala (<em>Phascolarctos cinereus</em>) blood samples and ticks <span class="citation">(<a href="references.html#ref-barbosaIncreasedGeneticDiversity2017" role="doc-biblioref">A. D. Barbosa et al. 2017</a>)</span> and has been since used to identify co-infections in woylie (brush-tailed bettong, <em>Bettongia penicillata</em>) blood samples <span class="citation">(<a href="references.html#ref-cooperNextGenerationSequencing2018" role="doc-biblioref">C. Cooper et al. 2018</a>)</span>.
In the present study, the assay was validated on additional controls using the phylogenetically distinct <em>Tr. binneyi</em> <span class="citation">(<a href="references.html#ref-papariniNovelGenotypesTrypanosoma2014" role="doc-biblioref">A. Paparini et al. 2014</a>)</span>, and members from the genus <em>Leishmania</em> including <em>L. macropodum</em> and <em>L. infantum</em>.
Validation of the assay on additional <em>Leishmania</em> species indicates that the absence of <em>Leishmania</em> in the present study most likely reflects a true negative finding.</p>
<p><strong>Amplification of non-target sequences</strong></p>
<p>The low rate of sequences retained after bioinformatic analysis in the present study is likely due to the co-amplification of non-target sequences.
In order to ensure final sequences generated are representative of true biological sequences it is important that amplicons undergo best practice bioinformatic analysis.
The level of sequences retained in the present study (~23.3%) were similar to that reported from Huggins et al. <span class="citation">(<a href="references.html#ref-hugginsNovelMetabarcodingDiagnostic2019" role="doc-biblioref">2019</a>)</span> (17.0%).
However, more widespread comparisons are difficult to make as in many cases sequences retained throughout bioinformatics are not adequately reported <span class="citation">(<a href="references.html#ref-squarreDiversityTrypanosomesWildlife2020" role="doc-biblioref">Squarre et al. 2020</a>; <a href="references.html#ref-wahabNovelMetabarcoded18S2020" role="doc-biblioref">Wahab et al. 2020</a>)</span>.
Due to the nature of the Illumina sequencing platform, artificial sequences or chimeras can be generated, particularly when libraries are prepared by PCR amplification methods.
Steps taken to avoid the impact of this included using primers without degenerate bases, including a G/C clamp at the 3’ end of primers, use of high fidelity Taq polymerases and minimising the number of PCR cycles <span class="citation">(<a href="references.html#ref-bradleyDesignEvaluationIllumina2016" role="doc-biblioref">I. M. Bradley, Pinto, and Guest 2016</a>)</span>.
The issue of abundance comparisons in amplicon sequencing has still not been well resolved. While attempts are beginning to improve in the case of bacterial 16S amplicon studies, there is still much work to be done for 18S amplicon studies.
Differences in amplification efficiencies among targets and gene copy number variations are two of the main factors impacting sequence abundance measures in amplicon sequencing <span class="citation">(<a href="references.html#ref-bradleyDesignEvaluationIllumina2016" role="doc-biblioref">I. M. Bradley, Pinto, and Guest 2016</a>)</span>.
Therefore, comparisons of sequence numbers as a measure for trypanosome abundance were not made in the present study.</p>
<p><strong><em>Trypanosoma</em> sp. identified</strong></p>
<p><em>Trypanosoma gilletti</em> was first described from koalas in Queensland and New South Wales <span class="citation">(<a href="references.html#ref-mcinnesNovelTrypanosomeTrypanosoma2011" role="doc-biblioref">McInnes, Hanger, et al. 2011</a>)</span>.
At the time of description, morphological characterisation was not possible as trypomastigotes were only identified in blood smears from one koala that was co-infected with <em>Tr. copemani</em>.
Since its description there have been four additional reports of <em>Tr. gilletti</em>; (i) further reports in koalas and their ticks (<em>Ix. holocyclus</em> and <em>Ix. tasmani</em>) <span class="citation">(<a href="references.html#ref-barbosaIncreasedGeneticDiversity2017" role="doc-biblioref">A. D. Barbosa et al. 2017</a>)</span>; (ii) woylies from Western Australia [<span class="citation"><a href="references.html#ref-cooperNextGenerationSequencing2018" role="doc-biblioref">C. Cooper et al.</a> (<a href="references.html#ref-cooperNextGenerationSequencing2018" role="doc-biblioref">2018</a>)</span>;<span class="citation"><a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">Amy S. Northover et al.</a> (<a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">2019</a>)</span>;]; (iii) swamp wallaby (<em>Wallabia bicolor</em>) and southern brown bandicoot (<em>Isoodon obesulus</em>) in New South Wales <span class="citation">(<a href="references.html#ref-ortiz-baezMetatranscriptomicIdentificationTrypanosoma2020" role="doc-biblioref">Ortiz-Baez et al. 2020</a>)</span>; and (iv) long-nosed potoroos from New South Wales <span class="citation">(<a href="references.html#ref-hallBaselineHealthParameters2021" role="doc-biblioref">Hall et al. 2021</a>)</span>.
Therefore, the present study reports the first host records of <em>Tr. gilletti</em> from black rats, brush-tailed possum, chuditch and long-nosed bandicoots and the first identification in <em>Ix. trichosuri</em>.
Despite its high prevalence from amplicon sequencing, <em>Tr. gilletti</em> was not readily detected in corresponding blood films from positive animals.
A slender trypomastigote was identified in one long-nosed bandicoot with a single infection of <em>Tr. gilletti</em>.
This slender form had a large central nucleus, with the kinetoplast located at the posterior end of the body and is consistent with findings of McInnes <span class="citation">(<a href="references.html#ref-mcinnesNovelTrypanosomeTrypanosoma2011" role="doc-biblioref">2011</a>)</span>.</p>
<p>Until recently, records of the <em>Tr. cyclops</em> clade were sparse. Since 2020 however, there have been at least four reports of this clade.
<em>Trypanosoma cyclops</em> genotypes have been identified from Tasmanian devil blood samples (Tasmania, Australia) <span class="citation">(<a href="references.html#ref-eganBloodParasitesEndangered2020" role="doc-biblioref">Siobhon L. Egan, Ruiz-Aravena, et al. 2020</a>)</span>, rodent liver samples (<em>Bunomys</em>, <em>Maxomys</em>, <em>Paucidentomys</em> and <em>Rattus</em>) (Indonesia) <span class="citation">(<a href="references.html#ref-winterhoffNativeIntroducedTrypanosome2020" role="doc-biblioref">Winterhoff et al. 2020</a>)</span>, swamp wallaby cranial tissue (New South Wales, Australia) <span class="citation">(<a href="references.html#ref-ortiz-baezMetatranscriptomicIdentificationTrypanosoma2020" role="doc-biblioref">Ortiz-Baez et al. 2020</a>)</span> and long-nosed potoroo blood samples (New South Wales, Australia) <span class="citation">(<a href="references.html#ref-hallBaselineHealthParameters2021" role="doc-biblioref">Hall et al. 2021</a>)</span>.
The results presented here provide further support for the widespread distribution of this trypanosome with regards to geographical location and hosts.
The evidence therefore indicates that the <em>Tr. cyclops</em> clade is endemic to the Asia-Pacific regions including Australia.
Trypanosome-like stages were detected in blood smears of two black rats.
The round circular stage was consistent with an amastigote form given the presence of both a kinetoplast and nuclear region and the absence of a free flagellum.
The trypomastigote-like stage contained two nuclear regions representing a kinetoplast and nucleus which were clearly separated.
This stage may represent a transitional stage as no obvious undulating membrane or free flagellum was detected.</p>
<p>The identification of a <em>Tr. lewisi</em>-like trypanosome from rodents at two sites in Sydney raises questions about the distribution of this exotic species.
This trypanosome was identified at the North Head site in two black rats and at the Manly Dam site in one black rat and one bush rat.
<em>Trypanosoma lewisi</em> infection brought in by invasive black rats, was thought responsible for the rapid extinction of two endemic rodents on Christmas Island <span class="citation">(<a href="references.html#ref-wyattHistoricalMammalExtinction2008" role="doc-biblioref">Wyatt et al. 2008</a>; <a href="references.html#ref-greenMammalExtinctionIntroduced2014" role="doc-biblioref">Green 2014</a>)</span>.
Subsequent genetic analysis shows that the genotype in the present study is distinct from the <em>Tr. lewisi</em> (<em>sensu stricto</em>) clade <span class="citation">(<a href="references.html#ref-eganMolecularIdentificationTrypanosoma2020" role="doc-biblioref">Siobhon L. Egan, Taylor, et al. 2020</a>)</span>.
The only other molecular identification of <em>Tr. lewisi</em>-like species in Australia was from small mammals; bush rat, ash-grey mouse (<em>Pseudomys albocinereus</em>) and a dibbler (<em>Parantechinus apicalis</em>) in south-west Australia <span class="citation">(<a href="references.html#ref-averisDiversityDistributionHostparasite2009" role="doc-biblioref">Averis et al. 2009</a>)</span>.
While molecular results of a 499 bp product confirmed genotypes were within the <em>Tr. lewisi</em> clade, an accurate phylogenetic reconstruction was not possible due to the short fragment <span class="citation">(<a href="references.html#ref-hamiltonResolvingRelationshipsAustralian2011" role="doc-biblioref">Patrick B. Hamilton and Stevens 2011</a>; <a href="references.html#ref-eganMolecularIdentificationTrypanosoma2020" role="doc-biblioref">Siobhon L. Egan, Taylor, et al. 2020</a>)</span>.
The clinical significance of this <em>Tr. lewisi</em>-like genotype for native Australian animals remains unknown.
Previous studies have shown that individuals may develop an immunity over their life <span class="citation">(<a href="references.html#ref-smithTrypanosomesFleasField2005" role="doc-biblioref">A. Smith et al. 2005</a>)</span>.
It is currently unclear if the arrival of this trypanosome predates the arrival of black rats to mainland Australia.
The identification of a unique genotype may suggest it has been present for a long period of time, and as such wildlife hosts may have elicited an immune response to infection.
Further investigation is needed to determine the pathogenicity of this trypanosome, particularly in endemic rodent species.</p>
<p>The low prevalence of <em>Tr. vegrandis</em> and <em>Tr. noyesi</em> in wildlife samples was unexpected.
These two species have been identified in numerous Australian mammal species including animals screened in the present study such as brush-tailed possums, bush rats and chuditch <span class="citation">(<a href="references.html#ref-barbosaPrevalenceGeneticDiversity2017a" role="doc-biblioref">A. Barbosa et al. 2017</a>; <a href="references.html#ref-averisDiversityDistributionHostparasite2009" role="doc-biblioref">Averis et al. 2009</a>; <a href="references.html#ref-cooperNextGenerationSequencing2018" role="doc-biblioref">C. Cooper et al. 2018</a>)</span>.
A recent study identified <em>Tr. noyesi</em> in 4.8% (6/125) of questing nymph <em>Am. triguttatum</em> using qPCR <span class="citation">(<a href="references.html#ref-krigeMolecularDetectionTrypanosoma2021" role="doc-biblioref">Krige et al. 2021</a>)</span>.
Additionally, Hamilton et al. <span class="citation">(<a href="references.html#ref-hamiltonInadvertentIntroductionAustralia2005" role="doc-biblioref">2005</a>)</span> described the introduction of the rabbit trypanosome <em>Tr. nabiasi</em>, from samples collected from New South Wales and Victoria.
All rabbit spleen samples in the present study were negative for any <em>Trypanosoma</em> species.
In respect to trypanosome data, a higher overall prevalence was also observed where blood samples showed there was a low proportion of ticks infected compared to blood samples.</p>
</div>
<div id="piroplasms-identified" class="section level3" number="4.5.2">
<h3>
<span class="header-section-number">4.5.2</span> Piroplasms identified<a class="anchor" aria-label="anchor" href="#piroplasms-identified"><i class="fas fa-link"></i></a>
</h3>
<p>Mackerras <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>)</span> described a novel species, <em>Theileria peramelis</em> with records from the long-nosed bandicoot and the southern brown bandicoot (<em>Isoodon obesulus</em>, syn. <em>Thylacis obesulus</em>); however, to-date no molecular data currently exists for this species.
In reports, Mackerras <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>)</span> noted that “The parasites seen in films from a rat-kangaroo, <em>Potorous tridactylus</em> (common name syn. long-nosed potoroo), have been tentatively assigned to this species. They are slightly larger on the average than forms in bandicoots….”
In the present study, <em>Theileria</em> sequences from the long-nosed bandicoot and black rat were all identical to each other and previously reported sequences from <em>Ix. tasmani</em> <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span>.
The morphological reports from Mackerras <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>)</span> are consistent with the molecular results from the present study, with the next most similar species being <em>Th. penicillata</em> ex. long-nosed potoroo.
Additionally, the vector of <em>Th. peramelis</em> has been identified as <em>Ix. tasmani</em> <span class="citation">(<a href="references.html#ref-weilgamaTransmissionTheileriaPeramelis1986" role="doc-biblioref">Weilgama 1986</a>)</span> and sequences from Loh et al. <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">2018</a>)</span> (ex. <em>Ix. tasmani</em>) are identical to those obtained here.
The type-specimen of <em>Th. peramelis</em>, in the form of a microscope slide, is housed at the Queensland Museum (Reg no. G2438, <a href="https://collections.qm.qld.gov.au/objects/PR121/theileria-peramelis" class="uri">https://collections.qm.qld.gov.au/objects/PR121/theileria-peramelis</a>).
While extraction of genetic material from tissue slides has been described <span class="citation">(<a href="references.html#ref-rameshDNAExtractionArchived2019" role="doc-biblioref">Ramesh et al. 2019</a>)</span>, the nondestructive sampling of blood smears from type-specimens is currently not practical.
Therefore, pending further evidence, it is likely the sequences identified in the present study can be assigned to <em>Th. peramelis</em> <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>)</span>, and the host record expanded to include black rats (<em>R. rattus</em>).
The geographical distribution of currently available sequences across three states (New South Wales, Queensland and Tasmania) shows an absence of genetic diversity.
This adds further support that sequences from the present study and those by Loh et al. <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">2018</a>)</span> represent a single species assigned as <em>Th. cf. peramelis</em>.</p>
<p><em>Babesia</em> sequences were identified exclusively from brush-tailed possum and grouped with sequences obtained from native Australian ticks.
The nearest named species, <em>B. lohae</em>, exhibited 98.3–98.6% similarity.
According to Barbosa <span class="citation">(<a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">2019</a>)</span> genetic distance thresholds within piroplasms at the 18S rRNA gene can be as little as 0.3% between named species.
Therefore, it is likely that <em>Babesia</em> sequences from the brush-tailed possum in the present study represent a novel species.
To the best of the authors’ knowledge this is the first identification of <em>Babesia</em> from brush-tailed possums and given the high prevalence in blood samples (72.2%), this is surprising as the species is common throughout Australia and New Zealand.
No <em>Babesia</em> was identified in any other hosts in the present study.
Previous records of <em>Babesia</em> from Australian marsupials include the northern quoll (<em>Dasyurus hallucatus</em>) <span class="citation">(<a href="references.html#ref-bangsBabesiaThylacisApicomplexa1996" role="doc-biblioref">Bangs 1996</a>)</span>, woylie <span class="citation">(<a href="references.html#ref-papariniIdentificationNovelBabesia2012" role="doc-biblioref">A. Paparini et al. 2012</a>; <a href="references.html#ref-northoverIncreasedTrypanosomaSpp2019" role="doc-biblioref">Amy S. Northover et al. 2019</a>)</span>, eastern grey kangaroo (<em>Macropus giganteus</em>) and agile wallaby (<em>Macropus agilis</em>) <span class="citation">(<a href="references.html#ref-donahoeRetrospectiveStudyBabesia2015" role="doc-biblioref">Donahoe et al. 2015</a>)</span>.
The absence of <em>Babesia</em> in cohabiting hosts raises interesting questions regarding the life-cycle and host specificity of the blood parasite. Further research including the clinical impact of <em>Babesia</em> infection in possums is recommended for future studies.</p>
<p>Human babesiosis is the most prevalent protozoal tick-borne disease in the northern hemisphere.
The unique <em>Babesia</em> reported on the Australian continent raise questions regarding the zoonotic potential of these unique microbes.
The taxonomic classification of the piroplasm group (i.e. genera <em>Babesia</em>, <em>Cytauxzoon</em> and <em>Theileria</em>) <span class="citation">(<a href="references.html#ref-papariniFirstMolecularCharacterization2015" role="doc-biblioref">A. Paparini et al. 2015</a>; <a href="references.html#ref-schreegMitochondrialGenomeSequences2016" role="doc-biblioref">Schreeg et al. 2016</a>; <a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>; <a href="references.html#ref-jaloveckaBabesiaLifeCycle2019" role="doc-biblioref">Jalovecka et al. 2019</a>)</span> is currently unresolved, with several groups displaying soft polytomies and a mixture of paraphyletic and polyphyletic clades.
The <em>B. microti</em> group is genetically distinct from the species of <em>Babesia</em> that have been identified from native Australian mammals <span class="citation">(<a href="references.html#ref-schreegMitochondrialGenomeSequences2016" role="doc-biblioref">Schreeg et al. 2016</a>; <a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>)</span>, and as such investigation is needed into the ability of these novel species to infect humans and their potential to cause disease.
No <em>B. microti</em> was identified in the present study.
In particular we note its absence from black rat samples, the vertebrate species that was previously hypothesised as a potential reservoir after an autochthonous case of human babesiosis was reported in Australia <span class="citation">(<a href="references.html#ref-senanayakeFirstReportHuman2012" role="doc-biblioref">Senanayake et al. 2012</a>)</span>.</p>
</div>
<div id="hepatozoon-identified" class="section level3" number="4.5.3">
<h3>
<span class="header-section-number">4.5.3</span> <em>Hepatozoon</em> identified<a class="anchor" aria-label="anchor" href="#hepatozoon-identified"><i class="fas fa-link"></i></a>
</h3>
<p>While <em>H. muris</em> (syn. <em>Hepatozoon perniciosum</em>) is the type-species for the genus, there are no official molecular data assigned to this species <span class="citation">(<a href="references.html#ref-hrazdilovaQuestTypeSpecies2021" role="doc-biblioref">Hrazdilova et al. 2021</a>)</span>.<br>
In most cases <em>Hepatozoon</em> sequences from rodents available on GenBank have not been allocated to species level.
Previous reports of <em>Hepatozoon</em> from Australian rodents are limited to morphological descriptions.
<em>Hepatozoon muris</em> has been reported from rodents including the native bush rat (syn. <em>Rattus assimilis</em> in Mackerras <span class="citation">(<a href="references.html#ref-mackerrasCatalogueAustralianMammals1958a" role="doc-biblioref">1958</a>, <a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>)</span> host records) and canefield rat (<em>Rattus sordidus</em>) and the introduced black rat, brown rat (<em>Rattus norvegicus</em>) and house mouse (<em>Mus musculus</em>) <span class="citation">(<a href="references.html#ref-mackerrasCatalogueAustralianMammals1958a" role="doc-biblioref">Mackerras 1958</a>, <a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>; <a href="references.html#ref-odonoghueCatalogueProtozoanParasites2000" role="doc-biblioref">P. J. O’Donoghue and Adlard 2000</a>)</span>.
O’Donoghue <span class="citation">(<a href="references.html#ref-odonoghueProtozoanParasitesWildlife1997" role="doc-biblioref">1997</a>)</span> reported a <em>Hepatozoon</em> sp. from bush rats, black rats and eastern barred bandicoots in Queensland and noted they were similar in appearance and described as “vacuolated elliptical gamonts within the cytoplasm of host leucocytes.”
However, in a later publication the record for the eastern barred bandicoot was listed as <em>Hepatozoon</em> sp. <span class="citation">(<a href="references.html#ref-odonoghueCatalogueProtozoanParasites2000" role="doc-biblioref">P. J. O’Donoghue and Adlard 2000</a>)</span>.</p>
<p>In comparison, <em>Hepatozoon</em> spp. from Australian marsupials have been described as intraerythrocytic in the morphological descriptions [<span class="citation"><a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras</a> (<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">1959</a>)</span>;<span class="citation"><a href="references.html#ref-bettiolFirstRecordMember1996" role="doc-biblioref">Silvana S. Bettiol et al.</a> (<a href="references.html#ref-bettiolFirstRecordMember1996" role="doc-biblioref">1996</a>)</span>;<span class="citation"><a href="references.html#ref-wicksMorphologicalMolecularCharacteristics2006" role="doc-biblioref">Wicks et al.</a> (<a href="references.html#ref-wicksMorphologicalMolecularCharacteristics2006" role="doc-biblioref">2006</a>)</span>;<span class="citation"><a href="references.html#ref-barbosaPrevalenceGeneticDiversity2017a" role="doc-biblioref">A. Barbosa et al.</a> (<a href="references.html#ref-barbosaPrevalenceGeneticDiversity2017a" role="doc-biblioref">2017</a>)</span>;] (assuming <em>H. muris</em> was misattributed to the eastern barred bandicoot in <span class="citation"><a href="references.html#ref-odonoghueProtozoanParasitesWildlife1997" role="doc-biblioref">P. J. O’Donoghue</a> (<a href="references.html#ref-odonoghueProtozoanParasitesWildlife1997" role="doc-biblioref">1997</a>)</span> as described above).
<em>Hepatozoon tachyglossi</em> from the monotreme echidna, was morphologically identified within monocytes; however, it was noted to be larger than <em>H. muris</em>.
The identity of <em>H. tachyglossi</em> however, remains uncertain after suggestions that its identification in the blood stream may have represented disseminated <em>Eimeria echidnae</em> stages <span class="citation">(<a href="references.html#ref-slapetaDeepsequencingResolveComplex2017" role="doc-biblioref">J. Slapeta et al. 2017</a>)</span>.
Since its morphological identification within monocytes of echidna blood smear <span class="citation">(<a href="references.html#ref-clarkHepatozoonTachyglossiSp2005" role="doc-biblioref">P. Clark, Holz, and Spratt 2005</a>; <a href="references.html#ref-ploegHepatozoonTachyglossiShortbeaked2008" role="doc-biblioref">Ploeg et al. 2008</a>)</span> there has not been any genetic information obtained about this species.</p>
<p>The analysis of <em>Hepatozoon</em> phylogeny was restricted due to previous studies which amplified different regions of the 18S rRNA gene
Alignments often resulted in poor overlap between sequences and as such a comparison with all genotypes was not possible.
The most similar sequences to the black rat genotype were from wild rodents in Thailand (Dantrakool et al. unpublished).
A study by the same authors noted morphological reports of the haemoparasite in mononucleated leukocytes of individuals positive for <em>Babesia</em> <span class="citation">(<a href="references.html#ref-dantrakoolIdentificationNewType2004" role="doc-biblioref">Dantrakool et al. 2004</a>)</span>.
However, there is no further published information on this <em>Hepatozoon</em>. The most similar named species was <em>Hepatozoon ayorgbor</em> identified from a ball python in Ghana <span class="citation">(<a href="references.html#ref-slobodaNEWSPECIESHEPATOZOON2007" role="doc-biblioref">Sloboda et al. 2007</a>)</span>.
A morphological evaluation reported a prevalence of 78.2% (43/55) in snakes.
Laboratory studies of <em>H. ayorgbor</em> concluded that pythons became infected after ingesting infected mosquitoes.
The bush rat genotype was closely related to sequences from the black rat but was most similar to a genotype identified from mice in South America <span class="citation">(<a href="references.html#ref-merinoMolecularCharacterizationAncient2009" role="doc-biblioref">S. Merino et al. 2009</a>)</span>.
The sequence for most closely related named species, <em>H. ophisauri</em> has limited associated metadata other than its location in Iran (GenBank: MN723845; Zechmeisterova, unpublished).
Without available sequences for the type <em>H. muris</em> the species identity of sequences in the present study remains unclear; however it is most likely that they represent a novel species.</p>
<p>It is intriguing that molecular results show rodent and reptilian <em>Hepatozoon</em> species group together phylogenetically.
While infection via means of ingestion of vectors has been shown in laboratory studies <span class="citation">(<a href="references.html#ref-slobodaNEWSPECIESHEPATOZOON2007" role="doc-biblioref">Sloboda et al. 2007</a>)</span>, the phylogenetic evidence presented here adds support that infection by carnivory (e.g. snakes ingesting infected rodents) is another possible alternative <span class="citation">(<a href="references.html#ref-harrisPrevalenceDiversityHepatozoon2015" role="doc-biblioref">Harris, Borges-Nojosa, and Maia 2015</a>)</span>.
Phylogenetic analysis also supports recent studies that have shown a lack of coevolutionary patterns between vertebrate hosts <span class="citation">(<a href="references.html#ref-maiaMOLECULARASSESSMENTHEPATOZOON2014" role="doc-biblioref">Maia et al. 2014</a>; <a href="references.html#ref-harrisPrevalenceDiversityHepatozoon2015" role="doc-biblioref">Harris, Borges-Nojosa, and Maia 2015</a>)</span>
that a clear distinction of <em>Hepatozoon</em> based on host species is not possible, and there is overlap between sequences detected.
Further investigations of <em>Hepatozoon</em> prevalence in both reptiles and rodents from the same environment will be helpful in understanding routes of infection.</p>
<p>No <em>Hepatozoon</em> species were identified from other hosts.
Previous records of <em>Hepatozoon</em> from the hosts species sampled include <em>Hepatozoon peramelis</em> from long-nosed bandicoots <span class="citation">(<a href="references.html#ref-mackerrasHaematozoaAustralianMammals1959" role="doc-biblioref">Mackerras 1959</a>)</span>, <em>Hepatozoon</em> sp. from quenda (syn. <em>Isoodon obesulus fusciventer</em>) <span class="citation">(<a href="references.html#ref-wicksMorphologicalMolecularCharacteristics2006" role="doc-biblioref">Wicks et al. 2006</a>)</span> and <em>Hepatozoon</em> sp. from brush-tailed possum (A.S. Northover, personal communication).
Additionally, <em>Hepatozoon</em> sp. have been described from Australian reptiles <span class="citation">(<a href="references.html#ref-ujvariHighPrevalenceHepatozoon2004" role="doc-biblioref">Ujvari, Madsen, and Olsson 2004</a>; <a href="references.html#ref-vilcinsMolecularMorphologicalDescription2009" role="doc-biblioref">Vilcins, Ujvari, et al. 2009</a>)</span> and more recently the exotic <em>Hepatozoon canis</em> was first reported in Australia from a domesticated dog <span class="citation">(<a href="references.html#ref-greayAustralianDogDiagnosed2018" role="doc-biblioref">Telleasha L. Greay et al. 2018</a>)</span>.</p>
</div>
<div id="host-perspective" class="section level3" number="4.5.4">
<h3>
<span class="header-section-number">4.5.4</span> Host perspective<a class="anchor" aria-label="anchor" href="#host-perspective"><i class="fas fa-link"></i></a>
</h3>
<p>Black rats showed the highest diversity of haemoprotozoans in the present study.
Globally, rodents are well known reservoirs of numerous zoonotic diseases.
Interestingly some studies have shown that the prevalence of pathogens is higher in native rodents compared to commensal invasive species (e.g. <em>R. rattus</em> and <em>M. musculus</em>) <span class="citation">(<a href="references.html#ref-dahmanaRodentsHostsPathogens2020" role="doc-biblioref">Dahmana et al. 2020</a>; <a href="references.html#ref-mangombiFirstInvestigationPathogenic2021" role="doc-biblioref">Mangombi et al. 2021</a>)</span>.
Given the disproportionate sampling between hosts in the present study, statistical comparisons of infection rates between hosts are not possible.
The habitat surveyed in the present study was likely biased towards the introduced rodents, which can replace native congeners in urban areas <span class="citation">(<a href="references.html#ref-banksReviewEvidencePotential2012" role="doc-biblioref">Banks and Hughes 2012</a>)</span>.
The higher diversity of haemoprotozoans identified might also reflect the much greater sampling overall for black rats compared to other species.
Future research on haemoprotozoan diversity in native Australian rodents is recommended.</p>
</div>
<div id="value-of-amplicon-sequencing---eukaryote-diagnostic" class="section level3" number="4.5.5">
<h3>
<span class="header-section-number">4.5.5</span> Value of amplicon sequencing - eukaryote diagnostic<a class="anchor" aria-label="anchor" href="#value-of-amplicon-sequencing---eukaryote-diagnostic"><i class="fas fa-link"></i></a>
</h3>
<p>The development of robust high-throughput sequencing methods targeting the bacterial 16S rRNA locus has resulted in a rapid uptake of bacterial microbiome studies.
Using short read sequencing platforms, the parallel sequencing method is ideal for profiling the suite of microbes present in almost any sample type.
However, the uptake for eukaryote community analysis has been hindered by the lack of a robust, universal method <span class="citation">(<a href="references.html#ref-hugerthSystematicDesign18S2014" role="doc-biblioref">Hugerth et al. 2014</a>)</span>.
At present, eukaryotic community sequencing is largely based on protocols outlined by the earth microbiome project (<a href="https://earthmicrobiome.org/" class="uri">https://earthmicrobiome.org/</a>) which uses generic eukaryote <em>18S rRNA</em> primers.
However, the co-amplification of host DNA and bacterial 16S rRNA sequences means that this method lacks specificity and sensitivity making it impractical in many situations <span class="citation">(<a href="references.html#ref-kounosuImproved18S28S2019" role="doc-biblioref">Kounosu et al. 2019</a>)</span>.
In addition, the amplification biases of primers and the ability to differentiate species varies between the hypervariable regions.
As a result, studies on pathogen or parasite prevalence still largely use these semi-targeted approaches <span class="citation">(<a href="references.html#ref-ghafarTargetedNextGenerationSequencing2020" role="doc-biblioref">Ghafar et al. 2020</a>; <a href="references.html#ref-wahabNovelMetabarcoded18S2020" role="doc-biblioref">Wahab et al. 2020</a>)</span>.
A semi-targeted molecular approach using primers targeting conserved regions of the nuclear 18S rRNA gene provides a number of advantages over using generic eukaryote primers <span class="citation">(<a href="references.html#ref-bradleyDesignEvaluationIllumina2016" role="doc-biblioref">I. M. Bradley, Pinto, and Guest 2016</a>; <a href="references.html#ref-cannonHighthroughputSequencingAssay2018a" role="doc-biblioref">Cannon et al. 2018</a>)</span>.
This approach is particularly favoured in a diagnostic setting, where there is a high priority for increased sensitivity and the ability to detect microbes that may be less abundant, as is often the case with pathogenic microorganisms.</p>
<p>The classification of haemogregarines is problematic, with the lack of a monophyletic group within the family Haemogregarinidae <span class="citation">(<a href="references.html#ref-al-quraishyHaemogregarinesCriteriaIdentification2021" role="doc-biblioref">Al-Quraishy et al. 2021</a>)</span>.
The group is likely to consist of at least three families.
Recent molecular discoveries have shown that morphological identification at the genus level may not be reliable for many haemoprotozoans.
For example, previous studies have shown that organisms considered to be <em>Hepatozoon</em> (-like) were actually members of distantly related groups <span class="citation">(<a href="references.html#ref-merinoSarcocystidMisidentifiedHepatozoon2008" role="doc-biblioref">Santiago Merino et al. 2008</a>; <a href="references.html#ref-zhuLooksCanDeceive2009" role="doc-biblioref">Zhu et al. 2009</a>)</span>.
These findings not only challenge taxonomic classification, but they also show that microbes may not be as tissue-specific (e.g. blood, liver) as previously thought, raising additional questions about the true life cycle of these organisms.
This diverse group of protozoans, therefore, raises an issue when attempting to design “generic” family-level primer sets for molecular barcoding.
This issue has been demonstrated in studies that have attempted to design such generic primers; for instance, Huggins et al. <span class="citation">(<a href="references.html#ref-hugginsNovelMetabarcodingDiagnostic2019" role="doc-biblioref">2019</a>)</span>, where apicomplexan primers amplified a 130 bp region of the <em>18S rRNA</em>, which proved to be incapable of differentiating beyond family level for many taxa.</p>
<p>With the advent of long read sequencing platforms (e.g. PacBio, Nanopore), it is now possible to reconstruct near full-length <em>16S rRNA</em> and <em>18S rRNA</em> sequences.
In the case of the eukaryotic communities, which are highly diverse, this technology will likely resolve many of the issues associated with primer bias and taxonomic resolution <span class="citation">(<a href="references.html#ref-jamyLongReadMetabarcoding2020" role="doc-biblioref">Jamy et al. 2020</a>)</span>.
However, in some cases, even with near full-length <em>18S rRNA</em> sequences, accurate species delimitation can still be challenging.
For piroplasms, mitochondrial genes such as cytochrome <em>c</em> oxidase subunit 3 (<em>cox3</em>) and cytochrome B (<em>cytB</em>), have shown to be more useful informative markers for species delimitation <span class="citation">(<a href="references.html#ref-schreegMitochondrialGenomeSequences2016" role="doc-biblioref">Schreeg et al. 2016</a>; <a href="references.html#ref-barbosaSequenceAnalysesMitochondrial2019" role="doc-biblioref">Amanda D. Barbosa et al. 2019</a>)</span>.
One major limitation to their use is the lack of adequate reference sequences available.
Additionally, there have been few studies to validate the sensitivity of these assays; therefore, their use to determine the prevalence of piroplasm infections in populations is uncertain.
Validation of high throughput sequencing assays for diagnostic use, particularly sensitivity, will be important and needs to be compared against current gold standard methods.</p>
</div>
<div id="limitations-and-recommendations-1" class="section level3" number="4.5.6">
<h3>
<span class="header-section-number">4.5.6</span> Limitations and recommendations<a class="anchor" aria-label="anchor" href="#limitations-and-recommendations-1"><i class="fas fa-link"></i></a>
</h3>
<p>The increased research into bacterial microbiomes of various samples has resulted in several benchmarking protocols and recommendations to deal with issues of contamination <span class="citation">(<a href="references.html#ref-salterReagentLaboratoryContamination2014" role="doc-biblioref">Salter et al. 2014</a>; <a href="references.html#ref-eisenhoferContaminationLowMicrobial2019" role="doc-biblioref">Eisenhofer et al. 2019</a>; <a href="references.html#ref-hoffmannAnalysisTickSurface2020" role="doc-biblioref">Hoffmann, Fingerle, and Noll 2020</a>)</span>.
In comparison, relatively little has been done on the eukaryotic microbiome.
While the recommendations from bacterial microbiome studies can be applied to eukaryote microbiomes, baseline data is needed to determine the profile and impact of these background eukaryote communities.
This is likely to be hindered by taxonomic information and understanding of species interactions.
For example, an increasing number of studies have identified parasitic eukaryotes outside of what is considered their usual hosts and environments <span class="citation">(<a href="references.html#ref-liDetectionHumanIntestinal2020" role="doc-biblioref">J. Li et al. 2020</a>)</span>.
The lack of consensus regarding limits of detection in high throughput sequencing also presents a confounding bias when comparing results between studies.
Documentation and explanation of detection threshold limits are largely up to the authors, and possibly reviewer feedback in the case of peer-reviewed publications.
In addition to stringent laboratory and bioinformatic processes, adequate documentation and data availability should be considered essential parts of the research <span class="citation">(<a href="references.html#ref-kumuthiniTenSimpleRules2020" role="doc-biblioref">Kumuthini et al. 2020</a>)</span>.</p>
<p><strong>Mixed infections</strong></p>
<p>One advantage of high-throughput metabarcoding is the ability to identify mixed infections. Recent research shows that the prevalence of piroplasms in Australian ticks was relatively low.
Using targeted assays, piroplasms were detected in 5.9% (12/205) of wildlife ticks <span class="citation">(<a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span> and 2.8% (20/711) of companion animal ticks <span class="citation">(<a href="references.html#ref-greayEndemicExoticNovel2018" role="doc-biblioref">T. L. Greay, Zahedi, et al. 2018</a>)</span>.
In the case of trypanosomes, very few mixed infections were identified with high-throughput sequencing in the present study.
Only 10% of positive blood samples contained more than one species of <em>Trypanosoma</em>.
In contrast, studies using the same assay on koalas and woylies identified mixed infections in 85.2% <span class="citation">(<a href="references.html#ref-barbosaIncreasedGeneticDiversity2017" role="doc-biblioref">A. D. Barbosa et al. 2017</a>)</span> and 50% <span class="citation">(<a href="references.html#ref-cooperNextGenerationSequencing2018" role="doc-biblioref">C. Cooper et al. 2018</a>)</span> of positive animals, respectively.
The lower incidence of <em>Trypanosoma</em> mixed infection observed in the present study may be due to decreased landscape heterogeneity in the urban and peri-urban sites.
This might result in decreased host movement, lower exposure to potential vectors and differences in animal density and interaction with cohabiting species.
Systematic studies across the landscape gradient and capturing seasonal components will help understand the dynamics of trypanosome infection at the urban-wildland interface.</p>
<p><span class="correction">Despite declining costs, amplicon metabarcoding is still more expensive than targeted Sanger sequencing approaches</span>; the main costs being additional labour required and bioinformatic analysis.
One of the main advantages of metabarcoding approaches is the ability to sequence a high number of positive samples (where a positive sample is defined as the presence of amplified PCR product visible via electrophoresis or nucleic acid quantification).
In the case of haemoprotozoans, the number of positive samples is much lower than comparable metabarcoding studies like bacterial microbiome profiling.
Given the low level of haemoprotozoan mixed infections reported from previous studies <span class="citation">(<a href="references.html#ref-hugginsNovelMetabarcodingDiagnostic2019" role="doc-biblioref">Huggins et al. 2019</a>)</span> and results presented here, we conclude that targeted Sanger sequencing approaches still remain a valuable tool for pathogen detection, and in many cases, the benefits of reduced cost and increased sensitivity makes them more suitable than metabarcoding studies.</p>
<p>A recent novel method has been developed using high-throughput sequencing that produces comparable sensitivity to gold standard methods. Protocols by Flaherty et al. <span class="citation">(<a href="references.html#ref-flahertyRestrictionEnzymeDigestion2018" role="doc-biblioref">2018</a>; <a href="references.html#ref-flahertySensitiveUniversalDetection2021" role="doc-biblioref">2021</a>)</span> used pan-eukaryote primers and restriction enzyme digestion to deplete host DNA. Using a nested PCR approach, they showed sensitivity was comparable to published qPCR assays <span class="citation">(<a href="references.html#ref-flahertySensitiveUniversalDetection2021" role="doc-biblioref">B. R. Flaherty et al. 2021</a>)</span>.
These assays provide a potential alternative to the current eukaryote profiling methods employed in the context of pathogen surveillance and diagnostics.
Incorporating this approach and optimising the host depletion of wildlife species would be of value for future research in this field.</p>
</div>
</div>
<div id="conclusions-1" class="section level2" number="4.6">
<h2>
<span class="header-section-number">4.6</span> Conclusions<a class="anchor" aria-label="anchor" href="#conclusions-1"><i class="fas fa-link"></i></a>
</h2>
<p>The present study provides important insight into the diversity of blood parasites in wildlife at the urban-wildland interface.
A total of eight haemoprotozoan species from four genera, i.e. <em>Trypanosoma</em>, <em>Babesia</em>, <em>Theileria</em> and <em>Hepatozoon</em> were identified.
No known northern hemisphere human haemoprotozoan pathogens (e.g. <em>B. microti</em>) were identified from samples analysed in the present study.
The clinical significance of the haemoprotozoans identified here is unknown with respect to both animal (wildlife and domestic) health and zoonotic potential.
Future studies on wildlife within the urban-wildland interface are needed, including sampling invasive species like black rats.
As a pest species in the environment, they can act as a practical ethical sampling alternative.
Black rats also move readily between urban and peri-urban areas, presenting a risk of potential zoonotic spillover events <span class="citation">(<a href="references.html#ref-banksReviewEvidencePotential2012" role="doc-biblioref">Banks and Hughes 2012</a>)</span>.
In the present study, black rats shared all haemoprotozoans identified from long-nosed bandicoots and native rodents, plus additional species.
As black rats move readily within a landscape, they are a suitable proxy to identify haemoprotozoa that may also be impacting native species.
The further identification of <em>Tr. lewisi</em> from rodents at two sites in Sydney raises important questions about the potential impact it may have on native animals, as discussed in Egan et al. <span class="citation">(<a href="references.html#ref-eganMolecularIdentificationTrypanosoma2020" role="doc-biblioref">2020</a>)</span>.
The health of wildlife and humans is interconnected, with growing recognition that the health of humans is dependent on a healthy environment.
An integrative approach is needed where co-operation on animal surveillance studies can share samples to better understand the health of wildlife and humans <span class="citation">(<a href="references.html#ref-johansenPrevalenceNeutralisingAntibodies2005" role="doc-biblioref">Johansen et al. 2005</a>; <a href="references.html#ref-cox-wittonEmergingInfectiousDiseases2014" role="doc-biblioref">Cox-Witton et al. 2014</a>; <a href="references.html#ref-hillmanUrbanEnvironmentsAlter2017" role="doc-biblioref">Hillman et al. 2017</a>)</span>.
Additionally, sampling of the wildlife hosts directly (e.g. blood or tissue) is recommended as it is more effective at characterising the suite of haemoprotozoa present rather than relying solely on the potential vector(s) (e.g. ticks).</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></div>
<div class="next"><a href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#wildlife-haemoprotozoa"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="nav-link" href="#preface-3">Preface</a></li>
<li><a class="nav-link" href="#abstract-3"><span class="header-section-number">4.1</span> Abstract</a></li>
<li><a class="nav-link" href="#introduction-2"><span class="header-section-number">4.2</span> Introduction</a></li>
<li>
<a class="nav-link" href="#methods-2"><span class="header-section-number">4.3</span> Methods</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#sample-collection-and-microscopy"><span class="header-section-number">4.3.1</span> Sample collection and microscopy</a></li>
<li><a class="nav-link" href="#dna-extractions-1"><span class="header-section-number">4.3.2</span> DNA extractions</a></li>
<li><a class="nav-link" href="#trypanosomes"><span class="header-section-number">4.3.3</span> Trypanosomes</a></li>
<li><a class="nav-link" href="#piroplasm-and-hepatozoon"><span class="header-section-number">4.3.4</span> Piroplasm and Hepatozoon</a></li>
<li><a class="nav-link" href="#sequence-identification-and-phylogenetics"><span class="header-section-number">4.3.5</span> Sequence identification and phylogenetics</a></li>
<li><a class="nav-link" href="#prevalence"><span class="header-section-number">4.3.6</span> Prevalence</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#results-1"><span class="header-section-number">4.4</span> Results</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#sample-information"><span class="header-section-number">4.4.1</span> Sample information</a></li>
<li><a class="nav-link" href="#trypanosomes-1"><span class="header-section-number">4.4.2</span> Trypanosomes</a></li>
<li><a class="nav-link" href="#piroplasms"><span class="header-section-number">4.4.3</span> Piroplasms</a></li>
<li><a class="nav-link" href="#hepatozoon"><span class="header-section-number">4.4.4</span> Hepatozoon</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#discussion-1"><span class="header-section-number">4.5</span> Discussion</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#trypanosomes-2"><span class="header-section-number">4.5.1</span> Trypanosomes</a></li>
<li><a class="nav-link" href="#piroplasms-identified"><span class="header-section-number">4.5.2</span> Piroplasms identified</a></li>
<li><a class="nav-link" href="#hepatozoon-identified"><span class="header-section-number">4.5.3</span> Hepatozoon identified</a></li>
<li><a class="nav-link" href="#host-perspective"><span class="header-section-number">4.5.4</span> Host perspective</a></li>
<li><a class="nav-link" href="#value-of-amplicon-sequencing---eukaryote-diagnostic"><span class="header-section-number">4.5.5</span> Value of amplicon sequencing - eukaryote diagnostic</a></li>
<li><a class="nav-link" href="#limitations-and-recommendations-1"><span class="header-section-number">4.5.6</span> Limitations and recommendations</a></li>
</ul>
</li>
<li><a class="nav-link" href="#conclusions-1"><span class="header-section-number">4.6</span> Conclusions</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/ulyngs/oxforddown/blob/master/04-wildlife-haemoprotozoa.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/ulyngs/oxforddown/edit/master/04-wildlife-haemoprotozoa.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></strong>" was written by Siobhon Lee Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
