<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>3 Microbiome of ticks and wildlife | Ecology of Ticks and Microbes in Australia Wildlife</title>
<meta name="author" content="Siobhon Egan">
<meta name="generator" content="bookdown 0.24 with bs4_book()">
<meta property="og:title" content="3 Microbiome of ticks and wildlife | Ecology of Ticks and Microbes in Australia Wildlife">
<meta property="og:type" content="book">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="3 Microbiome of ticks and wildlife | Ecology of Ticks and Microbes in Australia Wildlife">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/header-attrs-2.11.22/header-attrs.js"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="libs/kePrint-0.0.1/kePrint.js"></script><link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<link href="libs/bsTable-3.3.7/bootstrapTable.min.css" rel="stylesheet">
<script src="libs/bsTable-3.3.7/bootstrapTable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><link rel="stylesheet" href="templates/bs4_style.css">
<link rel="stylesheet" href="templates/corrections.css">
<meta name="description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Microbial Genomics. The following manuscript has been published: Egan, S., Taylor, C., Banks,...">
<meta property="og:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Microbial Genomics. The following manuscript has been published: Egan, S., Taylor, C., Banks,...">
<meta name="twitter:description" content="Preface Attribution Statement The following chapter has been drafted in accordance with the journal Microbial Genomics. The following manuscript has been published: Egan, S., Taylor, C., Banks,...">
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title=""><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome</a></li>
<li><a class="" href="preface-material.html">Preface material</a></li>
<li><a class="" href="litrev.html"><span class="header-section-number">1</span> Literature Review</a></li>
<li><a class="" href="thesis-aims.html">Thesis aims</a></li>
<li><a class="" href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></li>
<li><a class="active" href="wildlife-bacteria.html"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="" href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></li>
<li><a class="" href="black-rat.html"><span class="header-section-number">5</span> Black rat trypanosome</a></li>
<li><a class="" href="tas-devil.html"><span class="header-section-number">6</span> Tasmanian devil haemoprotozoa</a></li>
<li><a class="" href="gendiscussion.html"><span class="header-section-number">7</span> Discussion</a></li>
<li class="book-part">Appendix</li>
<li><a class="" href="supplementary-material.html"><span class="header-section-number">A</span> Supplementary material</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          <p><a id="book-repo" href="https://github.com/ulyngs/oxforddown">View book source <i class="fab fa-github"></i></a></p>
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="wildlife-bacteria" class="section level1" number="3">
<h1>
<span class="header-section-number">3</span> Microbiome of ticks and wildlife<a class="anchor" aria-label="anchor" href="#wildlife-bacteria"><i class="fas fa-link"></i></a>
</h1>
<div class="inline-figure"><img src="front-and-back-matter/preface/Julimar_fieldwork.png" width="95%"></div>
<div style="page-break-after: always;"></div>
<div id="preface-2" class="section level2 unnumbered">
<h2>Preface<a class="anchor" aria-label="anchor" href="#preface-2"><i class="fas fa-link"></i></a>
</h2>
<p><strong>Attribution Statement</strong></p>
<p>The following chapter has been drafted in accordance with the journal <em>Microbial Genomics</em>.</p>
<p>The following manuscript has been published: <strong>Egan, S.</strong>, Taylor, C., Banks, P., Northover, A., Alhstrom, L., Ryan, U., Irwin, P., and Oskam C. 2021. The bacterial biome of ticks and their wildlife hosts at the urban-wildland interface. <em>Microbial Genomics</em>, <strong>7</strong>(12): 000730. DOI: <a href="https://doi.org/10.1099/mgen.0.000730">10.1099/mgen.0.000730</a></p>
<p>The following authors contributed to this manuscript as outlined below<a class="footnote-ref" tabindex="0" data-toggle="popover" data-content="&lt;p&gt;Contribution indicates the total involvement the author has had in this project. Placing an ‘X’ in the remaining boxes indicates what aspect(s) of the project each author engaged in.&lt;/p&gt;"><sup>4</sup></a>.</p>
<div class="inline-table"><table class="table table" style="font-size: 7px; margin-left: auto; margin-right: auto; margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;">
Authorship order
</th>
<th style="text-align:right;">
Contribution (%)
</th>
<th style="text-align:left;">
Concept Development
</th>
<th style="text-align:left;">
Data Collection
</th>
<th style="text-align:left;">
Data Analysis
</th>
<th style="text-align:left;">
Draft
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Siobhon L. Egan
</td>
<td style="text-align:right;">
70.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
X
</td>
</tr>
<tr>
<td style="text-align:left;">
Casey L. Taylor
</td>
<td style="text-align:right;">
6.0
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter B. Banks
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Amy S. Northover
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Liisa A. Ahlstrom
</td>
<td style="text-align:right;">
2.5
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Una M. Ryan
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Peter J. Irwin
</td>
<td style="text-align:right;">
5.0
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Charlotte L. Oskam
</td>
<td style="text-align:right;">
6.5
</td>
<td style="text-align:left;">
X
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
</td>
</tr>
</tbody>
</table></div>
<p>By signing this document, the Candidate and Principal Supervisor acknowledge that the information provided is accurate and has been agreed to by all other authors.</p>
<div class="line-block">__________________       __________________<br>
  Candidate                                 Principal Supervisor</div>
<!-- 
>____________________________
>Candidate                            

\centering
____________________________

\centering 
Principal Supervisor                           

\raggedright
____________________________

Principal Supervisor 

\raggedright

| ______________ &nbsp; &nbsp; &nbsp; ______________
|   Candidate &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; Principal Supervisor
|   
|   __________  
|   Principal Supervisor  -->
<div style="page-break-after: always;"></div>
<p><strong>Chapter linking statement:</strong>
This chapter looks at the bacterial microbiome of wildlife samples. Using amplicon metabarcoding targeting the 16S rRNA gene to characterise the suite of bacteria in blood, tick and tissue samples. It provides an unbiased approach to investigate (i) what microbes are present in these wildlife samples and (ii) uncover overlap between sample types to provide insight into potential bacteria that maybe transmitted into wildlife which act as reservoir hosts. To obtain more detailed genetic information on ‘taxa of interest’ further sequencing was performed. This chapter provides important insights into tick (and potentially other vector) associated microbes in Australia.</p>
<p><strong>Acknowledgment statement:</strong>
This work was supported by resources provided by the Pawsey Supercomputing Centre with funding from the Australian Government and the Government of Western Australia.
We thank Dr David Chandler, Dr Christopher Noune and Dr Matthew Stevens at the Australian Genomics Research Facility for their assistance.
We thank the Department of Biodiversity, Conservation and Attractions staff for their invaluable assistance in Western Australian fieldwork, in particular Rebecca Kay, Douglas Giles and Hannah Kilian.
We also thank Jade Kelly and volunteers for assistance in fieldwork and sample collection.</p>
<p><strong>Funding statement:</strong>
This study was part-funded by the Australian Research Council (LP160100200), Bayer HealthCare (Germany) and Bayer Australia.
S.L.E. was supported by an Australian Government Research Training Program (RTP) Scholarship.
C.L.T. was supported by a scholarship from the Northern Beaches Council.
This project was also part supported by The Holsworth Wildlife Research Endowment from The Ecological Society of Australia (awarded to S.L.E.) and the Paddy Pallin Science Grant from The Royal Zoological Society (awarded to C.L.T.).</p>
<p><strong>Data availability:</strong>
Raw Illumina MiSeq data has been deposited in the European Nucleotide Archive under project number PRJEB46006 (16S rRNA bacteria) and PRJEB46056 (12S rRNA ticks).
Nucleotide data produced by Sanger sequencing has been submitted to NCBI GenBank nucleotide database under accession numbers MW633074, MW633076–MW633082, MW633160–MW633167.
A subset of representative zOTUs generated from tick 12S rRNA metabarcoding have been submitted to GenBank under accession numbers MW665133–MW665150.
Additional information is available within supplementary files including links to electronic files which have been deposited on FigShare <a href="https://doi.org/10.6084/m9.figshare.14363627.v1">repository</a>.
Code used for analysis and RData files used for bioinformatics and statistical analysis are available on GitHub repositories <a href="https://github.com/siobhon-egan/wildlife-bacteria">siobhon-egan/wildlife-bacteria</a> and <a href="https://github.com/siobhon-egan/wildlife-ticks">siobhon-egan/wildlife-ticks</a>.</p>
<p><strong>Animal ethics:</strong> Murdoch University Animal Ethics Committee permit number R3026/18. Department of Biodiversity Conservation and Attractions permit numbers 2018/54B, 2018/55B, 2018/57D and 2017/30. University of Sydney Animal Ethics Committee Permit number 2018/1429.</p>
<p><strong>Author contributions::</strong>
Conceptualization: S.L.E., U.M.R., P.J.I., C.L.O.
Data curation: S.L.E., C.L.T.
Formal Analysis: S.L.E.
Funding acquisition: S.L.E., C.L.T., P.B.B., L.A.A., U.M.R., P.J.I., C.L.O.
Investigation: S.L.E., C.L.T.
Methodology: S.L.E., C.L.T., A.S.N.
Project administration: P.J.I., C.L.O.
Resources: A.S.N., P.B.B., P.J.I., C.L.O.
Software: S.L.E.
Supervision: P.B.B., U.M.R., P.J.I., C.L.O.
Visualization: S.L.E.
Writing – original draft: S.L.E.
Writing – review &amp; editing: S.L.E., C.L.T., P.B.B., A.S.N., L.A.A., U.M.R., P.J.I., C.L.O.</p>
<p><strong>Keywords:</strong> Tick, microbiome, wildlife, sylvatic cycle, tick-borne diseases,</p>
<div style="page-break-after: always;"></div>
</div>
<div id="abstract-2" class="section level2" number="3.1">
<h2>
<span class="header-section-number">3.1</span> Abstract<a class="anchor" aria-label="anchor" href="#abstract-2"><i class="fas fa-link"></i></a>
</h2>
<p>Advances in sequencing technologies have revealed the complex and diverse microbial communities present in ticks (Ixodida).
As obligate blood feeding arthropods, ticks are responsible for a number of infectious diseases that can affect humans, livestock, domestic animals and wildlife.
While cases of human tick-borne diseases continue to increase in the northern hemisphere, there has been relatively little recognition of zoonotic tick-borne pathogens in Australia.
Over the past five years, studies using high-throughput sequencing technologies have shown that Australian ticks harbour unique and diverse bacterial communities.
In the present study, free-ranging wildlife (n = 203), representing ten mammal species, were sampled from urban and peri-urban areas in New South Wales (NSW), Queensland (QLD), and Western Australia (WA).
Bacterial metabarcoding targeting the 16S rRNA locus was used to characterise the microbiomes of three sample types collected from wildlife; blood, ticks and tissue samples.
Further sequence information was obtained for selected taxa of interest. Six tick species were identified from wildlife: <em>Amblyomma triguttatum</em>, <em>Ixodes antechini</em>, <em>Ixodes australiensis</em>, <em>Ixodes holocyclus</em>, <em>Ixodes tasmani</em> and <em>Ixodes trichosuri</em>.
Bacterial 16S rRNA metabarcoding was performed on 536 samples and 65 controls, generating over 100 million sequences.
Alpha diversity was significantly different between the three sample types, with tissue samples displaying the highest alpha diversity (P &lt; 0.001).
<em>Proteobacteria</em> was the most abundant taxon identified across all sample types (37.3%).
Beta diversity analysis and ordination revealed little overlap between the three sample types (P &lt; 0.001).
Taxa of interest included <em>Anaplasmataceae</em>, <em>Bartonella</em>, <em>Borrelia</em>, <em>Coxiellaceae</em>, <em>Francisella</em>, <em>Midichloria</em>, <em>Mycoplasma</em> and <em>Rickettsia.</em> <em>Anaplasmataceae</em> bacteria were detected in 17.7% (95/536) of samples and included <em>Anaplasma</em>, <em>Ehrlichia</em> and <em>Neoehrlichia</em> species.
In samples from NSW ‘<em>Ca</em>. Neoehrlichia australis,’ ‘<em>Ca.</em>. Neoehrlichia arcana,’ <em>Neoehrlichia</em> sp. and <em>Ehrlichia</em> sp were identified.
A putative novel <em>Ehrlichia</em> sp. was identified from WA and <em>Anaplasma platys</em> was identified from QLD.
Nine rodent tissue samples were positive for a novel <em>Borrelia</em> sp. which formed a phylogenetically distinct clade separate from the Lyme <em>Borrelia</em> and relapsing fever groups.
This novel clade included recently identified rodent-associated <em>Borrelia</em> genotypes, which were described from Spain and North America.
<em>Bartonella</em> was identified in 12.9% (69/536) of samples.
Over half of these positive samples were obtained from black rats (<em>Rattus rattus</em>), and the dominant bacterial species identified were <em>Bartonella coopersplainsensis</em> and <em>Bartonella queenslandensis</em>.
The results from the present study show the value of using unbiased high throughput sequencing applied to samples collected from wildlife.
In addition to understanding the sylvatic cycle of known vector associated pathogens, surveillance work is important to ensure preparedness for potential zoonotic spillover events.</p>
</div>
<div id="introduction-1" class="section level2" number="3.2">
<h2>
<span class="header-section-number">3.2</span> Introduction<a class="anchor" aria-label="anchor" href="#introduction-1"><i class="fas fa-link"></i></a>
</h2>
<p>Ticks carry a diverse range of infectious microbes such as viruses, piroplasms, spirochaetes and Rickettsiales <span class="citation">(<a href="references.html#ref-pfaffleEcologyTickborneDiseases2013" role="doc-biblioref">Pfaffle et al. 2013</a>)</span>.
Globally recognised infectious human tick-borne diseases include Lyme borreliosis, ehrlichiosis, babesiosis, tick-borne encephalitis and Powassan viral disease.
Additionally, tick bite can cause reactions such as paralysis and anaphylaxis <span class="citation">(<a href="references.html#ref-vannunenAssociationTickBite2009" role="doc-biblioref">Van Nunen et al. 2009</a>)</span>.
The incidence of tick-borne diseases is rapidly increasing in both prevalence and geographic distribution <span class="citation">(<a href="references.html#ref-madison-antenucciEmergingTickBorneDiseases2020" role="doc-biblioref">Madison-Antenucci et al. 2020</a>)</span>.
The sylvatic cycles of tick-borne pathogens in the northern hemisphere are generally well understood with respect to competent tick vectors and the reservoir hosts (e.g. <em>Babesia microti</em> <span class="citation">(<a href="references.html#ref-westbladeBabesiaMicrotiMice2017" role="doc-biblioref">Westblade et al. 2017</a>)</span> and <em>Borrelia burgdorferi</em> sensu lato <span class="citation">(<a href="references.html#ref-radolfTicksMiceMen2012" role="doc-biblioref">Radolf et al. 2012</a>)</span>).
The value of wildlife health surveillance as a tool for the detection of emerging zoonotic infectious disease has been demonstrated in outbreaks historically, such as malaria <span class="citation">(<a href="references.html#ref-rondonPrevalencePlasmodiumParasites2019" role="doc-biblioref">Rondon et al. 2019</a>)</span>, and more recently with SARS-CoV-2 <span class="citation">(<a href="references.html#ref-plowrightLandUseinducedSpillover2021" role="doc-biblioref">Plowright et al. 2021</a>)</span>.</p>
<p>In Australia, three human tick-borne diseases are recognised; Queensland tick typhus (<em>Rickettsia australis</em>), spotted fever (<em>Rickettsia honei</em> subspp.) and Q fever (<em>Coxiella burnetii</em>) <span class="citation">(<a href="references.html#ref-gravesTickborneInfectiousDiseases2017" role="doc-biblioref">S. R. Graves and Stenos 2017</a>)</span>.
Over the past decade there have been increasing concerns about unidentified tick-borne illness (syn. Lyme-like disease) in Australia <span class="citation">(<a href="references.html#ref-chaladaThereLymelikeDisease2016" role="doc-biblioref">Chalada, Stenos, and Bradbury 2016</a>)</span>, recently designated ‘Debilitating Symptom Complexes Attributed To Ticks’ (DSCATT) by the Australian Government Department of Health <span class="citation">(<a href="references.html#ref-radcliffeGrowingEvidenceEmerging2016" role="doc-biblioref">Radcliffe et al. 2016</a>)</span>.
Presently, 74 tick species have been described in Australia, including five introduced species <span class="citation">(<a href="references.html#ref-barkerIxodesBarkeriSp2019" role="doc-biblioref">D. Barker 2019</a>; <a href="references.html#ref-evansAutomaticBarcodeGap2019" role="doc-biblioref">Evans et al. 2019</a>)</span>, of which 22 species have been identified biting humans <span class="citation">(<a href="references.html#ref-kwakFirstRecordsHuman2018" role="doc-biblioref">Kwak 2018</a>)</span>.
However, data show that <em>Ixodes holocyclus</em> and <em>Amblyomma triguttatum</em> account for &gt;90% of human tick bites, with the remainder of reports largely attributed to <em>Ixodes tasmani</em>, <em>Haemaphysalis bancrofti</em>, <em>Haemaphysalis longicornis</em> and <em>Bothriocroton hydrosauri</em> <span class="citation">(<a href="references.html#ref-kwakFirstRecordsHuman2018" role="doc-biblioref">Kwak 2018</a>)</span>.
Small to medium-sized mammals are important in maintaining these tick populations, in particular during the immature life stages <span class="citation">(<a href="references.html#ref-murdochEcologyCommonMarsupial2005" role="doc-biblioref">Murdoch and Spratt 2005</a>; <a href="references.html#ref-lydeckerPeriurbanBlackRats2019" role="doc-biblioref">Lydecker, Hochuli, and Banks 2019</a>; <a href="references.html#ref-taylorInvasiveRabbitsHost2020" role="doc-biblioref">Taylor et al. 2020</a>)</span>.
While data on the incidence of human tick bites in Australia is limited, evidence suggests that tick bites are common in peri-urban areas, including residential yards (C Taylor, pers comms).</p>
<p>Over the past decade, due to new technologies there is mounting evidence that Australian ticks harbor a unique array of microbes <span class="citation">(<a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">Gofton, Oskam, et al. 2015</a>; <a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.
In addition to these molecular studies, more targeted screening of potential reservoirs such as dogs <span class="citation">(<a href="references.html#ref-irwinSearchingLymeBorreliosis2017" role="doc-biblioref">Irwin et al. 2017</a>)</span> and sea bird ticks <span class="citation">(<a href="references.html#ref-moonAustralianPenguinTicks2018" role="doc-biblioref">Moon et al. 2018</a>)</span> have added to the growing body of evidence suggesting that human tick-borne pathogens described in the northern hemisphere are not endemic in Australia.
Some of the microbes that have recently been identified from Australian ticks however include new species of <em>Borrelia</em> <span class="citation">(<a href="references.html#ref-lohMolecularCharacterizationCandidatus2017" role="doc-biblioref">S. M. Loh et al. 2017</a>; <a href="references.html#ref-panettaReptileassociatedBorreliaSpecies2017" role="doc-biblioref">Panetta et al. 2017</a>)</span>, <em>Ehrlichia</em> <span class="citation">(<a href="references.html#ref-goftonDetectionPhylogeneticCharacterisation2017" role="doc-biblioref">Gofton et al. 2017</a>)</span>, <em>Neoehrlichia</em> <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">Gofton et al. 2016</a>)</span>, <em>Babesia</em>, and <em>Theileria</em> <span class="citation">(<a href="references.html#ref-greayEndemicExoticNovel2018" role="doc-biblioref">T. L. Greay, Zahedi, et al. 2018</a>; <a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>)</span> and several viruses <span class="citation">(<a href="references.html#ref-obrienDiscoveryNovelIflavirus2018" role="doc-biblioref">O’Brien et al. 2018</a>; <a href="references.html#ref-harveyExtensiveDiversityRNA2019" role="doc-biblioref">Harvey et al. 2019</a>)</span>.
Little is known about the sylvatic life cycle of these microbes or their wildlife species reservoir hosts. Even for the few tick-borne pathogens recognised in Australia (e.g. <em>Rickettsia</em> spp.), much of this information remains elusive.</p>
<p>For a tick-borne disease to become established it requires a temporal and spatial overlap between the microbe (pathogens), competent vector tick(s), and one or more vertebrate reservoir hosts.
In the northern hemisphere studies have shown that there are important differences in the role of host species in maintaining the sylvatic cycle of tick-borne pathogens <span class="citation">(<a href="references.html#ref-leviQuantifyingDilutionAmplification2016" role="doc-biblioref">Levi et al. 2016</a>; <a href="references.html#ref-huangHighBurdensIxodes2019" role="doc-biblioref">Huang et al. 2019</a>; <a href="references.html#ref-stewartmerrillMechanisticUnderstandingCompetence2020" role="doc-biblioref">Stewart Merrill and Johnson 2020</a>)</span>.
Broadly, reservoir hosts can be classified as amplification and dilution hosts. Amplification hosts, in most cases, are small mammals that host immature ticks (larvae or nymph stages) and increase the environmental density of infected nymphs <span class="citation">(<a href="references.html#ref-leviQuantifyingDilutionAmplification2016" role="doc-biblioref">Levi et al. 2016</a>)</span>.
In contrast, dilution hosts are inefficient reservoirs of the microbial agent(s) <span class="citation">(<a href="references.html#ref-takumiImpactVertebrateCommunities2019" role="doc-biblioref">Takumi, Sprong, and Hofmeester 2019</a>; <a href="references.html#ref-ginsbergWhyLymeDisease2021" role="doc-biblioref">Ginsberg et al. 2021</a>)</span>.
An additional factor is the role of hosts throughout the tick life cycle.
Larger animals are considered important in maintaining adult stages of the tick; white-tailed deer (<em>Odocoileus virginianus</em>) and <em>Ixodes scapularis</em> <span class="citation">(<a href="references.html#ref-ostfeldTickborneDiseaseRisk2018" role="doc-biblioref">Ostfeld et al. 2018</a>)</span> for example.
In the Australian context it is generally assumed that small mammals such as bandicoots, possums and rodents are likely reservoirs for tick-associated microbes.
In turn, larger mammals such as kangaroos and wallabies may then play an important role in maintaining tick populations, filling a similar ecological niche as deer in the northern hemisphere.
While studies have used molecular tools to investigate microbes present within ticks in Australia <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">Gofton et al. 2016</a>; <a href="references.html#ref-lohMolecularSurveillancePiroplasms2018" role="doc-biblioref">S. M. Loh, Egan, et al. 2018</a>; <a href="references.html#ref-moonAustralianPenguinTicks2018" role="doc-biblioref">Moon et al. 2018</a>)</span>, very little research has been conducted at the vertebrate host level to further elucidate the sylvatic cycle of tick transmitted bacteria.
Bacterial amplicon sequencing is valuable in the setting of wildlife epidemiology surveys where investigations are not confined to existing presumptions of microbes <span class="citation">(<a href="references.html#ref-galan16SRRNAAmplicon2016" role="doc-biblioref">Galan et al. 2016</a>)</span>, however such studies of the ticks alone, without concurrent analysis of vertebrate host tissues, cannot distinguish infectious components of the tick’s total microbiome from those that are non-transmissible.<br>
Therefore, the search for tick-associated microbes in vertebrate host(s) is the next logical piece of information required to untangle the complex dynamics of tick-borne pathogens.
In addition to novel pathogen discovery, high throughput sequencing technologies applied concomitantly to tick and vertebrate host tissues can provide insights into the dynamics of microbial epidemiology and interactions within landscapes <span class="citation">(<a href="references.html#ref-titcombHighThroughputSequencingUnderstanding2019" role="doc-biblioref">Titcomb, Jerde, and Young 2019</a>)</span>.
Identification of microbes may inform our understanding of the tick-borne disease risk and potential differences among temporal and spatial gradients.</p>
<p>The present study used metabarcoding to explore the bacterial communities present in Australian wildlife and their ticks, with a focus on bacteria related to known (vector-borne) pathogens, to provide insights into the sylvatic life cycle of tick associated organisms.
This research was conducted in an ecological zone referred to as the urban-wildland interface, where there is a transition between wilderness and land developed by human activity – an area where a built environment meets or intermingles with a natural environment.</p>
</div>
<div id="methods-1" class="section level2" number="3.3">
<h2>
<span class="header-section-number">3.3</span> Methods<a class="anchor" aria-label="anchor" href="#methods-1"><i class="fas fa-link"></i></a>
</h2>
<div id="study-sites-and-sample-collection" class="section level3" number="3.3.1">
<h3>
<span class="header-section-number">3.3.1</span> Study sites and sample collection<a class="anchor" aria-label="anchor" href="#study-sites-and-sample-collection"><i class="fas fa-link"></i></a>
</h3>
<p><strong>Small mammal trapping</strong></p>
<p>Small mammal trapping was performed at various sites in Perth, Western Australia and Sydney, New South Wales (Figure <a href="wildlife-bacteria.html#fig:F3map">3.1</a>).
Each site was targeted in urban and peri-urban areas to reflect the urban-wildland interface.
Elliot and cage traps were set and baited with universal bait (rolled oats, peanut butter and sardines).
Traps were set at dusk and cleared at sunrise over three or four consecutive nights. To ensure a comprehensive assessment and sampling, target mammals were briefly anaesthetised in the field using isoflurane (I.S.O. 1mL/mL) vaporised in medical oxygen.
Animals were weighed and examined systematically for ectoparasites.
Up to 1 mL of blood was collected from either the caudal (tail) vein, femoral vein or ear capillary and stored in Mini Collect EDTA tubes.
If practical, a 2 mm punch biopsy was taken at the tick bite site; where possible, a biopsy was taken from the ear and stored in RNAlater or 80% ethanol.
Animals were systematically examined for ectoparasites which were removed and stored in 80% ethanol.
Animals were recovered from anaesthesia by providing medical oxygen, and once fully alert were released at trap point.
An individual mark was applied to identify animals using either a microchip administered subcutaneously or a unique patch of hair was removed.
The number of animals and samples analysed, and locations from the present study are available in Table <a href="wildlife-bacteria.html#tab:T3hosts">3.1</a> and Figure <a href="wildlife-bacteria.html#fig:F3map">3.1</a> respectively.</p>
<p><strong>Opportunistic collection</strong></p>
<p>In a small number of cases, animal carcasses were also obtained opportunistically. These specimens were collected through incidental findings during fieldwork or from situations where animals were humanely euthanised in accordance with animal ethics and state and federal guidelines.</p>
<p><strong>Tick identification</strong></p>
<p>Ticks were collected and stored in tubes containing 70–90% ethanol. Samples were visualised using an Olympus SZ61 stereomicroscope (Olympus, Centre Valley, PA, United States) with an external Schott KL 1500 LED (Schott AG Mainz, Germany) light source.
Photographs of tick specimens were taken using an Olympus SC30 digital camera (Olympus, Centre Valley, PA, United States) and analysis getIT software (Olympus, Centre Valley, PA, United States).
Instar, sex and species was identified using a combination of available morphological keys and species descriptions <span class="citation">(<a href="references.html#ref-robertsAustralianTicks1970" role="doc-biblioref">F. S. H. Roberts 1970</a>; <a href="references.html#ref-jacksonMorphologicalComparisonAdult2002" role="doc-biblioref">J. Jackson et al. 2002</a>; <a href="references.html#ref-laanObservationsBiologyDistribution2011" role="doc-biblioref">Laan, Handasyde, and Beveridge 2011a</a>; <a href="references.html#ref-barkerTicksAustraliaSpecies2014" role="doc-biblioref">Stephen C. Barker and Walker 2014</a>; <a href="references.html#ref-kwakPhylogeneticAnalysisAustralian2017" role="doc-biblioref">Kwak et al. 2017</a>)</span>.</p>
<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T3hosts">Table 3.1: </span>Animal sampling numbers for bacterial microbiome sequencing. Number of individual animals () by host species with number of genomic DNA samples analysed in parentheses.
</caption>
<thead><tr>
<th style="text-align:left;">
Common name
</th>
<th style="text-align:left;">
Scientific name
</th>
<th style="text-align:center;">
n (individuals)
</th>
<th style="text-align:center;">
Blood
</th>
<th style="text-align:center;">
Tick
</th>
<th style="text-align:center;">
Tissue
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Black rat
</td>
<td style="text-align:left;font-style: italic;">
Rattus rattus
</td>
<td style="text-align:center;">
88
</td>
<td style="text-align:center;">
68 (84)
</td>
<td style="text-align:center;">
54 (79)
</td>
<td style="text-align:center;">
71 (80)
</td>
</tr>
<tr>
<td style="text-align:left;">
Brown antechinus
</td>
<td style="text-align:left;font-style: italic;">
Antechinus stuartii
</td>
<td style="text-align:center;">
5
</td>
<td style="text-align:center;">
0 (0)
</td>
<td style="text-align:center;">
4 (6)
</td>
<td style="text-align:center;">
2 (2)
</td>
</tr>
<tr>
<td style="text-align:left;">
Brush tail possum
</td>
<td style="text-align:left;font-style: italic;">
Trichosurus vulpecula
</td>
<td style="text-align:center;">
27
</td>
<td style="text-align:center;">
18 (21)
</td>
<td style="text-align:center;">
15 (33)
</td>
<td style="text-align:center;">
16 (20)
</td>
</tr>
<tr>
<td style="text-align:left;">
Bush rat
</td>
<td style="text-align:left;font-style: italic;">
Rattus fuscipes
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
3 (4)
</td>
<td style="text-align:center;">
3 (3)
</td>
</tr>
<tr>
<td style="text-align:left;">
Chuditch
</td>
<td style="text-align:left;font-style: italic;">
Dasyurus geoffroii
</td>
<td style="text-align:center;">
22
</td>
<td style="text-align:center;">
22 (24)
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
9 (9)
</td>
</tr>
<tr>
<td style="text-align:left;">
Deer
</td>
<td style="text-align:left;font-style: italic;">
Rusa timorensis
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
0 (0)
</td>
<td style="text-align:center;">
0 (0)
</td>
<td style="text-align:center;">
3 (6)
</td>
</tr>
<tr>
<td style="text-align:left;">
Long-nosed bandicoot
</td>
<td style="text-align:left;font-style: italic;">
Perameles nasuta
</td>
<td style="text-align:center;">
44
</td>
<td style="text-align:center;">
20 (25)
</td>
<td style="text-align:center;">
31 (60)
</td>
<td style="text-align:center;">
40 (44)
</td>
</tr>
<tr>
<td style="text-align:left;">
Quenda
</td>
<td style="text-align:left;font-style: italic;">
Isoodon fusciventer
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
0 (0)
</td>
</tr>
<tr>
<td style="text-align:left;">
Rabbit
</td>
<td style="text-align:left;font-style: italic;">
Oryctolagus cuniculus
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
0 (0)
</td>
<td style="text-align:center;">
7 (20)
</td>
<td style="text-align:center;">
7 (7)
</td>
</tr>
<tr>
<td style="text-align:left;">
Swamp rat
</td>
<td style="text-align:left;font-style: italic;">
Rattus lutreolus
</td>
<td style="text-align:center;">
2
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
0 (0)
</td>
<td style="text-align:center;">
1 (1)
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Total
</td>
<td style="text-align:left;font-weight: bold;font-style: italic;">
</td>
<td style="text-align:center;font-weight: bold;">
203
</td>
<td style="text-align:center;font-weight: bold;">
132 (159)
</td>
<td style="text-align:center;font-weight: bold;">
117 (205)
</td>
<td style="text-align:center;font-weight: bold;">
152 (172)
</td>
</tr>
</tbody>
</table></div>
<div class="figure">
<span style="display:block;" id="fig:F3map"></span>
<img src="figures/ms-figs/Ch3-map.png" alt="Map of study sites for collection of wildlife samples used in bacterial profiling. Sampling sites denoted by blue triangles and capital cities by pink circles (for geographical reference). Insert maps of sites in (1) Perth, Western Australia and (2) Sydney, New South Wales." width="95%"><p class="caption">
Figure 3.1: Map of study sites for collection of wildlife samples used in bacterial profiling. Sampling sites denoted by blue triangles and capital cities by pink circles (for geographical reference). Insert maps of sites in (1) Perth, Western Australia and (2) Sydney, New South Wales.
</p>
</div>
</div>
<div id="dna-extractions" class="section level3" number="3.3.2">
<h3>
<span class="header-section-number">3.3.2</span> DNA Extractions<a class="anchor" aria-label="anchor" href="#dna-extractions"><i class="fas fa-link"></i></a>
</h3>
<p>Total genomic DNA (gDNA) was extracted from 200 <span class="math inline">\(\mu L\)</span> of blood using a MasterPure DNA purification kit (EpicentreBiotechnologies, Madison, Wisconsin, U.S.A) following the manufacturer’s recommendations. Where 200 <span class="math inline">\(\mu L\)</span> of blood was not available, sterile DNA free phosphate-buffered saline (PBS) was used to make samples up to 200 <span class="math inline">\(\mu L\)</span>.
Genomic DNA was eluted in 30–40 <span class="math inline">\(\mu L\)</span> of TE buffer and stored at -20<span class="math inline">\(^\circ C\)</span> until further processing.</p>
<p>Tissue samples (skin and spleen) were first rinsed in sterile, DNA-free PBS and cut into small pieces (&lt;1-2 mm) using a sterile scalpel blade.
Samples were homogenised in 180 <span class="math inline">\(\mu L\)</span> of buffer ATL and 20 <span class="math inline">\(\mu L\)</span> of proteinase K was added and incubated at 56<span class="math inline">\(^\circ C\)</span> for ~ 16 hours. gDNA was extracted using the QIAamp DNA Mini Kit (QIAGEN, Germany) following the manufacturer’s protocols with the exception that final elution volume was decreased to 40–50 <span class="math inline">\(\mu L\)</span> to increase gDNA yield.</p>
<p>Once ticks were identified, they were surface-sterilised by washing in 10 % hypochlorite solution, rinsed in 70 % ethanol and DNA-free PBS, and then air-dried.
Genomic DNA was extracted using the DNeasy Blood and Tissue kit (QIAGEN, Germany) for adults, or the QIAamp DNA Mini Kit (QIAGEN, Germany) for nymphs and larvae.
Due to the large number of immature tick stages collected from some animals and the expected low DNA yield, up to 5 specimens were pooled for extraction based on host, instar, engorgement status and species identification as determined by morphological methods.
Ticks were placed in a 2 mL safe lock Eppendorf tube with a 5 mm steel bead, frozen in liquid nitrogen for 1 min and homogenised by shaking at 40 Hz in a Tissue Lyser LT (QIAGEN, Germany).
Final elution of DNA was adjusted according to tick size, between 30–150 <span class="math inline">\(\mu L\)</span> of AE buffer was added to the silicon membrane.</p>
<p>Extraction controls (EXBs) consisting of 200 <span class="math inline">\(\mu L\)</span> sterile DNA-free PBS were included randomly in each extraction batch (total EXBs = 34).
Blood and tissue samples were extracted in a separate laboratory, away from where ticks were processed, to avoid cross-contamination between sample types. Sterile procedures were followed throughout the laboratory process.</p>
</div>
<div id="metabarcoding-sequencing" class="section level3" number="3.3.3">
<h3>
<span class="header-section-number">3.3.3</span> Metabarcoding sequencing<a class="anchor" aria-label="anchor" href="#metabarcoding-sequencing"><i class="fas fa-link"></i></a>
</h3>
<p>A high-throughput metabarcoding approach was used to sequence samples using the Illumina MiSeq platform. Libraries were built following the 16S Metagenomic Sequencing Library Preparation (Illumina Part # 15044223 Rev. B), with amplicon PCR primers containing Illumina MiSeq adapter sequences (Table <a href="wildlife-bacteria.html#tab:T32">3.2</a>).
A total of 536 samples from 203 individuals (159 blood, 205 tick pools, and 172 tissue samples) underwent bacterial profiling.</p>
<p>Bacterial 16S rRNA libraries were built targeting the 16S hypervariable region 1-2 using primers 27F-Y and 338R <span class="citation">(<a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">Gofton, Oskam, et al. 2015</a>)</span>.
Reactions were carried out in 25 <span class="math inline">\(\mu L\)</span> volumes each containing: 1X buffer (KAPA Biosystems, South Africa), 1.5 mM MgCl<sub>2</sub>, 0.4 mg/mL of bovine serum albumin (Fisher Biotech), 0.4 <span class="math inline">\(\mu M\)</span> of each primer, 0.25 mM of each dNTP, 0.5 U of Taq (KAPA Biosystems, South Africa) and 2 <span class="math inline">\(\mu L\)</span> of gDNA.
Thermal cycling conditions were as follows; 95<span class="math inline">\(^\circ C\)</span> for 5 mins, followed by 35 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 55<span class="math inline">\(^\circ C\)</span> for 30 secs, 72<span class="math inline">\(^\circ C\)</span> for 45 secs; and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 5 mins.
For a subset of samples where microbes of interest were identified after initial screening at the v1-2 hypervariable region, additional bacterial 16S rRNA libraries targeting the v3-4 hypervariable region were also prepared.
Primers 338F and 806R were used to target an ~450 bp product.
Reactions and thermal cycling conditions were carried out as per the 27F-Y / 338R assay, except for an increased concentration of MgCl<sub>2</sub> to 2.0 mM.</p>
<p>To confirm morphological identification tick gDNA underwent amplicon metabarcoding approach targeting the 12S rDNA gene.
Pan-Ixodida primers T1B/T2A <span class="citation">(<a href="references.html#ref-beatiAnalysisSystematicRelationships2001" role="doc-biblioref">Beati and Keirans 2001</a>)</span> were used to amplify an ~370 bp product of the 12S rRNA locus.
Reactions were carried out as per the bacteria 27F-Y / 338R assay, except only 1 <span class="math inline">\(\mu M\)</span> of tick gDNA template was added.
Thermal cycling conditions were as follows; 95<span class="math inline">\(^\circ C\)</span> for 5 mins, followed by 5 cycles of 95<span class="math inline">\(^\circ C\)</span> for 15 secs, 51<span class="math inline">\(^\circ C\)</span> for 30 secs, 68<span class="math inline">\(^\circ C\)</span> for 30 secs; 25 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 53<span class="math inline">\(^\circ C\)</span> for 30 secs, 72<span class="math inline">\(^\circ C\)</span> for 1 min; and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 5 mins.</p>
<p>Amplicon PCR products were then indexed using the Nextera XT DNA library preparation kit in 25 <span class="math inline">\(\mu L\)</span> volumes following the manufacturer’s recommendations.
All PCRs included no-template controls (NTC; total = 31) during each reaction set up and PCRs were performed under strict laboratory conditions.
Amplicons were then dual-indexed using the Nextera XT index kit.
Reactions were performed in 25 <span class="math inline">\(\mu L\)</span> volumes following manufacturers recommendations.
Libraries were purified with Axygen PCR clean up beads and quantified using Qubit High Sensitivity dsDNA assay kit (Thermo Fisher Scientific, Waltham, MA, USA) and pooled in equimolar amounts.
Libraries were shipped to the Australian Genomic Research Facility (Melbourne, Australia) for final QC and sequenced on an Illumina MiSeq using v2 chemistry (2 x 250 paired-end).</p>

<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T32">Table 3.2: </span>List of sequencing used for metabarcoding and target PCRs to screen blood, tissue and tick samples for bacteria. References <strong>1.</strong> Gofton et al. <span class="citation">(<a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">2015</a>)</span>, <strong>2.</strong> Turner et al. <span class="citation">(<a href="references.html#ref-turnerInvestigatingDeepPhylogenetic1999" role="doc-biblioref">1999</a>)</span>, <strong>3.</strong> Lopez et al. <span class="citation">(<a href="references.html#ref-lopezDesignEvaluationPCR2003" role="doc-biblioref">2003</a>)</span>, <strong>4.</strong> Caporaso et al. <span class="citation">(<a href="references.html#ref-caporasoGlobalPatterns16S2011" role="doc-biblioref">2011</a>)</span>, <strong>5.</strong> Beard et al. <span class="citation">(<a href="references.html#ref-beardBartonellaSppFeral2011" role="doc-biblioref">2011</a>)</span>, <strong>6.</strong> Anderson et al. <span class="citation">(<a href="references.html#ref-andersonEhrlichiaChaffeensisNew1991" role="doc-biblioref">1991</a>)</span>, <strong>7.</strong> Paddock et al. <span class="citation">(<a href="references.html#ref-paddockIsolationCharacterisationEhrlichia1997" role="doc-biblioref">1997</a>)</span>, <strong>8.</strong> Kawahara et al. <span class="citation">(<a href="references.html#ref-kawaharaUltrastructurePhylogeneticAnalysis2004" role="doc-biblioref">2004</a>)</span>, <strong>9.</strong> Beati and Keirans 2001 <span class="citation">(<a href="references.html#ref-beatiAnalysisSystematicRelationships2001" role="doc-biblioref">2001</a>)</span>.
</caption>
<thead><tr>
<th style="text-align:left;">
Primer
</th>
<th style="text-align:left;">
Sequence (5’-3’)
</th>
<th style="text-align:left;">
Ref
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;font-weight: bold;">
Bacteria 16S rRNA
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
27F-Y
</td>
<td style="text-align:left;">
AGAGTTTGATCCTGGCTYAG
</td>
<td style="text-align:left;">
1
</td>
</tr>
<tr>
<td style="text-align:left;">
338R
</td>
<td style="text-align:left;">
TGCTGCCTCCCGTAGGAGT
</td>
<td style="text-align:left;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
338F
</td>
<td style="text-align:left;">
ACTCCTACGGGAGGCAGCAG
</td>
<td style="text-align:left;">
3
</td>
</tr>
<tr>
<td style="text-align:left;">
806R
</td>
<td style="text-align:left;">
GGACTACHVGGGTWTCTAAT
</td>
<td style="text-align:left;">
4
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Bartonella 16S rRNA/ ITS
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
438s
</td>
<td style="text-align:left;">
GGTTTTCCGGTTTATCCCGGAGGGC
</td>
<td style="text-align:left;">
5
</td>
</tr>
<tr>
<td style="text-align:left;">
1100as
</td>
<td style="text-align:left;">
GAACCGACGACCCCCTGCTTGCAAAGC
</td>
<td style="text-align:left;">
5
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;">
Anaplasmataceae 16S rRNA
</td>
<td style="text-align:left;font-weight: bold;">
</td>
<td style="text-align:left;font-weight: bold;">
</td>
</tr>
<tr>
<td style="text-align:left;">
EC9
</td>
<td style="text-align:left;">
TACCTTGTTACGACTT
</td>
<td style="text-align:left;">
6
</td>
</tr>
<tr>
<td style="text-align:left;">
EC12
</td>
<td style="text-align:left;">
TGATCCTGGCTCAGAACGAACG
</td>
<td style="text-align:left;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
A171a
</td>
<td style="text-align:left;">
GCGGCAAGCCTCCCACAT
</td>
<td style="text-align:left;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
IS58-1345r
</td>
<td style="text-align:left;">
CACCAGCTTCGAGTTAAACC
</td>
<td style="text-align:left;">
8
</td>
</tr>
</tbody>
</table></div>
</div>
<div id="bioinformatics" class="section level3" number="3.3.4">
<h3>
<span class="header-section-number">3.3.4</span> Bioinformatics<a class="anchor" aria-label="anchor" href="#bioinformatics"><i class="fas fa-link"></i></a>
</h3>
<p>Metabarcoding 16S rRNA sequence data was analysed using Quantitative Insights into Microbial Ecology (QIIME 2 2020.11) <span class="citation">(<a href="references.html#ref-bolyenReproducibleInteractiveScalable2019" role="doc-biblioref">Bolyen et al. 2019</a>)</span>. Briefly, raw sequences were demultiplexed and quality filtered using the q2-demux plugin, followed by denoising using DADA2 (via q2-dada2) <span class="citation">(<a href="references.html#ref-callahanDADA2HighresolutionSample2016" role="doc-biblioref">Callahan et al. 2016</a>)</span> resulting in amplicon sequence variations (ASVs) (i.e. 100% identical operational taxonomic units or OTUs <span class="citation">(<a href="references.html#ref-callahanExactSequenceVariants2017" role="doc-biblioref">Callahan, McMurdie, and Holmes 2017</a>)</span>).
Bacterial data taxonomy was assigned to ASVs using the q2‐feature‐classifier <span class="citation">(<a href="references.html#ref-bokulichOptimizingTaxonomicClassification2018" role="doc-biblioref">Bokulich et al. 2018</a>)</span> classify‐sklearn naive Bayes taxonomy classifier against the SILVA database <span class="citation">(<a href="references.html#ref-quastSILVARibosomalRNA2013" role="doc-biblioref">Quast et al. 2013</a>)</span> (release 132).
Taxonomic assignments were confirmed using BLAST analysis (BLASTN 2.11.0+ <span class="citation">(<a href="references.html#ref-zhangGreedyAlgorithmAligning2000" role="doc-biblioref">Zhang et al. 2000</a>; <a href="references.html#ref-morgulisDatabaseIndexingProduction2008" role="doc-biblioref">Morgulis et al. 2008</a>)</span>) against the NCBI nucleotide collection (nt) database (accessed January 2021).
Taxonomic lineage was then retrieved from NCBI taxonomy database using TaxonKit <span class="citation">(<a href="references.html#ref-weissHostReproductiveCycle2021" role="doc-biblioref">Weiss et al. 2021</a>)</span> and adjusted to the lowest common ancestor based on percentage identity and e-value score.</p>
<p>Tick 12S metabarcoding sequences generated were analysed USEARCH v11 <span class="citation">(<a href="references.html#ref-edgarSearchClusteringOrders2010" role="doc-biblioref">R. C. Edgar 2010</a>)</span> pipeline.
Briefly paired end reads were merged and sequences matching forward and reverse primers were retrieved (max number of mismatches = 2).
Sequences were then quality filtered and singletons were removed.
The unoise3 <span class="citation">(<a href="references.html#ref-edgarUNOISE2ImprovedErrorcorrection2016" role="doc-biblioref">Robert C. Edgar 2016</a>)</span> algorithm was used to perform denoising (error-correction) and generate zero-radius taxonomic units (zOTUs) equivalent to amplicon sequence variants (ASVs).
Taxonomy was assigned using BLAST analysis and lineage retrieved as outlined above for 16S metabarcoding.</p>
<p>Data visualization and statistical analysis was performed in RStudio (v1.4) with R version 4.0.2. The main R packages used were ampvis2 v2.6.7, phyloseq v1.34.0, microbiomeutilities v1.00.12 and vegan v2.5-7. The R package decontam v1.10.0 <span class="citation">(<a href="references.html#ref-davisSimpleStatisticalIdentification2018" role="doc-biblioref">Davis et al. 2018</a>)</span> was used to inspect data for cross-contamination and cross-talk between samples and to establish read cut-off thresholds (prevalence threshold = 0.05). Identified contaminant ASVs (n = 104), controls samples (n = 65) and sequence values &lt; 100 were removed. Rarefaction curves were generated using a step size of 100. Alpha diversity was measured using four indexes (number of observed ASVs, Chao1, Shannon and invSimpson) and statistical analysis was calculated using Wilcoxon pairwise test (non-parametric) between sample types (blood, tick and tissue). Heatmaps were generated using ampvis2, where data was first aggregated to family level and transformed to relative abundance.</p>
<p>Constrained ordination analysis was performed using two methods, canonical correspondence analysis and redundancy analysis (constrained principal component analysis), to investigate the relationships between sample types.
Prior to the analysis ASVs &lt; 0.1 relative abundance and samples with &lt; 1000 sequences were removed.
Data was first transformed using the Hellinger transformation and distance was measured using the Bray-Curtis method.
Beta diversity statistical analysis was performed using analysis of variance (ANOVA) and permutational multivariate analysis of variance (PERMANOVA) (999 permutations) via the adonis function in the vegan package <span class="citation">(<a href="references.html#ref-oksanenVeganCommunityEcology2020" role="doc-biblioref">Oksanen et al. 2020</a>)</span>, (Hellinger transformation and Bray-Curtis distance measure), the F statistic and P value are presented (full statistical output available in Supplementary Material <a href="supplementary-material.html#ch3stats">A.2.1</a>).
Hierarchical cluster analysis was performed using euclidean distance measure (average), where data was first transformed using Hellinger transformation.</p>
<p>The composition for taxa of interest was aggregated to family level and compared between the three sample types.
Taxa of interest are defined as bacteria related to known pathogens associated with ticks and other vectors described globally, as outlined in Egan et al. <span class="citation">(<a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">2020</a>)</span>.
Statistical analysis was performed using the Wilcoxon pairwise test (non-parametric).
To identify shared ASVs (i.e. core microbiome) between sample types and species, data was transformed to relative abundance and assigned to best taxonomic hit using the microbiomeutilities R package.
Similarities of the core microbiome between the three sample types were investigated using thresholds of abundance and prevalence between 0.0001–-0.001 and 0.001–-0.05 respectively.
For subsequent comparisons of the core microbiome between host and tick species, abundance and prevalence levels were set at 0.001 and 0.05 respectively.
Prevalence data for taxa of interest is presented for samples based on 16S rRNA metabarcoding. Where ticks have been pooled, prevalence is reported based on minimum infection rates (i.e. assumes only one positive tick in each pool) <span class="citation">(<a href="references.html#ref-estrada-penaPitfallsTickTickBorne2021" role="doc-biblioref">Estrada-Pena et al. 2021</a>)</span>.</p>
<p>Scripts for data analysis are available at <a href="https://github.com/siobhon-egan/wildlife-bacteria">siobhon-egan/wildlife-bacteria</a> and <a href="https://github.com/siobhon-egan/wildlife-ticks">siobhon-egan/wildlife-ticks</a> and raw Illumina MiSeq data has been deposited in the European Nucleotide Archive under project accession number PRJEB46056 (16S rRNA bacteria) and PRJEB46056 (12S rRNA ticks).</p>
</div>
<div id="target-pcrs" class="section level3" number="3.3.5">
<h3>
<span class="header-section-number">3.3.5</span> Target PCRs<a class="anchor" aria-label="anchor" href="#target-pcrs"><i class="fas fa-link"></i></a>
</h3>
<p>A nested PCR was used to amplify ~1.4 kb fragment of 16S rRNA locus for <em>Neoehrlichia</em> and <em>Ehrlichia.</em>
Amplicon PCRs were carried out in 25 <span class="math inline">\(\mu L\)</span> reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.5 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu M\)</span> of each primer, 0.25 mM of each dNTP, 0.5 U of Taq (KAPA Biosystems, South Africa) and 2 <span class="math inline">\(\mu L\)</span> of gDNA or 1 <span class="math inline">\(\mu L\)</span> of primary product.
Thermal cycling conditions were as follows; 95<span class="math inline">\(^\circ C\)</span> for 3 mins, followed by 40 cycles of 95<span class="math inline">\(^\circ C\)</span> for 30 secs, 48<span class="math inline">\(^\circ C\)</span> (primary) or 54<span class="math inline">\(^\circ C\)</span> (secondary) for 1 min, 72<span class="math inline">\(^\circ C\)</span> for 2 mins; and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 5 mins.</p>
<p>For <em>Bartonella</em>, a PCR was used to amplify a ~370-450 bp fragment of the 16S rRNA - 23S internal transcribed spacer (ITS) region.
Amplicon PCRs were carried out in 25 <span class="math inline">\(\mu L\)</span> reactions each containing: 1X buffer (KAPA Biosystems, South Africa), 2.0 mM MgCl<sub>2</sub>, 0.4 <span class="math inline">\(\mu M\)</span> of each primer, 0.25 mM of each dNTP, 0.5 U of Taq (KAPA Biosystems, South Africa) and 2 <span class="math inline">\(\mu L\)</span> of gDNA.
Thermal cycling conditions were as follows; 95<span class="math inline">\(^\circ C\)</span> for 3 mins, followed by 40 cycles of 95<span class="math inline">\(^\circ C\)</span> for 15 secs, 66<span class="math inline">\(^\circ C\)</span> 15 secs, 72<span class="math inline">\(^\circ C\)</span> for 18 secs; and a final extension of 72<span class="math inline">\(^\circ C\)</span> for 5 mins.</p>
<p>Amplicons were visualised on agarose gel and products of the expected size were excised with a sterile scalpel blade and purified using an in-house filtered pipette tip method <span class="citation">(<a href="references.html#ref-yangSpecificQuantitativeDetection2013" role="doc-biblioref">R. Yang et al. 2013</a>)</span>.
Sanger sequencing was performed at the Australian Genome Research Facility (Perth, Western Australia) on an Applied Biosystems 3730xl DNA Analyzer using BigDye(TM) Terminator v3.1 Cycle Sequencing Kit.</p>
</div>
<div id="phylogeny" class="section level3" number="3.3.6">
<h3>
<span class="header-section-number">3.3.6</span> Phylogeny<a class="anchor" aria-label="anchor" href="#phylogeny"><i class="fas fa-link"></i></a>
</h3>
<p>Nucleotide sequences were inspected and quality filtered using Geneious 10.2.6 (<a href="https://www.geneious.com" class="uri">https://www.geneious.com</a>).
Identity was confirmed using BLAST analysis (BLASTN 2.11.0+ <span class="citation">(<a href="references.html#ref-zhangGreedyAlgorithmAligning2000" role="doc-biblioref">Zhang et al. 2000</a>; <a href="references.html#ref-morgulisDatabaseIndexingProduction2008" role="doc-biblioref">Morgulis et al. 2008</a>)</span>) against NCBI nucleotide collection (nt) database.
Sequences generated were aligned with references retrieved from GenBank <span class="citation">(<a href="references.html#ref-bensonGenBank2017" role="doc-biblioref">Benson et al. 2017</a>)</span> using MUSCLE <span class="citation">(<a href="references.html#ref-edgarMUSCLEMultipleSequence2004" role="doc-biblioref">R. C. Edgar 2004</a>)</span> (<em>Anaplasmataceae</em> and <em>Borrelia</em>) or clustalW <span class="citation">(<a href="references.html#ref-larkinClustalClustalVersion2007" role="doc-biblioref">Larkin et al. 2007</a>)</span> (<em>Bartonella</em>).
The clustalW aligment method was identified as most suitable for Bartonella analysis due to variable length of the 16S rRNA and ITS regions targeted.
Phylogenies were inferred using the maximum likelihood (ML) method.
The optimal evolutionary model was selected using ModelFinder <span class="citation">(<a href="references.html#ref-kalyaanamoorthyModelFinderFastModel2017" role="doc-biblioref">Kalyaanamoorthy et al. 2017</a>)</span> based on the Bayesian information criterion. Phylogenetic analysis was performed in IQ-TREE v1.6.11 <span class="citation">(<a href="references.html#ref-nguyenIQTREEFastEffective2015" role="doc-biblioref">Nguyen et al. 2015</a>)</span> and bootstrap support was calculated using ultrafast (UFBoot2) method with 10,000 replicates <span class="citation">(<a href="references.html#ref-hoangUFBoot2ImprovingUltrafast2018" role="doc-biblioref">Hoang et al. 2018</a>)</span>.</p>
</div>
</div>
<div id="results" class="section level2" number="3.4">
<h2>
<span class="header-section-number">3.4</span> Results<a class="anchor" aria-label="anchor" href="#results"><i class="fas fa-link"></i></a>
</h2>
<div id="next-generation-sequencing" class="section level3" number="3.4.1">
<h3>
<span class="header-section-number">3.4.1</span> Next-generation sequencing<a class="anchor" aria-label="anchor" href="#next-generation-sequencing"><i class="fas fa-link"></i></a>
</h3>
<p>The hosts species sampled in the present study were the black rat (<em>Rattus rattus</em>), brown antechinus (<em>Antechinus stuartii</em>), brush-tailed possum (<em>Trichosurus vulpecula</em>), bush rat (<em>Rattus fuscipes</em>), chuditch (<em>Dasyurus geoffroii</em>), Rusa deer (<em>Rusa timorensis</em>), long-nosed bandicoot (<em>Perameles nasuta</em>), quenda (<em>Isoodon fusciventer</em>), rabbit (<em>Oryctolagus cuniculus</em>) and swamp rat (<em>Rattus lutreolus</em>) (Table <a href="wildlife-bacteria.html#tab:T3hosts">3.1</a>).
Ticks identified from wildlife hosts were <em>Am. triguttatum</em>, <em>Ix. antechini</em>, <em>Ix. australiensis</em>, <em>Ix. holocyclus</em>, <em>Ix. tasmani</em> and <em>Ix. trichosuri</em>.
Molecular screening was performed on 257 ticks pooled into 205 gDNA extracts.
Metabarcoding of ticks at the 12S rRNA locus confirmed morphological analysis and accurate differentiation of morphologically similar species.
These results identified that mixed tick species were detected in five gDNA pools and in all cases, these were larvae of <em>Ix. holocyclus</em> and <em>Ix. trichosuri</em> (Table <a href="wildlife-bacteria.html#tab:T3ticks">3.3</a>).
A subset of representative 12S rRNA tick zOTU sequences were deposited under accession numbers MW665133–MW665150.</p>
<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
<span id="tab:T3ticks">Table 3.3: </span>Summary of tick species collected from wildlife. Sample numbers in table refer to number of genomic DNA extracts with total number of tick specimens included in parenthesis.
</caption>
<thead><tr>
<th style="text-align:left;">
Host
</th>
<th style="text-align:left;">
Tick species
</th>
<th style="text-align:center;">
Larvae
</th>
<th style="text-align:center;">
Nymph
</th>
<th style="text-align:center;">
Male
</th>
<th style="text-align:center;">
Female
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
Black rat
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:center;">
6 (12)
</td>
<td style="text-align:center;">
5 (5)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:center;">
30 (38)
</td>
<td style="text-align:center;">
31 (34)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. trichosuri
</td>
<td style="text-align:center;">
4 (9)
</td>
<td style="text-align:center;">
2 (3)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus; Ix. trichosuri
</td>
<td style="text-align:center;">
1 (3)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
Brown antechinus
</td>
<td style="text-align:left;font-style: italic;">
Ix. antechini
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1 (1)
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
Brush-tailed possum
</td>
<td style="text-align:left;font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:center;">
2 (3)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
3 (3)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
5 (5)
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2 (2)
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. trichosuri
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
18 (18)
</td>
</tr>
<tr>
<td style="text-align:left;">
Bush rat
</td>
<td style="text-align:left;font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
3 (3)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
Chuditch
</td>
<td style="text-align:left;font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
Long-nosed bandicoot
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:center;">
8 (9)
</td>
<td style="text-align:center;">
5 (5)
</td>
<td style="text-align:center;">
5 (5)
</td>
<td style="text-align:center;">
20 (20)
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:center;">
5 (7)
</td>
<td style="text-align:center;">
12 (12)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. trichosuri
</td>
<td style="text-align:center;">
3 (3)
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
Quenda
</td>
<td style="text-align:left;font-style: italic;">
Am. triguttatum
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. australiensis
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
1 (1)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
Rabbit
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus
</td>
<td style="text-align:center;">
3 (6)
</td>
<td style="text-align:center;">
6 (10)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. tasmani
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
2 (2)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. trichosuri
</td>
<td style="text-align:center;">
1 (4)
</td>
<td style="text-align:center;">
4 (6)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
</td>
<td style="text-align:left;font-style: italic;">
Ix. holocyclus; Ix. trichosuri
</td>
<td style="text-align:center;">
3 (12)
</td>
<td style="text-align:center;">
1 (3)
</td>
<td style="text-align:center;">
0
</td>
<td style="text-align:center;">
0
</td>
</tr>
</tbody>
</table></div>
</div>
<div id="bacteria-16s-metabarcoding" class="section level3" number="3.4.2">
<h3>
<span class="header-section-number">3.4.2</span> Bacteria 16S metabarcoding<a class="anchor" aria-label="anchor" href="#bacteria-16s-metabarcoding"><i class="fas fa-link"></i></a>
</h3>
<p>A total of 536 gDNA extracts from 203 individual animals (159 blood, 205 tick pools, and 172 tissue samples) underwent bacterial profiling.
An additional 65 control samples (EXB and NTC) were also sequenced to obtain background laboratory and reagent profiles.
Bacterial 16S metabarcoding produced 102,503,643 raw sequences (Table <a href="supplementary-material.html#tab:TA31">A.4</a> and Figure <a href="supplementary-material.html#fig:FA31">A.2</a>; samples = 96,969,355, controls = 5,534,288).
Blood samples produced 35,160,160 sequences (mean = 185,053), tick samples produced 24,844,688 sequences (mean = 95,556) and tissue samples produced 36,964,507 sequences (mean = 197,671).
Following quality filtering, denoising and merging a total of 64,186,307 sequences (66.2%) were obtained from the samples; mean for blood samples 121,376 (65.6%) mean for tick samples 70,650 (73.9%) and mean for tissue samples 121,689 (61.6%).</p>
<p>After contaminating taxa and controls were removed, a total of 31,194 bacterial ASVs were identified and, 5,243 ASVs had an abundance of over 1000 sequences.
Rarefaction curves (Figure <a href="supplementary-material.html#fig:FA32">A.3</a>) showed the number of ASVs generally plateaued at a depth of 60,000, 40,000 and 50,000 sequences for blood, tick and tissue samples respectively.
Alpha diversity analysis (Figure <a href="wildlife-bacteria.html#fig:F3alpha">3.2</a>) identified significant variation between sample types (Wilcoxon pairwise test, P &lt; 0.001).
The highest number of observed ASVs was in tissue samples (21,811), followed by blood (9,401) and ticks (3,568).
Tissue samples showed the highest level of alpha diversity across all measures of alpha diversity.
<em>Ixodes australiensis</em> and <em>Ix. trichosuri</em> had the highest alpha diversity of tick species sampled (Figure <a href="supplementary-material.html#fig:FA33">A.4</a>).
Once identified, contaminant taxa and controls were removed and the dominant phyla identified were Proteobacteria (37.3%), Firmicutes (27.8%), Actinobacteria (14.7%) and Bacteroidetes (10.7%) (Figures <a href="supplementary-material.html#fig:FA34">A.5</a> and <a href="supplementary-material.html#fig:FA35">A.6</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F3alpha"></span>
<img src="figures/ms-figs/Ch3-alphadiv.png" alt="Boxplot of Alpha-diversity indices. Diversity indexes (A) Observed number of ASVs, (B) Chao1 index, (C) Shannon index and (D) inverse Simpson index. Boxplots and violin plots represent the distribution of diversity among samples within their category - blood (159), tick (205 pools) and tissue (172). Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test with significance values indicated as follows: NS for P &gt; 0.05; * for P &lt;= 0.05; ** for P &lt;= 0.01; *** for P &lt;= 0.001; **** for P &lt;= 0.0001." width="95%"><p class="caption">
Figure 3.2: Boxplot of Alpha-diversity indices. Diversity indexes (A) Observed number of ASVs, (B) Chao1 index, (C) Shannon index and (D) inverse Simpson index. Boxplots and violin plots represent the distribution of diversity among samples within their category - blood (159), tick (205 pools) and tissue (172). Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test with significance values indicated as follows: NS for P &gt; 0.05; * for P &lt;= 0.05; ** for P &lt;= 0.01; *** for P &lt;= 0.001; **** for P &lt;= 0.0001.
</p>
</div>
<p>In blood samples, the most abundant families were <em>Bacillaceae</em> (22.9%), <em>Bartonellaceae</em> (11.3%), <em>Anaplasmataceae</em> (7.3%) and <em>Mycobacteriaceae</em> (2.6%) (Figure <a href="supplementary-material.html#fig:FA36">A.7</a>).
In tick samples the most abundant families were Coxiellaceae (33.8%), <em>Midichloriaceae</em> (31.0%), <em>Mycobacteriaceae</em> (6.0%), Flavobacteriaceae (2.6%) and <em>Anaplasmataceae</em> (1.8%) (Figure <a href="supplementary-material.html#fig:FA37">A.8</a>).
In tissue samples the most abundant families were Staphylococcaceae (8.2%), <em>Mycobacteriaceae</em> (6.5%), Ruminococcaceae (5.5%), Flavobacteriaceae (4.7%) and <em>Anaplasmataceae</em> (2.7%) (Figure <a href="supplementary-material.html#fig:FA38">A.9</a>).
Comparison of family taxa showed differences in abundance among sample types and species (and tick instar) (Figures <a href="wildlife-bacteria.html#fig:F3heatmap">3.3</a> and <a href="wildlife-bacteria.html#fig:F3heatmaptick">3.4</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F3heatmap"></span>
<img src="figures/ms-figs/Ch3-heatmap.png" alt="Heatmap of the top 40 most prevalent bacterial family taxa identified in wildlife blood, tick and tissue samples. Data first transformed to relative sequence abundance." width="95%"><p class="caption">
Figure 3.3: Heatmap of the top 40 most prevalent bacterial family taxa identified in wildlife blood, tick and tissue samples. Data first transformed to relative sequence abundance.
</p>
</div>
<div class="figure">
<span style="display:block;" id="fig:F3heatmaptick"></span>
<img src="figures/ms-figs/Ch3-heatmap-tick.png" alt="Heatmap of the top 15 most prevalent bacterial family taxa identified in tick samples, showing abundance in species and instar. Data first transformed to relative sequence abundance. Tick samples consisting of mixed species pools were excluded." width="95%"><p class="caption">
Figure 3.4: Heatmap of the top 15 most prevalent bacterial family taxa identified in tick samples, showing abundance in species and instar. Data first transformed to relative sequence abundance. Tick samples consisting of mixed species pools were excluded.
</p>
</div>
<p>The overlap of samples at the ASV level was investigated using various thresholds to define the ‘core microbiome,’ with two out of three models showing only seven shared ASVs between all three sample types (Figure <a href="wildlife-bacteria.html#fig:F3venn">3.5</a>).
Shared core microbiome ASVs of blood and tissue samples were compared between the four most abundant hosts samples and tick species (Figure <a href="wildlife-bacteria.html#fig:F3venn">3.5</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F3venn"></span>
<img src="figures/ms-figs/Ch3-venn_combine.png" alt="Venn diagram of core taxa present within samples presented as detection of amplicon sequence variants (ASVs) using abundance and prevalence. Data first transformed to relative abundance and ASVs assigned to best hit. (A) Impact of similarity based on detection thresholds of abundance and prevalence as (i) 0.001 and 0.05, (ii) 0.001 and 0.001, (iii) 0.0001 and 0.05. (B) Core taxa present within sample types using detection levels set at 0.001 and 0.05 for abundance and prevalence respectively for (i) blood (ii) tissue and (iii) ticks. For within sample comparison only the four most abundant vertebrate host/tick species were selected. Tick species abbreviations: \textit{Amblyomma triguttatum} (\textit{Am. tri}), \textit{Ixodes holocyclus} (\textit{Ix. hol}), \textit{Ixodes tasmani} (\textit{Ix. tas}), \textit{Ixodes trichosuri} (\textit{Ix. tri}). Tick samples consisting of mixed species pools were excluded." width="95%"><p class="caption">
Figure 3.5: Venn diagram of core taxa present within samples presented as detection of amplicon sequence variants (ASVs) using abundance and prevalence. Data first transformed to relative abundance and ASVs assigned to best hit. (A) Impact of similarity based on detection thresholds of abundance and prevalence as (i) 0.001 and 0.05, (ii) 0.001 and 0.001, (iii) 0.0001 and 0.05. (B) Core taxa present within sample types using detection levels set at 0.001 and 0.05 for abundance and prevalence respectively for (i) blood (ii) tissue and (iii) ticks. For within sample comparison only the four most abundant vertebrate host/tick species were selected. Tick species abbreviations:  (),  (),  (),  (). Tick samples consisting of mixed species pools were excluded.
</p>
</div>
<p>Ordination analysis revealed a distinct difference between the bacterial composition of blood, tick and tissue samples (Figure <a href="wildlife-bacteria.html#fig:F3ord">3.6</a>). Hierarchical cluster analysis (Figure <a href="supplementary-material.html#fig:FA39">A.10</a>) and statistical analysis (PERMANOVA F = 36.209, P = 0.001, see Supplementary Material <a href="supplementary-material.html#ch3stats">A.2.1</a>) also supported this finding showing that there was a statistical difference in sample types, and there was little overlap in the microbiome composition of blood and tick samples.
Tissue samples were identified as an intermediate sample type and showed some (albeit low) similarities to blood and tick samples. Ordination analysis of tick samples showed distinction between tick species (Figure <a href="wildlife-bacteria.html#fig:F3ord">3.6</a>).
Tick species <em>Ix. holocyclus</em> and <em>Ix. trichosuri</em> showed the most similar bacterial composition with ordination analysis demonstrating overlap between these two species (PERMANOVA, F = 14.01, P = 0.001, see Supplementary Material <a href="supplementary-material.html#ch3stats">A.2.1</a>).
Although sampling numbers were uneven among host species, which limited statistical inference, hierarchical cluster analysis did show that samples grouped were based on host (and tick) species (Figure <a href="supplementary-material.html#fig:FA310">A.11</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F3ord"></span>
<img src="figures/ms-figs/Ch3-ord_combine.png" alt="Constrained ordination plot of microbiome composition from using constrained methods (A) Canonical Correspondence Analysis and (B) Redundancy Analysis (constrained Principal Component Analysis). (i) Ordination analysis of all samples N = 536 (159 blood, 205 tick pools, and 172 tissue) by sample type. (ii) Ordination of all tick samples by tick species (iii) Ordination of three co-habiting tick species in Sydney Northern Beaches area. Prior to the analysis ASVs &lt; 0.1 relative abundance were removed. Analysis using the Hellinger transformation and Bray-Curtis distance measure. The relative contribution (eigen value) of each axis to the total inertia in the data as well as to the constrained space only, respectively, are indicated in percent at the axis titles. Tick species abbreviations: \textit{Amblyomma triguttatum} (\textit{Am. tri}),\textit{Ixodes antechinus} (\textit{Ix. ant}), \textit{Ixodes australiensis} (\textit{Ix. aus}), \textit{Ixodes holocyclus} (\textit{Ix. hol}), \textit{Ixodes tasmani} (\textit{Ix. tas}), \textit{Ixodes trichosuri} (\textit{Ix. tri})." width="95%"><p class="caption">
Figure 3.6: Constrained ordination plot of microbiome composition from using constrained methods (A) Canonical Correspondence Analysis and (B) Redundancy Analysis (constrained Principal Component Analysis). (i) Ordination analysis of all samples N = 536 (159 blood, 205 tick pools, and 172 tissue) by sample type. (ii) Ordination of all tick samples by tick species (iii) Ordination of three co-habiting tick species in Sydney Northern Beaches area. Prior to the analysis ASVs &lt; 0.1 relative abundance were removed. Analysis using the Hellinger transformation and Bray-Curtis distance measure. The relative contribution (eigen value) of each axis to the total inertia in the data as well as to the constrained space only, respectively, are indicated in percent at the axis titles. Tick species abbreviations:  (), (),  (),  (),  (),  ().
</p>
</div>
</div>
<div id="taxa-of-interest" class="section level3" number="3.4.3">
<h3>
<span class="header-section-number">3.4.3</span> Taxa of interest<a class="anchor" aria-label="anchor" href="#taxa-of-interest"><i class="fas fa-link"></i></a>
</h3>
<p>Nine family taxa were chosen for comparison among sample types (Figure <a href="wildlife-bacteria.html#fig:F3tois">3.7</a>). <em>Anaplasmataceae</em> bacteria were most prevalent in blood samples (both by relative abundance and prevalence). Blood and tissue samples showed no statistical difference in relative abundance (Wilcoxon pairwise test, P = 0.26), with significantly lower abundance in tick samples (Wilcoxon pairwise test, P &lt; 0.001).
There was weak support for differences in <em>Mycoplasmataceae</em> between sample types (Wilcoxon pairwise test, P = 0.023–0.43).
<em>Midichloriaceae</em>, <em>Coxielleaceae</em>, and <em>Rickettsiaceae</em> were all significantly more abundant in ticks than blood or tissue samples.
Statistical comparisons of <em>Borreliaceae</em> and <em>Francisellaceae</em> were not made due to the small number of positive samples.</p>
<p>For subsequent prevalence analysis only taxa of interest that could be accurately defined to the genus level were retained (as per BLAST results). This identified 235 ASVs from 364 samples and 159 individuals see Table <a href="wildlife-bacteria.html#tab:T3prev">3.4</a> and full sequence data and BLAST results in <a href="supplementary-material.html#supplementary-table-e1.1">Supplementary Table E1.1</a>.</p>
<div class="figure">
<span style="display:block;" id="fig:F3tois"></span>
<img src="figures/ms-figs/Ch3-toiboxplot.png" alt="Relative abundance for select bacteria taxa of interest (aggregated to family level) from wildlife samples. Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test with significance values indicated as follows: NS for P &gt; 0.05; * for P &lt;= 0.05; ** for P &lt;= 0.01; *** for P &lt;= 0.001; **** for P &lt;= 0.0001." width="95%"><p class="caption">
Figure 3.7: Relative abundance for select bacteria taxa of interest (aggregated to family level) from wildlife samples. Statistical analysis between sample types calculated using Wilcoxon pairwise (non-parametric) test with significance values indicated as follows: NS for P &gt; 0.05; * for P &lt;= 0.05; ** for P &lt;= 0.01; *** for P &lt;= 0.001; **** for P &lt;= 0.0001.
</p>
</div>
<div class="inline-table"><table class="table" style="font-size: 8.5px; margin-left: auto; margin-right: auto;border-bottom: 0;">
<caption style="font-size: initial !important;">
<span id="tab:T3prev">Table 3.4: </span>Presence of ‘taxa of interest’(a) by bacteria metabarcoding. Prevalence reported as proportion of individual hosts positive and proportion of samples positive in parenthesis (accounting for instances where more than one sample type was collected from an individual).
</caption>
<thead><tr>
<th style="text-align:left;">
Taxa of interest
</th>
<th style="text-align:left;">
Sample type
</th>
<th style="text-align:left;">
Black rat
</th>
<th style="text-align:left;">
Brown antechinus
</th>
<th style="text-align:left;">
Brush tail possum
</th>
<th style="text-align:left;">
Bush rat
</th>
<th style="text-align:left;">
Chuditch
</th>
<th style="text-align:left;">
Deer
</th>
<th style="text-align:left;">
Long-nosed bandicoot
</th>
<th style="text-align:left;">
Quenda
</th>
<th style="text-align:left;">
Rabbit
</th>
<th style="text-align:left;">
Swamp rat
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;font-style: italic;">
Anaplasmataceae
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
4/68 (4/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
5/18 (6/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
7/22 (9/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
18/20 (22/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
3/54 (3/79)
</td>
<td style="text-align:left;">
1/4 (1/6)
</td>
<td style="text-align:left;">
4/15 (5/33)
</td>
<td style="text-align:left;">
0/3 (0/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
8/31 (9/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/7 (0/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
6/71 (8/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
4/16 (5/20)
</td>
<td style="text-align:left;">
0/3 (0/3)
</td>
<td style="text-align:left;">
1/9 (1/9)
</td>
<td style="text-align:left;">
3/3 (3/6)
</td>
<td style="text-align:left;">
18/40 (19/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Bartonella
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
22/68 (27/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
7/18 (8/21)
</td>
<td style="text-align:left;">
2/2 (2/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
4/20 (4/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
3/54 (4/79)
</td>
<td style="text-align:left;">
0/4 (0/6)
</td>
<td style="text-align:left;">
0/15 (0/33)
</td>
<td style="text-align:left;">
1/3 (1/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
1/31 (1/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/7 (0/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
4/71 (5/80)
</td>
<td style="text-align:left;">
1/2 (1/2)
</td>
<td style="text-align:left;">
9/16 (11/20)
</td>
<td style="text-align:left;">
2/3 (2/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
3/40 (3/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Borrelia
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
0/68 (0/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/18 (0/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/20 (0/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
0/54 (0/79)
</td>
<td style="text-align:left;">
0/4 (0/6)
</td>
<td style="text-align:left;">
0/15 (0/33)
</td>
<td style="text-align:left;">
0/3 (0/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/31 (0/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/7 (0/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
7/71 (7/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/16 (0/20)
</td>
<td style="text-align:left;">
1/3 (1/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
0/40 (0/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
1/1 (1/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Coxiellaceae
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
7/68 (7/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
1/18 (1/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
3/20 (5/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
50/54 (63/79)
</td>
<td style="text-align:left;">
3/4 (4/6)
</td>
<td style="text-align:left;">
8/15 (9/33)
</td>
<td style="text-align:left;">
3/3 (4/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
22/31 (24/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
2/7 (2/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
33/71 (38/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
2/16 (2/20)
</td>
<td style="text-align:left;">
2/3 (2/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
20/40 (22/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Francisellaceae
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
0/68 (0/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/18 (0/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/20 (0/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
0/54 (0/79)
</td>
<td style="text-align:left;">
0/4 (0/6)
</td>
<td style="text-align:left;">
2/15 (2/33)
</td>
<td style="text-align:left;">
0/3 (0/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/31 (0/60)
</td>
<td style="text-align:left;">
2/2 (2/2)
</td>
<td style="text-align:left;">
0/7 (0/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
0/71 (0/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/16 (0/20)
</td>
<td style="text-align:left;">
0/3 (0/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
0/40 (0/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Midichloria
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
0/68 (0/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/18 (0/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/20 (0/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
17/54 (21/79)
</td>
<td style="text-align:left;">
2/4 (2/6)
</td>
<td style="text-align:left;">
12/15 (20/33)
</td>
<td style="text-align:left;">
0/3 (0/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
19/31 (41/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
5/7 (10/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
5/71 (5/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
1/16 (1/20)
</td>
<td style="text-align:left;">
0/3 (0/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
3/40 (3/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Mycoplasma
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
1/68 (1/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/18 (0/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/20 (0/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
1/1 (1/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
0/54 (0/79)
</td>
<td style="text-align:left;">
0/4 (0/6)
</td>
<td style="text-align:left;">
0/15 (0/33)
</td>
<td style="text-align:left;">
0/3 (0/4)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/31 (0/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/7 (0/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
1/71 (1/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/16 (0/20)
</td>
<td style="text-align:left;">
0/3 (0/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
0/40 (0/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
Rickettsia
</td>
<td style="text-align:left;font-weight: bold;">
Blood
</td>
<td style="text-align:left;">
0/68 (0/84)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/18 (0/21)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/22 (0/24)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/20 (0/25)
</td>
<td style="text-align:left;">
0/1 (0/2)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tick
</td>
<td style="text-align:left;">
7/54 (7/79)
</td>
<td style="text-align:left;">
0/4 (0/6)
</td>
<td style="text-align:left;">
2/15 (2/33)
</td>
<td style="text-align:left;">
2/3 (3/4)
</td>
<td style="text-align:left;">
1/1 (1/1)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
9/31 (10/60)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
1/7 (1/20)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
</tr>
<tr>
<td style="text-align:left;font-style: italic;">
</td>
<td style="text-align:left;font-weight: bold;">
Tissue
</td>
<td style="text-align:left;">
0/71 (0/80)
</td>
<td style="text-align:left;">
0/2 (0/2)
</td>
<td style="text-align:left;">
0/16 (0/20)
</td>
<td style="text-align:left;">
0/3 (0/3)
</td>
<td style="text-align:left;">
0/9 (0/9)
</td>
<td style="text-align:left;">
0/3 (0/6)
</td>
<td style="text-align:left;">
0/40 (0/44)
</td>
<td style="text-align:left;">
0/0 (0/0)
</td>
<td style="text-align:left;">
0/7 (0/7)
</td>
<td style="text-align:left;">
0/1 (0/1)
</td>
</tr>
</tbody>
<tfoot><tr>
<td style="padding: 0; " colspan="100%">
<sup>a</sup> Taxa of interest are defined as bacteria related to known pathogens associated with ticks and other vectors described globally, as outlined in Egan et al. 2020.
</td>
</tr></tfoot>
</table></div>
<div id="anaplasmataceae-1" class="section level4" number="3.4.3.1">
<h4>
<span class="header-section-number">3.4.3.1</span> <em>Anaplasmataceae</em><a class="anchor" aria-label="anchor" href="#anaplasmataceae-1"><i class="fas fa-link"></i></a>
</h4>
<p>The <em>Anaplasmataceae</em> family was identified in 95 samples from 59 individuals.
The most prevalent genera were <em>Neoehrlichia</em> (43 individuals), followed by <em>Ehrlichia</em> (eight individuals) than <em>Anaplasma</em> (three individuals).</p>
<p>‘<em>Candidatus</em> Neoehrlichia arcana’ was identified in 52 samples: 29 blood, seven ticks (<em>Ix. holocyclus</em>), and 16 tissue. ‘<em>Candidatus</em> Neoehrlichia australis’ was identified in 33 samples: eight blood, six ticks (<em>Ix. holocyclus</em>, <em>Ix. trichosuri</em>), and 19 tissue.
Sixteen samples had mixed infections of ‘<em>Ca</em>. N. arcana’ and ‘<em>Ca</em>. N. australis’ (six blood and ten tissue).
No ticks had mixed <em>Neoehrlichia</em> infections.
Three putative novel <em>Anaplasmataceae</em> species were identified from the present study, two <em>Ehrlichia</em> spp. and a single <em>Neoehrlichia</em> sp.
The novel <em>Neoehrlichia</em> sp. was identified in three samples: a tissue sample from a black rat and a corresponding tick from the same individual (<em>Ix. tasmani</em>); and a tick from a brown antechinus (<em>Ix. antechini</em>).
One novel <em>Ehrlichia</em> sp. was identified in five samples from two brush-tailed possums: two blood, one tick (<em>Ix. holocyclus</em>) and two tissue.
A second novel <em>Ehrlichia</em> sp. was identified in seven samples from six chuditch (six blood and one tissue). Only one chuditch was positive in corresponding tissue and blood samples.
Three deer tissue samples were infected with <em>Anaplasma platys</em> (<a href="supplementary-material.html#supplementary-table-e1.1">Supplementary Table E1.1</a>).
Detection of <em>Anaplasmataceae</em> bacteria differed between tick species and instars.
<em>Ixodes holocyclus</em> females had the highest prevalence of this taxon, followed by nymph and larval ticks, while no males were positive (Figure <a href="wildlife-bacteria.html#fig:F3heatmaptick">3.4</a>).
Only <em>Ix. tasmani</em> nymph samples were positive and in the case of <em>Ix. trichosuri</em>, females and nymphs were identified with <em>Anaplasmataceae</em> and all larvae were negative.</p>
<p>Phylogenetic analysis was performed using a 1,244 bp fragment of the 16S rRNA locus (Figure <a href="wildlife-bacteria.html#fig:F3anaplas">3.8</a>).
All ‘<em>Ca</em>. N. arcana’ amplified were identical to each other and showed 100% similarity to ‘<em>Ca</em>. Neoehrlichia arcana’ isolate HT94 from <em>Ix. holocyclus</em>, Australia (KU865447).
<em>Ehrlichia</em> sequences from chuditch blood samples were 100% identical to each other and most similar to <em>Ehrlichia</em> sp. Anan from <em>Ix. ovatus</em>, Japan (AB028319, 98.94% similarity).
The nearest named species was <em>Ehrlichia muris</em> (CP006917, 98.86% similarity) isolated from a wild mouse (<em>Eothenomys kageus</em>) in Japan (genetic distances <a href="supplementary-material.html#supplementary-table-e1.2">Supplementary Table E1.2</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F3anaplas"></span>
<img src="figures/ms-figs/Ch3-anaplasmataceaetree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of \textit{Anaplasmataceae} based on a 1,244 bp alignment of the 16S rRNA locus. Substitution model K80 + F with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequences generated in the present study are in bold." width="95%"><p class="caption">
Figure 3.8: Maximum likelihood (ML) phylogenetic reconstruction of  based on a 1,244 bp alignment of the 16S rRNA locus. Substitution model K80 + F with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 are indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequences generated in the present study are in bold.
</p>
</div>
</div>
<div id="midichloria" class="section level4" number="3.4.3.2">
<h4>
<span class="header-section-number">3.4.3.2</span> <em>Midichloria</em><a class="anchor" aria-label="anchor" href="#midichloria"><i class="fas fa-link"></i></a>
</h4>
<p>A total of 103 samples were positive for <em>Midichloria</em> which were identified in 94 tick samples from 62 individual animals.
Positive tick species included <em>Ix. holocyclus</em>, <em>Ix. tasmani</em>, <em>Ix. trichosuri</em> from black rats, brown antechinuses, brush-tailed possums, long-nosed bandicoots and rabbits from New South Wales.
While <em>Midichloria</em> was identified in 13 <em>Ix. tasmani</em>, the relative abundance was low (108–363 sequences; 0.1-0.4% relative abundance).
Nine tissue samples (nine individuals) were positive for <em>Midichloria</em> however the number of sequences was generally low (111–14,002) when compared to tick samples; hosts included black rats (five), a brush-tailed possum (one) and long-nosed bandicoots (two).
Only two individuals had positive tissue and tick samples, one brush-tailed possum and one long-nosed bandicoot.
<em>Midichloria</em> was identified in all instars of <em>Ix. holocyclus</em> but only female and nymph stages of <em>Ix. trichosuri</em> (Figure <a href="wildlife-bacteria.html#fig:F3heatmaptick">3.4</a>).
No blood samples were positive for <em>Midichloria</em> (Figure <a href="wildlife-bacteria.html#fig:F3heatmap">3.3</a>).</p>
</div>
<div id="coxiellaceae" class="section level4" number="3.4.3.3">
<h4>
<span class="header-section-number">3.4.3.3</span> Coxiellaceae<a class="anchor" aria-label="anchor" href="#coxiellaceae"><i class="fas fa-link"></i></a>
</h4>
<p>The Coxiellaceae family was identified in 183 samples from 113 individuals. Eleven individuals had positive blood samples, although there was a relatively low number of sequences (100–2,819), from hosts including the black rat (seven), brush-tailed possum (one) and long-nosed bandicoot (three).
One hundred and six ticks from 88 individuals were positive; <em>Am triguttatum</em> (one), <em>Ix. tasmani</em> (90) <em>Ix. holocyclus</em> (nine), and <em>Ix. trichosuri</em>.
Fifty-seven individuals were positive for Coxiellaceae in tissue samples, including the black rat (33), brush-tailed possum (two), bush rat (two) and long-nosed bandicoot (20).
Forty-five individuals had at least two sample types positive for Coxiellaceae, and only two black rats had positive blood, tick and tissue samples.
All instars of <em>Ix. tasmani</em> sampled (female, nymph, larva) were positive for Coxiellaceae, but only female <em>Ix. holocyclus</em> were positive.</p>
</div>
<div id="rickettsiaceae" class="section level4" number="3.4.3.4">
<h4>
<span class="header-section-number">3.4.3.4</span> <em>Rickettsiaceae</em><a class="anchor" aria-label="anchor" href="#rickettsiaceae"><i class="fas fa-link"></i></a>
</h4>
<p><em>Rickettsia</em> was identified exclusively in 24 tick samples from 22 individuals.
Positive samples from Western Australia were all identified as originating in <em>Am. triguttatum</em> (three) from brush-tailed possums and quenda.
Ticks identified as positive from New South Wales were: <em>Ix. holocyclus</em> (nine) from black rats and long-nosed bandicoots, <em>Ix. tasmani</em> (15) from black rats, bush rats, long-nosed bandicoots and rabbits, and <em>Ix. trichosuri</em> (three) from long nosed bandicoots.
No tissue or blood samples were positive for <em>Rickettsia</em>.</p>
</div>
<div id="borrelia-2" class="section level4" number="3.4.3.5">
<h4>
<span class="header-section-number">3.4.3.5</span> <em>Borrelia</em><a class="anchor" aria-label="anchor" href="#borrelia-2"><i class="fas fa-link"></i></a>
</h4>
<p>A novel <em>Borrelia</em> sp. was identified in nine tissue samples: seven black rats, one bush rat and one swamp rat. The identity of these <em>Borrelia</em> sequences showed they were most similar to <em>Borrelia</em> R57 (AY626138, 95.68–96.70% similarity, <a href="supplementary-material.html#supplementary-table-e1.3">Supplementary Table E1.1</a>).
Corresponding blood and ticks from these individuals were negative and no other <em>Borrelia</em> sequences were identified from any other samples.</p>
<p>Attempts to amplify and perform Sanger sequencing on the <em>flaB</em> gene using pan-<em>Borrelia</em> primers <span class="citation">(<a href="references.html#ref-lohNovelBorreliaSpecies2016" role="doc-biblioref">S. M. Loh et al. 2016</a>)</span> were unsuccessful.
Given these failed attempts and the small volume of gDNA from samples (tissue biopsy punch only 2 mm), an alternative approach was employed to amplify additional fragments of the 16S rRNA v3-4 hypervariable region on the Illumina MiSeq.</p>
<p>After bioinformatic analysis outlined above, a single ASV was generated from all nine samples.
Phylogeny was reconstructed using the maximum likelihood method (GTR + R model) based on 426 bp alignment of the 16S rRNA locus (hypervariable region v3-4) (Figure <a href="wildlife-bacteria.html#fig:F3borrelia">3.9</a>). The novel <em>Borrelia</em> sequence showed 98.4% similarity with <em>Borrelia</em> sp. ALEPB216 (KF957671) from <em>R. rattus</em> in California, U.S.A (genetic distances <a href="supplementary-material.html#supplementary-table-e1.3">Supplementary Table E1.3</a>). The next most similar sequences were <em>Borrelia</em> sp. CA682 (KF957670, similarity 98.1%) and <em>Borrelia</em> sp. R57 (AY626138, similarity 97.6%).
The topology showed these sequences formed a distinct ‘rodent’ clade, basal to the three major groups currently described (Figure <a href="wildlife-bacteria.html#fig:F3borrelia">3.9</a>). The nearest named species was <em>Borrelia theileri</em> (U38375, similarity 94.50%).</p>
<div class="figure">
<span style="display:block;" id="fig:F3borrelia"></span>
<img src="figures/ms-figs/Ch3-borreliatree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of \textit{Borrelia} genus based on a 431 bp alignment of the 16S rRNA locus (hypervariable region 3-4). Substitution model K2P + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequence generated in the present study in bold." width="95%"><p class="caption">
Figure 3.9: Maximum likelihood (ML) phylogenetic reconstruction of  genus based on a 431 bp alignment of the 16S rRNA locus (hypervariable region 3-4). Substitution model K2P + G4 with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequence generated in the present study in bold.
</p>
</div>
</div>
<div id="bartonella" class="section level4" number="3.4.3.6">
<h4>
<span class="header-section-number">3.4.3.6</span> <em>Bartonella</em><a class="anchor" aria-label="anchor" href="#bartonella"><i class="fas fa-link"></i></a>
</h4>
<p><em>Bartonella</em> sp. was identified in 69 samples from 47 individuals. Blood samples represented the majority of <em>Bartonella</em> positives with 35 individuals which included black rats (22), brush-tailed possums (seven), bush rats (two), and long-nosed bandicoots (four).
Nineteen individuals had positive tissue samples; black rats (four), brown antechinus (one), brush-tailed possums (nine), bush rats (two) and long-nosed bandicoots (three); seven individuals had positive tissue and blood samples.
Six tick samples were positive for <em>Bartonella</em> (from five individuals) with three tick species (<em>Ix. holocyclus</em>, <em>Ix. tasmani</em> and <em>Ix. trichosuri</em>) from black rats and long-nosed bandicoots. All tick samples that were positive also had corresponding positive blood or tissue samples.</p>
<p>Targeted sequencing on a subset of positive samples showed two genotypes of <em>Bartonella queenslandensis</em> that were 97.2% similar to each other with ten SNPs (genetic distances <a href="supplementary-material.html#supplementary-table-e1.4">Supplementary Table E1.4</a>.
The next most similar sequence to genotype BR025 and BR048 was <em>Ba. queenslandensis</em> strain AUST/NH8 (EU111767) at 96.1% and 94.8% similarity respectively.
Comparison of <em>Ba. queenslandensis</em> type sequences showed divergence up to 7.2%% (between EU111765 and EU111766) within the species. Five genotypes of <em>Ba. coopersplainsensis</em> were identified with 94.4–99.2% similarity to each other; and between 96.7–98.7% similarity to <em>Ba. coopersplainsensis</em> type sequence (EU111770) (Figure <a href="wildlife-bacteria.html#fig:F3bartonella">3.10</a>).</p>
<div class="figure">
<span style="display:block;" id="fig:F3bartonella"></span>
<img src="figures/ms-figs/Ch3-bartonellatree.png" alt="Maximum likelihood (ML) phylogenetic reconstruction of \textit{Bartonella} genus based on a 533 bp alignment of the 16S-23S internal transcribed space (ITS) region. Substitution model HKY + F with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequences generated in the present study in bold." width="95%"><p class="caption">
Figure 3.10: Maximum likelihood (ML) phylogenetic reconstruction of  genus based on a 533 bp alignment of the 16S-23S internal transcribed space (ITS) region. Substitution model HKY + F with 10,000 replicates. Node values correspond to bootstrap support where values &gt; 0.7 indicated by shaded circles. Number of substitutions per nucleotide position is represented by the scale bar. Sequences generated in the present study in bold.
</p>
</div>
</div>
<div id="francisella-and-mycoplasma" class="section level4" number="3.4.3.7">
<h4>
<span class="header-section-number">3.4.3.7</span> <em>Francisella</em> and <em>Mycoplasma</em><a class="anchor" aria-label="anchor" href="#francisella-and-mycoplasma"><i class="fas fa-link"></i></a>
</h4>
<p><em>Francisella</em> was identified in four ticks, <em>Am. triguttatum</em> (three) and <em>Ix. australiensis</em> (one) from brush-tailed possums and quenda in Western Australia (Table <a href="wildlife-bacteria.html#tab:T3prev">3.4</a>, <a href="supplementary-material.html#supplementary-table-e1.1">Supplementary Table E1.1</a>).
<em>Mycoplasma</em> was present in three samples, from three individuals: two blood samples from a black rat and a swamp rat and one tissue sample from a black rat (Table <a href="wildlife-bacteria.html#tab:T3prev">3.4</a>, <a href="supplementary-material.html#supplementary-table-e1.1">Supplementary Table E1.1</a>).</p>
</div>
</div>
</div>
<div id="discussion" class="section level2" number="3.5">
<h2>
<span class="header-section-number">3.5</span> Discussion<a class="anchor" aria-label="anchor" href="#discussion"><i class="fas fa-link"></i></a>
</h2>
<p>This body of work represents the first large scale investigation into tick (and vector)-associated bacteria concurrently with their vertebrate hosts in urban and peri-urban locations, and identified novel tick-associated bacteria in Australian wildlife.
Bacterial amplicon sequencing showed that there were limited taxa shared between blood, tick and tissue samples.
The identification of bacterial taxa of interest was generally confined to certain hosts and/or sample types.
To the best of the authors’ knowledge, this study represents the first report of <em>Neoehrlichia</em> and <em>Ehrlichia</em> from Australian wildlife (blood and/or tissue samples).
The first identification of the rodent-associated <em>Borrelia</em> clade in Australia is also described.
These findings show that geographic boundaries and host associations influence the prevalence of tick-associated bacteria.
Importantly, these findings also provide support for the absence of northern hemisphere tick-borne pathogens, adding to the overwhelming body of evidence that any potential human tick-borne pathogens in Australia are likely endemic and genetically distinct from currently described pathogens in parts of North America and Europe.</p>
<p>Although a comparatively deep level of sequencing was achieved compared to similar studies <span class="citation">(<a href="references.html#ref-rynkiewiczConcordanceBacterialCommunities2015" role="doc-biblioref">Rynkiewicz et al. 2015</a>; <a href="references.html#ref-geSpleenMicrobiotaSmall2018" role="doc-biblioref">Ge et al. 2018</a>)</span>, rarefaction plots showed that blood samples did not reach the same asymptote as tick and tissue, indicating that deeper sequencing of these samples is needed to ensure complete coverage of bacterial diversity.
Across all four alpha diversity measures, tissue samples showed the highest diversity and variance.
In comparison, tick samples showed a less diverse bacterial composition.
This is likely due to the dominance of endosymbiont bacteria in the tick microbiome, which has been demonstrated in previous studies of the same tick species as identified in the present study <span class="citation">(<a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">Gofton, Oskam, et al. 2015</a>; <a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>, and in tick species globally <span class="citation">(<a href="references.html#ref-lalzarCompositionSeasonalVariation2012" role="doc-biblioref">Lalzar et al. 2012</a>; <a href="references.html#ref-gilIxodesScapularisMicrobiome2020" role="doc-biblioref">J. C. Gil et al. 2020</a>)</span>.
The overall lack of similarity in bacterial communities between sample types observed in the present work has been reported in previous studies on small mammals and their vectors (ticks and fleas), indicating the complexity of microbial assemblages inhabiting these biological niches <span class="citation">(<a href="references.html#ref-cohenSimilaritiesSeasonalVariations2015" role="doc-biblioref">Cohen et al. 2015</a>; <a href="references.html#ref-rynkiewiczConcordanceBacterialCommunities2015" role="doc-biblioref">Rynkiewicz et al. 2015</a>; <a href="references.html#ref-liStochasticProcessesGovern2018" role="doc-biblioref">H. Li, Li, and Qu 2018</a>)</span>.
The diagnostic value of 16S rRNA amplicon sequencing methods has not been thoroughly validated for tick-borne pathogens.
One study however showed that ticks which tested positive for <em>Borrelia</em> using qPCR methods also had a high abundance of <em>Borrelia</em> using amplicon bacterial metabarcoding <span class="citation">(<a href="references.html#ref-sperlingMicrobiomeCompositionBorrelia2020" role="doc-biblioref">J. L. H. Sperling et al. 2020</a>)</span>.
Therefore, even though <em>Ix. holocyclus</em> and <em>Ix. tasmani</em> had abundant endosymbionts, and the presence of bacterial species in lower abundance cannot be ruled out completely, it is likely that our finding of the absence of some bacterial species here reflects the true situation.</p>
<p>The microbiome of <em>Ix. holocyclus</em> and <em>Ix. tasmani</em> was dominated by a limited number of highly abundant ASVs.
Other tick species, such as <em>Ix. trichosuri</em> and <em>Ix. australiensis</em>, showed a more diverse bacterial composition.
The interaction between bacterial endosymbionts and pathogens in ticks is not yet fully understood.
The presence of endosymbiont bacteria is thought to inhibit the colonisation by other potentially pathogenic bacterial species.
For example, colonisation by the rickettsial endosymbiont (<em>Rickettsia peacockii</em>) in <em>Dermacentor andersoni</em> tick ovaries is thought to render the ticks more resistant to co-infection by a second rickettsial species (<em>Rickettsia rickettsii</em>), which is the main cause of Rocky Mountain Spotted Fever
<span class="citation">(<a href="references.html#ref-burgdorferNonpathogenicRickettsiaeDermacentor1981" role="doc-biblioref">W. Burgdorfer, Hayes, and Mavros 1981</a>)</span>
<span class="citation">(<a href="references.html#ref-burgdorferNonpathogenicRickettsiaeDermacentor1981" role="doc-biblioref">W. Burgdorfer, Hayes, and Mavros 1981</a>; <a href="references.html#ref-macalusoRickettsialInfectionDermacentor2002" role="doc-biblioref">Macaluso et al. 2002</a>)</span>.</p>
<p>Likewise, <em>Dermacentor variabilis</em> ticks infected with one strain of Rickettsia were shown to be less susceptible to secondary rickettsial infections <span class="citation">(<a href="references.html#ref-macalusoRickettsialInfectionDermacentor2002" role="doc-biblioref">Macaluso et al. 2002</a>)</span>.
However, this theory is not uniformly accepted, and there is some rejection of Burgdorfer’s <span class="citation">(<a href="references.html#ref-burgdorferNonpathogenicRickettsiaeDermacentor1981" role="doc-biblioref">W. Burgdorfer, Hayes, and Mavros 1981</a>)</span> initial findings.
Instead, it is hypothesised that differences in <em>R. peacockii</em> prevalence in <em>De. andersoni</em> are due to microclimatic factors <span class="citation">(<a href="references.html#ref-telfordStatusEastSide2009" role="doc-biblioref">Sam R. Telford 2009</a>)</span>.
Additionally, in recent years microbiome profiling has led to conflicting conclusions about the interaction between endosymbiont and pathogens <span class="citation">(<a href="references.html#ref-bonnetUpdateIntricateTango2020" role="doc-biblioref">Sarah Irene Bonnet and Pollet 2020</a>)</span>.
Based on what is known from the northern hemisphere examples, tick species without a dominant endosymbiont may pose a greater risk of harbouring potentially pathogenic microbes <span class="citation">(<a href="references.html#ref-bonnetUpdateIntricateTango2020" role="doc-biblioref">Sarah Irene Bonnet and Pollet 2020</a>)</span>.</p>
<p>Equivalent studies from the northern hemisphere using 16S rRNA metabarcoding methods and wildlife surveillance to identify tick-borne pathogens are limited.
Instead, pathogen surveillance studies are generally targeted to well-known pathogens.
For example, studies of <em>B. burgdorferi</em> sensu lato have shown infection is not detrimental to the health of natural rodent reservoirs <span class="citation">(<a href="references.html#ref-voordouwLymeDiseasePathogen2015" role="doc-biblioref">Voordouw, Lachish, and Dolan 2015</a>)</span>, which have been shown to harbour a diverse range of <em>Borrelia</em> species <span class="citation">(<a href="references.html#ref-bunikisThirdBorreliaSpecies2005" role="doc-biblioref">Bunikis and Barbour 2005</a>)</span>.
In the present study, chuditch and long-nosed bandicoots had a high abundance of <em>Anaplasmataceae</em> in blood samples, while brush-tailed possums had the highest abundance of this taxon in tissue samples.
In addition, <em>Anaplasmataceae</em> bacteria were mostly absent from larval ticks.</p>
<p>This suggests that the <em>Anaplasmataceae</em> bacteria identified are likely maintained via horizontal transmission (i.e. between host and tick) and transstadial routes, which has been observed with related bacteria in companion animals <span class="citation">(<a href="references.html#ref-fourieTransmissionEhrlichiaCanis2013" role="doc-biblioref">Fourie et al. 2013</a>; <a href="references.html#ref-almazanExperimentalIxodesRicinusSheep2020" role="doc-biblioref">Almazan et al. 2020</a>)</span>.
In contrast, the identification of <em>Midichloria</em> across all life stages of <em>Ix. holocyclus</em> shows evidence of vertical (transovarial) transmission.
Both horizontal and vertical transmission have been identified previously in other tick species with <em>Midichloria</em> <span class="citation">(<a href="references.html#ref-maricontiHumansParasitizedHard2012" role="doc-biblioref">Mariconti et al. 2012</a>; <a href="references.html#ref-serraMolecularSerologicalEvidence2018" role="doc-biblioref">Serra et al. 2018</a>)</span>.
The absence of <em>Midichloria</em> in vertebrate hosts (blood and skin) in the present study suggests the horizontal infection route is not a major contributor to the presence of this endosymbiont in <em>Ix. holocyclus</em> and is consistent with previous studies <span class="citation">(<a href="references.html#ref-mukhachevaBacteriaFamilyCandidatus2017" role="doc-biblioref">Mukhacheva and Kovalev 2017</a>)</span>.</p>
<div id="taxa-of-interest-1" class="section level3" number="3.5.1">
<h3>
<span class="header-section-number">3.5.1</span> Taxa of interest<a class="anchor" aria-label="anchor" href="#taxa-of-interest-1"><i class="fas fa-link"></i></a>
</h3>
<div id="anaplasmataceae-2" class="section level4" number="3.5.1.1">
<h4>
<span class="header-section-number">3.5.1.1</span> <em>Anaplasmataceae</em><a class="anchor" aria-label="anchor" href="#anaplasmataceae-2"><i class="fas fa-link"></i></a>
</h4>
<p>The present study has provided the first identification of <em>Neoehrlichia</em> spp. from Australian wildlife. Two <em>Neoehrlichia</em> species, ‘<em>Ca</em>. N. arcana’ and ‘<em>Ca</em>. N. australis,’ were recently characterised from <em>Ix. holocyclus</em> along the east coast of Australia at a prevalence of 11.3% (44/391) <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">Gofton et al. 2016</a>)</span>.
This finding is consistent with other species of <em>Neoehrlichia</em> globally, which have been described circulating in a range of mammals and associated ticks <span class="citation">(<a href="references.html#ref-kawaharaUltrastructurePhylogeneticAnalysis2004" role="doc-biblioref">Kawahara et al. 2004</a>; <a href="references.html#ref-mullerCandidatusNeoehrlichiaChilensis2018" role="doc-biblioref">Muller et al. 2018</a>)</span>.
In contrast to Gofton et al. <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">2016</a>)</span>, the results presented here show that ‘<em>Ca</em>. N. arcana’ is more prevalent than ‘<em>Ca</em>. N. australis’ in small mammals, which is consistent with a recent study of Australian wildlife ticks <span class="citation">(<a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.
The main reservoir host for ‘<em>Ca</em>. N australis’ is likely to be medium to large macropods (e.g. eastern grey kangaroo (<em>Macropus giganteus</em>) or swamp wallaby (<em>Wallabia bicolor</em>)), which were not sampled in the present study.</p>
<p>The identification of three putative novel <em>Anaplasmataceae</em> species further demonstrates the high diversity of <em>Anaplasmataceae</em> from Australian ticks and wildlife.
This finding adds to recent discoveries identifying new species from <em>Am. triguttatum</em> <span class="citation">(<a href="references.html#ref-goftonDetectionPhylogeneticCharacterisation2017" role="doc-biblioref">Gofton et al. 2017</a>)</span>, <em>Bothriocroton concolor</em>, <span class="citation">(<a href="references.html#ref-lohIdentificationCharacterisationMicroorganisms2018" role="doc-biblioref">Siew-May Loh 2018</a>)</span>, <em>Ix. holocyclus</em> <span class="citation">(<a href="references.html#ref-goftonPhylogeneticCharacterisationTwo2016" role="doc-biblioref">Gofton et al. 2016</a>)</span>, and from the platypus (<em>Ornithorhynchus anatinus</em>) and its associated ticks <em>Ix. ornithorhynchi</em> <span class="citation">(<a href="references.html#ref-goftonNovelEhrlichiaSpecies2018" role="doc-biblioref">Gofton et al. 2018</a>)</span>.
The novel <em>Anaplasmataceae</em> species were less readily identified in tick vectors compared to ‘<em>Ca</em>. N. arcana’ and ‘<em>Ca</em>. N. australis’; however more targeted studies are needed to understand their prevalence and distribution.</p>
<p>The first identification of <em>Anaplasma platys</em> (formerly <em>Ehrlichia platys</em>) in Australia was made in 2001 <span class="citation">(<a href="references.html#ref-brownDetectionEhrlichiaPlatys2001" role="doc-biblioref">G. K. Brown, Roberts, and Aitken 2001</a>)</span>, and has since been described from dogs throughout the country <span class="citation">(<a href="references.html#ref-brownDetectionAnaplasmaPlatys2006" role="doc-biblioref">G. Brown et al. 2006</a>; <a href="references.html#ref-hiiCanineVectorborneDisease2012" role="doc-biblioref">Hii et al. 2012</a>)</span>.
To date it has not been identified in any native Australian wildlife, although <em>Anaplasma platys</em> has been reported from deer overseas <span class="citation">(<a href="references.html#ref-yuPrevalenceCommonTickborne2020" role="doc-biblioref">Yu et al. 2020</a>)</span>.
To the best of the authors’ knowledge, this is the first report of <em>A. platys</em> from wild deer in Australia. The vector of <em>A. platys</em> overseas, <em>Rhipicephalus sanguineus</em> <span class="citation">(<a href="references.html#ref-snellgroveVectorCompetenceRhipicephalus2020" role="doc-biblioref">Snellgrove et al. 2020</a>)</span>, is likely also the vector in Australia.
The brown dog tick, syn. <em>Rh. sanguineus</em> tropical lineage was recently named <em>Rhipicephalus linnaei</em> <span class="citation">(<a href="references.html#ref-slapetaTropicalLineageBrown2021" role="doc-biblioref">Jan Slapeta, Chandra, and Halliday 2021</a>)</span>. Importantly, <em>Ehrlichia canis</em>, which was first identified in dogs and brown dog ticks in northern Australia during 2020, was not identified in ticks, tissue or blood from wildlife in this study.</p>
</div>
<div id="midichloria-1" class="section level4" number="3.5.1.2">
<h4>
<span class="header-section-number">3.5.1.2</span> <em>Midichloria</em><a class="anchor" aria-label="anchor" href="#midichloria-1"><i class="fas fa-link"></i></a>
</h4>
<p>While ‘<em>Ca</em>. M. mitochondrii’ (CMm) is known to be highly prevalent and abundant in ticks identified in the present study, the present study did not employ a blocking primer as used in previous studies <span class="citation">(<a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">Gofton, Oskam, et al. 2015</a>; <a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.
The justification for this was a deliberate choice to include CMm in the ‘taxa of interest’ investigation, i.e. provide insight into the ecology of this microbe.
The absence of CMm in both blood and tissue was surprising, especially given the high proportion and abundance of the bacteria in <em>Ix. holocyclus</em> in the present study.
Studies in France have identified CMm from roe deer blood samples (<em>Capreolus capreolus</em>) using molecular and serology techniques <span class="citation">(<a href="references.html#ref-serraMolecularSerologicalEvidence2018" role="doc-biblioref">Serra et al. 2018</a>)</span>; while that study reported a higher sensitivity for serology, molecular methods were also successful at identifying infection.
In vitro studies of <em>Ix. ricinus</em> and rabbits suggest that the endosymbiont replicates within the vertebrate host <span class="citation">(<a href="references.html#ref-cafisoMidichloriaMitochondriiEndosymbiont2019" role="doc-biblioref">Cafiso et al. 2019</a>)</span>.
The bacterial microbiome of many tick species is generally dominated by the presence of one (or a few) highly abundant endosymbiont microbes <span class="citation">(<a href="references.html#ref-claytonCharacterizationManipulationBacterial2015" role="doc-biblioref">Clayton et al. 2015</a>; <a href="references.html#ref-goftonInhibitionEndosymbiontCandidatus2015" role="doc-biblioref">Gofton, Oskam, et al. 2015</a>; <a href="references.html#ref-varela-stokesMicrobialCommunitiesNorth2017" role="doc-biblioref">Varela-Stokes et al. 2017</a>)</span>.
This suggests that rare, less abundant bacteria are not easily identified using standard 16S rRNA bacterial metabarcoding methods.
While the possibility cannot be ruled out, given the high abundance of CMm in tick samples, it is likely that the methods employed here would have identified it in wildlife hosts (either blood or tissue) if present.
Therefore, given the absence of CMm in the hosts sampled in the present study, vertical transmission may be the primary source of <em>Midichloria</em> infection for <em>Ix. holocyclus</em>. Alternatively, the reservoir host for CMm may not have been sampled.</p>
</div>
<div id="coxiellaceae-1" class="section level4" number="3.5.1.3">
<h4>
<span class="header-section-number">3.5.1.3</span> Coxiellaceae<a class="anchor" aria-label="anchor" href="#coxiellaceae-1"><i class="fas fa-link"></i></a>
</h4>
<p>All Coxiellaceae ASVs identified were most similar to members of the <em>Rickettsiella</em> genus and dominated the microbiome of <em>Ix. tasmani</em>.
<em>Coxiella</em> was not detected in the present study.
The absence of <em>Coxiella</em> from host blood and tissue samples likely reflects a low abundance of this bacteria in the small mammal hosts sampled.
Serological methods are more commonly used to detect exposure to <em>C. burnetii</em> <span class="citation">(<a href="references.html#ref-cooperSerologicalEvidenceCoxiella2012" role="doc-biblioref">A. Cooper et al. 2012</a>)</span> but lack the ability to detect acute infections.
A recent study using molecular techniques detected <em>C. burnetii</em> in kangaroo meat samples <span class="citation">(<a href="references.html#ref-shapiroMolecularDetectionCoxiella2020" role="doc-biblioref">Shapiro et al. 2020</a>)</span>. This suggests that animals sampled in the present study were negative for an active infection with <em>Coxiella</em>.</p>
</div>
<div id="rickettsiaceae-1" class="section level4" number="3.5.1.4">
<h4>
<span class="header-section-number">3.5.1.4</span> <em>Rickettsiaceae</em><a class="anchor" aria-label="anchor" href="#rickettsiaceae-1"><i class="fas fa-link"></i></a>
</h4>
<p>While several forms of rickettsiosis are amongst the few recognised human tick-borne diseases in Australia, remarkably little is known about the sylvatic life cycle of these intracellular bacteria.
At least five agents of human rickettsial disease have been described in Australia <span class="citation">(<a href="references.html#ref-tadepalliMolecularEvidenceNovel2021" role="doc-biblioref">Tadepalli et al. 2021</a>)</span>, together with several additional species and genotypes with unknown pathogenicity <span class="citation">(<a href="references.html#ref-vilcinsDetectionSpottedFever2008" role="doc-biblioref">Vilcins, Old, and Deane 2008</a>; <a href="references.html#ref-vilcinsEvidencePresenceFrancisella2009" role="doc-biblioref">Vilcins, Fournier, et al. 2009</a>; <a href="references.html#ref-tadepalliMolecularEvidenceNovel2021" role="doc-biblioref">Tadepalli et al. 2021</a>)</span>.
Bandicoots and other small mammals have previously been described as reservoirs for Queensland tick typhus (<em>R. australis</em>), and <em>Ix. holocyclus</em> and <em>Ix. tasmani</em> are known vectors <span class="citation">(<a href="references.html#ref-campbellRickettsiosesAustraliaIsolation1974" role="doc-biblioref">Campbell and Domrow 1974</a>; <a href="references.html#ref-sextonSpottedFeverGroup1991" role="doc-biblioref">Sexton et al. 1991</a>)</span>.
Although the relationship between bandicoots and rickettsial infections is largely accepted there has been little validation of this association since the early publications.</p>
<p>In the present study, the genus <em>Rickettsia</em> was predominately identified from ticks, with a low prevalence and abundance in four tissue samples and was absent from all blood samples.
Areas sampled in Sydney’s northern beaches are endemic for Queensland tick typhus and spotted fever.
Therefore, the absence of spotted fever or typhus group <em>Rickettsia</em> from mammal hosts in the present study was unexpected.
Recent studies utilising qPCR have reported <em>Rickettsia</em> in 15.4% (23/149) <em>Ix. holocyclus</em> from NSW <span class="citation">(<a href="references.html#ref-gravesIxodesHolocyclusTicktransmitted2016" role="doc-biblioref">Stephen Graves et al. 2016</a>)</span> and 6.4% (13/203) ticks collected from wildlife in north-east Queensland <span class="citation">(<a href="references.html#ref-hussain-yusufScreeningRickettsiaCoxiella2020" role="doc-biblioref">Hussain-Yusuf et al. 2020</a>)</span>.
Additional reports also recognise the presence of spotted fever group <em>Rickettsia</em> in Australian ticks <span class="citation">(<a href="references.html#ref-cookScrubTyphusOther1967" role="doc-biblioref">Cook, Scott, and Campbell 1967</a>; <a href="references.html#ref-campbellRickettsiosesAustraliaIsolation1974" role="doc-biblioref">Campbell and Domrow 1974</a>; <a href="references.html#ref-gravesRickettsiosesAustralia2006" role="doc-biblioref">S. Graves, Unsworth, and Stenos 2006</a>)</span>.</p>
<p>The lack of <em>Rickettsia</em> identified in ticks may be due to the highly abundant endosymbionts <em>Midichloria</em> or Coxiellaceae in <em>Ix. holocyclus</em> and <em>Ix. tasmani</em> respectively.
While there are limited comparable studies that have used bacterial metabarcoding on wildlife blood samples, previous research has shown that molecular methods successfully identify <em>Rickettsia</em> in reservoir hosts <span class="citation">(<a href="references.html#ref-ndeerehMolecularSurveillanceSpotted2017" role="doc-biblioref">Ndeereh et al. 2017</a>; <a href="references.html#ref-martelloBorreliaBurgdorferiSensu2019" role="doc-biblioref">Martello et al. 2019</a>; <a href="references.html#ref-chaorattanakaweeAmpliconBasedNextGeneration2021" role="doc-biblioref">Chaorattanakawee et al. 2021</a>)</span>. Flinders Island Spotted Fever, caused by <em>R. honei</em> is associated with reptiles and the southern reptile tick <em>Bo. hydrosauri</em> <span class="citation">(<a href="references.html#ref-stenosAponommaHydrosauriReptileassociated2003" role="doc-biblioref">Stenos et al. 2003</a>)</span>.
A closely related strain, named <em>R. honei marmionii</em>, has been identified from various regions in Australia and is associated with <em>Haemaphysalis novaeguineae</em> <span class="citation">(<a href="references.html#ref-laneEvidenceSpottedFeverlike2005a" role="doc-biblioref">Lane et al. 2005</a>; <a href="references.html#ref-unsworthFlindersIslandSpotted2007" role="doc-biblioref">N. Unsworth et al. 2007</a>)</span>.
Interestingly no <em>Haemaphysalis</em> ticks were identified from hosts in the present study.
Further investigation into the sylvatic cycle of <em>Rickettsia</em> in Australia would benefit from the inclusion of samples from reptile hosts and vertebrate hosts of <em>Haemaphysalis</em> ticks.</p>
</div>
<div id="borrelia-3" class="section level4" number="3.5.1.5">
<h4>
<span class="header-section-number">3.5.1.5</span> <em>Borrelia</em><a class="anchor" aria-label="anchor" href="#borrelia-3"><i class="fas fa-link"></i></a>
</h4>
<p>The novel <em>Borrelia</em> identified here came exclusively from black rats and was found at two sites in Sydney’s Northern Beaches area. All samples that tested positive were tissue, and no <em>Borrelia</em> sequences were identified from corresponding blood or tick samples.</p>
<p>The <em>Borrelia</em> species identified from black rats formed a clade with other described <em>Borrelia</em> species identified from rodents. <em>Borrelia</em> sp. R57 was identified from bank voles (<em>Myodes glareolus</em>; formerly <em>Clethrionomys glareolus</em>) and wood mice (<em>Apodemus sylvaticus</em>) in Spain at a prevalence rate of 8.5–12.0% using PCR methods <span class="citation">(<a href="references.html#ref-gilIdentificationNewBorrelia2005" role="doc-biblioref">H. Gil et al. 2005</a>)</span>.
That study identified serological cross-reactivity of this species with <em>B. burgdorferi</em> sensu lato using western blot testing.
A follow-up study (in Spain) further identified the genotype from 24.5% (62/253) of rodent tissue samples <span class="citation">(<a href="references.html#ref-barandikaTickBorneZoonoticBacteria2007" role="doc-biblioref">Barandika et al. 2007</a>)</span>.
Identification of similar sequences was subsequently made from black rats (<em>R. rattus</em>) in California, U.S.A., with a prevalence of 43.5% (10/23) in tissue (skin) biopsy samples <span class="citation">(<a href="references.html#ref-fedorovaRemarkableDiversityTick2014" role="doc-biblioref">Fedorova et al. 2014</a>)</span>.
The absence of this <em>Borrelia</em> species from our tick samples is interesting and mirrors the previous studies in Europe and the U.S.A <span class="citation">(<a href="references.html#ref-gilIdentificationNewBorrelia2005" role="doc-biblioref">H. Gil et al. 2005</a>; <a href="references.html#ref-fedorovaRemarkableDiversityTick2014" role="doc-biblioref">Fedorova et al. 2014</a>)</span>.
Despite multiple attempts during this study, amplification at the <em>flaB</em> locus was unsuccessful, consistent with previous research of members within the clade <span class="citation">(<a href="references.html#ref-gilIdentificationNewBorrelia2005" role="doc-biblioref">H. Gil et al. 2005</a>)</span>.</p>
<p>The extensive geographical range of this clade, yet tight host associations identified to date (i.e. confined to the Order Rodentia), raises questions regarding its origin.
Particularly interesting is the paucity of reports globally, despite small mammals being the most widely studied reservoir host of tick-borne diseases <span class="citation">(<a href="references.html#ref-zikeliWhyResearchLyme2020" role="doc-biblioref">Zikeli and Zohdy 2020</a>)</span>.
Our phylogenetic analysis shows strong support for these <em>Borrelia</em> sequences forming a distinct clade and is consistent with other studies <span class="citation">(<a href="references.html#ref-margosControversiesBacterialTaxonomy2019" role="doc-biblioref">Margos et al. 2019</a>)</span>.
Its basal phylogenetic position may provide important insights into the evolutionary history of the <em>Borrelia</em> genus.
The lack of its identification in tick samples means that the vector for this <em>Borrelia</em> sp. remains unknown, and the transmissibility and pathogenicity of this microbe requires further research.
Importantly, no identifications were made of the recently characterised <em>Borrelia</em> species from Australia, ‘<em>Ca</em>. B. tachyglossi’ <span class="citation">(<a href="references.html#ref-lohMolecularCharacterizationCandidatus2017" role="doc-biblioref">S. M. Loh et al. 2017</a>)</span> or reptile <em>Borrelia</em> sp. <span class="citation">(<a href="references.html#ref-panettaReptileassociatedBorreliaSpecies2017" role="doc-biblioref">Panetta et al. 2017</a>)</span>, nor any sequences from the <em>B. burgdorferi</em> sensu lato group.</p>
<p>Currently there is no evidence that the novel rodent-associated <em>Borrelia</em> genotype identified in the present study is transmissible to humans or capable of causing disease.
Phylogenetic reconstruction indicates that it is distinct from the <em>B. burgdorferi</em> sensu lato and relapsing fever groups <span class="citation">(<a href="references.html#ref-margosControversiesBacterialTaxonomy2019" role="doc-biblioref">Margos et al. 2019</a>)</span>.
The absence of detection from tick samples and its identification only in rodents, more likely implies another vector that presumably has a tight host relationship with <em>Rattus</em> species.</p>
</div>
<div id="bartonella-1" class="section level4" number="3.5.1.6">
<h4>
<span class="header-section-number">3.5.1.6</span> <em>Bartonella</em><a class="anchor" aria-label="anchor" href="#bartonella-1"><i class="fas fa-link"></i></a>
</h4>
<p><em>Bartonella</em> spp. was identified in a relatively high proportion of wildlife blood samples, and most prevalent in blood samples from black rats (31.0%).
<em>Bartonella coopersplainsensis</em> and <em>Ba. queenslandensis</em> were described from native Australian rodents (<em>Melomys</em> spp. and <em>Rattus</em> spp.) in Queensland <span class="citation">(<a href="references.html#ref-gundiBartonellaRattaustralianiSp2009" role="doc-biblioref">Gundi et al. 2009</a>)</span>.
Since then, a number of novel <em>Bartonella</em> strains have been identified from Australia, including novel species described from marsupials and their ectoparasites (ticks and fleas) <span class="citation">(<a href="references.html#ref-vilcinsBartonellalikeDNADetected2009" role="doc-biblioref">Vilcins, Kosoy, et al. 2009</a>; <a href="references.html#ref-kaewmongkolCandidatusBartonellaAntechini2011" role="doc-biblioref">Kaewmongkol, Kaewmongkol, Owen, et al. 2011</a>; <a href="references.html#ref-kaewmongkolGeneticCharacterizationFleaderived2011" role="doc-biblioref">Kaewmongkol, Kaewmongkol, McInnes, et al. 2011</a>)</span>.
While generally considered to be transmitted by fleas, reports of <em>Bartonella</em> species isolated from ticks are increasing, as studies take a broader approach to surveillance of vector-associated microbes <span class="citation">(<a href="references.html#ref-rynkiewiczConcordanceBacterialCommunities2015" role="doc-biblioref">Rynkiewicz et al. 2015</a>; <a href="references.html#ref-asyikhaDetectionBartonellaSp2020" role="doc-biblioref">Asyikha, Sulaiman, and Mohd-Taib 2020</a>; <a href="references.html#ref-krolEvaluatingTransmissionPaths2021" role="doc-biblioref">Krol et al. 2021</a>)</span>.
Identification of <em>Bartonella</em> in fed ticks does not mean they are a viable vector, but it does highlight that future studies should include ticks when exploring candidate vectors.
The high identification of <em>Bartonella</em> in rodents has been reported in other metabarcoding studies from blood samples <span class="citation">(<a href="references.html#ref-cohenSimilaritiesSeasonalVariations2015" role="doc-biblioref">Cohen et al. 2015</a>)</span>.</p>
</div>
<div id="francisella-and-mycoplasma-1" class="section level4" number="3.5.1.7">
<h4>
<span class="header-section-number">3.5.1.7</span> <em>Francisella</em> and <em>Mycoplasma</em><a class="anchor" aria-label="anchor" href="#francisella-and-mycoplasma-1"><i class="fas fa-link"></i></a>
</h4>
<p><em>Francisella</em> was identified exclusively from ticks in Western Australia, <em>Am. triguttatum</em> and <em>Ix. australiensis</em>.
Previous bacterial profiling of <em>Am. triguttatum</em> has shown a high proportion of <em>Francisella</em> species <span class="citation">(<a href="references.html#ref-goftonBacterialProfilingReveals2015" role="doc-biblioref">Gofton, Doggett, et al. 2015</a>; <a href="references.html#ref-eganBacterialCommunityProfiling2020" role="doc-biblioref">Siobhon L. Egan et al. 2020</a>)</span>.
Given its high abundance in tick samples, the absence of <em>Francisella</em> in corresponding host may indicate a lack of transmission.
Tularemia (caused by <em>Francisella tularensis</em>) can cause disease in humans and animals.
While the infection is more commonly acquired via handling infected animal material or aerosol inhalation, <em>F. tularensis</em> has been reported in ticks <span class="citation">(<a href="references.html#ref-regierCombinationMicrobiomeAnalysis2019" role="doc-biblioref">Regier et al. 2019</a>)</span>.
<em>Francisella</em> has been reported in several tick species <span class="citation">(<a href="references.html#ref-garcia-vozmedianoDermacentorMarginatusDermacentor2020" role="doc-biblioref">Garcia-Vozmediano et al. 2020</a>)</span>.
While the species identified in the present study is not closely related to <em>F. tularensis</em>, considering the significance of this genera <span class="citation">(<a href="references.html#ref-regierCombinationMicrobiomeAnalysis2019" role="doc-biblioref">Regier et al. 2019</a>)</span>, further characterisation should be of high importance.</p>
<p>The present study did not identify any haemoprotic <em>Mycoplasma</em> species. The <em>Mycoplasma</em> sequences identified were most closely related to <em>Mycoplasma arthritidis</em>, which is recognised as a common pathogen in rats <span class="citation">(<a href="references.html#ref-dybvigGenomeMycoplasmaArthritidis2008" role="doc-biblioref">Dybvig et al. 2008</a>)</span>.
It was identified in two black rats and a single identification in the native swamp rat.</p>
</div>
</div>
<div id="limitations-and-recommendations" class="section level3" number="3.5.2">
<h3>
<span class="header-section-number">3.5.2</span> Limitations and recommendations<a class="anchor" aria-label="anchor" href="#limitations-and-recommendations"><i class="fas fa-link"></i></a>
</h3>
<p>While the cost of next-generation or second-generation sequencing platforms continues to decline, analysis of large numbers of samples is still constrained by resources and funds.
Amplicon high throughput sequencing remains a cost-effective approach for characterising bacterial microbiomes.
With the advent of long-read sequencing technologies (e.g. PacBio), more studies are moving towards obtaining full-length 16S rRNA sequences which allow for more robust phylogenetic reconstruction and in many cases identification of bacteria to strain/genotype level <span class="citation">(<a href="references.html#ref-earlSpecieslevelBacterialCommunity2018" role="doc-biblioref">Earl et al. 2018</a>; <a href="references.html#ref-callahanHighthroughputAmpliconSequencing2019" role="doc-biblioref">Callahan et al. 2019</a>)</span>.
In addition to describing the bacteria present, the idea of the functional tick microbiome has been suggested <span class="citation">(<a href="references.html#ref-estrada-penaTaxonomicVariabilityFunctional2020" role="doc-biblioref">Estrada-Pena, Cabezas-Cruz, and Obregon 2020</a>)</span>.
While adaption of these methods can seem promising, their use outside of human microbiome studies is questionable due to the reliability of databases <span class="citation">(<a href="references.html#ref-bonnetUpdateIntricateTango2020" role="doc-biblioref">Sarah Irene Bonnet and Pollet 2020</a>; <a href="references.html#ref-sunInferencebasedAccuracyMetagenome2020" role="doc-biblioref">Sun, Jones, and Fodor 2020</a>)</span>.
An alternative approach such as the “common core” or ecological “core microbiome” may be more suitable in the context of tick microbiome research <span class="citation">(<a href="references.html#ref-riselyApplyingCoreMicrobiome2020" role="doc-biblioref">Risely 2020</a>)</span>.
Studies incorporating transcriptomics will likely also prove useful for understanding the functional components of the microbiome.</p>
<p>While the present study reported on the relative abundance of microbes between ticks sampled, it is well known that differences in the 16S copy number limit inferences <span class="citation">(<a href="references.html#ref-brooksTruthMetagenomicsQuantifying2015" role="doc-biblioref">Brooks et al. 2015</a>; <a href="references.html#ref-loucaCorrecting16SRRNA2018" role="doc-biblioref">Louca, Doebeli, and Parfrey 2018</a>)</span>.
In the context of the present study, relative abundance of taxa was compared among samples, as it was considered a suitable measure given samples were processed in the same manner.
Calculations of absolute abundance using 16S rRNA metabarcoding are challenging. While copy number is one factor that can be modelled it relies on information that in many cases is not accurately known.
Using mock communities can help assess bias without baseline data on copy number, however, these analysis methods are still limited.</p>
<p>The sensitivity of molecular diagnostic testing for tick-borne pathogens has been shown to vary greatly.
Factors that can influence molecular detection of pathogens include level of parasitaemia/bacteremia, sample type, time of sample collection from initial infection, and and pre-analytical preparation techniques; additionally, this is compounded by differences between taxa due to host-pathogen interactions <span class="citation">(<a href="references.html#ref-delafuenteTickPathogenInteractionsVector2017" role="doc-biblioref">Fuente et al. 2017</a>; <a href="references.html#ref-estrada-penaFunctionalRedundancyEcological2017" role="doc-biblioref">Estrada-Pena, Fuente, and Cabezas-Cruz 2017</a>; <a href="references.html#ref-bonnetEditorialTickHostPathogenInteractions2018" role="doc-biblioref">Sarah I. Bonnet, Nijhof, and Fuente 2018</a>)</span>.
A strong advantage of using generic metabarcoding assays is their unbiased approach; in the context of the present study, the most important factor was to achieve unbiased characterisation of bacterial communities from Australian wildlife.
However, it is acknowledged that the use of bacterial metabarcoding for the identification of low abundant and rare organisms is limited.
The lack of recognised endemic bacterial pathogens such as <em>Coxiella burnetii</em>, and <em>Rickettsia</em> spp. was surprising.
Future studies should use serological and culture methods in addition to these highly specific and sensitive molecular tools (e.g. qPCR).</p>
<p>Another limitation of the present study was the bias in animals sampled.
Small mammals were chosen as it was hypothesised they would have the highest prevalence of potential tick-borne pathogens based on what is known from the northern hemisphere <span class="citation">(<a href="references.html#ref-bastianAntibodyPrevalenceMolecular2012" role="doc-biblioref">Bastian et al. 2012</a>; <a href="references.html#ref-bouchardHarvestedWhitetailedDeer2013" role="doc-biblioref">Bouchard et al. 2013</a>; <a href="references.html#ref-sweiEvidenceTransmissionZoonotic2019" role="doc-biblioref">Swei et al. 2019</a>)</span>.
Further sampling with expanded host and spatial aspects are vital to understand the full epidemiology of tick (and vector) associated microbes. Likewise, alternative trapping methods (e.g., differing trap type, bait) may expand the suite of hosts captured, importantly those with territorial overlap (e.g., semi-arboreal marsupials).</p>
<p>In the present study, most taxa of interest were confined to specific sample types. For example, <em>Borrelia</em> was identified from tissue samples of rats but absent from any blood or tick samples.
This finding has also been reported in white-footed mice (<em>Peromyscus leucopus</em>) <span class="citation">(<a href="references.html#ref-zawadaOptimizationTissueSampling2020" role="doc-biblioref">Zawada et al. 2020</a>)</span>.
While in many cases disease surveillance is targeted to known pathogens where sampling protocols and diagnostics are well defined, in scoping studies such as the present one, a range of sample types are essential.
This is evident in the different ways pathogens act within their host.
As the first line of defense, the innate immune system has a key role in protecting against cutaneous and systemic infection <span class="citation">(<a href="references.html#ref-coatesInnateAntimicrobialImmunity2018" role="doc-biblioref">Coates, Blanchard, and MacLeod 2018</a>)</span>.
Alternatively pathogens can quickly evade host innate immune responses by rapidly disseminating into the blood stream after invasion <span class="citation">(<a href="references.html#ref-kumarIntravitalImagingVascular2015" role="doc-biblioref">D. Kumar et al. 2015</a>; <a href="references.html#ref-hydeBorreliaBurgdorferiKeeps2017" role="doc-biblioref">Hyde 2017</a>)</span>.
In the context of Australia however, many wildlife species are endangered and therefore their capture and sampling in the wild is subject to strict ethical conditions and government regulations.
As such, collection of multiple sample types is restricted to those requiring only a minimally invasive method and is also dependent on the availability of trained personnel and resources.
In many instances it is standard procedure to collect a tissue biopsy for animal genetic (population) purposes, this presents a simple alternative to other procedures which may require field anaesthesia.
The results here highlight the value of this sample type for use in pathogen surveillance.</p>
</div>
</div>
<div id="conclusions" class="section level2" number="3.6">
<h2>
<span class="header-section-number">3.6</span> Conclusions<a class="anchor" aria-label="anchor" href="#conclusions"><i class="fas fa-link"></i></a>
</h2>
<p>Increased land-use and climate change will continue to affect the spread of vectors and associated diseases globally.
There is an urgent need for expanded disease surveillance, particularly at the urban interface with wildlife.
Through collaborative approaches, discoveries of novel microbes and important epidemiological links will be critical to ensure adequate and timely responses to human and animal diseases.
Investigations into taxa of interest in the present study showed <em>Anaplasmataceae</em> bacteria had a relatively high prevalence in the vertebrate hosts examined here and a wide geographical distribution.
This group of bacteria is therefore a viable candidate for continued research into Australian zoonotic tick-borne diseases.</p>

</div>
</div>

  <div class="chapter-nav">
<div class="prev"><a href="austicks.html"><span class="header-section-number">2</span> An update on records of Australian ticks</a></div>
<div class="next"><a href="wildlife-haemoprotozoa.html"><span class="header-section-number">4</span> Haemoprotozoa surveillance</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#wildlife-bacteria"><span class="header-section-number">3</span> Microbiome of ticks and wildlife</a></li>
<li><a class="nav-link" href="#preface-2">Preface</a></li>
<li><a class="nav-link" href="#abstract-2"><span class="header-section-number">3.1</span> Abstract</a></li>
<li><a class="nav-link" href="#introduction-1"><span class="header-section-number">3.2</span> Introduction</a></li>
<li>
<a class="nav-link" href="#methods-1"><span class="header-section-number">3.3</span> Methods</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#study-sites-and-sample-collection"><span class="header-section-number">3.3.1</span> Study sites and sample collection</a></li>
<li><a class="nav-link" href="#dna-extractions"><span class="header-section-number">3.3.2</span> DNA Extractions</a></li>
<li><a class="nav-link" href="#metabarcoding-sequencing"><span class="header-section-number">3.3.3</span> Metabarcoding sequencing</a></li>
<li><a class="nav-link" href="#bioinformatics"><span class="header-section-number">3.3.4</span> Bioinformatics</a></li>
<li><a class="nav-link" href="#target-pcrs"><span class="header-section-number">3.3.5</span> Target PCRs</a></li>
<li><a class="nav-link" href="#phylogeny"><span class="header-section-number">3.3.6</span> Phylogeny</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#results"><span class="header-section-number">3.4</span> Results</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#next-generation-sequencing"><span class="header-section-number">3.4.1</span> Next-generation sequencing</a></li>
<li><a class="nav-link" href="#bacteria-16s-metabarcoding"><span class="header-section-number">3.4.2</span> Bacteria 16S metabarcoding</a></li>
<li><a class="nav-link" href="#taxa-of-interest"><span class="header-section-number">3.4.3</span> Taxa of interest</a></li>
</ul>
</li>
<li>
<a class="nav-link" href="#discussion"><span class="header-section-number">3.5</span> Discussion</a><ul class="nav navbar-nav">
<li><a class="nav-link" href="#taxa-of-interest-1"><span class="header-section-number">3.5.1</span> Taxa of interest</a></li>
<li><a class="nav-link" href="#limitations-and-recommendations"><span class="header-section-number">3.5.2</span> Limitations and recommendations</a></li>
</ul>
</li>
<li><a class="nav-link" href="#conclusions"><span class="header-section-number">3.6</span> Conclusions</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
<li><a id="book-source" href="https://github.com/ulyngs/oxforddown/blob/master/03-wildlife-bacteria.Rmd">View source <i class="fab fa-github"></i></a></li>
          <li><a id="book-edit" href="https://github.com/ulyngs/oxforddown/edit/master/03-wildlife-bacteria.Rmd">Edit this page <i class="fab fa-github"></i></a></li>
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong><p>Ecology of Ticks and Microbes in<br>
Australia Wildlife</p></strong>" was written by Siobhon Egan. </p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
